<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.56">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Dashboard: Packages At-Risk of Being Archived – CRANhaven</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<link href="site_libs/htmltools-fill-0.5.8/fill.css" rel="stylesheet">
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="site_libs/datatables-binding-0.32/datatables.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">CRANhaven</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-articles" role="link" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Articles</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-articles">    
        <li>
    <a class="dropdown-item" href="./dashboard-live.html">
 <span class="dropdown-text">Dashboard: Recently Archived CRAN Packages</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./dashboard-at-risk.html">
 <span class="dropdown-text">Dashboard: Packages At-Risk of Being Archived</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./cran-archiving-stats.html">
 <span class="dropdown-text">Study: Many Archived Packages Return to CRAN</span></a>
  </li>  
    </ul>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">About</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/cranhaven/cranhaven.org/"> <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar"><div class="quarto-margin-header"><div class="margin-header-item">
<p><a href="https://cranhaven.r-universe.dev/builds"><img src="https://cranhaven.r-universe.dev/badges/:total" alt="CRANhaven number of packages"></a></p>
</div></div>
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Dashboard: Packages At-Risk of Being Archived</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<style>
tr { vertical-align: top; }
</style>
<p>The below table lists 123 R packages that are at risk of being archived<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a> on CRAN, because they have outstanding unresolved issues.</p>
<p>Some of them might have already submitted a new version to fix the issues identified by CRAN (check the event column).</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="live-dashboard" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="live-dashboard">{"x":{"filter":"none","vertical":false,"data":[["2024-09-02<br><small>(in 13 days)<\/small>","2024-08-18<br><small>(2 days overdue)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-18<br><small>(2 days overdue)<\/small>","2024-08-31<br><small>(in 11 days)<\/small>","2024-08-27<br><small>(in 7 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-23<br><small>(in 3 days)<\/small>","2024-08-19<br><small>(1 day overdue)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-19<br><small>(1 day overdue)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-27<br><small>(in 7 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-08-31<br><small>(in 11 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-29<br><small>(in 9 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-18<br><small>(2 days overdue)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-24<br><small>(in 4 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-31<br><small>(in 11 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-09-01<br><small>(in 12 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-08-18<br><small>(2 days overdue)<\/small>","2024-09-01<br><small>(in 12 days)<\/small>","2024-09-01<br><small>(in 12 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-19<br><small>(1 day overdue)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-01<br><small>(in 12 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-19<br><small>(1 day overdue)<\/small>","2024-08-19<br><small>(1 day overdue)<\/small>","2024-08-18<br><small>(2 days overdue)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-31<br><small>(in 11 days)<\/small>","2024-08-23<br><small>(in 3 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-20<br><small>(today)<\/small>","2024-08-18<br><small>(2 days overdue)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-19<br><small>(1 day overdue)<\/small>","2024-08-19<br><small>(1 day overdue)<\/small>","2024-08-19<br><small>(1 day overdue)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-18<br><small>(2 days overdue)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-09-01<br><small>(in 12 days)<\/small>","2024-08-19<br><small>(1 day overdue)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-19<br><small>(1 day overdue)<\/small>","2024-08-22<br><small>(in 2 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-08-25<br><small>(in 5 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-25<br><small>(in 5 days)<\/small>","2024-08-19<br><small>(1 day overdue)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-29<br><small>(in 9 days)<\/small>","2024-08-21<br><small>(in 1 day)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-23<br><small>(in 3 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-08-21<br><small>(in 1 day)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-21<br><small>(in 1 day)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-08-29<br><small>(in 9 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-01<br><small>(in 12 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-28<br><small>(in 8 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-09-02<br><small>(in 13 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-08-20<br><small>(today)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>","2024-09-01<br><small>(in 12 days)<\/small>","2024-09-20<br><small>(in 31 days)<\/small>"],["<a href=\"https://cran.r-project.org/package=admiralvaccine\">admiralvaccine<\/a>","<a href=\"https://cran.r-project.org/package=algaeClassify\">algaeClassify<\/a>","<a href=\"https://cran.r-project.org/package=AlgDesign\">AlgDesign<\/a>","<a href=\"https://cran.r-project.org/package=amap\">amap<\/a>","<a href=\"https://cran.r-project.org/package=archivist\">archivist<\/a>","<a href=\"https://cran.r-project.org/package=arrow\">arrow<\/a>","<a href=\"https://cran.r-project.org/package=BayesMassBal\">BayesMassBal<\/a>","<a href=\"https://cran.r-project.org/package=bggAnalytics\">bggAnalytics<\/a>","<a href=\"https://cran.r-project.org/package=bit\">bit<\/a>","<a href=\"https://cran.r-project.org/package=bite\">bite<\/a>","<a href=\"https://cran.r-project.org/package=bold\">bold<\/a>","<a href=\"https://cran.r-project.org/package=caTools\">caTools<\/a>","<a href=\"https://cran.r-project.org/package=celltrackR\">celltrackR<\/a>","<a href=\"https://cran.r-project.org/package=circular\">circular<\/a>","<a href=\"https://cran.r-project.org/package=collapse\">collapse<\/a>","<a href=\"https://cran.r-project.org/package=ContourFunctions\">ContourFunctions<\/a>","<a href=\"https://cran.r-project.org/package=coxme\">coxme<\/a>","<a href=\"https://cran.r-project.org/package=cplm\">cplm<\/a>","<a href=\"https://cran.r-project.org/package=cregg\">cregg<\/a>","<a href=\"https://cran.r-project.org/package=cwbtools\">cwbtools<\/a>","<a href=\"https://cran.r-project.org/package=data.table\">data.table<\/a>","<a href=\"https://cran.r-project.org/package=datelife\">datelife<\/a>","<a href=\"https://cran.r-project.org/package=DescTools\">DescTools<\/a>","<a href=\"https://cran.r-project.org/package=dhlabR\">dhlabR<\/a>","<a href=\"https://cran.r-project.org/package=distr\">distr<\/a>","<a href=\"https://cran.r-project.org/package=distrEx\">distrEx<\/a>","<a href=\"https://cran.r-project.org/package=dlm\">dlm<\/a>","<a href=\"https://cran.r-project.org/package=dparser\">dparser<\/a>","<a href=\"https://cran.r-project.org/package=energy\">energy<\/a>","<a href=\"https://cran.r-project.org/package=ergm\">ergm<\/a>","<a href=\"https://cran.r-project.org/package=evd\">evd<\/a>","<a href=\"https://cran.r-project.org/package=extras\">extras<\/a>","<a href=\"https://cran.r-project.org/package=fastICA\">fastICA<\/a>","<a href=\"https://cran.r-project.org/package=fdrtool\">fdrtool<\/a>","<a href=\"https://cran.r-project.org/package=ff\">ff<\/a>","<a href=\"https://cran.r-project.org/package=fftw\">fftw<\/a>","<a href=\"https://cran.r-project.org/package=FNN\">FNN<\/a>","<a href=\"https://cran.r-project.org/package=gam\">gam<\/a>","<a href=\"https://cran.r-project.org/package=gap\">gap<\/a>","<a href=\"https://cran.r-project.org/package=geepack\">geepack<\/a>","<a href=\"https://cran.r-project.org/package=GenSA\">GenSA<\/a>","<a href=\"https://cran.r-project.org/package=geometry\">geometry<\/a>","<a href=\"https://cran.r-project.org/package=gggenomes\">gggenomes<\/a>","<a href=\"https://cran.r-project.org/package=glmmTMB\">glmmTMB<\/a>","<a href=\"https://cran.r-project.org/package=gmp\">gmp<\/a>","<a href=\"https://cran.r-project.org/package=grpreg\">grpreg<\/a>","<a href=\"https://cran.r-project.org/package=gtfstools\">gtfstools<\/a>","<a href=\"https://cran.r-project.org/package=hydroloom\">hydroloom<\/a>","<a href=\"https://cran.r-project.org/package=intervals\">intervals<\/a>","<a href=\"https://cran.r-project.org/package=IPDFileCheck\">IPDFileCheck<\/a>","<a href=\"https://cran.r-project.org/package=khisr\">khisr<\/a>","<a href=\"https://cran.r-project.org/package=LDLcalc\">LDLcalc<\/a>","<a href=\"https://cran.r-project.org/package=LOGANTree\">LOGANTree<\/a>","<a href=\"https://cran.r-project.org/package=ltsa\">ltsa<\/a>","<a href=\"https://cran.r-project.org/package=malariaAtlas\">malariaAtlas<\/a>","<a href=\"https://cran.r-project.org/package=matrixStats\">matrixStats<\/a>","<a href=\"https://cran.r-project.org/package=mboost\">mboost<\/a>","<a href=\"https://cran.r-project.org/package=MCMCpack\">MCMCpack<\/a>","<a href=\"https://cran.r-project.org/package=microeco\">microeco<\/a>","<a href=\"https://cran.r-project.org/package=MLEcens\">MLEcens<\/a>","<a href=\"https://cran.r-project.org/package=mlr3learners\">mlr3learners<\/a>","<a href=\"https://cran.r-project.org/package=mlt.docreg\">mlt.docreg<\/a>","<a href=\"https://cran.r-project.org/package=mnis\">mnis<\/a>","<a href=\"https://cran.r-project.org/package=msm\">msm<\/a>","<a href=\"https://cran.r-project.org/package=mstDIF\">mstDIF<\/a>","<a href=\"https://cran.r-project.org/package=mvMORPH\">mvMORPH<\/a>","<a href=\"https://cran.r-project.org/package=ncvreg\">ncvreg<\/a>","<a href=\"https://cran.r-project.org/package=neotoma2\">neotoma2<\/a>","<a href=\"https://cran.r-project.org/package=netgsa\">netgsa<\/a>","<a href=\"https://cran.r-project.org/package=NMF\">NMF<\/a>","<a href=\"https://cran.r-project.org/package=noaaoceans\">noaaoceans<\/a>","<a href=\"https://cran.r-project.org/package=odbr\">odbr<\/a>","<a href=\"https://cran.r-project.org/package=otargen\">otargen<\/a>","<a href=\"https://cran.r-project.org/package=pbdZMQ\">pbdZMQ<\/a>","<a href=\"https://cran.r-project.org/package=phangorn\">phangorn<\/a>","<a href=\"https://cran.r-project.org/package=phenofit\">phenofit<\/a>","<a href=\"https://cran.r-project.org/package=pkgdown\">pkgdown<\/a>","<a href=\"https://cran.r-project.org/package=polyglotr\">polyglotr<\/a>","<a href=\"https://cran.r-project.org/package=portfolio\">portfolio<\/a>","<a href=\"https://cran.r-project.org/package=PreciseSums\">PreciseSums<\/a>","<a href=\"https://cran.r-project.org/package=psidR\">psidR<\/a>","<a href=\"https://cran.r-project.org/package=PublicWorksFinanceIT\">PublicWorksFinanceIT<\/a>","<a href=\"https://cran.r-project.org/package=qtl\">qtl<\/a>","<a href=\"https://cran.r-project.org/package=quanteda\">quanteda<\/a>","<a href=\"https://cran.r-project.org/package=quanteda.textmodels\">quanteda.textmodels<\/a>","<a href=\"https://cran.r-project.org/package=randomForest\">randomForest<\/a>","<a href=\"https://cran.r-project.org/package=RANN\">RANN<\/a>","<a href=\"https://cran.r-project.org/package=rb3\">rb3<\/a>","<a href=\"https://cran.r-project.org/package=ready4\">ready4<\/a>","<a href=\"https://cran.r-project.org/package=rgenoud\">rgenoud<\/a>","<a href=\"https://cran.r-project.org/package=robotstxt\">robotstxt<\/a>","<a href=\"https://cran.r-project.org/package=rPDBapi\">rPDBapi<\/a>","<a href=\"https://cran.r-project.org/package=RPostgreSQL\">RPostgreSQL<\/a>","<a href=\"https://cran.r-project.org/package=rrcov\">rrcov<\/a>","<a href=\"https://cran.r-project.org/package=RRphylo\">RRphylo<\/a>","<a href=\"https://cran.r-project.org/package=sas7bdat\">sas7bdat<\/a>","<a href=\"https://cran.r-project.org/package=sched\">sched<\/a>","<a href=\"https://cran.r-project.org/package=sem\">sem<\/a>","<a href=\"https://cran.r-project.org/package=SFSI\">SFSI<\/a>","<a href=\"https://cran.r-project.org/package=SIMplyBee\">SIMplyBee<\/a>","<a href=\"https://cran.r-project.org/package=sna\">sna<\/a>","<a href=\"https://cran.r-project.org/package=som\">som<\/a>","<a href=\"https://cran.r-project.org/package=spatstat.explore\">spatstat.explore<\/a>","<a href=\"https://cran.r-project.org/package=spotifyr\">spotifyr<\/a>","<a href=\"https://cran.r-project.org/package=SRTsim\">SRTsim<\/a>","<a href=\"https://cran.r-project.org/package=stplanr\">stplanr<\/a>","<a href=\"https://cran.r-project.org/package=SuppDists\">SuppDists<\/a>","<a href=\"https://cran.r-project.org/package=svd\">svd<\/a>","<a href=\"https://cran.r-project.org/package=tensorEVD\">tensorEVD<\/a>","<a href=\"https://cran.r-project.org/package=tidytuesdayR\">tidytuesdayR<\/a>","<a href=\"https://cran.r-project.org/package=timereg\">timereg<\/a>","<a href=\"https://cran.r-project.org/package=transport\">transport<\/a>","<a href=\"https://cran.r-project.org/package=TreatmentPatterns\">TreatmentPatterns<\/a>","<a href=\"https://cran.r-project.org/package=unmarked\">unmarked<\/a>","<a href=\"https://cran.r-project.org/package=VGAM\">VGAM<\/a>","<a href=\"https://cran.r-project.org/package=VPdtw\">VPdtw<\/a>","<a href=\"https://cran.r-project.org/package=WeightIt\">WeightIt<\/a>","<a href=\"https://cran.r-project.org/package=wgaim\">wgaim<\/a>","<a href=\"https://cran.r-project.org/package=WGCNA\">WGCNA<\/a>","<a href=\"https://cran.r-project.org/package=WH\">WH<\/a>","<a href=\"https://cran.r-project.org/package=wk\">wk<\/a>","<a href=\"https://cran.r-project.org/package=worldbank\">worldbank<\/a>","<a href=\"https://cran.r-project.org/package=yaImpute\">yaImpute<\/a>"],["0.2.0<br><small>(2024-01-08)<small>","2.0.2<br><small>(2024-01-12)<small>","1.2.1<br><small>(2022-05-25)<small>","0.8-19<br><small>(2022-10-28)<small>","2.3.6<br><small>(2021-05-20)<small>","17.0.0<br><small>(2024-08-17)<small>","1.1.0<br><small>(2022-06-17)<small>","0.2.1<br><small>(2022-08-31)<small>","4.0.5<br><small>(2022-11-15)<small>","0.3<br><small>(2020-04-22)<small>","1.3.0<br><small>(2023-05-02)<small>","1.18.2<br><small>(2021-03-28)<small>","1.2.0<br><small>(2024-03-26)<small>","0.5-0<br><small>(2023-09-05)<small>","2.0.15<br><small>(2024-07-08)<small>","0.1.1<br><small>(2019-05-20)<small>","2.2-20<br><small>(2024-03-05)<small>","0.7-12<br><small>(2024-01-14)<small>","0.4.0<br><small>(2020-06-28)<small>","0.4.2<br><small>(2024-04-28)<small>","1.15.4<br><small>(2024-03-30)<small>","0.6.8<br><small>(2023-06-19)<small>","0.99.55<br><small>(2024-07-29)<small>","1.0.6<br><small>(2024-07-26)<small>","2.9.3<br><small>(2024-01-29)<small>","2.9.2<br><small>(2024-01-30)<small>","1.1-6<br><small>(2022-11-28)<small>","1.3.1-11<br><small>(2023-12-08)<small>","1.7-11<br><small>(2022-12-22)<small>","4.6.0<br><small>(2023-12-18)<small>","2.3-7<br><small>(2024-04-23)<small>","0.6.1<br><small>(2023-05-10)<small>","1.2-5<br><small>(2024-08-19)<small>","1.2.17<br><small>(2021-11-13)<small>","4.0.12<br><small>(2024-01-12)<small>","1.0-8<br><small>(2024-01-08)<small>","1.1.4<br><small>(2024-01-12)<small>","1.22-4<br><small>(2024-07-18)<small>","1.5-3<br><small>(2023-08-26)<small>","1.3.11<br><small>(2024-06-06)<small>","1.1.14<br><small>(2024-01-22)<small>","0.4.7<br><small>(2023-02-03)<small>","1.0.0<br><small>(2024-06-28)<small>","1.1.9<br><small>(2024-03-20)<small>","0.7-4<br><small>(2024-01-15)<small>","3.4.0<br><small>(2021-07-26)<small>","1.2.0<br><small>(2022-11-24)<small>","1.0.2<br><small>(2024-01-15)<small>","0.15.4<br><small>(2023-06-29)<small>","0.8.1<br><small>(2024-03-08)<small>","1.0.5<br><small>(2024-06-27)<small>","2.1<br><small>(2024-02-11)<small>","0.1.1<br><small>(2022-06-22)<small>","1.4.6<br><small>(2015-12-21)<small>","1.6.1<br><small>(2024-07-18)<small>","1.3.0<br><small>(2024-04-11)<small>","2.9-10<br><small>(2024-04-29)<small>","1.7-0<br><small>(2024-01-18)<small>","1.9.0<br><small>(2024-08-17)<small>","0.1-7<br><small>(2022-10-18)<small>","0.7.0<br><small>(2024-06-28)<small>","1.1-8<br><small>(2024-04-29)<small>","0.3.1<br><small>(2021-04-03)<small>","1.7.1<br><small>(2023-11-23)<small>","0.1.7<br><small>(2022-12-02)<small>","1.1.9<br><small>(2023-12-09)<small>","3.14.2<br><small>(2024-04-20)<small>","1.0.4<br><small>(2024-07-28)<small>","4.0.5<br><small>(2023-11-14)<small>","0.27<br><small>(2024-02-08)<small>","0.3.0<br><small>(2021-04-10)<small>","0.1.0<br><small>(2023-11-20)<small>","1.1.1<br><small>(2024-02-21)<small>","0.3-11<br><small>(2024-01-15)<small>","2.11.1<br><small>(2023-01-23)<small>","0.3.9<br><small>(2024-01-23)<small>","2.1.0<br><small>(2024-07-06)<small>","1.5.1<br><small>(2024-07-27)<small>","0.5-2<br><small>(2021-07-10)<small>","0.6<br><small>(2023-04-22)<small>","2.2<br><small>(2024-06-04)<small>","0.2.0<br><small>(2024-03-07)<small>","1.66<br><small>(2023-11-27)<small>","4.0.2<br><small>(2024-04-24)<small>","0.9.7<br><small>(2024-04-11)<small>","4.7-1.1<br><small>(2022-05-23)<small>","2.6.1<br><small>(2019-01-08)<small>","0.0.11<br><small>(2024-07-13)<small>","0.1.14<br><small>(2024-07-04)<small>","5.9-0.10<br><small>(2023-12-15)<small>","0.7.13<br><small>(2020-09-03)<small>","1.3<br><small>(2024-06-06)<small>","0.7-6<br><small>(2024-01-11)<small>","1.7-6<br><small>(2024-08-19)<small>","2.8.0<br><small>(2023-06-04)<small>","0.7<br><small>(2023-05-02)<small>","1.0.1<br><small>(2024-06-14)<small>","3.1-15<br><small>(2022-04-10)<small>","1.4<br><small>(2024-06-25)<small>","0.3.0<br><small>(2022-12-15)<small>","2.7-2<br><small>(2023-12-06)<small>","0.3-5.1<br><small>(2016-07-06)<small>","3.3-1<br><small>(2024-07-15)<small>","2.2.4<br><small>(2022-12-15)<small>","0.99.6<br><small>(2023-01-13)<small>","1.2.1<br><small>(2024-05-01)<small>","1.1-9.7<br><small>(2022-01-03)<small>","0.5.5<br><small>(2023-11-28)<small>","0.1.3<br><small>(2024-05-30)<small>","1.0.3<br><small>(2023-12-13)<small>","2.0.5<br><small>(2023-01-17)<small>","0.15-2<br><small>(2024-05-10)<small>","2.6.7<br><small>(2024-05-24)<small>","1.4.1<br><small>(2024-01-09)<small>","1.1-11<br><small>(2024-05-15)<small>","2.1-14<br><small>(2022-07-14)<small>","1.2.0<br><small>(2024-07-26)<small>","2.0-1<br><small>(2019-10-03)<small>","1.72-5<br><small>(2023-12-07)<small>","1.1.1<br><small>(2024-04-10)<small>","0.9.2<br><small>(2024-07-09)<small>","0.2.0<br><small>(2024-07-31)<small>","1.0-34<br><small>(2023-12-12)<small>"],["<a href=\"mailto:sukalpo.saha-at-pfizer.com\">Sukalpo Saha<\/a>","<a href=\"mailto:vpatil-at-usgs.gov\">Vijay Patil<\/a>","<a href=\"mailto:jvbraun.statistics-at-gmail.com\">Jerome Braun<\/a>","<a href=\"mailto:antoinelucas-at-gmail.com\">Antoine Lucas<\/a>","<a href=\"mailto:przemyslaw.biecek-at-gmail.com\">Przemyslaw Biecek<\/a>","<a href=\"mailto:jkeane-at-gmail.com\">Jonathan Keane<\/a>","<a href=\"mailto:skoermer-at-vt.edu\">Scott Koermer<\/a>","<a href=\"mailto:bujnowiczgithub-at-gmail.com\">Jakub Bujnowicz<\/a>","<a href=\"mailto:Jens.Oehlschlaegel-at-truecluster.com\">Jens Oehlschlägel<\/a>","<a href=\"mailto:theo.gaboriau-at-unil.ch\">Theo Gaboriau<\/a>","<a href=\"mailto:salixdubois+bold-at-gmail.com\">Salix Dubois<\/a>","<a href=\"mailto:mdietze-at-gfz-potsdam.de\">Michael Dietze<\/a>","<a href=\"mailto:johannes.textor-at-gmx.de\">Johannes Textor<\/a>","<a href=\"mailto:edgarcia-at-est-econ.uc3m.es\">Eduardo García-Portugués<\/a>","<a href=\"mailto:sebastian.krantz-at-graduateinstitute.ch\">Sebastian Krantz<\/a>","<a href=\"mailto:collinberickson-at-gmail.com\">Collin Erickson<\/a>","<a href=\"mailto:therneau.terry-at-mayo.edu\">Terry M. Therneau<\/a>","<a href=\"mailto:actuary_zhang-at-hotmail.com\">Yanwei (Wayne) Zhang<\/a>","<a href=\"mailto:thosjleeper-at-gmail.com\">Thomas J. Leeper<\/a>","<a href=\"mailto:andreas.blaette-at-uni-due.de\">Andreas Blaette<\/a>","<a href=\"mailto:t.barrett88-at-gmail.com\">Tyson Barrett<\/a>","<a href=\"mailto:sanchez.reyes.luna-at-gmail.com\">Luna L. Sanchez Reyes<\/a>","<a href=\"mailto:andri-at-signorell.net\">Andri Signorell<\/a>","<a href=\"mailto:ingerid.dale-at-nb.no\">Ingerid Dale<\/a>","<a href=\"mailto:peter.ruckdeschel-at-uni-oldenburg.de\">Peter Ruckdeschel<\/a>","<a href=\"mailto:Matthias.Kohl-at-stamats.de\">Matthias Kohl<\/a>","<a href=\"mailto:GPetris-at-uark.edu\">Giovanni Petris<\/a>","<a href=\"mailto:matthew.fidler-at-gmail.com\">Matthew Fidler<\/a>","<a href=\"mailto:mrizzo-at-bgsu.edu\">Maria Rizzo<\/a>","<a href=\"mailto:pavel-at-statnet.org\">Pavel N. Krivitsky<\/a>","<a href=\"mailto:alec_stephenson-at-hotmail.com\">Alec Stephenson<\/a>","<a href=\"mailto:joe-at-poissonconsulting.ca\">Joe Thorley<\/a>","<a href=\"mailto:ripley-at-stats.ox.ac.uk\">Brian Ripley<\/a>","<a href=\"mailto:strimmerlab-at-gmail.com\">Korbinian Strimmer<\/a>","<a href=\"mailto:Jens.Oehlschlaegel-at-truecluster.com\">Jens Oehlschlägel<\/a>","<a href=\"mailto:ligges-at-statistik.tu-dortmund.de\">Uwe Ligges<\/a>","<a href=\"mailto:lishengqiao-at-yahoo.com\">Shengqiao Li<\/a>","<a href=\"mailto:hastie-at-stanford.edu\">Trevor Hastie<\/a>","<a href=\"mailto:jinghuazhao-at-hotmail.com\">Jing Hua Zhao<\/a>","<a href=\"mailto:sorenh-at-math.aau.dk\">Søren Højsgaard<\/a>","<a href=\"mailto:DL.RSupport-at-pmi.com\">Sylvain Gubian<\/a>","<a href=\"mailto:david.c.sterratt-at-ed.ac.uk\">David C. Sterratt<\/a>","<a href=\"mailto:t.hackl-at-rug.nl\">Thomas Hackl<\/a>","<a href=\"mailto:mollieebrooks-at-gmail.com\">Mollie Brooks<\/a>","<a href=\"mailto:antoinelucas-at-gmail.com\">Antoine Lucas<\/a>","<a href=\"mailto:patrick-breheny-at-uiowa.edu\">Patrick Breheny<\/a>","<a href=\"mailto:dhersz-at-gmail.com\">Daniel Herszenhut<\/a>","<a href=\"mailto:dblodgett-at-usgs.gov\">David Blodgett<\/a>","<a href=\"mailto:edzer.pebesma-at-uni-muenster.de\">Edzer Pebesma<\/a>","<a href=\"mailto:sheejamk-at-gmail.com\">Sheeja Manchira Krishnan<\/a>","<a href=\"mailto:hello-at-damurka.com\">David Kariuki<\/a>","<a href=\"mailto:P.paplomatas-at-hotmail.com\">Petros Paplomatas<\/a>","<a href=\"mailto:logantreeqq-at-gmail.com\">Qi Qin<\/a>","<a href=\"mailto:aimcleod-at-uwo.ca\">A.I. McLeod<\/a>","<a href=\"mailto:mauricio.vandenberg-at-telethonkids.org.au\">Mauricio van den Berg<\/a>","<a href=\"mailto:henrikb-at-braju.com\">Henrik Bengtsson<\/a>","<a href=\"mailto:Torsten.Hothorn-at-R-project.org\">Torsten Hothorn<\/a>","<a href=\"mailto:jongheepark-at-snu.ac.kr\">Jong Hee Park<\/a>","<a href=\"mailto:liuchi0426-at-126.com\">Chi Liu<\/a>","<a href=\"mailto:marloesmaathuis-at-gmail.com\">Marloes Maathuis<\/a>","<a href=\"mailto:marcbecker-at-posteo.de\">Marc Becker<\/a>","<a href=\"mailto:Torsten.Hothorn-at-R-project.org\">Torsten Hothorn<\/a>","<a href=\"mailto:evanodell91-at-gmail.com\">Evan Odell<\/a>","<a href=\"mailto:chris.jackson-at-mrc-bsu.cam.ac.uk\">Christopher Jackson<\/a>","<a href=\"mailto:rudolf.debelak-at-gmail.com\">Rudolf Debelak<\/a>","<a href=\"mailto:julien.clavel-at-hotmail.fr\">Julien Clavel<\/a>","<a href=\"mailto:patrick-breheny-at-uiowa.edu\">Patrick Breheny<\/a>","<a href=\"mailto:s.dominguez-at-ht-data.com\">Dominguez Vidana Socorro<\/a>","<a href=\"mailto:mikeh1-at-uw.edu\">Michael Hellstern<\/a>","<a href=\"mailto:nicolas.sauwen-at-openanalytics.eu\">Nicolas Sauwen<\/a>","<a href=\"mailto:warlick.sean-at-gmail.com\">Sean Warlick<\/a>","<a href=\"mailto:hsvab-at-hsvab.eng.br\">Haydee Svab<\/a>","<a href=\"mailto:afeizi-at-gmail.com\">Amir Feizi<\/a>","<a href=\"mailto:wccsnow-at-gmail.com\">Wei-Chen Chen<\/a>","<a href=\"mailto:klaus.schliep-at-gmail.com\">Klaus Schliep<\/a>","<a href=\"mailto:kongdd.sysu-at-gmail.com\">Dongdong Kong<\/a>","<a href=\"mailto:hadley-at-posit.co\">Hadley Wickham<\/a>","<a href=\"mailto:t.iwan-at-vu.nl\">Tomer Iwan<\/a>","<a href=\"mailto:dan-at-gerlanc.com\">Daniel Gerlanc<\/a>","<a href=\"mailto:matthew.fidler-at-gmail.com\">Matthew Fidler<\/a>","<a href=\"mailto:florian.oswald-at-gmail.com\">Florian Oswald<\/a>","<a href=\"mailto:lorena.ricciotti-at-uniba.it\">Lorena Ricciotti<\/a>","<a href=\"mailto:broman-at-wisc.edu\">Karl W Broman<\/a>","<a href=\"mailto:kbenoit-at-lse.ac.uk\">Kenneth Benoit<\/a>","<a href=\"mailto:kbenoit-at-lse.ac.uk\">Kenneth Benoit<\/a>","<a href=\"mailto:andy_liaw-at-merck.com\">Andy Liaw<\/a>","<a href=\"mailto:jefferis-at-gmail.com\">Gregory Jefferis<\/a>","<a href=\"mailto:wilson.freitas-at-gmail.com\">Wilson Freitas<\/a>","<a href=\"mailto:matthew.hamilton1-at-monash.edu\">Matthew Hamilton<\/a>","<a href=\"mailto:jas.sekhon-at-yale.edu\">Jasjeet Singh Sekhon<\/a>","<a href=\"mailto:retep.meissner-at-gmail.com\">Peter Meissner<\/a>","<a href=\"mailto:selcukorkmaz-at-gmail.com\">Selcuk Korkmaz<\/a>","<a href=\"mailto:tomoakin-at-staff.kanazawa-u.ac.jp\">Tomoaki Nishiyama<\/a>","<a href=\"mailto:valentin.todorov-at-chello.at\">Valentin Todorov<\/a>","<a href=\"mailto:pasquale.raia-at-unina.it\">Pasquale Raia<\/a>","<a href=\"mailto:matt.shotwell-at-vanderbilt.edu\">Matt Shotwell<\/a>","<a href=\"mailto:pierrick.roger-at-cea.fr\">Pierrick Roger<\/a>","<a href=\"mailto:jfox-at-mcmaster.ca\">John Fox<\/a>","<a href=\"mailto:maraloc-at-gmail.com\">Marco Lopez-Cruz<\/a>","<a href=\"mailto:obsteter.jana-at-gmail.com\">Jana Obšteter<\/a>","<a href=\"mailto:buttsc-at-uci.edu\">Carter T. Butts<\/a>","<a href=\"mailto:jyan-at-stat.uconn.edu\">Jun Yan<\/a>","<a href=\"mailto:Adrian.Baddeley-at-curtin.edu.au\">Adrian Baddeley<\/a>","<a href=\"mailto:daniel.antal-at-dataobservatory.eu\">Daniel Antal<\/a>","<a href=\"mailto:jiaqiang-at-umich.edu\">Jiaqiang Zhu<\/a>","<a href=\"mailto:rob00x-at-gmail.com\">Robin Lovelace<\/a>","<a href=\"mailto:thorsten.pohlert-at-gmx.de\">Thorsten Pohlert<\/a>","<a href=\"mailto:anton-at-korobeynikov.info\">Anton Korobeynikov<\/a>","<a href=\"mailto:maraloc-at-gmail.com\">Marco Lopez-Cruz<\/a>","<a href=\"mailto:ellishughes-at-live.com\">Ellis Hughes<\/a>","<a href=\"mailto:ts-at-biostat.ku.dk\">Thomas Scheike<\/a>","<a href=\"mailto:dominic.schuhmacher-at-mathematik.uni-goettingen.de\">Dominic Schuhmacher<\/a>","<a href=\"mailto:m.l.vankessel-at-erasmusmc.nl\">Maarten van Kessel<\/a>","<a href=\"mailto:contact-at-kenkellner.com\">Ken Kellner<\/a>","<a href=\"mailto:t.yee-at-auckland.ac.nz\">Thomas Yee<\/a>","<a href=\"mailto:ethanbass-at-gmail.com\">Ethan Bass<\/a>","<a href=\"mailto:noah.greifer-at-gmail.com\">Noah Greifer<\/a>","<a href=\"mailto:julian.taylor-at-adelaide.edu.au\">Julian Taylor<\/a>","<a href=\"mailto:Peter.Langfelder-at-gmail.com\">Peter Langfelder<\/a>","<a href=\"mailto:guillaume.biessy78-at-gmail.com\">Guillaume Biessy<\/a>","<a href=\"mailto:dewey-at-fishandwhistle.net\">Dewey Dunnington<\/a>","<a href=\"mailto:muecke.maximilian-at-gmail.com\">Maximilian Mücke<\/a>","<a href=\"mailto:jeffrey_evans-at-tnc.org\">Jeffrey S. Evans<\/a>"],["<a href=\"https://cran.r-project.org/web/checks/check_results_admiralvaccine.html\">checks<\/a><br><a href=\"https://pharmaverse.github.io/admiralvaccine/\">url<\/a><br><a href=\"https://github.com/pharmaverse/admiralvaccine/issues/\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_algaeClassify.html\">checks<\/a><br><a href=\"https://doi.org/10.5066/F7S46Q3F\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_AlgDesign.html\">checks<\/a><br><a href=\"https://github.com/jvbraun/AlgDesign\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_amap.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_archivist.html\">checks<\/a><br><a href=\"https://pbiecek.github.io/archivist/\">url<\/a><br><a href=\"https://github.com/pbiecek/archivist/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_arrow.html\">checks<\/a><br><a href=\"https://github.com/apache/arrow/\">url<\/a><br><a href=\"https://github.com/apache/arrow/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_BayesMassBal.html\">checks<\/a><br><a href=\"https://github.com/skoermer/BayesMassBal\">url<\/a><br><a href=\"https://github.com/skoermer/BayesMassBal/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_bggAnalytics.html\">checks<\/a><br><a href=\"https://github.com/JakubBujnowicz/bggAnalytics\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_bit.html\">checks<\/a><br><a href=\"https://github.com/truecluster/bit\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_bite.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_bold.html\">checks<\/a><br><a href=\"https://docs.ropensci.org/bold/\">url<\/a><br><a href=\"https://github.com/ropensci/bold/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_caTools.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_celltrackR.html\">checks<\/a><br><a href=\"http://www.motilitylab.net\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_circular.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_collapse.html\">checks<\/a><br><a href=\"https://sebkrantz.github.io/collapse/\">url<\/a><br><a href=\"https://github.com/SebKrantz/collapse/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ContourFunctions.html\">checks<\/a><br><a href=\"https://github.com/CollinErickson/contour\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_coxme.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_cplm.html\">checks<\/a><br><a href=\"https://github.com/actuaryzhang/cplm\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_cregg.html\">checks<\/a><br><a href=\"https://github.com/leeper/cregg\">url<\/a><br><a href=\"https://github.com/leeper/cregg/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_cwbtools.html\">checks<\/a><br><a href=\"https://github.com/PolMine/cwbtools\">url<\/a><br><a href=\"https://github.com/PolMine/cwbtools/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_data.table.html\">checks<\/a><br><a href=\"https://r-datatable.com\">url<\/a><br><a href=\"https://github.com/Rdatatable/data.table/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_datelife.html\">checks<\/a><br><a href=\"https://github.com/phylotastic/datelife\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_DescTools.html\">checks<\/a><br><a href=\"https://andrisignorell.github.io/DescTools/\">url<\/a><br><a href=\"https://github.com/AndriSignorell/DescTools/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_dhlabR.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_distr.html\">checks<\/a><br><a href=\"http://distr.r-forge.r-project.org/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_distrEx.html\">checks<\/a><br><a href=\"http://distr.r-forge.r-project.org/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_dlm.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_dparser.html\">checks<\/a><br><a href=\"https://nlmixr2.github.io/dparser-R/\">url<\/a><br><a href=\"https://github.com/nlmixr2/dparser-R/issues/\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_energy.html\">checks<\/a><br><a href=\"https://github.com/mariarizzo/energy\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ergm.html\">checks<\/a><br><a href=\"https://statnet.org\">url<\/a><br><a href=\"https://github.com/statnet/ergm/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_evd.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_extras.html\">checks<\/a><br><a href=\"https://poissonconsulting.github.io/extras/\">url<\/a><br><a href=\"https://github.com/poissonconsulting/extras/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_fastICA.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_fdrtool.html\">checks<\/a><br><a href=\"https://strimmerlab.github.io/software/fdrtool/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ff.html\">checks<\/a><br><a href=\"https://github.com/truecluster/ff\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_fftw.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_FNN.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gam.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gap.html\">checks<\/a><br><a href=\"https://github.com/jinghuazhao/R\">url<\/a><br><a href=\"https://github.com/jinghuazhao/R/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_geepack.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_GenSA.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_geometry.html\">checks<\/a><br><a href=\"https://davidcsterratt.github.io/geometry/\">url<\/a><br><a href=\"https://github.com/davidcsterratt/geometry/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gggenomes.html\">checks<\/a><br><a href=\"https://github.com/thackl/gggenomes\">url<\/a><br><a href=\"https://github.com/thackl/gggenomes/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_glmmTMB.html\">checks<\/a><br><a href=\"https://github.com/glmmTMB/glmmTMB\">url<\/a><br><a href=\"https://github.com/glmmTMB/glmmTMB/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gmp.html\">checks<\/a><br><a href=\"https://forgemia.inra.fr/sylvain.jasson/gmp\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_grpreg.html\">checks<\/a><br><a href=\"https://pbreheny.github.io/grpreg/\">url<\/a><br><a href=\"https://github.com/pbreheny/grpreg/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gtfstools.html\">checks<\/a><br><a href=\"https://ipeagit.github.io/gtfstools/\">url<\/a><br><a href=\"https://github.com/ipeaGIT/gtfstools/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hydroloom.html\">checks<\/a><br><a href=\"https://github.com/DOI-USGS/hydroloom\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_intervals.html\">checks<\/a><br><a href=\"https://github.com/edzer/intervals\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_IPDFileCheck.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_khisr.html\">checks<\/a><br><a href=\"https://khisr.damurka.com\">url<\/a><br><a href=\"https://github.com/damurka/khisr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_LDLcalc.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_LOGANTree.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ltsa.html\">checks<\/a><br><a href=\"http://www.stats.uwo.ca/faculty/aim\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_malariaAtlas.html\">checks<\/a><br><a href=\"https://github.com/malaria-atlas-project/malariaAtlas\">url<\/a><br><a href=\"https://github.com/malaria-atlas-project/malariaAtlas/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_matrixStats.html\">checks<\/a><br><a href=\"https://github.com/HenrikBengtsson/matrixStats\">url<\/a><br><a href=\"https://github.com/HenrikBengtsson/matrixStats/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mboost.html\">checks<\/a><br><a href=\"https://github.com/boost-R/mboost\">url<\/a><br><a href=\"https://github.com/boost-R/mboost/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_MCMCpack.html\">checks<\/a><br><a href=\"https://CRAN.R-project.org/package=MCMCpack\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_microeco.html\">checks<\/a><br><a href=\"https://github.com/ChiLiubio/microeco\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_MLEcens.html\">checks<\/a><br><a href=\"http://stat.ethz.ch/~maathuis/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mlr3learners.html\">checks<\/a><br><a href=\"https://mlr3learners.mlr-org.com\">url<\/a><br><a href=\"https://github.com/mlr-org/mlr3learners/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mlt.docreg.html\">checks<\/a><br><a href=\"http://ctm.R-forge.R-project.org\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mnis.html\">checks<\/a><br><a href=\"https://docs.evanodell.com/mnis\">url<\/a><br><a href=\"https://github.com/EvanOdell/mnis/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_msm.html\">checks<\/a><br><a href=\"https://github.com/chjackson/msm\">url<\/a><br><a href=\"https://github.com/chjackson/msm/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mstDIF.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mvMORPH.html\">checks<\/a><br><a href=\"https://github.com/JClavel/mvMORPH\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ncvreg.html\">checks<\/a><br><a href=\"https://pbreheny.github.io/ncvreg/\">url<\/a><br><a href=\"https://github.com/pbreheny/ncvreg/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_neotoma2.html\">checks<\/a><br><a href=\"https://github.com/NeotomaDB/neotoma2\">url<\/a><br><a href=\"https://github.com/NeotomaDB/neotoma2/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_netgsa.html\">checks<\/a><br><a href=\"https://github.com/mikehellstern/netgsa\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_NMF.html\">checks<\/a><br><a href=\"http://renozao.github.io/NMF/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_noaaoceans.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_odbr.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_otargen.html\">checks<\/a><br><a href=\"https://amirfeizi.github.io/otargen/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_pbdZMQ.html\">checks<\/a><br><a href=\"https://pbdr.org/\">url<\/a><br><a href=\"https://github.com/snoweye/pbdZMQ/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_phangorn.html\">checks<\/a><br><a href=\"https://github.com/KlausVigo/phangorn\">url<\/a><br><a href=\"https://github.com/KlausVigo/phangorn/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_phenofit.html\">checks<\/a><br><a href=\"https://github.com/eco-hydro/phenofit\">url<\/a><br><a href=\"https://github.com/eco-hydro/phenofit/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_pkgdown.html\">checks<\/a><br><a href=\"https://pkgdown.r-lib.org/\">url<\/a><br><a href=\"https://github.com/r-lib/pkgdown/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_polyglotr.html\">checks<\/a><br><a href=\"https://github.com/Tomeriko96/polyglotr/\">url<\/a><br><a href=\"https://github.com/Tomeriko96/polyglotr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_portfolio.html\">checks<\/a><br><a href=\"https://github.com/dgerlanc/portfolio\">url<\/a><br><a href=\"https://github.com/dgerlanc/portfolio/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_PreciseSums.html\">checks<\/a><br><a href=\"https://github.com/nlmixr2/PreciseSums\">url<\/a><br><a href=\"https://github.com/nlmixr2/PreciseSums/issues/\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_psidR.html\">checks<\/a><br><a href=\"https://github.com/floswald/psidR\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_PublicWorksFinanceIT.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_qtl.html\">checks<\/a><br><a href=\"https://rqtl.org\">url<\/a><br><a href=\"https://github.com/kbroman/qtl/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_quanteda.html\">checks<\/a><br><a href=\"https://quanteda.io\">url<\/a><br><a href=\"https://github.com/quanteda/quanteda/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_quanteda.textmodels.html\">checks<\/a><br><a href=\"https://github.com/quanteda/quanteda.textmodels\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_randomForest.html\">checks<\/a><br><a href=\"https://www.stat.berkeley.edu/~breiman/RandomForests/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_RANN.html\">checks<\/a><br><a href=\"https://github.com/jefferis/RANN\">url<\/a><br><a href=\"https://github.com/jefferis/RANN/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rb3.html\">checks<\/a><br><a href=\"https://github.com/ropensci/rb3\">url<\/a><br><a href=\"https://github.com/ropensci/rb3/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ready4.html\">checks<\/a><br><a href=\"https://ready4-dev.github.io/ready4/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rgenoud.html\">checks<\/a><br><a href=\"https://github.com/JasjeetSekhon/rgenoud\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_robotstxt.html\">checks<\/a><br><a href=\"https://docs.ropensci.org/robotstxt/\">url<\/a><br><a href=\"https://github.com/ropensci/robotstxt/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rPDBapi.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_RPostgreSQL.html\">checks<\/a><br><a href=\"https://github.com/tomoakin/RPostgreSQL\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rrcov.html\">checks<\/a><br><a href=\"https://github.com/valentint/rrcov\">url<\/a><br><a href=\"https://github.com/valentint/rrcov/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_RRphylo.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_sas7bdat.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_sched.html\">checks<\/a><br><a href=\"https://gitlab.com/cnrgh/databases/r-sched\">url<\/a><br><a href=\"https://gitlab.com/cnrgh/databases/r-sched/-/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_sem.html\">checks<\/a><br><a href=\"https://www.r-project.org\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_SFSI.html\">checks<\/a><br><a href=\"https://github.com/MarcooLopez/SFSI\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_SIMplyBee.html\">checks<\/a><br><a href=\"https://github.com/HighlanderLab/SIMplyBee\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_sna.html\">checks<\/a><br><a href=\"https://statnet.org\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_som.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_spatstat.explore.html\">checks<\/a><br><a href=\"http://spatstat.org/\">url<\/a><br><a href=\"https://github.com/spatstat/spatstat.explore/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_spotifyr.html\">checks<\/a><br><a href=\"https://github.com/charlie86/spotifyr\">url<\/a><br><a href=\"https://github.com/charlie86/spotifyr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_SRTsim.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_stplanr.html\">checks<\/a><br><a href=\"https://github.com/ropensci/stplanr\">url<\/a><br><a href=\"https://github.com/ropensci/stplanr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_SuppDists.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_svd.html\">checks<\/a><br><a href=\"https://github.com/asl/svd\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tensorEVD.html\">checks<\/a><br><a href=\"https://github.com/MarcooLopez/tensorEVD\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tidytuesdayR.html\">checks<\/a><br><a href=\"https://github.com/thebioengineer/tidytuesdayR\">url<\/a><br><a href=\"https://github.com/thebioengineer/tidytuesdayR/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_timereg.html\">checks<\/a><br><a href=\"https://github.com/scheike/timereg\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_transport.html\">checks<\/a><br><a href=\"https://dschuhm1.pages.gwdg.de/software\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_TreatmentPatterns.html\">checks<\/a><br><a href=\"https://github.com/darwin-eu/TreatmentPatterns\">url<\/a><br><a href=\"https://github.com/darwin-eu/TreatmentPatterns/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_unmarked.html\">checks<\/a><br><a href=\"https://groups.google.com/d/forum/unmarked\">url<\/a><br><a href=\"https://github.com/rbchan/unmarked/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_VGAM.html\">checks<\/a><br><a href=\"https://www.stat.auckland.ac.nz/~yee/VGAM/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_VPdtw.html\">checks<\/a><br><a href=\"https://github.com/ethanbass/VPdtw/\">url<\/a><br><a href=\"https://github.com/ethanbass/VPdtw/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_WeightIt.html\">checks<\/a><br><a href=\"https://ngreifer.github.io/WeightIt/\">url<\/a><br><a href=\"https://github.com/ngreifer/WeightIt/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_wgaim.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_WGCNA.html\">checks<\/a><br><a href=\"http://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_WH.html\">checks<\/a><br><a href=\"https://github.com/GuillaumeBiessy/WH\">url<\/a><br><a href=\"https://github.com/GuillaumeBiessy/WH/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_wk.html\">checks<\/a><br><a href=\"https://paleolimbot.github.io/wk/\">url<\/a><br><a href=\"https://github.com/paleolimbot/wk/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_worldbank.html\">checks<\/a><br><a href=\"https://m-muecke.github.io/worldbank/\">url<\/a><br><a href=\"https://github.com/m-muecke/worldbank/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_yaImpute.html\">checks<\/a><br><a href=\"https://github.com/jeffreyevans/yaImpute\">url<\/a><br><a href=\"https://github.com/jeffreyevans/yaImpute/issues\">url2<\/a>"],[null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#arrow\">resubmitted/pending<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#distr\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#hydroloom\">resubmitted/pending<\/a><\/small>",null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#mboost\">resubmitted/archive<\/a><\/small>",null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#microeco\">resubmitted/inspect<\/a><\/small>",null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#msm\">resubmitted/waiting<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#polyglotr\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#quanteda\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#sched\">resubmitted/pending<\/a><\/small>",null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#spatstat.explore\">resubmitted/inspect<\/a><\/small>",null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#SRTsim\">resubmitted/archive<\/a><\/small>","<small><a href=\"https://nx10.github.io/cransubs/pkg#stplanr\">resubmitted/inspect<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#WeightIt\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null],[0,0,11,13,2,25,0,0,9,0,3,46,0,33,23,0,10,5,0,0,1653,0,68,0,24,9,9,2,31,25,37,4,11,39,23,9,101,26,5,30,18,60,0,16,57,13,1,1,11,0,1,0,0,10,0,441,19,99,3,3,9,0,0,80,0,2,30,0,0,23,0,0,0,1,48,0,3,1,0,2,0,0,22,31,0,200,67,0,0,21,2,0,8,37,0,3,0,10,0,0,52,7,46,0,0,2,14,11,1,0,5,13,0,5,113,0,3,0,30,0,14,0,8],["","","agricolae, bioCancer, cbcTools, choiceDes, conjoint, crossdes, DoE.wrapper, edibble, radiant.design, SensoMineR, STPGA","bios2mds, CluMP, Clustering, cogena, ctc, DiffBind, genefu, Linnorm, MetCirc, moc.gapbk, multiClust, plspm, swamp","archivist.github, modelDown","APCalign, bigrquerystorage, censobr, ClickHouseHTTP, diffdfs, disk.frame, DominoDataR, ebirdst, Eunomia, foundry, gbifdb, geslaR, IGoRRR, MolgenisArmadillo, MSstatsBig, nhdplusTools, overturemapsr, parqr, parquetize, pysparklyr, radiant.data, RcisTarget, sfarrow, SQL, strand","","","ADaCGH2, BGData, bit64, ETLUtils, ff, FLightR, flowClean, symDMatrix, TOC","","datelife, refdb, taxize","alookr, azlogr, Basic4Cseq, BayesPostEst, cape, compcodeR, CondIndTests, convergEU, countsimQC, CSMES, DNLC, EMMAgeo, eseis, factset.analyticsapi.engines, flowPloidy, flux, FuzzyClass, GAMens, GCalcium, gplots, iglu, ImpactEffectsize, infercnv, m2b, MBMethPred, meta4diag, MIMSunit, mirTarRnaSeq, mmaqshiny, nonlinearICP, numbat, opdisDownsampling, OpenSpecy, opGMMassessment, PharmacoGx, pheble, R2Addhaz, RadioGx, RSelenium, RTextTools, SAFARI, sSeq, SVMDO, TFBSTools, ToxicoGx, vulcan","","actel, AeRobiology, amt, ascotraceR, bayes4psy, bpDir, CircMLE, CircOutlier, CircSpaceTime, circularEV, cplots, CSHShydRology, cylcop, FLightR, GInSARCorW, HDiR, isocir, KarsTS, kernplus, monographaR, movementsync, NPCirc, Ostats, peco, pgKDEsphere, phase, PPTcirc, ridgetorus, samc, sharpshootR, SOPIE, tricycle, VoxR","africamonitor, aorsf, blocs, cheapr, deduped, DepecheR, dfms, dggridR, DisImpact, fastverse, FuncDiv, joyn, multistateQTL, osmclass, penppml, plm, QTLExperiment, samadb, summclust, timeplyr, ugatsdb, versus, zoomerjoin","","bigQF, famSKATRC, highMLR, jstable, MAGMWAR, mlma, mlmm.gwas, multisite.accuracy, MXM, PenCoxFrail","actuaRE, ChainLadder, GlmSimulatoR, HeritSeq, Maaslin2","","","Ac3net, accessibility, acdcR, Achilles, actel, ActivePathways, ActivityIndex, actuaRE, ADAMgui, adductomicsR, AdhereR, AdhereRViz, ADImpute, AeRobiology, AF, AFM, africamonitor, agcounts, AggregateR, agriutilities, ahw, AICcPermanova, akc, akiFlagger, aLFQ, alpaca, AlphaBeta, amapGeocode, AMAPVox, AMIM, amplican, amt, Anaconda, animint2, AnxietySleep, aopdata, aorsf, AOV1R, APCI, aPEAR, APIS, aqp, arf, artMS, asciiSetupReader, ascotraceR, aslib, ASpli, ASRgenomics, assertable, atime, AtmChile, atSNP, attrib, AUCell, aum, autoCovariateSelection, autonewsmd, autonomics, AutoPlots, autostats, autostsm, aweSOM, bagged.outliertrees, bakR, bambu, BANDITS, Banksy, baseballr, basecallQC, BatchExperiments, BatchJobs, BatchQC, batchtools, bayesLife, BayesLN, bayesMeanScale, bayesPop, bayesTFR, BBmisc, bbmix, bbotk, bcputility, bdots, bea.R, BEARscc, BEclear, bedr, BeeGUTS, behavr, benford.analysis, bfw, BgeeCall, BgeeDB, bggAnalytics, BGmisc, BiasCorrector, biblionetwork, bife, BIGL, bigreadr, bigsnpr, binancer, binomialRF, binsegRcpp, biomartr, BIOMASS, biopixR, BioR, bioregion, birankr, biscuiteer, bkmrhat, BKTR, BLA, blorr, BMisc, bnbc, bold, BootstrapQTL, BPRMeth, bRacatus, brainGraph, branchpointer, brandwatchR, bratteli, bread, BREADR, bridger2, BRINDA, brmsmargins, bruceR, bsseq, BTSPAS, BTYDplus, bupaR, BuyseTest, c3, CAGEr, CALF, calmr, CamelUp, camtrapR, CaMutQC, card, caretEnsemble, cartogramR, casebase, CAST, CATALYST, categoryEncodings, CatEncoders, causalCmprsk, CausalGPS, cbl, ccdR, ccmap, ccostr, cdparcoord, ceas, celda, CellBarcode, cellbaseR, CelliD, cellKey, CEMiTool, Census2016, Certara.NLME8, Certara.R, cffdrs, cglm, changepoint.influence, ChannelAttributionApp, ChemoSpecUtils, Chicago, chicane, chimeraviz, ChineseNames, ChIPexoQual, ChIPpeakAnno, CHO, chronicle, cicero, circhelp, citationchaser, citecorp, ciw, classifierplots, cleanTS, ClickHouseHTTP, clickstream, clinDataReview, clinspacy, clinUtils, CluMP, Clustering, clustra, CLVTools, cmapR, cmsaf, CNVRanger, CNVScope, COCOA, coder, CoDi, coffee, cohorts, coinmarketcapr, coloc, colocPropTest, colorednoise, Colossus, comorbidity, CompareCausalNetworks, compareDF, COMPASS, ComPrAn, compSPOT, CONFESS, CoNI, conleyreg, consensusDE, consortr, ConsReg, constellation, container, CoOL, cooltools, CooRTweet, cophescan, coRdon, CoreGx, CornerstoneR, corporaexplorer, corpustools, corrcoverage, correspondenceTables, CoSMoS, counterfactuals, coveffectsplot, COVID19, covid19br, covid19india, covidnor, CovRegRF, cppRouting, CRE, creditmodel, crew, crfsuite, CRISPRseek, CropScapeR, crosshap, crossmeta, csa, csalert, csdata, csmaps, CSMES, cstidy, cstime, CSTools, csutil, csv, csvy, ctbi, ctmm, cTRAP, ctsem, ctxR, cubar, curtailment, cusum, cusumcharter, cvAUC, cvms, cwbtools, CWT, cxhull, cyclestreets, cylcop, cytoKernel, CytoML, D3partitionR, daiquiri, daiR, damr, dartR, dartR.base, dartR.popgen, dartR.sim, dartR.spatial, dat, data360r, DataCombine, datadictionary, DataExplorer, dataframeexplorer, datagovsgR, datamods, dataMojo, dataprep, dataPreparation, DataSpaceR, DATAstudio, datazoom.amazonia, datelife, dateutils, dblr, dbnR, dbWebForms, ddtlcm, decompTumor2Sig, deconvR, dedupewider, deepdive, DEFormats, DegNorm, delayed, deltaccd, DemoKin, DeProViR, DEScan2, DescTools, designit, DESpace, detectRUNS, DEWSeq, dfmeta, DGM, DIAlignR, diathor, did2s, Diderot, DiDforBigData, didimputation, DIDmultiplegtDYN, difconet, DifferentialRegulation, DiscoRhythm, DiSCos, discSurv, DisImpact, disk.frame, distantia, distillML, DIZtools, DIZutils, DMCFB, do, DominoEffect, doremi, DoubleML, DQAgui, DQAstats, dQTG.seq, DramaAnalysis, dreamlet, drgee, dropR, droptest, dsos, DTAT, dtplyr, DTSg, dtts, DTwrappers, DTwrappers2, dumbbell, dvmisc, dynamite, dynsurv, DysPIA, DYTE, E4tools, Eagle, easycsv, easyr, easysurv, eatGADS, eatTools, EBPRS, eclust, ecocomDP, ecoregime, edeaR, edl, EdSurvey, eeptools, efdm, EGM, EHR, EIAapi, EIX, eixport, electionsBR, eltr, EmiR, EmissV, ENCODExplorerData, endogeneity, endoSwitch, enmSdmX, ensr, enviGCMS, eoR, Epi, epialleleR, epidm, EpiForsk, epigraHMM, EpiMix, EpiNow2, epiomics, epistasisGA, eplusr, epwshiftr, eSIR, estadistica, EthSEQ, etm, etwfe, eurocordexr, eurodata, europeanaR, EurosarcBayes, eurostat, evapoRe, eventstudyr, evoper, EWCE, exiftoolr, explainer, exploratory, expowo, expss, exvatools, eyeRead, eyeTrackR, ezr, facmodCS, facmodTS, FACT, factR, faers, faersquarterlydata, FairMclus, familiar, FAOSTAT, farrell, fastDummies, fastLink, fastreeR, fastRhockey, fastText, fastverse, fbRads, fcfdr, FCO, fdq, FeatureImpCluster, fedmatch, fedstatAPIr, fedup, felp, fgdr, Fgmutils, fgsea, fhircrackr, FIESTA, FIESTAutils, finch, fishmethods, fitbitViz, FitHiC, flextable, flightsbr, flippant, FLOPART, florabr, flowGraph, flowml, flowWorkspace, Fluidigm, FMAT, fmeffects, FOCI, forceplate, forecastML, forestError, formatdown, formulaic, FORTLS, fossilbrush, FourWayHMM, fplot, fplyr, FRASER, fruclimadapt, fsbrain, FuncDiv, funchir, FuzzyDBScan, fwlplot, fy, GALLO, GARCOM, gasfluxes, gatom, gbp, gcite, gde, gDRcore, gDRimport, gDRtestData, gDRutils, gdxdt, gemma.R, gems, GeneGeneInteR, geneplast, GENESIS, GeneStructureTools, geneXtendeR, genomation, GenomAutomorphism, genomic.autocorr, GenomicDistributions, GenomicDistributionsData, GenomicInteractions, GenomicTools.fileHandler, GenomicTuples, GenoScan, GenVisR, geobr, geocausal, GeoModels, GeoMongo, GeomxTools, geoperu, GEOquery, GeoTcgaData, germinationmetrics, getDTeval, GFE, gfer, gfoRmula, ggbrain, ggcyto, ggdmc, ggetho, ggfacto, GGIR, ggkegg, ggmugs, ggperiodic, ggPMX, ggpval, ggquickeda, ggsom, ggTimeSeries, GHap, githubinstall, gittargets, GladiaTOX, GmicR, GMMAT, gmoTree, gnrprod, goat, gompertztrunc, googleCloudVisionR, GOTHiC, gpboost, GRaNIE, grattan, grattanInflators, greatR, GREENeR, greport, growthcleanr, GRShiny, GSALightning, gsDesign2, GSODR, gtfs2emis, gtfs2gps, gtfsio, gtfsrouter, gtfstools, GUIDEseq, gunsales, gwaRs, gwascatData, gwasforest, GWASinspector, GWASTools, gwid, HACSim, HaDeX, hagis, hahmmr, hal9001, haldensify, hBayesDM, HCD, HCR, hdd, hdpGLM, healthdb, healthyAddress, heatwaveR, heims, hereR, HERON, hesim, heuristicsmineR, HicAggR, HiCBricks, HiCcompare, HiCDCPlus, HiCDOC, hicp, highfrequency, hilldiv, himach, hisse, Hmisc, HMMcopy, HMP2Data, hoardeR, HodgesTools, hoopR, HospitalNetwork, hot.deck, HPLB, hR, HS, hsrecombi, httk, hudr, hurricaneexposure, hutils, hutilscpp, HVT, hwig, hydroloom, hyfo, hypervolume, HYPEtools, i2extras, IBRtools, iCAMP, ICAMS, iCellR, IceSat2R, iCNV, IDEATools, IDmining, ie2misc, ifCNVR, iGC, iimi, imabc, imcRtools, iml, immunarch, immuneSIM, importinegi, imputeMulti, inbreedR, IncDTW, incidence2, inferr, InflectSSP, influential, Information, insurancerating, InterCellar, intervalaverage, inTrees, IOHanalyzer, ips, irg, IrregLong, iscoCrosswalks, isdparser, IsoBayes, isobxr, isotracer, iSTATS, itol.toolkit, itscalledsoccer, ivdoctr, ivtools, jage, JointFPM, joinXL, journalabbr, joyn, jrSiCKLSNMF, JSmediation, jsmodule, jstable, jsTree, jubilee, JWileymisc, kangar00, kdry, kerastuneR, kesernetwork, kgen, kibior, kim, KinSwingR, kisopenapi, KMD, KnockoffScreen, koRpus, kosis, KPC, LabourMarketAreas, labourR, LACE, lacunr, lar, latrend, LCMSQA, ldamatch, ldaPrototype, LDLcalc, ldsr, leafR, leakyIV, LearningStats, lexicon, LexisNexisTools, lgrExtra, libbib, libr, lidR, lightgbm, LilRhino, limexhub, limorhyde2, lineagespot, LinkHD, lipidr, lisaClust, live, LKT, llama, LMPdata, lmtp, loadeR, loci2path, logiBin, LOLA, LOMAR, lookupTable, LOPART, lori, LSPFP, Luminescence, LWFBrook90R, lwqs, LymphoSeq, m5, Maaslin2, Macarron, maditr, maftools, MAGAR, magclass, MAGEE, maicChecks, MANOVA.RM, mapi, marginaleffects, margins, mariner, marked, maser, MAST, matman, mau, mbr, mbRes, mcauchyd, mcboost, Melissa, memisc, MesKit, metabolomicsWorkbenchR, MetaComp, metaforest, metaGE, metagene2, MetaIntegrator, MetAlyzer, MetaScope, meteo, methimpute, methodical, methrix, methylKit, MethylMix, methylPipe, methylscaper, metR, mggd, mgi.report.reader, miceFast, miceRanger, MicrobiotaProcess, microeco, microseq, midfieldr, miesmuschel, miLineage, MIMER, mimi, mind, minfi, MinimumDistance, minSNPs, mipplot, MIRA, mirTarRnaSeq, miscset, missCompare, mixgb, mixIndependR, mknapsack, mlbplotR, mldr.resampling, mlexperiments, mllrnrs, mlr, mlr3, mlr3batchmark, mlr3benchmark, mlr3cluster, mlr3db, mlr3fairness, mlr3fda, mlr3filters, mlr3fselect, mlr3hyperband, mlr3learners, mlr3mbo, mlr3misc, mlr3oml, mlr3pipelines, mlr3resampling, mlr3shiny, mlr3spatial, mlr3spatiotempcv, mlr3summary, mlr3torch, mlr3tuning, mlr3tuningspaces, mlr3verse, mlr3viz, mlrintermbo, mlrMBO, mlsurvlrnrs, mltools, mmaqshiny, mnem, mobileR, MOCHA, modACDC, ModCon, ModelMetrics, modelsummary, MoleculeExperiment, MouseFM, MplusAutomation, mpwR, mrMLM, mrMLM.GUI, mrregression, MsBackendSql, mschart, msImpute, mslp, MSmix, msmtools, MSnID, MSstats, MSstatsConvert, MSstatsLiP, MSstatsPTM, MSstatsShiny, MSstatsTMT, mstate, mstclustering, mstrio, mstudentd, mtb, mulea, multicrispr, multifamm, MultiFit, multiHiCcompare, multilevelcoda, multilevelTools, multistateQTL, multiWGC, MungeSumstats, muscat, musica, musicatk, mutationtypes, mutualinf, mvQuad, mvSUSY, mxnorm, myClim, naaccr, NanoMethViz, naptanr, nc, ndjson, neatR, neatRanges, neatStats, neo4r, NEONiso, neonOS, neonPlantEcology, neonUtilities, nestedcv, net4pg, netgsa, NetLogoR, netSmooth, networkR, netZooR, neuroimaGene, nflfastR, nflplotR, nflreadr, nflseedR, NFLSimulatoR, nlmixr2extra, nlpred, NMcalc, NMdata, NMsim, nngeo, NNS, noaastormevents, noctua, node2vec, noncompliance, nonmem2rx, normfluodbf, nosoi, npcurePK, nRegression, nullranges, numbat, o2geosocial, OBIC, obliqueRSF, occupationMeasurement, octad, octopus, odbr, oddsapiR, ofpetrial, ogrdbstats, OGRE, okxAPI, OmicNavigator, OMICsPCA, omicwas, ompr, OnboardClient, OncoSimulR, ondisc, One4All, openCyto, openEBGM, OpenML, OpenSpecy, opentripplanner, optiSel, ORFhunteR, ORFik, orgR, origami, orthogene, osmclass, OUTRIDER, overviewR, owdbr, owidR, paar, packageRank, pairkat, PakPMICS2014Ch, PakPMICS2014HH, PakPMICS2014HL, PakPMICS2014Wm, PAMpal, PAMscapes, panelaggregation, PanelMatch, panelWranglR, panstarrs, PanViz, paradox, ParBayesianOptimization, parseRPDR, parSim, partCNV, partools, pathviewr, pathwayTMB, patternator, pbox, pcadapt, PCBS, PCMBase, PCMBaseCpp, PCRA, pda, PDATK, pdynmc, PeakSegDisk, pedquant, penaltyLearning, pepr, pepStat, PGRdup, pgTools, phantasus, phantasusLite, PharmacoGx, pheble, phenofit, phenomis, PhenotypeSimulator, phers, phm, phonics, PhyloProfile, phyloseq, pingers, pinyin, pivottabler, pk4adi, pkggraph, pkgnet, PKPDsim, pksensi, PlanetNICFI, plinkQC, plnr, plotgardener, plotly, pmartR, pmd, pmparser, pmxpartab, pointdensityP, poisFErobust, politeness, polle, polmineR, polypharmacy, pomp, POMS, poolfstat, pooling, popEpi, portvine, posologyr, POSSA, potools, POUMM, powerbiR, ppmlasso, PPtreeregViz, pram, pRecipe, precrec, predhy.GUI, prediction, PReMiuM, PreProcessRecordLinkage, preputils, PressPurt, previsionio, pricelevels, proActiv, processmapR, processpredictR, prodlim, prolific.api, PROSPER, Prostar, protGear, protti, prrd, prt, psborrow, PSF, psichomics, psidR, psmineR, psp, psycCleaning, PsychWordVec, ptable, ptairMS, pTITAN2, PTXQC, Publish, PureCN, puremoe, pureseqtmr, pvda, Pviz, pvLRT, pwOmics, qad, QCAcluster, qckitfastq, qdapTools, qgg, qreport, qte, QTL.gCIMapping, QTL.gCIMapping.GUI, qtl2, QuartPAC, quickcheck, quickPlot, r2dii.match, R2DT, r2glmm, R3CPET, r3Cseq, r5r, R62S3, R6causal, RABR, RadioGx, radlibs, RALSA, rAmCharts, RaMS, randnet, randomForestExplainer, randomNames, rangeMapper, RankAggSIgFUR, RapidoPGS, RARfreq, rassta, rasterDT, RAthena, ratios, Ravages, RawHummus, rbacon, RBaseX, rBayesianOptimization, rbi, rbi.helpers, rBiasCorrection, rbin, rBiopaxParser, rblt, Rborist, rbtt, Rcan, RCAS, RcextTools, rchemo, RcisTarget, RcisTarget.hg19.motifDBs.cisbpOnly.500bp, RcmdrPlugin.RiskDemo, rCNV, RcppGreedySetCover, Rcrawler, rdbnomics, rdfp, Rdiagnosislist, Rdice, RDML, rdracor, rE, ReactomeContentService4R, ReactomeGraph4R, readNSx, readtext, realTimeloads, reappraised, rechaRge, reclin2, recorder, RecordLinkage, recount3, REDCapExporter, Rediscover, RegParallel, regport, regtools, ReinforcementLearning, reinsureR, remiod, REndo, replacer, repmis, reproducible, Require, rerddap, resemble, reslr, RESOLVE, respR, restatapi, rfaRm, rfars, rfisheries, rfPred, RFpredInterval, rgbif, RGMQL, rhdf5client, RIbench, RiboCrypt, RiboDiPA, RiboProfiling, ricu, Rilostat, RImmPort, rintcal, rio, rIsing, riskRegression, RITCH, ritis, rlas, rlist, rllama, RLumShiny, RMassBank, rmcfs, rMIDAS, RNewsflow, rnpn, Rnvd3, roads, robCompositions, RobinCar, robis, robotoolbox, ROCket, rODE, rollinglda, rorcid, rPackedBar, rprev, rqdatatable, rqPen, Rrepest, RSauceLabs, rSPARCS, rsparse, rsurveycto, rsyntax, RTCGA, RTCGAToolbox, rtmpt, RTN, RTNsurvival, rtry, rtsdata, RTTWebClient, rTwig, runcharter, rush, RVA, RWDataPlyr, rWikiPathways, RWildbook, rworkflows, RxnSim, rxode2, rxode2parse, ryandexdirect, rzentra, RZooRoH, s3fs, SafeVote, salesforcer, samadb, sampcompR, samplingin, sangeranalyseR, sanityTracker, sansa, SARC, sarp.snowprofile, sarp.snowprofile.alignment, sarp.snowprofile.pyface, saseR, SC2API, scanMiR, scanMiRApp, scATAC.Explorer, scCustomize, SCdeconR, scDiffCom, scifer, scMET, scMitoMut, SCnorm, scoper, scorecard, ScoreEB, scoringutils, scPipe, scraEP, scRseqApp, scruff, scTreeViz, sdcHierarchies, sdcLog, sdcMicro, sdcTable, SDLfilter, SEA, SeaVal, seedr, seeker, segregation, SELF, semgram, semtree, sendigR, sense, sensobol, sentiment.ai, sentimentr, SenTinMixt, sentometrics, sentopics, seqsetvis, SeqVarTools, servosphereR, SEtools, sevenbridges, sevenbridges2, sevenC, sgat, SGP, SGPdata, SHAPforxgboost, shapr, shinyChatR, shinyepico, shinyExprPortal, shinyHugePlot, shinylogs, shinyML, ShinyQuickStarter, shinySbm, siconvr, sigminer, Signac, signatureSearch, siie, SimBu, simcausal, simDAG, SimEngine, simfinapi, simitation, simPH, simphony, simplePHENOTYPES, simPop, simstudy, SimSurvey, simtrial, SingleCellSignalR, singleCellTK, SingleMoleculeFootprinting, SIPmg, sitepickR, siteymlgen, sitree, sitreeE, sjtable2df, skm, skynet, sleepr, slim, smallstuff, SmartEDA, smartsnp, smoothic, snapcount, SNPannotator, SNPhood, snpsettest, socialh, socialmixr, soilDB, SoilHyP, SoilTaxonomy, soiltestcorr, solitude, SOMNiBUS, SOMnmR, somspace, soundgen, spacyr, SpaDES.core, SpaDES.tools, sparrow, sparseFLMM, SPARSEMODr, SparseSignatures, spaths, SpatialCPie, SpatialFeatureExperiment, spatialHeatmap, SpatialOmicsOverlay, spatialrisk, SpATS, spatsoc, spDates, speaq, spectr, spectralAnalysis, spellcheckr, spicyR, SpliceWiz, splitstackshape, splutil, spNetwork, spocc, SPONGE, sport, sqlHelpers, SQMtools, squid, SSHAARP, standartox, staRdom, statar, statcanR, statgenGWAS, statgenIBD, statgenMPP, statgenQTLxT, Statial, Statsomat, stdReg, stenR, SticsRFiles, stm, stochvolTMB, stopdetection, stplanr, stppSim, strvalidator, subscreen, subSeq, SUMMER, suncalc, SUNGEO, SunsVoc, superml, surveyplanning, surveyPrev, surveysd, survMisc, susographql, SVMDO, SVN, SWATH2stats, sweater, SWMPr, symphony, synlet, synr, synthACS, table.express, tablecompare, tabledown, tableMatrix, tabulator, tabxplor, takos, TangledFeatures, targeted, targets, taskscheduleR, tatoo, taxize, taxonomizr, TCA, TCGAbiolinks, tcgaViz, tcpl, tdsc, TeXCheckR, text2vec, textclean, textplot, textrank, textshape, textTinyR, textTools, TheOpenAIR, TideCurves, TideTables, tidyDenovix, TidyDensity, tidyfast, tidyfst, tidyft, tidygapminder, tidyrules, tidytable, tidyterra, tidytransit, tidyvpc, timeplyr, timeseriesdb, TIN, TitanC, tLOH, toolStability, topdowntimeratio, tosca, totalcensus, ToxicoGx, toxSummary, TPAC, TPACData, tpm, TPP, tracee, trackdf, tradestatistics, Trading, TrafficBDE, trainsplit, traits, transcriptogramer, transport, traveltimeR, treediff, treekoR, treemap, TreeMineR, treeshap, triact, TrialEmulation, tRigon, trimmer, tripr, TrumpetPlots, tsbox, tsdistributions, tsgarch, TSstudio, tstests, tstools, TTAinterfaceTrendAnalysis, ttgsea, Tushare, twang, twangRDC, twl, twopartm, txshift, uavRmp, UCell, uci, udpipe, ugatsdb, UKB.COVID19, ukbnmr, ukbtools, UKgrid, Ularcirc, Unico, Uniquorn, uptasticsearch, utilsIPEA, validata, VanillaICE, vardpoor, vein, versioning, versus, VIM, vimp, ViSEAGO, vivainsights, VMDecomp, VOSONDash, vosonSML, vote, votesys, VoxR, vvtableau, walkboutr, wally, weatherOz, WeatherSentiment, WebAnalytics, webtrackR, wehoop, WGScan, WhatsR, whSample, wikitaxa, wilson, word.alignment, worrms, wpa, wpp2017, wpp2019, wTO, wyz.code.metaTesting, wyz.code.offensiveProgramming, wyz.code.rdoc, wyz.code.testthat, xefun, xesreadR, xgb2sql, xgboost, XString, xVA, yamss, youngSwimmers, zebu, zeitgebr, ZIprop, zoltr, zoomGroupStats","","ANCOMBC, ANOVAShiny, ANOVAShiny2, AntibodyTiters, atable, bfsMaps, catfun, CGPfunctions, ClusTCR2, cryptography, dad, DescrTab2, DescToolsAddIns, DGEAR, drugprepr, DynForest, epifitter, geosimilarity, gerbil, GNOSIS, gosset, gpuMagic, HLAtools, IFAA, ILoReg, InteRD, jack, matRiks, mcradds, meteo, miceRanger, mlmts, ModTools, morphomap, mrbayes, negligible, nmrrr, npcurePK, parTimeROC, pbANOVA, pointRes, popstudy, ProAE, qspray, rcompanion, RGMM, robin, RobRegression, rties, rTwig, SARC, SCOPE, SEIRfansy, Semblance, sendigR, sevenbridges2, SGCP, smvgraph, SSHAARP, tabxplor, tidyrules, TipDatingBeast, Toothnroll, tspredit, vvdoctor, WaverideR, wrappedtools, xgxr","","distrDoc, distrEllipse, distrEx, distrMod, distrRmetrics, distrSim, distrTeach, EDOIF, fastglmpca, funStatTest, genlogis, isobar, mediationsens, meteR, miCoPTCM, moderate.mediation, RandVar, RcmdrPlugin.TeachStat, RobAStBase, RobExtremes, RobLox, RobLoxBioC, ROptEst, scMerge","distrDoc, distrEllipse, distrMod, distrTeach, RandVar, RcmdrPlugin.TeachStat, RobAStBase, RobExtremes, ROptEst","CureDepCens, DepCens, exdqlm, ExGaussEstim, longitudinalAL, MfUSampler, NTS, RGAP, TPMplt","nonmem2rx, rxode2parse","ANCOMBC, biosensors.usc, cassowaryr, CircMLE, Compositional, EDMeasure, EnergyOnlineCPM, etree, fAssets, GiniDistance, HellCor, hySAINT, itdr, kpcalg, linkspotter, MBC, metrica, mgc, miRLAB, miRSM, MVN, MXM, nda, pgraph, PhyloProfile, scperturbR, semidist, Statsomat, stops, TDAkit, VariableScreening","Bergm, BFpack, bigergm, btergm, dnr, EpiModel, ergm.count, ergm.ego, ergm.multi, ergm.rank, ergMargins, ergmgp, ergmito, fergm, latentnet, mlergm, nemBM, netmediate, RDS, statnet, statnetWeb, tergm, tergmLite, VBLPCM, xergm.common","cape, climwin, criticality, ddst, DEEVD, ensembleMOS, evt0, evtclass, ExtremalDep, extremeIndex, ExtremeRisks, extremeStat, extremis, flood, GJRM, HydroPortailStats, IDF, INQC, intamap, lmboot, lookout, metafuse, mvPot, PWMEnrich, RecordLinkage, Renext, ReturnCurves, RobExtremes, ROlogit, rSHAPE, rtdists, RtsEva, SimCorMultRes, SpatialGEV, truncdist, tsxtreme, weird","mcmcderive, mcmcr, nlist, term","consICA, fGarch, gogarch, isva, MineICA, monocle, oposSOM, pcalg, psichomics, sincell, TSCAN","admix, asm, carSurv, crossmeta, DEP, DiffCorr, DMCHMM, DNLC, EGRNi, ERP, FitHiC, GeneCycle, GeneNet, ggisotonic, GSRI, HDMT, HybridMTest, IHW, IHWpaper, InvStablePrior, les, medScan, meshr, metapone, MixTwice, NetworkToolbox, nlnet, oposSOM, OVESEG, qgraph, rags2ridges, rnmamod, Rtpca, sda, SimSeq, SOHPIE, st, TBEST, vaccine","ADaCGH2, BGData, bootSVD, cn.farms, crlmm, dualScale, ETLUtils, geoTS, Haplin, LOMAR, Macarron, MAGAR, MinimumDistance, oligo, oligoClasses, omicwas, propagate, RecordLinkage, rmgarch, rmio, RnBeads, SFtools, symDMatrix","eseis, matrixProfile, MixfMRI, ohun, PAMmisc, pmultinom, qusage, SuperGauss, warbleR","adamethods, agghoo, amt, Anthropometry, atakrig, autoFRK, bapred, BayesNSGP, bigdatadist, Canek, CAST, cccd, cicero, clap, cmsaf, cmsafops, colors3d, cooltools, crmReg, customizedTraining, daltoolbox, damAOI, deepgp, densityClust, DepecheR, DSWE, ECoL, EmbedSOM, evclass, evclust, exdqlm, ExNRuleEnsemble, fCI, first, fpcb, freqparcoord, gainML, GloScope, GpGp, GPvecchia, gstat, GWmodel, HDoutliers, HellCor, hextri, Hmsc, ider, imbalance, IndepTest, intRinsic, KCSKNNShiny, KNNShiny, ks, less, LilRhino, LOMAR, mateable, MBC, meshed, metamer, miceRanger, missRanger, mlquantify, MOEADr, msImpute, musclesyneRgies, noisemodel, OkNNE, outForest, PINSPlus, pmlbr, probout, pRoloc, RaceID, RaSEn, rchemo, regtools, rgnoisefilt, sambia, sansa, scCAN, scgwr, Sconify, semidist, shattering, SmartMeterAnalytics, smotefamily, SMOTEWB, SpatialDDLS, spmoran, SRTsim, stray, supercompress, TDA, toweran, trafo, uwot, varrank, VoxR, waywiser, xkcdcolors","Ball, bandit, CausalGAM, CausalGPS, DeepLearningCausal, diversityForest, drpop, dtComb, flexOR, GAMens, glam, MapGAM, mixcure, mtsdi, MultiKink, netmediate, PAMpal, precmed, ProDenICA, RPPanalyzer, scalpel, singR, sufficientForecasting, SuperLearner, vaccine, xpose4","BayesianTools, DHARMa, FamAgg, PopGenReport, pould","aftgee, batchtma, CausalModels, CorrBin, CrossCarry, CRTspat, discSurv, eventglm, geeasy, geecure, ipw, IrregLong, jsmodule, jstable, LGRF, LiquidAssociation, MESS, miLineage, MRTAnalysis, MXM, pseudo, recurrentpseudo, reportRmd, SLTCA, snSMART, spass, spind, swgee, tern.gee, trajmsm","basket, calibrar, chillR, ConsReg, cops, corHMM, crmPack, eulerr, FBMS, gripp, hisse, MSTest, nearfar, netcom, nlrx, PortfolioAnalytics, spatialrisk, TropFishR","alphashape3d, archetypal, archiDART, Arothron, BAT, bayesGAM, betapart, blackbox, caRamel, cati, ClaimsProblems, contoureR, CoopGame, curvHDR, ddalpha, debCAM, DepthProc, DiceView, dispRity, EthSEQ, EvolutionaryGames, FD, FEA, fundiversity, gensphere, GeomArchetypal, gMOIP, GWSDAT, habtools, HybridMicrobiomes, hypervolume, latentcor, Maeswrap, MBHdesign, mFD, modQR, Momocs, morphomap, mrfDepth, MultBiplotR, mvGPS, mvmesh, nonparaeff, pavo, pdfCluster, ProjectionBasedClustering, quadmesh, randomGLM, RegCombin, RepeatedHighDim, RoundAndRound, Rpolyhedra, spaMM, SpatialfdaR, STEPCAM, TCIU, Toothnroll, ucie, Umatrix, VoxR","","cv, DEGRE, GLMMcosinor, glmmSeq, gmvjoint, iccCounts, lefko3, LongDat, Maaslin2, mecoturn, mtscr, muscat, pastaPlot, pdR, predictmeans, twosigma","andrews, arrangements, bigBits, bratteli, cgwtools, Clustering, ClusterR, collatz, CommEcol, confreq, contFracR, convertBCD, cooccur, CryptRndTest, CVXR, cyclotomic, distcomp, DPQmpfr, FLAME, FOCI, FunWithNumbers, gmpoly, go2bigq, homomorpheR, ICRanks, iterpc, jack, kantorovich, ldamatch, m2r, MSmix, mvPot, nos, OPDOE, partitions, PMCMRplus, polyhedralCubature, qspray, rain, RankProd, RationalMatrix, ratioOfQsprays, rbch, rbtc, RcppAlgos, RcppBigIntAlgos, resultant, Rfractran, Rmpfr, robustrao, slr, spaMM, STraTUS, symbolicQspray, treebalance, unifDAG, Zseq","bestglm, DMRnet, fsemipar, geoGAM, kko, mixedLSR, MTAFT, naivereg, NVCSSL, PCLassoReg, refund, sparseGAM, SSGL","gtfs2gps","nhdplusTools","cta, eyelinker, genomeIntervals, girafe, MaximinInfer, mombf, prototest, RobustIV, selectiveInference, spacetime, stacomiR","","cancerscreening","","","arfima, artfima, beyondWhittle, ConfZIC, garma, gmwmx, sarima, slm, tscopula, tscount","","abcrf, ablasso, ACNE, ActivityIndex, adproclus, airpart, AlpsNMR, AMARETTO, amplican, antiProfiles, apollo, aroma.affymetrix, aroma.cn, aroma.core, aroma.light, aSPU, atena, autonomics, bahc, Banksy, BASiCS, BatchQC, BayesfMRI, BayesianPlatformDesignTimeTrend, BayesTwin, baystability, bcf, bigstep, BindingSiteFinder, bingat, binsreg, BioNERO, biscuiteer, blackbox, BloodGen3Module, bmm, bnbc, bnclassify, bnem, BREADR, BRISC, brms, BSgenome, bspcov, BSW, bulkAnalyseR, bumphunter, calmate, Canek, CARBayesST, cardelino, CATALYST, causalOT, CDI, celda, CelliD, cellmigRation, cellWise, CEMiTool, ChAMP, Chicago, ChIPpeakAnno, CIPerm, cliqueMS, Clomial, clusterExperiment, clustifyr, cmapR, cna, cnaOpt, CNVScope, coconots, coin, cointmonitoR, cointReg, cola, compcodeR, ComplexHeatmap, condiments, conformalbayes, conquer, consensus, consensusOV, cophescan, CopulaGAMM, corrcoverage, cosinor2, countts, CpGFilter, CRISPRball, crlmm, crossmeta, csdR, cSEM, CureDepCens, cvCovEst, cytofQC, cytomapper, cytoMEM, DAMOCLES, DCATS, DCLEAR, ddtlcm, dearseq, decompr, deconvR, DELocal, DeMixT, DepCens, DepecheR, DepInfeR, DEqMS, DESeq2, DHS.rates, diffUTR, Dino, disbayes, DiscoRhythm, DisHet, DMCFB, dmrseq, DMtest, Doscheda, doseR, dplR, DSWE, dwp, eaf, easier, ecospat, eGST, EMDomics, EMMAgeo, ENmix, EnrichedHeatmap, EpiDISH, epimutacions, epistasisGA, eseis, evaluomeR, EventPointer, ExceedanceTools, ExCluster, expss, extraChIPs, FADA, fairsubset, familial, FastPCS, FastRCS, FateID, FEAST, felp, fergm, fipp, fishpond, flexmsm, flocker, flowCore, flowGraph, flowWorkspace, fMRItools, footBayes, FRASER, funcharts, funtooNorm, GAD, gcapc, gDx, GenEst, geneticae, GeneTonic, genomation, GenomAutomorphism, geocmeans, geva, ggdmc, GGPA, GJRM, glmGamPoi, GLMMadaptive, GNET2, GPGame, GRaNIE, graper, graphicalMCP, GUIDEseq, GUniFrac, Gviz, gwasurvivr, GWE, HACSim, haldensify, Harman, hbamr, HDSpatialScan, hermes, HERON, hipathia, Hmsc, Holomics, HonestDiD, ICBioMark, icpack, IMIFA, ImpactEffectsize, incidental, InDisc, iNETgrate, InfiniumPurify, Infusion, iPath, iperform, iraceplot, JMbayes2, JointFPM, jointseg, kgschart, kissDE, kpmt, latrend, LDM, ldsep, lemur, LFDREmpiricalBayes, lfproQC, liminal, localgauss, locaR, loo, LS2Wstat, lspartition, ltmle, Luminescence, M3Drop, magpie, maic, MatrixGenerics, matrixTests, mbsts, mcmcsae, MEAL, MEDseq, memes, metabCombiner, MetabolicSurv, metagenomeSeq, metamicrobiomeR, MetaNeighbor, methrix, MethylAid, methylumi, mgcViz, miclust, microbiomeExplorer, MicrobiomeStat, miesmuschel, migest, miloR, MiMIR, MinimumDistance, miRecSurv, mixOmics, mnem, moanin, MOCHA, MoEClust, monocle, MOSim, motifbreakR, motifTestR, mrfDepth, msImpute, mult.latent.reg, multinomialLogitMix, multistateQTL, multivarious, muscat, mvpd, mzipmed, NanoStringDiff, nbconv, nearBynding, nempi, nestedcv, netZooR, neurobase, nn2poly, nnSVG, NormalyzerDE, nparMD, obfuscatoR, occupancy, omicplotR, omicsPrint, omicsViewer, omicwas, OncoBayes2, OptHoldoutSize, OptimalDesign, Ostats, OUTRIDER, pandaR, pARI, PathoStat, PDATK, pecora, penppml, PepsNMR, pgKDEsphere, phosphonormalizer, Pigengene, PINSPlus, pipeComp, PlackettLuce, pmp, POMaSPU, posterior, priorsense, ProteoMM, PSCBS, QDseq, Qest, Qindex, RAC, RaceID, rADA, RadioGx, ramr, RandomForestsGLS, randomGLM, randomizationInference, RBesT, RCarb, recommenderlab, reconsi, Rediscover, reservr, RGCCA, rhierbaps, RiboDiPA, RJcluster, RmodR, RnBeads, robustlm, Rogue, RolDE, rpm, RseqQC, rsvddpd, RTCC, rTLsDeep, rtrend, rvec, samr, SANvi, scDataviz, scDesign3, scDHA, SCFA, scISR, scmap, scMET, scMultiSim, scone, scp, scPCA, scPloidy, scpoisson, sctransform, sechm, seeds, sensobol, seqArchR, Seurat, SGP, SIAMCAT, SICtools, signifinder, simona, simPIC, simPop, sindyr, singleCellTK, singscore, sizeMat, slasso, slingshot, SpaceMarkers, SPARRAfairness, SparseArray, sparseMatrixStats, spathial, spatzie, spdesign, splatter, SpliceWiz, SPOTlight, spqn, SQI, SRTsim, stabiliser, stability, stagedtrees, stapler, staRdom, statar, stm, stoppingrule, summarytools, sumR, surveyPrev, susieR, sva, target, TaxaNorm, TCA, templateICAr, tenXplore, ThurMod, TopDom, topGO, tradeSeq, TRESS, ttScreening, UBayFS, Unico, UPG, VanillaICE, variancePartition, VaSP, Voyager, vsclust, wateRmelon, WaverideR, weakARMA, WeMix, WGC, winputall, Wrench, yarn","biospear, boostrq, bujar, carSurv, censored, DIFboost, EnMCB, expectreg, FDboost, gamboostLSS, gamboostMSM, GeDS, geoGAM, gfboost, InvariantCausalPrediction, mgwrsar, sgboost, tbm, visaOTR","adaptsmoFMRI, animalEKF, bacr, bartcs, Bayenet, bayesanova, BayesDissolution, bayesDP, BayesESS, BayesianFactorZoo, BayesLCA, BayesMFSurv, bayesmove, BayesPostEst, bayest, BCClong, Bergm, bgumbel, BHAI, bmet, BMRMM, bmstdr, BMTAR, brxx, CARBayes, CARBayesST, ccpsyc, clickb, coarseDataTools, ConsReg, DCATS, DPtree, epigenomix, evolqg, factor.switching, faers, FAVAR, fdrDiscreteNull, frenchFISH, fungible, gJLS2, GraphAT, hdpGLM, HKprocess, HMMmlselect, Hmsc, lchemix, llbayesireg, lsirm12pl, manet, maxcombo, MBSGS, MBSP, mbsts, MCPAN, Melissa, mfa, mHMMbayes, midas2, miscF, miWQS, mixAR, MixSIAR, multilevelmediation, mvst, netcmc, NetworkChange, NMADiagT, noncomplyR, NVCSSL, optDesignSlopeInt, pCalibrate, PICBayes, PLMIX, poolHelper, popdemo, PortRisk, prolsirm, PRSPGx, qgg, quid, Rdta, riAFTBART, robustsae, RSGHB, scorematchingad, sectorgap, SensMap, SizeEstimation, sizeMat, sorocs, sparsereg, spikeSlabGAM, ssmsn, StReg, SurrogateBMA, telescope, twl, uskewFactors","file2meco, meconetcomp, mecoturn","EventPredInCure, icenReg, interval","DoubleML, GenericML, highMLR, mlr3fairness, mlr3shiny, mlr3verse, sense, SIAMCAT, spFSR","","","ammiBayes, ARCensReg, ARDL, ASSET, ASSIGN, BayesianTools, BayesRGMM, biodosetools, breakpoint, bsam, ccostr, ccrs, CensSpatial, CEoptim, cfda, clustMD, CopCTS, cosa, decisionSupport, DFIT, difNLR, drugdevelopR, EAinference, eatRep, ei, eiPack, elect, endoSwitch, epiphy, EventPredInCure, extrafrail, Families, frailtyEM, gems, gmnl, grmsem, hesim, hmsr, horseshoenlm, httk, hysteresis, ICcalib, interactionR, interactionRCS, ltm, methInheritSim, msmtools, OneSampleMR, optBiomarker, oro.pet, PanCanVarSel, parfm, Petersen, pexm, PRIMEplus, Rchoice, riAFTBART, RMark, rriskDistributions, rtdists, scDECO, score, semmcmc, SightabilityModel, simExam, SizeEstimation, sparsereg, SPAS, spBFA, spCP, stochLAB, Surrogate, surrosurv, tradepolicy, TrenchR, TropFishR, varTestnlme, VirtualPop, womblR, wxgenR","","ratematrix, RPANDA","AteMeVs, biglasso, FarmSelect, fastTS, glmmPen, glmvsd, HCTR, hdmed, hdnom, HIMA, HSDiC, L2E, mcb, mombf, mrMLM, mrMLM.GUI, msaenet, naivereg, nonprobsvy, PDMIF, polywog, RCTS, simode, SIS, SOIL, sparseR, sparsevar, SPSP, srlTS, stabiliser","","","clusterExperiment, DBNMFrank, debCAM, diceR, fabisearch, hNMF, InterSIM, IntNMF, miRSM, musicatk, MutationalPatterns, pcaExplorer, projectR, SCdeconR, scITD, shattering, sigminer, signeR, SomaticSignatures, SparseSignatures, spectralAnalysis, SpNMF, SPOTlight","","","","IRkernel","apex, babette, beastier, BoSSA, BSS, coalescentMCMC, corHMM, datelife, DCLEAR, DIAlignR, dispRity, dowser, EnvNJ, genBaRcode, haplotypes, hisse, immunarch, indelmiss, ips, LDM, LymphoSeq, markophylo, MesKit, microbial, MonoPhy, MSCquartets, nodeSub, OUwie, paleotree, pcmabc, philr, phyloregion, phytools, POMS, ppgm, rapidphylo, recluster, RPANDA, rwty, scistreer, SeqFeatR, sketching, STraTUS, tanggle, taxize, TML, treespace, windex","","CALANGO, devtools, preferably","vvconverter","","ciuupi2, rxode2","","","ASMap, cape, ctl, dQTG.seq, fssemR, lineup, LinkageMapView, mppR, netgwas, PopVar, qpgraph, QTL.gCIMapping, QTL.gCIMapping.GUI, qtl2convert, qtlcharts, qtlhot, qtlnet, rNeighborQTL, statgenSTA, trigger, wgaim, xoi","AutoPlots, conText, corpustools, doc2concrete, eHDPrep, grafzahl, highlightr, keyATM, LDABiplots, LDAShiny, LexisNexisTools, LSX, newsmap, oolong, poldis, politeness, quanteda.textmodels, quanteda.textstats, rainette, RNewsflow, seededlda, sentometrics, sentopics, stm, stylest2, sweater, textstem, tosca, Twitmo, ulex, wordmap","","aLFQ, alookr, aloom, aRtsy, assignPOP, ATACseqQC, AutoScore, aVirtualTwins, BAGofT, banter, bartMachine, BASiNET, BASiNETEntropy, binomialRF, biosigner, BPRMeth, BWGS, C443, CALIBERrfimpute, cem, CERFIT, CiteFuse, classmap, CompMix, CondIndTests, conformalClassification, consensusOV, countSTAR, cpfa, cpt, CRE, crisprScore, curvir, cytofQC, daltoolbox, DaMiRseq, DeepLearningCausal, dendroTools, dmlalg, DMTL, driveR, easyalluvial, ebmc, EFS, EnsembleBase, EpiSemble, EPX, evaluomeR, ExpImage, FateID, FeatureTerminatoR, FFTrees, ForestDisc, FSelector, fuzzyforest, gamclass, GB5mcPred, geomod, ggRandomForests, ggscidca, GIC, GiniDistance, gINTomics, GMDH2, GOexpress, HaploCatcher, iCheck, iimi, ImHD, ImML, interpretR, ivitr, JQL, kernelFactory, KnowSeq, less, LINselect, LongituRF, LSDirf, m2b, MBMethPred, metaEnsembleR, MethPed, microbial, mikropml, missForest, MixRF, mlearning, mlmts, mlquantify, Modeler, ModelMap, ModTools, Moonlight2R, MoonlightR, MPV, MSclassifR, mslp, mt, NCSampling, needmining, nempi, NicheBarcoding, nlcv, nlnet, nonet, nonlinearICP, nproc, omu, OncoSubtype, OOBCurve, OpEnHiMR, optBiomarker, optimalFlow, ORFhunteR, ORION, OTE, PAA, pathwayTMB, pdi, permimp, pheble, PheVis, POMA, pomodoro, POPInf, practicalSigni, preciseTAD, PredPsych, pRF, pRoloc, psica, quantregForest, RaceID, randomForestExplainer, RandomForestsGLS, randomForestVIP, rasclass, RaSEn, RclusTool, regclass, RegEnrich, REPTILE, RFclust, RFlocalfdr, rfPermute, rfVarImpOOB, rfviz, rgnoisefilt, riAFTBART, Ricrt, rminer, RPtests, rQSAR, RSDA, RTextTools, Rtrack, scmap, scorecardModelUtils, scReClassify, SDMtune, seer, sigInt, signeR, simputation, smdi, soilassessment, sovereign, SpatialML, spikeslab, spm, spm2, SPONGE, SSDM, Sstack, stablelearner, statTarget, statVisual, steprf, STGS, SweepDiscovery, SynDI, synthesizer, synthpop, tashu, TIGERr, traineR, tRigon, TriMatch, TSPred, UBL, varSelRF, visaOTR, vita, VMDML, VSURF, vtype, WaveletRF, waves, xLLiM","alphashape3d, AnalysisLin, anticlust, BRISC, CCMMR, celestial, condiments, ElevDistr, EntropyMCMC, evtclass, FOCI, gjam, GloScope, GrabSVG, HGC, hoodscanR, hopkins, hydroloom, iCellR, imbalanceDatRel, infercnv, insect, isokernel, kanjistat, kldest, KMD, KPC, langevitour, ldbod, leapgp, less, LOMAR, lookout, magicaxis, maotai, mastif, monocle, nonprobsvy, ProbitSpatial, ptools, RapidPolygonLookup, rcccd, Rdimtools, recalibratiNN, rliger, RsimMosaic, saasCNV, scBSP, scINSIGHT, SCORPION, SCORPIUS, Seurat, SMOTEWB, spAbundance, spaSim, SpaTopic, SPIAT, spNNGP, spOccupancy, sr, SSLR, SUNGEO, symphony, themis, tidySEM, uSORT, Vicus","","","calibrar, CatPredi, cops, DiceOptim, DynTxRegime, flipr, glmnetr, GPareto, KrigInv, LSDsensitivity, mlpwr, NFCP, otrKM, pbcc, pGPx, QTOCen, quantoptr, RobPer, Synth, tidysynth, ZIprop","polite, ralger","","BIEN, birdscanR, lucas, NetSimR, occCite, rpostgis, RRedshiftSQL, sejmRP","bigdatadist, BMA, cellWise, crmReg, DEGraph, DepthProc, flowBeads, flowMeans, flowMerge, flowStats, funcharts, geneticae, groc, GSE, ICSClust, ICSShiny, MAINT.Data, mcrPioda, MVTests, pmartR, QuadratiK, RcmdrPlugin.PcaRobust, riv, RobAStBase, robCompositions, RobStatTM, robust, robustfa, rospca, rrcov, rrcov3way, rrcovHD, rstanbdp, saseR, SC3, statTarget, TCIU","","BEACH, rfars, stpm","","bmem, bmemLavaan, dsem, faoutlier, fungible, MBESS, phylosem, qtlnet, semPlot, stablespec","","","BAM, BasketballAnalyzeR, BGGM, bipartite, bootcluster, btergm, CEMiTool, CEoptim, CIN, CluMSID, ClusTCR2, concorR, conserveR, cssTools, dnr, ecoCopula, econet, EGAnet, EloRating, ergMargins, fglsnet, ftsspec, gatom, GGMncv, GGMnonreg, ggnetwork, GGPA, haplotypes, intensitynet, ITNr, keyplayer, latentnet, Linkage, metanetwork, mlergm, MRPC, ndtv, NetCluster, netdiffuseR, netmap, netmediate, netplot, networkABC, NetworkExtinction, relevent, sfc, spaceNet, SSrat, statnet, statnetWeb, VBLPCM, WOTPLY","CAGEr, chemometrics, CINdex, earlywarnings, FateID, som.nn, wTO","dbmss, ecespa, ETAS, ForestGapR, funkycells, geocausal, globalKinhom, highriskzone, idar, IDSpatialStats, lacunaritycovariance, lgcp, lisaClust, mxfda, NTSS, onpoint, ppmlasso, rcarbon, replicatedpp2w, scFeatures, scider, selectspm, Seurat, shar, smacpod, SpaceMarkers, sparr, spatialTIME, spatstat, spatstat.gui, spatstat.Knet, spatstat.linnet, spatstat.local, spatstat.model, spatsurv, SPIAT, spicyR, SPUTNIK, Statial, stlnpp, stopp, stpp, TCIU, tectonicr, trip, ttbary","","","agricolaeplotr, pct","Bayenet, causal.decomp, glmtoolbox, holiglm, kSamples, NSM3, PHENIX, PMCMRplus, psbcGroup, PTSR, sft, survcomp, ufs, vartest","AncestryMapper, bigmds, bigQF, BSPBSS, cellWise, lori, mogsa, multivarious, pcLasso, RItools, Rssa","SFSI","","ahw, mets, mixcure, pec, riskRegression","corral, cylcop, CytoMDS, drf, Ecume, epiphy, GMPro, kanjistat, LOMAR, nearBynding, optimalFlow, otinference, SBCK","","AHMbook, AICcmodavg, AMModels, nmixgof, ubms","AICcmodavg, AnaCoDa, assessor, AutoTransQF, Bayenet, bayesCureRateModel, BayesGOF, binspp, BioPET, CAGEr, CalcThemAll.PRM, calibmsm, Cascade, casebase, casper, cg, childsds, cicero, ClusPred, collin, CompDist, corncob, Countr, crlmm, crov, deepSNV, DEsingle, difNLR, DIFtree, discFA, discSurv, DPBBM, Dpit, drord, dwp, EffectStars, EffectStars2, eoa3, EpiForsk, ERPeq, EurosarcBayes, fiberLD, fitPS, fmx, ForestGapR, FRASER, GeoModels, GJRM, glm.predict, glmxdiag, GlobalAncova, gofcat, ib, iccCounts, jmdem, JWileymisc, l1ball, lefko3, list, MADSEQ, marp, MigConnectivity, mixvlmc, monocle, multgee, netmediate, new.dist, ordDisp, orders, PAFit, PAsso, Patterns, ph2bye, PLreg, powerTCR, proteus, PScr, PureCN, r3Cseq, RCM, refitME, regclass, RelDists, renz, robmixglm, rSHAPE, rxSeq, sads, sampleSelection, scMET, scRepertoire, scShapes, semisup, signeR, SimCorrMix, SimMultiCorrData, SLTCA, smcfcs, SOMNiBUS, sparsereg, ssdtools, sssc, svyVGAM, TBFmultinomial, TempStable, tpn, TPP, uSORT, vanquish, VGAMdata, VGAMextra, zipfextR, ZIR1","","CIMTx, MatchThem, mvGPS","","BioM2, BioNERO, BioR, CEMiTool, csdR, diffcoexp, DiPALM, DrDimont, eclust, fastLiquidAssociation, FREEtree, GmicR, GWE, iNETgrate, Macarron, MCbiclust, miRSM, MODA, MRPC, multiWGC, netboost, NetSAM, Patterns, Pigengene, RegEnrich, scpoisson, seq2pathway, SPsimSeq, TIN, TOmicsVis","","apcf, arcgisplaces, ceramic, fasterize, geoarrow, geos, neotoma2, PROJ, redistmetrics, rgbif, rosm, s2, spocc, wkutils","","ale, ALEPlot, climetrics, gamlss.ggplots, intrinsicDimension, NCSampling, neuroim, stlplus"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>deadline<\/th>\n      <th>package<\/th>\n      <th>version<\/th>\n      <th>maintainer<\/th>\n      <th>links<\/th>\n      <th>event<\/th>\n      <th>reverse packages (n)<\/th>\n      <th>reverse packages<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":50,"order":[[0,"asc"],[6,"desc"],[1,"asc"]],"columnDefs":[{"className":"dt-right","targets":6},{"name":"deadline","targets":0},{"name":"package","targets":1},{"name":"version","targets":2},{"name":"maintainer","targets":3},{"name":"links","targets":4},{"name":"event","targets":5},{"name":"reverse packages (n)","targets":6},{"name":"reverse packages","targets":7}],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>




<div id="quarto-appendix" class="default"><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><h2 class="anchored quarto-appendix-heading">Footnotes</h2>

<ol>
<li id="fn1"><p>When a package is <em>archived</em>, the archived version is still available in the <a href="https://cran.r-project.org/src/contrib/Archive/">CRAN archive area</a> together with all previous versions.<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




<footer class="footer"><div class="nav-footer"><div class="nav-footer-center"><div class="toc-actions"><ul><li><a href="https://github.com/cranhaven/cranhaven.org/blob/main/dashboard-at-risk.qmd" class="toc-action"><i class="bi bi-github"></i>View source</a></li><li><a href="https://github.com/cranhaven/cranhaven.org/edit/main/dashboard-at-risk.qmd" class="toc-action"><i class="bi empty"></i>Edit this page</a></li><li><a href="https://github.com/cranhaven/cranhaven.org/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></div></div></footer></body></html>