<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.40">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Dashboard: Packages At-Risk of Being Archived – CRANhaven</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-549806ee2085284f45b00abea8c6df48.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-5c67a7444eebc356e54fdce0a63e8750.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">CRANhaven</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-articles" role="link" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Articles</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-articles">    
        <li>
    <a class="dropdown-item" href="./dashboard-live.html">
 <span class="dropdown-text">Dashboard: Recently Archived CRAN Packages</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./dashboard-at-risk.html">
 <span class="dropdown-text">Dashboard: Packages At-Risk of Being Archived</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./cran-archiving-stats.html">
 <span class="dropdown-text">Study: Many Archived Packages Return to CRAN</span></a>
  </li>  
    </ul>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">About</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/cranhaven/cranhaven.org/"> <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar"><div class="quarto-margin-header"><div class="margin-header-item">
<p><a href="https://cranhaven.r-universe.dev/builds"><img src="https://cranhaven.r-universe.dev/badges/:total" alt="CRANhaven number of packages"></a></p>
</div></div>
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Dashboard: Packages At-Risk of Being Archived</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<style>
tr { vertical-align: top; }
</style>
<p>The below table lists 88 R packages that are at risk of being archived<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a> on CRAN, because they have outstanding unresolved issues.</p>
<p>Some of them might have already submitted a new version to fix the issues identified by CRAN (check the event column).</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="live-dashboard" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="live-dashboard">{"x":{"filter":"none","vertical":false,"data":[["2025-03-06<br><small>(in 18 days)<\/small>","2025-03-02<br><small>(in 14 days)<\/small>","2025-02-16<br><small>(today)<\/small>","2025-02-26<br><small>(in 10 days)<\/small>","2025-02-26<br><small>(in 10 days)<\/small>","2025-03-02<br><small>(in 14 days)<\/small>","2025-03-06<br><small>(in 18 days)<\/small>","2025-02-26<br><small>(in 10 days)<\/small>","2025-02-26<br><small>(in 10 days)<\/small>","2025-02-18<br><small>(in 2 days)<\/small>","2025-03-08<br><small>(in 20 days)<\/small>","2025-03-02<br><small>(in 14 days)<\/small>","2025-02-22<br><small>(in 6 days)<\/small>","2025-03-08<br><small>(in 20 days)<\/small>","2025-02-20<br><small>(in 4 days)<\/small>","2025-02-23<br><small>(in 7 days)<\/small>","2025-03-08<br><small>(in 20 days)<\/small>","2025-02-16<br><small>(today)<\/small>","2025-02-20<br><small>(in 4 days)<\/small>","2025-03-07<br><small>(in 19 days)<\/small>","2025-02-19<br><small>(in 3 days)<\/small>","2025-02-16<br><small>(today)<\/small>","2025-02-22<br><small>(in 6 days)<\/small>","2025-02-16<br><small>(today)<\/small>","2025-03-07<br><small>(in 19 days)<\/small>","2025-03-06<br><small>(in 18 days)<\/small>","2025-03-08<br><small>(in 20 days)<\/small>","2025-03-08<br><small>(in 20 days)<\/small>","2025-02-21<br><small>(in 5 days)<\/small>","2025-03-07<br><small>(in 19 days)<\/small>","2025-02-26<br><small>(in 10 days)<\/small>","2025-02-18<br><small>(in 2 days)<\/small>","2025-03-07<br><small>(in 19 days)<\/small>","2025-02-17<br><small>(in 1 day)<\/small>","2025-02-24<br><small>(in 8 days)<\/small>","2025-02-26<br><small>(in 10 days)<\/small>","2025-03-07<br><small>(in 19 days)<\/small>","2025-03-07<br><small>(in 19 days)<\/small>","2025-03-07<br><small>(in 19 days)<\/small>","2025-02-24<br><small>(in 8 days)<\/small>","2025-02-17<br><small>(in 1 day)<\/small>","2025-03-13<br><small>(in 25 days)<\/small>","2025-02-20<br><small>(in 4 days)<\/small>","2025-03-06<br><small>(in 18 days)<\/small>","2025-03-07<br><small>(in 19 days)<\/small>","2025-03-08<br><small>(in 20 days)<\/small>","2025-02-22<br><small>(in 6 days)<\/small>","2025-03-02<br><small>(in 14 days)<\/small>","2025-02-16<br><small>(today)<\/small>","2025-02-26<br><small>(in 10 days)<\/small>","2025-03-08<br><small>(in 20 days)<\/small>","2025-02-23<br><small>(in 7 days)<\/small>","2025-02-17<br><small>(in 1 day)<\/small>","2025-03-03<br><small>(in 15 days)<\/small>","2025-02-19<br><small>(in 3 days)<\/small>","2025-02-27<br><small>(in 11 days)<\/small>","2025-02-23<br><small>(in 7 days)<\/small>","2025-02-23<br><small>(in 7 days)<\/small>","2025-02-28<br><small>(in 12 days)<\/small>","2025-02-24<br><small>(in 8 days)<\/small>","2025-02-24<br><small>(in 8 days)<\/small>","2025-02-27<br><small>(in 11 days)<\/small>","2025-02-26<br><small>(in 10 days)<\/small>","2025-03-01<br><small>(in 13 days)<\/small>","2025-03-07<br><small>(in 19 days)<\/small>","2025-02-20<br><small>(in 4 days)<\/small>","2025-03-07<br><small>(in 19 days)<\/small>","2025-03-07<br><small>(in 19 days)<\/small>","2025-03-02<br><small>(in 14 days)<\/small>","2025-02-17<br><small>(in 1 day)<\/small>","2025-02-28<br><small>(in 12 days)<\/small>","2025-02-17<br><small>(in 1 day)<\/small>","2025-02-21<br><small>(in 5 days)<\/small>","2025-03-06<br><small>(in 18 days)<\/small>","2025-02-21<br><small>(in 5 days)<\/small>","2025-02-18<br><small>(in 2 days)<\/small>","2025-02-18<br><small>(in 2 days)<\/small>","2025-02-18<br><small>(in 2 days)<\/small>","2025-02-26<br><small>(in 10 days)<\/small>","2025-02-20<br><small>(in 4 days)<\/small>","2025-02-16<br><small>(today)<\/small>","2025-03-02<br><small>(in 14 days)<\/small>","2025-02-22<br><small>(in 6 days)<\/small>","2025-02-20<br><small>(in 4 days)<\/small>","2025-03-05<br><small>(in 17 days)<\/small>","2025-02-26<br><small>(in 10 days)<\/small>","2025-02-24<br><small>(in 8 days)<\/small>","2025-02-26<br><small>(in 10 days)<\/small>"],["<a href=\"https://cran.r-project.org/package=AICcmodavg\">AICcmodavg<\/a>","<a href=\"https://cran.r-project.org/package=autogam\">autogam<\/a>","<a href=\"https://cran.r-project.org/package=autothresholdr\">autothresholdr<\/a>","<a href=\"https://cran.r-project.org/package=baf\">baf<\/a>","<a href=\"https://cran.r-project.org/package=BaseSet\">BaseSet<\/a>","<a href=\"https://cran.r-project.org/package=BCDAG\">BCDAG<\/a>","<a href=\"https://cran.r-project.org/package=BHSBVAR\">BHSBVAR<\/a>","<a href=\"https://cran.r-project.org/package=BRugs\">BRugs<\/a>","<a href=\"https://cran.r-project.org/package=btergm\">btergm<\/a>","<a href=\"https://cran.r-project.org/package=causalBatch\">causalBatch<\/a>","<a href=\"https://cran.r-project.org/package=CDMConnector\">CDMConnector<\/a>","<a href=\"https://cran.r-project.org/package=censobr\">censobr<\/a>","<a href=\"https://cran.r-project.org/package=cfr\">cfr<\/a>","<a href=\"https://cran.r-project.org/package=CohortConstructor\">CohortConstructor<\/a>","<a href=\"https://cran.r-project.org/package=ColombiAPI\">ColombiAPI<\/a>","<a href=\"https://cran.r-project.org/package=cpmBigData\">cpmBigData<\/a>","<a href=\"https://cran.r-project.org/package=curl\">curl<\/a>","<a href=\"https://cran.r-project.org/package=dartR.base\">dartR.base<\/a>","<a href=\"https://cran.r-project.org/package=data.table\">data.table<\/a>","<a href=\"https://cran.r-project.org/package=DataFakeR\">DataFakeR<\/a>","<a href=\"https://cran.r-project.org/package=DATAstudio\">DATAstudio<\/a>","<a href=\"https://cran.r-project.org/package=DDoutlier\">DDoutlier<\/a>","<a href=\"https://cran.r-project.org/package=dendrometeR\">dendrometeR<\/a>","<a href=\"https://cran.r-project.org/package=detrendr\">detrendr<\/a>","<a href=\"https://cran.r-project.org/package=DGP4LCF\">DGP4LCF<\/a>","<a href=\"https://cran.r-project.org/package=divo\">divo<\/a>","<a href=\"https://cran.r-project.org/package=dodgr\">dodgr<\/a>","<a href=\"https://cran.r-project.org/package=dtmapi\">dtmapi<\/a>","<a href=\"https://cran.r-project.org/package=duckplyr\">duckplyr<\/a>","<a href=\"https://cran.r-project.org/package=EFAfactors\">EFAfactors<\/a>","<a href=\"https://cran.r-project.org/package=fasterRaster\">fasterRaster<\/a>","<a href=\"https://cran.r-project.org/package=fastQR\">fastQR<\/a>","<a href=\"https://cran.r-project.org/package=fields\">fields<\/a>","<a href=\"https://cran.r-project.org/package=figpatch\">figpatch<\/a>","<a href=\"https://cran.r-project.org/package=GDILM.SEIRS\">GDILM.SEIRS<\/a>","<a href=\"https://cran.r-project.org/package=gimap\">gimap<\/a>","<a href=\"https://cran.r-project.org/package=hmmr\">hmmr<\/a>","<a href=\"https://cran.r-project.org/package=isotracer\">isotracer<\/a>","<a href=\"https://cran.r-project.org/package=logrx\">logrx<\/a>","<a href=\"https://cran.r-project.org/package=MatchIt\">MatchIt<\/a>","<a href=\"https://cran.r-project.org/package=mbest\">mbest<\/a>","<a href=\"https://cran.r-project.org/package=MicroSEC\">MicroSEC<\/a>","<a href=\"https://cran.r-project.org/package=MiscMetabar\">MiscMetabar<\/a>","<a href=\"https://cran.r-project.org/package=mixtools\">mixtools<\/a>","<a href=\"https://cran.r-project.org/package=mlr3pipelines\">mlr3pipelines<\/a>","<a href=\"https://cran.r-project.org/package=nestedmodels\">nestedmodels<\/a>","<a href=\"https://cran.r-project.org/package=NewmanOmics\">NewmanOmics<\/a>","<a href=\"https://cran.r-project.org/package=noegletalR\">noegletalR<\/a>","<a href=\"https://cran.r-project.org/package=nplr\">nplr<\/a>","<a href=\"https://cran.r-project.org/package=o2geosocial\">o2geosocial<\/a>","<a href=\"https://cran.r-project.org/package=offsetreg\">offsetreg<\/a>","<a href=\"https://cran.r-project.org/package=ormBigData\">ormBigData<\/a>","<a href=\"https://cran.r-project.org/package=OwenQ\">OwenQ<\/a>","<a href=\"https://cran.r-project.org/package=pander\">pander<\/a>","<a href=\"https://cran.r-project.org/package=Pandora\">Pandora<\/a>","<a href=\"https://cran.r-project.org/package=PL94171\">PL94171<\/a>","<a href=\"https://cran.r-project.org/package=PResiduals\">PResiduals<\/a>","<a href=\"https://cran.r-project.org/package=ProtE\">ProtE<\/a>","<a href=\"https://cran.r-project.org/package=rangr\">rangr<\/a>","<a href=\"https://cran.r-project.org/package=rbmi\">rbmi<\/a>","<a href=\"https://cran.r-project.org/package=readsparse\">readsparse<\/a>","<a href=\"https://cran.r-project.org/package=readxl\">readxl<\/a>","<a href=\"https://cran.r-project.org/package=RISCA\">RISCA<\/a>","<a href=\"https://cran.r-project.org/package=rllama\">rllama<\/a>","<a href=\"https://cran.r-project.org/package=rmzqc\">rmzqc<\/a>","<a href=\"https://cran.r-project.org/package=ScottKnott\">ScottKnott<\/a>","<a href=\"https://cran.r-project.org/package=scRNAstat\">scRNAstat<\/a>","<a href=\"https://cran.r-project.org/package=shiny.info\">shiny.info<\/a>","<a href=\"https://cran.r-project.org/package=skellam\">skellam<\/a>","<a href=\"https://cran.r-project.org/package=soilassessment\">soilassessment<\/a>","<a href=\"https://cran.r-project.org/package=spectralGraphTopology\">spectralGraphTopology<\/a>","<a href=\"https://cran.r-project.org/package=squat\">squat<\/a>","<a href=\"https://cran.r-project.org/package=stratifyR\">stratifyR<\/a>","<a href=\"https://cran.r-project.org/package=synthpop\">synthpop<\/a>","<a href=\"https://cran.r-project.org/package=tableschema.r\">tableschema.r<\/a>","<a href=\"https://cran.r-project.org/package=teal\">teal<\/a>","<a href=\"https://cran.r-project.org/package=teal.modules.clinical\">teal.modules.clinical<\/a>","<a href=\"https://cran.r-project.org/package=teal.modules.general\">teal.modules.general<\/a>","<a href=\"https://cran.r-project.org/package=tinytiger\">tinytiger<\/a>","<a href=\"https://cran.r-project.org/package=TukeyC\">TukeyC<\/a>","<a href=\"https://cran.r-project.org/package=twitteR\">twitteR<\/a>","<a href=\"https://cran.r-project.org/package=urlshorteneR\">urlshorteneR<\/a>","<a href=\"https://cran.r-project.org/package=viralmodels\">viralmodels<\/a>","<a href=\"https://cran.r-project.org/package=vistla\">vistla<\/a>","<a href=\"https://cran.r-project.org/package=wallace\">wallace<\/a>","<a href=\"https://cran.r-project.org/package=waywiser\">waywiser<\/a>","<a href=\"https://cran.r-project.org/package=WeightIt\">WeightIt<\/a>","<a href=\"https://cran.r-project.org/package=zippeR\">zippeR<\/a>"],["2.3-3<br><small>(2023-11-16)<small>","0.0.1<br><small>(2024-10-02)<small>","1.4.2<br><small>(2023-12-13)<small>","0.0.1<br><small>(2024-06-04)<small>","0.9.0<br><small>(2023-08-22)<small>","1.1.2<br><small>(2025-01-30)<small>","3.1.1<br><small>(2022-11-05)<small>","0.9-2.1<br><small>(2024-02-25)<small>","1.10.12<br><small>(2024-03-31)<small>","1.3.0<br><small>(2025-01-07)<small>","1.7.0<br><small>(2024-12-19)<small>","0.4.0<br><small>(2024-09-20)<small>","0.1.2<br><small>(2024-09-26)<small>","0.3.4<br><small>(2025-01-17)<small>","0.1.0<br><small>(2025-01-09)<small>","0.0.1<br><small>(2023-02-13)<small>","6.2.0<br><small>(2025-01-23)<small>","0.98<br><small>(2024-09-19)<small>","1.16.4<br><small>(2024-12-06)<small>","0.1.3<br><small>(2023-02-12)<small>","1.1<br><small>(2022-04-28)<small>","0.1.0<br><small>(2018-05-30)<small>","1.1.0<br><small>(2024-10-14)<small>","0.6.15<br><small>(2023-03-04)<small>","1.0.0<br><small>(2024-05-28)<small>","1.0.1<br><small>(2019-12-16)<small>","0.4.1<br><small>(2024-07-07)<small>","0.0.2<br><small>(2024-09-17)<small>","1.0.0<br><small>(2025-02-07)<small>","1.2.0<br><small>(2025-01-07)<small>","8.4.0.3<br><small>(2024-12-17)<small>","1.0.0<br><small>(2025-02-04)<small>","16.3<br><small>(2024-09-30)<small>","0.2<br><small>(2022-05-03)<small>","0.0.3<br><small>(2025-01-13)<small>","1.0.1<br><small>(2025-02-03)<small>","1.0-0<br><small>(2021-05-27)<small>","1.1.7<br><small>(2024-11-05)<small>","0.3.1<br><small>(2024-04-12)<small>","4.7.0<br><small>(2025-01-12)<small>","0.6<br><small>(2018-05-25)<small>","2.1.6<br><small>(2024-08-25)<small>","0.12.1<br><small>(2025-01-29)<small>","2.0.0<br><small>(2022-12-05)<small>","0.7.1<br><small>(2024-11-14)<small>","1.1.0<br><small>(2023-09-30)<small>","1.0.11<br><small>(2024-04-28)<small>","0.2.1<br><small>(2024-09-12)<small>","0.1-7<br><small>(2016-12-28)<small>","1.1.3<br><small>(2024-06-21)<small>","1.1.0<br><small>(2024-04-11)<small>","0.0.1<br><small>(2021-06-10)<small>","1.0.7<br><small>(2023-04-11)<small>","0.6.5<br><small>(2022-03-18)<small>","24.2.0<br><small>(2024-02-26)<small>","1.1.2<br><small>(2022-09-11)<small>","1.0-1<br><small>(2021-06-24)<small>","1.0.1<br><small>(2025-01-15)<small>","1.0.5<br><small>(2025-02-14)<small>","1.4.0<br><small>(2025-02-07)<small>","0.1.5-6<br><small>(2023-11-27)<small>","1.4.3<br><small>(2023-07-06)<small>","1.0.6<br><small>(2025-01-20)<small>","0.4.4<br><small>(2024-02-02)<small>","0.5.4<br><small>(2024-04-16)<small>","1.3-2<br><small>(2023-08-30)<small>","0.1.1<br><small>(2021-09-22)<small>","0.2.0<br><small>(2020-03-23)<small>","0.2.3<br><small>(2024-03-25)<small>","0.3.0<br><small>(2025-01-13)<small>","0.2.3<br><small>(2022-03-14)<small>","0.3.0<br><small>(2024-01-10)<small>","1.0-3<br><small>(2021-12-07)<small>","1.9-0<br><small>(2024-12-20)<small>","1.1.2<br><small>(2022-09-29)<small>","0.15.2<br><small>(2024-03-07)<small>","0.9.1<br><small>(2024-04-27)<small>","0.3.0<br><small>(2024-03-05)<small>","0.0.9<br><small>(2024-06-04)<small>","1.3-42<br><small>(2023-08-31)<small>","1.1.9<br><small>(2015-07-29)<small>","1.5.7<br><small>(2022-08-22)<small>","1.3.1<br><small>(2024-10-18)<small>","2.1.0<br><small>(2025-02-06)<small>","2.1.3<br><small>(2024-07-18)<small>","0.6.0<br><small>(2024-06-27)<small>","1.3.2<br><small>(2024-11-05)<small>","0.1.0<br><small>(2024-07-14)<small>"],["<a href=\"mailto:marc.mazerolle-at-sbf.ulaval.ca\">Marc J. Mazerolle<\/a>","<a href=\"mailto:Chitu.Okoli-at-skema.edu\">Chitu Okoli<\/a>","<a href=\"mailto:rorynoolan-at-gmail.com\">Rory Nolan<\/a>","<a href=\"mailto:christopherkenny-at-fas.harvard.edu\">Christopher T. Kenny<\/a>","<a href=\"mailto:lluis.revilla-at-gmail.com\">Lluís Revilla Sancho<\/a>","<a href=\"mailto:alessandro.mascaro-at-upf.edu\">Alessandro Mascaro<\/a>","<a href=\"mailto:p.richardson.54391-at-gmail.com\">Paul Richardson<\/a>","<a href=\"mailto:ligges-at-statistik.tu-dortmund.de\">Uwe Ligges<\/a>","<a href=\"mailto:philip.leifeld-at-manchester.ac.uk\">Philip Leifeld<\/a>","<a href=\"mailto:ericwb95-at-gmail.com\">Eric W. Bridgeford<\/a>","<a href=\"mailto:black-at-ohdsi.org\">Adam Black<\/a>","<a href=\"mailto:rafa.pereira.br-at-gmail.com\">Rafael H. M. Pereira<\/a>","<a href=\"mailto:adam.kucharski-at-lshtm.ac.uk\">Adam Kucharski<\/a>","<a href=\"mailto:edward.burn-at-ndorms.ox.ac.uk\">Edward Burn<\/a>","<a href=\"mailto:arenzocaceresrossi-at-gmail.com\">Renzo Caceres Rossi<\/a>","<a href=\"mailto:cli77199-at-usc.edu\">Chun Li<\/a>","<a href=\"mailto:jeroenooms-at-gmail.com\">Jeroen Ooms<\/a>","<a href=\"mailto:bernd.gruber-at-canberra.edu.au\">Bernd Gruber<\/a>","<a href=\"mailto:t.barrett88-at-gmail.com\">Tyson Barrett<\/a>","<a href=\"mailto:krystian8207-at-gmail.com\">Krystian Igras<\/a>","<a href=\"mailto:Miguel.deCarvalho-at-ed.ac.uk\">Miguel de Carvalho<\/a>","<a href=\"mailto:jacob.madsen1-at-mail.com\">Jacob H. Madsen<\/a>","<a href=\"mailto:marko.smiljanic313-at-gmail.com\">Marko Smiljanic<\/a>","<a href=\"mailto:rorynoolan-at-gmail.com\">Rory Nolan<\/a>","<a href=\"mailto:jiachen.cai-at-mrc-bsu.cam.ac.uk\">Jiachen Cai<\/a>","<a href=\"mailto:pietrzak.20-at-osu.edu\">Maciej Pietrzak<\/a>","<a href=\"mailto:mark.padgham-at-email.com\">Mark Padgham<\/a>","<a href=\"mailto:lutran-at-iom.int\">Luong Bang Tran<\/a>","<a href=\"mailto:kirill-at-cynkra.com\">Kirill Müller<\/a>","<a href=\"mailto:haijiang133-at-outlook.com\">Haijiang Qin<\/a>","<a href=\"mailto:adam.smith-at-mobot.org\">Adam B. Smith<\/a>","<a href=\"mailto:mauro.bernardi-at-unipd.it\">Mauro Bernardi<\/a>","<a href=\"mailto:douglasnychka-at-gmail.com\">Douglas Nychka<\/a>","<a href=\"mailto:brady.johnston-at-me.com\">Brady Johnston<\/a>","<a href=\"mailto:abeda-at-myumanitoba.ca\">Amin Abed<\/a>","<a href=\"mailto:cansav09-at-gmail.com\">Candace Savonen<\/a>","<a href=\"mailto:i.visser-at-uva.nl\">Ingmar Visser<\/a>","<a href=\"mailto:matthieu.bruneaux-at-gmail.com\">Matthieu Bruneaux<\/a>","<a href=\"mailto:nhkosiba-at-gmail.com\">Nathan Kosiba<\/a>","<a href=\"mailto:noah.greifer-at-gmail.com\">Noah Greifer<\/a>","<a href=\"mailto:patperry-at-gmail.com\">Patrick O. Perry<\/a>","<a href=\"mailto:ikegamitky-at-gmail.com\">Masachika Ikegami<\/a>","<a href=\"mailto:adrien.taudiere-at-zaclys.net\">Adrien Taudière<\/a>","<a href=\"mailto:derek.young-at-uky.edu\">Derek Young<\/a>","<a href=\"mailto:mlr.developer-at-mb706.com\">Martin Binder<\/a>","<a href=\"mailto:ashbythorpe-at-gmail.com\">Ashby Thorpe<\/a>","<a href=\"mailto:krc-at-silicovore.com\">Kevin R. Coombes<\/a>","<a href=\"mailto:frlars21-at-student.aau.dk\">Frederik Larsen<\/a>","<a href=\"mailto:fredcommo-at-gmail.com\">Frederic Commo<\/a>","<a href=\"mailto:alexis.robert-at-lshtm.ac.uk\">Alexis Robert<\/a>","<a href=\"mailto:mattrmattrs-at-gmail.com\">Matt Heaphy<\/a>","<a href=\"mailto:cli77199-at-usc.edu\">Chun Li<\/a>","<a href=\"mailto:laurent_step-at-outlook.fr\">Stéphane Laurent<\/a>","<a href=\"mailto:daroczig-at-rapporter.net\">Gergely Daróczi<\/a>","<a href=\"mailto:jan.abel-at-inwt-statistics.de\">Jan Abel<\/a>","<a href=\"mailto:cmccartan-at-g.harvard.edu\">Cory McCartan<\/a>","<a href=\"mailto:cli77199-at-usc.edu\">Chun Li<\/a>","<a href=\"mailto:ted.margelos02-at-gmail.com\">Theodoros Margelos<\/a>","<a href=\"mailto:katarzyna.markowska-at-amu.edu.pl\">Katarzyna Markowska<\/a>","<a href=\"mailto:craig.gower-page-at-roche.com\">Craig Gower-Page<\/a>","<a href=\"mailto:david.cortes.rivera-at-gmail.com\">David Cortes<\/a>","<a href=\"mailto:jenny-at-posit.co\">Jennifer Bryan<\/a>","<a href=\"mailto:Yohann.Foucher-at-univ-poitiers.fr\">Yohann Foucher<\/a>","<a href=\"mailto:twor.2r-at-gmail.com\">Alexey Ivanov<\/a>","<a href=\"mailto:chris.bielow-at-bsc.fu-berlin.de\">Chris Bielow<\/a>","<a href=\"mailto:ivanalaman-at-gmail.com\">Ivan Bezerra Allaman<\/a>","<a href=\"mailto:xiayh17-at-gmail.com\">Yonghe Xia<\/a>","<a href=\"mailto:kuba-at-appsilon.com\">Jakub Nowicki<\/a>","<a href=\"mailto:patrick.brown-at-utoronto.ca\">Patrick E. Brown<\/a>","<a href=\"mailto:thineomuto-at-yahoo.com\">Christian Thine Omuto<\/a>","<a href=\"mailto:jvmirca-at-gmail.com\">Ze Vinicius<\/a>","<a href=\"mailto:aymeric.stamm-at-cnrs.fr\">Aymeric Stamm<\/a>","<a href=\"mailto:karuna.reddy-at-usp.ac.fj\">Karuna G. Reddy<\/a>","<a href=\"mailto:beata.nowok-at-gmail.com\">Beata Nowok<\/a>","<a href=\"mailto:koupidis.okfgr-at-gmail.com\">Kleanthis Koupidis<\/a>","<a href=\"mailto:dawid.kaledkowski-at-roche.com\">Dawid Kaledkowski<\/a>","<a href=\"mailto:joe.zhu-at-roche.com\">Joe Zhu<\/a>","<a href=\"mailto:dawid.kaledkowski-at-roche.com\">Dawid Kaledkowski<\/a>","<a href=\"mailto:christopherkenny-at-fas.harvard.edu\">Christopher T. Kenny<\/a>","<a href=\"mailto:ivanalaman-at-gmail.com\">Ivan Bezerra Allaman<\/a>","<a href=\"mailto:geoffjentry-at-gmail.com\">Jeff Gentry<\/a>","<a href=\"mailto:cincenko-at-outlook.com\">John Malc<\/a>","<a href=\"mailto:acua6307-at-gmail.com\">Juan Pablo Acuña González<\/a>","<a href=\"mailto:m-at-mbq.me\">Miron B. Kursa<\/a>","<a href=\"mailto:mblair1-at-amnh.org\">Mary E. Blair<\/a>","<a href=\"mailto:mike.mahoney.218-at-gmail.com\">Michael Mahoney<\/a>","<a href=\"mailto:noah.greifer-at-gmail.com\">Noah Greifer<\/a>","<a href=\"mailto:Christopher.Prener-at-pfizer.com\">Christopher Prener<\/a>"],["<a href=\"https://cran.r-project.org/web/checks/check_results_AICcmodavg.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_autogam.html\">checks<\/a><br><a href=\"https://github.com/tripartio/autogam\">url<\/a><br><a href=\"https://github.com/tripartio/autogam/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_autothresholdr.html\">checks<\/a><br><a href=\"https://rorynolan.github.io/autothresholdr/\">url<\/a><br><a href=\"https://github.com/rorynolan/autothresholdr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_baf.html\">checks<\/a><br><a href=\"http://christophertkenny.com/baf/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_BaseSet.html\">checks<\/a><br><a href=\"https://github.com/ropensci/BaseSet\">url<\/a><br><a href=\"https://github.com/ropensci/BaseSet/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_BCDAG.html\">checks<\/a><br><a href=\"https://github.com/alesmascaro/BCDAG\">url<\/a><br><a href=\"https://github.com/alesmascaro/BCDAG/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_BHSBVAR.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_BRugs.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_btergm.html\">checks<\/a><br><a href=\"https://github.com/leifeld/btergm\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_causalBatch.html\">checks<\/a><br><a href=\"https://github.com/neurodata/causal_batch\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_CDMConnector.html\">checks<\/a><br><a href=\"https://darwin-eu.github.io/CDMConnector/\">url<\/a><br><a href=\"https://github.com/darwin-eu/CDMConnector/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_censobr.html\">checks<\/a><br><a href=\"https://github.com/ipeaGIT/censobr\">url<\/a><br><a href=\"https://github.com/ipeaGIT/censobr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_cfr.html\">checks<\/a><br><a href=\"https://github.com/epiverse-trace/cfr\">url<\/a><br><a href=\"https://github.com/epiverse-trace/cfr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_CohortConstructor.html\">checks<\/a><br><a href=\"https://ohdsi.github.io/CohortConstructor/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ColombiAPI.html\">checks<\/a><br><a href=\"https://github.com/lightbluetitan/colombiapi\">url<\/a><br><a href=\"https://github.com/lightbluetitan/colombiapi/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_cpmBigData.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_curl.html\">checks<\/a><br><a href=\"https://jeroen.r-universe.dev/curl\">url<\/a><br><a href=\"https://github.com/jeroen/curl/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_dartR.base.html\">checks<\/a><br><a href=\"https://green-striped-gecko.github.io/dartR/\">url<\/a><br><a href=\"https://groups.google.com/g/dartr?pli=1\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_data.table.html\">checks<\/a><br><a href=\"https://r-datatable.com\">url<\/a><br><a href=\"https://github.com/Rdatatable/data.table/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_DataFakeR.html\">checks<\/a><br><a href=\"https://github.com/openpharma/DataFakeR\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_DATAstudio.html\">checks<\/a><br><a href=\"https://www.maths.ed.ac.uk/~mdecarv/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_DDoutlier.html\">checks<\/a><br><a href=\"https://github.com/jhmadsen/DDoutlier\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_dendrometeR.html\">checks<\/a><br><a href=\"https://github.com/smiljanicm/dendrometeR\">url<\/a><br><a href=\"https://github.com/smiljanicm/dendrometeR/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_detrendr.html\">checks<\/a><br><a href=\"https://rorynolan.github.io/detrendr/\">url<\/a><br><a href=\"https://github.com/rorynolan/detrendr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_DGP4LCF.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_divo.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_dodgr.html\">checks<\/a><br><a href=\"https://github.com/UrbanAnalyst/dodgr\">url<\/a><br><a href=\"https://github.com/UrbanAnalyst/dodgr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_dtmapi.html\">checks<\/a><br><a href=\"https://github.com/Displacement-Tracking-Matrix/dtmapi-R\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_duckplyr.html\">checks<\/a><br><a href=\"https://duckplyr.tidyverse.org\">url<\/a><br><a href=\"https://github.com/tidyverse/duckplyr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_EFAfactors.html\">checks<\/a><br><a href=\"https://haijiangqin.com/EFAfactors/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_fasterRaster.html\">checks<\/a><br><a href=\"https://github.com/adamlilith/fasterRaster\">url<\/a><br><a href=\"https://github.com/adamlilith/fasterRaster/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_fastQR.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_fields.html\">checks<\/a><br><a href=\"https://github.com/dnychka/fieldsRPackage\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_figpatch.html\">checks<\/a><br><a href=\"https://github.com/BradyAJohnston/figpatch\">url<\/a><br><a href=\"https://github.com/BradyAJohnston/figpatch/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_GDILM.SEIRS.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gimap.html\">checks<\/a><br><a href=\"https://github.com/FredHutch/gimap\">url<\/a><br><a href=\"https://github.com/FredHutch/gimap/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hmmr.html\">checks<\/a><br><a href=\"<https://depmix.github.io/hmmr/>\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_isotracer.html\">checks<\/a><br><a href=\"https://gitlab.com/matthieu-bruneaux/isotracer\">url<\/a><br><a href=\"https://gitlab.com/matthieu-bruneaux/isotracer/-/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_logrx.html\">checks<\/a><br><a href=\"https://github.com/pharmaverse/logrx\">url<\/a><br><a href=\"https://github.com/pharmaverse/logrx/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_MatchIt.html\">checks<\/a><br><a href=\"https://kosukeimai.github.io/MatchIt/\">url<\/a><br><a href=\"https://github.com/kosukeimai/MatchIt/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mbest.html\">checks<\/a><br><a href=\"https://github.com/patperry/r-mbest\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_MicroSEC.html\">checks<\/a><br><a href=\"https://github.com/MANO-B/MicroSEC/\">url<\/a><br><a href=\"https://github.com/MANO-B/MicroSEC/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_MiscMetabar.html\">checks<\/a><br><a href=\"https://github.com/adrientaudiere/MiscMetabar\">url<\/a><br><a href=\"https://github.com/adrientaudiere/MiscMetabar/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mixtools.html\">checks<\/a><br><a href=\"https://github.com/dsy109/mixtools\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mlr3pipelines.html\">checks<\/a><br><a href=\"https://mlr3pipelines.mlr-org.com\">url<\/a><br><a href=\"https://github.com/mlr-org/mlr3pipelines/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_nestedmodels.html\">checks<\/a><br><a href=\"https://github.com/ashbythorpe/nestedmodels\">url<\/a><br><a href=\"https://github.com/ashbythorpe/nestedmodels/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_NewmanOmics.html\">checks<\/a><br><a href=\"http://oompa.r-forge.r-project.org/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_noegletalR.html\">checks<\/a><br><a href=\"https://github.com/FrLars21/noegletalR\">url<\/a><br><a href=\"https://github.com/FrLars21/noegletalR/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_nplr.html\">checks<\/a><br><a href=\"https://github.com/fredcommo/nplr\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_o2geosocial.html\">checks<\/a><br><a href=\"https://github.com/alxsrobert/o2geosocial\">url<\/a><br><a href=\"https://github.com/alxsrobert/o2geosocial/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_offsetreg.html\">checks<\/a><br><a href=\"https://github.com/mattheaphy/offsetreg/\">url<\/a><br><a href=\"https://github.com/mattheaphy/offsetreg/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ormBigData.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_OwenQ.html\">checks<\/a><br><a href=\"https://github.com/stla/OwenQ\">url<\/a><br><a href=\"https://github.com/stla/OwenQ/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_pander.html\">checks<\/a><br><a href=\"https://rapporter.github.io/pander/\">url<\/a><br><a href=\"https://github.com/rapporter/pander/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_Pandora.html\">checks<\/a><br><a href=\"https://github.com/Pandora-IsoMemo/pandora-data\">url<\/a><br><a href=\"https://github.com/Pandora-IsoMemo/pandora-data/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_PL94171.html\">checks<\/a><br><a href=\"https://corymccartan.com/PL94171/\">url<\/a><br><a href=\"https://github.com/CoryMcCartan/PL94171/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_PResiduals.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ProtE.html\">checks<\/a><br><a href=\"https://github.com/theomargel/ProtE\">url<\/a><br><a href=\"https://github.com/theomargel/ProtE/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rangr.html\">checks<\/a><br><a href=\"https://github.com/ropensci/rangr\">url<\/a><br><a href=\"https://github.com/ropensci/rangr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rbmi.html\">checks<\/a><br><a href=\"https://insightsengineering.github.io/rbmi/\">url<\/a><br><a href=\"https://github.com/insightsengineering/rbmi/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_readsparse.html\">checks<\/a><br><a href=\"https://github.com/david-cortes/readsparse\">url<\/a><br><a href=\"https://github.com/david-cortes/readsparse/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_readxl.html\">checks<\/a><br><a href=\"https://readxl.tidyverse.org\">url<\/a><br><a href=\"https://github.com/tidyverse/readxl/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_RISCA.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rllama.html\">checks<\/a><br><a href=\"https://github.com/AlexTwoR/rllama\">url<\/a><br><a href=\"https://github.com/AlexTwoR/rllama/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rmzqc.html\">checks<\/a><br><a href=\"https://github.com/MS-Quality-hub/rmzqc\">url<\/a><br><a href=\"https://github.com/MS-Quality-hub/rmzqc/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ScottKnott.html\">checks<\/a><br><a href=\"https://github.com/ivanalaman/ScottKnott\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_scRNAstat.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_shiny.info.html\">checks<\/a><br><a href=\"https://github.com/Appsilon/shiny.info/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_skellam.html\">checks<\/a><br><a href=\"https//r-forge.r-project.org/projects/healthqueues/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_soilassessment.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_spectralGraphTopology.html\">checks<\/a><br><a href=\"https://github.com/dppalomar/spectralGraphTopology\">url<\/a><br><a href=\"https://github.com/dppalomar/spectralGraphTopology/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_squat.html\">checks<\/a><br><a href=\"https://github.com/LMJL-Alea/squat\">url<\/a><br><a href=\"https://github.com/LMJL-Alea/squat/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_stratifyR.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_synthpop.html\">checks<\/a><br><a href=\"<https://www.synthpop.org.uk/>\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tableschema.r.html\">checks<\/a><br><a href=\"https://github.com/frictionlessdata/tableschema-r\">url<\/a><br><a href=\"https://github.com/frictionlessdata/tableschema-r/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_teal.html\">checks<\/a><br><a href=\"https://insightsengineering.github.io/teal/\">url<\/a><br><a href=\"https://github.com/insightsengineering/teal/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_teal.modules.clinical.html\">checks<\/a><br><a href=\"https://insightsengineering.github.io/teal.modules.clinical/\">url<\/a><br><a href=\"https://github.com/insightsengineering/teal.modules.clinical/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_teal.modules.general.html\">checks<\/a><br><a href=\"https://insightsengineering.github.io/teal.modules.general/\">url<\/a><br><a href=\"https://github.com/insightsengineering/teal.modules.general/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tinytiger.html\">checks<\/a><br><a href=\"https://github.com/alarm-redist/tinytiger\">url<\/a><br><a href=\"https://github.com/alarm-redist/tinytiger/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_TukeyC.html\">checks<\/a><br><a href=\"https://github.com/jcfaria/TukeyC\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_twitteR.html\">checks<\/a><br><a href=\"http://lists.hexdump.org/listinfo.cgi/twitter-users-hexdump.org\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_urlshorteneR.html\">checks<\/a><br><a href=\"https://github.com/dmpe/urlshorteneR\">url<\/a><br><a href=\"https://github.com/dmpe/urlshorteneR/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_viralmodels.html\">checks<\/a><br><a href=\"https://github.com/juanv66x/viralmodels\">url<\/a><br><a href=\"https://github.com/juanv66x/viralmodels/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_vistla.html\">checks<\/a><br><a href=\"https://gitlab.com/mbq/vistla\">url<\/a><br><a href=\"https://gitlab.com/mbq/vistla/-/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_wallace.html\">checks<\/a><br><a href=\"http://wallaceecomod.github.io/wallace/,\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_waywiser.html\">checks<\/a><br><a href=\"https://github.com/ropensci/waywiser\">url<\/a><br><a href=\"https://github.com/ropensci/waywiser/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_WeightIt.html\">checks<\/a><br><a href=\"https://ngreifer.github.io/WeightIt/\">url<\/a><br><a href=\"https://github.com/ngreifer/WeightIt/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_zippeR.html\">checks<\/a><br><a href=\"https://github.com/pfizer-opensource/zippeR\">url<\/a><br><a href=\"https://github.com/pfizer-opensource/zippeR/issues\">url2<\/a>"],[null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#CDMConnector\">resubmitted/archive<\/a><\/small>",null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#CohortConstructor\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#divo\">resubmitted/pretest<\/a><\/small>","<small><a href=\"https://nx10.github.io/cransubs/pkg#dodgr\">resubmitted/UL<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#nplr\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#teal\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#waywiser\">resubmitted/archive<\/a><\/small>","<small><a href=\"https://nx10.github.io/cransubs/pkg#WeightIt\">resubmitted/archive<\/a><\/small>",null],[8,0,4,0,0,0,0,0,2,0,10,0,0,2,0,0,429,4,1725,0,0,1,0,1,0,0,1,0,1,0,0,0,186,0,0,0,0,0,0,14,1,0,0,43,8,0,0,0,2,0,0,0,0,39,0,2,0,0,0,1,0,295,0,0,2,1,0,0,3,0,2,0,0,0,0,2,0,0,5,0,1,1,0,0,0,0,3,1],["AdIsMF, enmSdmX, longit, OpenStats, Petersen, sars, soiltestcorr, temperatureresponse","","coveR2, detrendr, hemispheR, nandb","","","","","","ergMargins, netmediate","","CohortCharacteristics, CohortConstructor, CohortSurvival, CohortSymmetry, DrugExposureDiagnostics, DrugUtilisation, IncidencePrevalence, OmopSketch, PatientProfiles, TestGenerator","","","OmopSketch, PhenotypeR","","","addinslist, AIscreenR, alarmdata, allcontributors, AlphaMissenseR, AMAPVox, ami, AnnoProbe, AnnotationHub, AnVILAz, anyflights, anyLib, aopdata, APCalign, arcgisgeocode, ARPALData, asciicast, ausplotsR, autoEnsemble, avidaR, aws.ec2metadata, aws.s3, AzureCosmosR, AzureGraph, baf, baizer, bambooHR, basemaps, basifoR, BatchGetSymbols, bbk, BFS, BgeeDB, bigrquery, BiocFileCache, biocompute, biomaRt, biomartr, biometryassist, bioRad, biscuiteerData, bitmexr, blastula, brickster, BridgeDbR, bulkreadr, captr, cartography, CBRT, CDCPLACES, cder, cellxgenedp, censobr, ceramic, chatAI4R, chemodiv, chromote, cimir, ciw, clarifai, clidatajp, climate, cloudstoR, coinmarketcapr, completejourney, conflr, CongressData, contentid, covid19.analytics, covid19br, covid19us, cpp11qpdf, CRANsearcher, credentials, crul, crunch, cryptoQuotes, ctrdata, cyclestreets, CytobankAPI, czechrates, czso, dadjokeapi, dataRetrieval, DataSpaceR, dawaR, deeptime, depmap, diseasystore, dverse, ebvcube, ecb, ecce, ech, echor, EDIutils, EEAaq, eiopaR, elastic, electionsBR, elevatr, emayili, enrichR, ensembldb, eph, epitweetr, EpiTxDb, eurlex, eurostat, evolMap, EWSmethods, excluder, exifr, exiftoolr, ExperimentHubData, faers, FedData, federalregister, felp, feltr, fgga, fHMM, finna, finreportr, fishualize, FjordLight, flashr, flightsbr, foghorn, gasper, gbfs, geneHummus, geniusr, genomes, geoAr, geobr, geofi, geoperu, GEOquery, geouy, GetBCBData, GetDFPData, GetLattesData, GetTDData, ggmapcn, GHCNr, gie, GIFT, gimms, gitgadget, githubinstall, gitignore, googleCloudStorageR, GoogleKnowledgeGraphR, googlesheets4, googleway, gpg, gptstudio, gptzeror, gridtext, GSODR, gtfstools, gtrendsR, h2otools, healthatlas, heapsofpapers, helsinki, hereR, hfhub, HiCBricks, hlidacr, hsrecombi, httptest, httr, httr2, hubUtils, hyd1d, hydflood, ifo, ihpdr, imf.data, immunotation, InterVA5, ipeadatar, IsoMemo, ispdata, jgsbook, jsonld, khisr, kmeRtone, KoboconnectR, LACE, LAGOSNE, languagelayeR, legislatoR, libbib, lifeR, lifx, magick, mapboxapi, mapping, mapscanner, maptiles, MassWateR, mcbette, medrxivr, meteospain, Microsoft365R, misuvi, mlim, mlr3oml, modisfast, MouseFM, mRpostman, MsBackendMetaboLights, natmanager, ncaavolleyballr, neonOS, neonUtilities, NFCP, nfl4th, nflfastR, nflreadr, ngramr, nhdR, njgeo, njtr1, NLPclient, npi, nser, ntdr, nzffdr, OAIHarvester, occCite, ocs4R, omicsViewer, OmnipathR, openaistream, opencpu, opendatatoronto, OpenML, opentripplanner, osmapiR, osmdata, osrm, ottrpal, owidR, packageRank, pacs, pafdR, pageviews, palaeoverse, paleobioDB, Pandora, panstarrs, parquetize, pastclim, paws.common, pedquant, peermodels, pestr, phantasus, pivotaltrackR, pkgcache, pkgdepends, pkgfilecache, pkggraph, pkgsearch, planscorer, plotgardener, plumbertableau, pmetar, pmparser, PortalHacienda, postdoc, promptr, protti, psychmeta, ptm, pxweb, pyMTurkR, qpdf, Qtlizer, quantmod, r.proxy, r311, r4googleads, radiant.data, radous, ralger, random, randquotes, RapidoPGS, rATTAINS, Ravages, rcites, rcmdcheck, Rcpi, RCzechia, rdbnomics, rdfp, rdian, rdomains, rdryad, rdtLite, ready4, receptiviti, redcapAPI, REDCapExporter, remmy, request, resmush, resourcecode, restez, rgho, rglobi, rgoogleclassroom, rhub, ricu, rinat, RInno, rio, rirods, riskmetric, rivnet, rix, rJavaEnv, RKorAPClient, rmacrostrat, rnbp, rnpn, rnrfa, robis, rosm, rotl, Rpadrino, rpaleoclim, rphylopic, RPublica, RPushbullet, rpx, rqti, Rrdap, rredlist, rsat, rsconnect, rsocialwatcher, rsoi, rsurveycto, rtika, rtoot, rtracklayer, rtry, rtsdata, rtweet, rversions, s3fs, salesforcer, sbtools, SchoolDataIT, secret, selenider, sevenbridges, sevenbridges2, shapley, shinyloadtest, shinyscholar, shopifyr, SMMT, socialmixr, soilDB, sooty, sotkanet, spanishoddata, statcanR, statnipokladna, statswalesr, stepcount, stevedore, SticsRFiles, stplanr, subincomeR, susographql, SWMPrExtension, tall, tapLock, taxize, taxonomizr, telegram, telegram.bot, tesseract, threesixtygiving, thunder, tidyllm, tidyquant, tinyscholar, tinytiger, tower, trelloR, TTR, TwitterAutomatedTrading, uklr, UniprotR, updateme, urlchecker, usethis, V8, vegdata, VicmapR, virtuoso, vmr, W3CMarkupValidator, waffle, warbleR, washex, wdpar, weatherOz, webdav, webdriver, webmockr, webutils, wikitaxa, wikiTools, worldfootballR, xmlrpc2, yahoofinancer, zendown, zenstats, zipcodeR","dartR.popgen, dartR.sexlinked, dartR.sim, dartR.spatial","Ac3net, accessibility, acdcR, Achilles, acoRn, actel, ActivePathways, ActivityIndex, actuaRE, ADAMgui, adductomicsR, AdhereR, AdhereRViz, ADImpute, AeRobiology, AF, AFM, africamonitor, agcounts, AggregateR, agriutilities, ahw, AICcPermanova, airship, akc, akiFlagger, aLFQ, alpaca, AlphaBeta, amadeus, amapGeocode, AMAPVox, AMIM, amplican, amt, Anaconda, animint2, antaresEditObject, antaresProcessing, antaresRead, antaresViz, AnxietySleep, aopdata, aorsf, AOV1R, APCI, APIS, AQEval, aqp, arf, artMS, asciiSetupReader, ascotraceR, aslib, ASpli, ASRgenomics, assertable, atime, AtmChile, atSNP, attrib, AUCell, aum, autoCovariateSelection, autonewsmd, autonomics, AutoPlots, autostats, autostsm, aweSOM, bagged.outliertrees, bakR, bambu, BANDITS, Banksy, baseballr, basecallQC, BatchExperiments, BatchJobs, BatchQC, batchtools, bayesLife, BayesLN, bayesMeanScale, bayesMig, bayesPop, bayesTFR, bbk, BBmisc, bbmix, bbotk, bcputility, bdots, bea.R, BEclear, bedr, BeeGUTS, behavr, benford.analysis, bfw, BgeeCall, BgeeDB, BGmisc, BiasCorrector, biblionetwork, bife, BIGL, bigreadr, bigsnpr, binancer, binomialRF, binsegRcpp, biomartr, BIOMASS, biopixR, BioR, bioregion, birankr, biscuiteer, bkmrhat, BKTR, BLA, blorr, BMisc, bnbc, BootstrapQTL, BPRMeth, bRacatus, brainGraph, branchpointer, brandwatchR, bratteli, bread, BREADR, bridger2, BRINDA, brmsmargins, brpop, bruceR, bsitar, bsseq, BTSPAS, BTYDplus, bupaR, BuyseTest, c3, CAGEr, CALF, calmr, CamelUp, camtrapR, CaMutQC, CareDensity, caretEnsemble, cartogramR, casebase, CAST, CATALYST, categoryEncodings, CatEncoders, CatsCradle, causalCmprsk, CausalGPS, cbl, CBRT, ccdR, ccmap, ccostr, cdparcoord, ceas, celda, CellBarcode, cellbaseR, CelliD, cellKey, CEMiTool, Census2016, cercospoRa, Certara.NLME8, Certara.R, Certara.RsNLME, Certara.RsNLME.ModelBuilder, Certara.VPCResults, cffdrs, cglm, changepoint.influence, ChannelAttributionApp, ChemoSpecUtils, Chicago, chicane, chimeraviz, ChineseNames, ChIPexoQual, ChIPpeakAnno, CHO, chronicle, cicero, circhelp, citationchaser, citecorp, ciw, clam, classifierplots, cleanTS, ClickHouseHTTP, clickstream, climate, clinDataReview, clinspacy, clinUtils, CluMP, Clustering, clustra, CLVTools, cmapR, cmsaf, CNVRanger, CNVScope, COCOA, coder, CoDi, coffee, cohorts, coinmarketcapr, coloc, colocPropTest, colorednoise, Colossus, comorbidity, CompareCausalNetworks, compareDF, COMPASS, ComPrAn, compSPOT, CONFESS, CoNI, conleyreg, consensusDE, consortr, ConsReg, constellation, container, CoOL, cooltools, CooRTweet, cophescan, coRdon, CoreGx, CornerstoneR, corporaexplorer, corpustools, corrcoverage, correspondenceTables, CoSMoS, counterfactuals, coveffectsplot, COVID19, covid19br, covid19india, covidnor, cppRouting, CRE, creditmodel, crew, crfsuite, CRISPRseek, CropScapeR, crosshap, crumble, csa, csalert, csdata, csmaps, CSMES, cstidy, cstime, CSTools, csutil, csv, csvy, ctbi, ctmm, cTRAP, ctsem, ctxR, cubar, curtailment, cusum, cusumcharter, cvAUC, cvms, CWT, cxhull, cyclestreets, cylcop, cytoKernel, CytoML, D3partitionR, daiquiri, daiR, damr, dartR.base, dartR.popgen, dartR.sim, dartR.spatial, dat, data.table.threads, data360r, DataCombine, dataCompare, datadictionary, DataExplorer, dataframeexplorer, datamods, dataMojo, dataprep, dataPreparation, DataSpaceR, DATAstudio, datazoom.amazonia, dateutils, dblr, dbnR, dbWebForms, ddtlcm, decompTumor2Sig, deconvR, dedupewider, deepdive, deeptrafo, DEFormats, DegNorm, delayed, deltaccd, DemoKin, DEScan2, DescTools, designit, DESpace, detectRUNS, DEWSeq, dfmeta, DGM, diathor, diceplot, did2s, Diderot, DiDforBigData, didimputation, DIDmultiplegtDYN, difconet, DifferentialRegulation, DiscoRhythm, DiSCos, discSurv, DisImpact, disk.frame, distillML, DIZtools, DIZutils, DMCFB, dMrs, do, DominoEffect, doremi, DoubleML, DQAgui, DQAstats, dQTG.seq, DramaAnalysis, dreamlet, drgee, dropR, droptest, dsos, DTAT, dtplyr, DTSg, dtts, DTwrappers, DTwrappers2, dumbbell, dvmisc, dynamite, dynsurv, DysPIA, DYTE, E4tools, Eagle, easybio, easycsv, easyr, easysurv, eatGADS, eatTools, EBPRS, eclust, ecocomDP, ecoregime, edeaR, edl, EdSurvey, eeptools, efdm, EGM, EHR, EIAapi, EIX, eixport, electionsBR, eltr, EmiR, EmissV, ENCODExplorerData, enderecobr, endogeneity, endoSwitch, enmSdmX, ensr, enviGCMS, eoR, Epi, epialleleR, EpiCompare, epidm, EpiForsk, epigraHMM, EpiMix, EpiNow2, epiomics, epistasisGA, eplusr, epwshiftr, equateMultiple, eSIR, estadistica, EthSEQ, etm, etwfe, eurocordexr, eurodata, europeanaR, eurostat, evapoRe, eventstudyr, evoper, EWCE, exiftoolr, explainer, exploratory, expowo, expss, exvatools, eyeRead, eyeTrackR, ezr, facmodCS, facmodTS, FACT, factR, faers, faersquarterlydata, FairMclus, familiar, FAOSTAT, farrell, fastdid, fastDummies, fasterRaster, fastLink, fastreeR, FastRet, fastRhockey, fastText, fastverse, fastVoteR, fbRads, fcfdr, FCO, fdq, FeatureImpCluster, fedmatch, fedstatAPIr, fedup, felp, fgdr, Fgmutils, fgsea, fhircrackr, FIESTA, FIESTAutils, finch, finnsurveytext, fishmethods, fitbitViz, FitHiC, flextable, flightsbr, flippant, FLOPART, florabr, flowGraph, flowml, flowWorkspace, Fluidigm, FMAT, fmeffects, FOCI, forceplate, forecastML, forestError, formatdown, formulaic, FORTLS, fossilbrush, FourWayHMM, fplot, fplyr, FRASER, fruclimadapt, fsbrain, FuncDiv, funchir, FuzzyDBScan, fwlplot, fy, GALLO, GARCOM, gasfluxes, gatom, gbp, gcite, gde, gDRcore, gDRimport, gDRtestData, gDRutils, gdxdt, gemma.R, gems, GeneGeneInteR, geneplast, GENESIS, GeneStructureTools, geneXtendeR, genomation, GenomAutomorphism, genomic.autocorr, GenomicDistributions, GenomicDistributionsData, GenomicInteractions, GenomicTools.fileHandler, GenomicTuples, GenoScan, GenVisR, geobr, geocausal, geocodebr, GeoMongo, GeomxTools, geoperu, GEOquery, GeoTcgaData, GeoThinneR, germinationmetrics, getDTeval, GFE, gfer, gfoRmula, gfoRmulaICE, ggbrain, ggcyto, ggdmc, ggetho, ggfacto, ggfigdone, GGIR, GGIRread, ggkegg, ggmugs, ggperiodic, ggPMX, ggpval, ggquickeda, ggsom, ggTimeSeries, GHap, githubinstall, gittargets, GladiaTOX, GmicR, GMMAT, gmoTree, gnrprod, goat, gompertztrunc, googleCloudVisionR, GOTHiC, gpboost, GRaNIE, grattan, grattanInflators, greatR, GREENeR, greport, GRIN2, growthcleanr, GRShiny, GSALightning, gsDesign2, GSODR, gtfs2emis, gtfs2gps, gtfsio, gtfsrouter, gtfstools, GTFSwizard, GUIDEseq, gunsales, gwaRs, gwascatData, gwasforest, GWASinspector, GWASTools, gwid, HACSim, HaDeX, hagis, hahmmr, hal9001, haldensify, hBayesDM, HCD, HCR, hdd, hdpGLM, healthdb, healthyAddress, heatwaveR, heims, hereR, HERON, hesim, heuristicsmineR, HicAggR, HiCBricks, HiCcompare, HiCDCPlus, HiCDOC, hicp, highfrequency, hilldiv, himach, hisse, Hmisc, HMMcopy, HMP2Data, hoardeR, HodgesTools, hollr, hoopR, HospitalNetwork, hot.deck, HPLB, hR, hsrecombi, httk, hudr, hurricaneexposure, hutils, hutilscpp, hwig, hydroloom, hyfo, hypervolume, HYPEtools, i2extras, IBRtools, iCAMP, ICAMS, iCellR, IceSat2R, iCNV, ideanet, IDEATools, IDmining, ie2misc, ieegio, ifCNVR, iGC, iimi, imabc, imcRtools, iml, immunarch, immuneSIM, importinegi, imputeMulti, inbreedR, IncDTW, incidence2, inferr, InflectSSP, influential, Information, insurancerating, InterCellar, intervalaverage, inTrees, IOHanalyzer, ips, irace, iraceplot, irg, IrregLong, ISCA, iscoCrosswalks, isdparser, IsoBayes, isobxr, isotracer, iSTATS, itol.toolkit, itscalledsoccer, ivdoctr, ivtools, jage, JointFPM, joinXL, journalabbr, joyn, jrSiCKLSNMF, JSmediation, jsmodule, jstable, jsTree, jubilee, JWileymisc, kangar00, kdry, kerastuneR, kesernetwork, kgen, kgraph, kibior, kim, KinSwingR, kisopenapi, KMD, kmeRtone, knfi, KnockoffScreen, koRpus, kosis, KPC, lab2clean, LabourMarketAreas, labourR, LACE, lacunr, lar, latrend, LCMSQA, ldamatch, ldaPrototype, ldsr, leafR, leakyIV, LearningStats, LegATo, lexicon, LexisNexisTools, lgrExtra, libbib, libr, lidR, lightgbm, LilRhino, limexhub, limorhyde2, lineagespot, LinkHD, lipidr, lisaClust, live, LKT, llama, LMPdata, lmtp, loadeR, loci2path, logiBin, LOLA, LOMAR, lookupTable, LOPART, lori, LPDynR, LSPFP, Luminescence, LWFBrook90R, lwqs, LymphoSeq, m5, Maaslin2, Macarron, maditr, maftools, MAGAR, magclass, MAGEE, maicChecks, MAICtools, MANOVA.RM, MantaID, manydist, mapi, marginaleffects, margins, mariner, marked, maser, MAST, matman, MatrixHMM, mau, maxaltall, maxent.ot, mbr, mbRes, mcauchyd, mcboost, medrxivr, Melissa, memisc, MesKit, metabodecon, metabolomicsWorkbenchR, MetaComp, metaforest, metaGE, metagene2, MetAlyzer, MetaScope, metasnf, meteo, methimpute, methodical, methrix, methylKit, MethylMix, methylPipe, methylscaper, metR, mggd, mgi.report.reader, MIC, miceFast, miceRanger, MicrobiotaProcess, microeco, microseq, midfieldr, miesmuschel, miLineage, MIMER, mimi, mind, minfi, MinimumDistance, minSNPs, mintyr, mipplot, MIRA, mirTarRnaSeq, miscset, missCompare, mixgb, mixIndependR, mknapsack, mlbplotR, mldr.resampling, mlexperiments, mllrnrs, mlr, mlr3, mlr3batchmark, mlr3benchmark, mlr3cluster, mlr3db, mlr3fairness, mlr3fda, mlr3filters, mlr3fselect, mlr3hyperband, mlr3inferr, mlr3learners, mlr3mbo, mlr3misc, mlr3oml, mlr3pipelines, mlr3resampling, mlr3shiny, mlr3spatial, mlr3spatiotempcv, mlr3summary, mlr3superlearner, mlr3torch, mlr3tuning, mlr3tuningspaces, mlr3verse, mlr3viz, mlrintermbo, mlrMBO, mlsurvlrnrs, mltools, mmaqshiny, mnem, mobileR, MOCHA, modACDC, ModCon, ModelMetrics, modelsummary, MoleculeExperiment, MouseFM, MPAC, mpactr, MplusAutomation, mpwR, mrMLM, mrMLM.GUI, mrregression, MsBackendSql, mschart, msImpute, mslp, MSmix, MSnID, MSstats, MSstatsConvert, MSstatsLiP, MSstatsPTM, MSstatsShiny, MSstatsTMT, mstate, mstclustering, mstrio, mstudentd, mtb, mulea, multicrispr, multifamm, MultiFit, multiHiCcompare, multilevelcoda, multilevelTools, multistateQTL, multiWGC, multvardiv, MungeSumstats, muscat, musica, musicatk, mutationtypes, mutualinf, mvQuad, mvSUSY, mxnorm, myClim, naaccr, naptanr, nc, ndjson, neatR, neatRanges, neatStats, neo4r, NEONiso, neonOS, neonPlantEcology, neonUtilities, nestedcv, net4pg, netgsa, NetLogoR, netSmooth, networkR, netZooR, neuroimaGene, nflfastR, nflplotR, nflreadr, nflseedR, NFLSimulatoR, nlmixr2extra, nlpembeds, nlpred, NMcalc, NMdata, NMsim, nngeo, NNS, noctua, node2vec, noncompliance, nonmem2rx, normfluodbf, nosoi, npcurePK, nRegression, nullranges, numbat, o2geosocial, OBIC, obliqueRSF, occupationMeasurement, octad, octopus, odbr, oddsapiR, ofpetrial, ogrdbstats, OGRE, okxAPI, OlinkAnalyze, OmicNavigator, OMICsPCA, omicwas, ompr, OnboardClient, OncoSimulR, One4All, openCyto, opendataformat, openEBGM, OpenML, OpenSpecy, opentripplanner, optiSel, orderanalyzer, ORFhunteR, ORFik, orgR, origami, orthogene, osmclass, OUTRIDER, overviewR, owdbr, owidR, paar, packageRank, pairkat, PakPMICS2014Ch, PakPMICS2014HH, PakPMICS2014HL, PakPMICS2014Wm, pam, PAMpal, PAMscapes, panelaggregation, PanelMatch, panelWranglR, panstarrs, paradox, paramix, ParBayesianOptimization, parseRPDR, parSim, partCNV, partools, parttree, pathviewr, pathwayTMB, patternator, pbox, pcadapt, PCBS, PCMBase, PCMBaseCpp, PCRA, pcvr, pda, pdynmc, PeakSegDisk, Pedixplorer, pedquant, penaltyLearning, PepMapViz, pepr, pepStat, PGRdup, pgTools, phantasus, phantasusLite, PharmacoGx, pheble, phenofit, phenomis, PhenotypeSimulator, phers, phm, phonics, PhyloProfile, phyloseq, pingers, pinyin, pipeflow, pivottabler, pk4adi, pkggraph, pkgnet, PKPDsim, pksensi, PlanetNICFI, plinkQC, plmmr, plnr, plotgardener, plotly, pmartR, pmparser, pmxpartab, pointdensityP, poisFErobust, politeness, polle, polmineR, polypharmacy, pomp, POMS, poolfstat, pooling, popEpi, PopulateR, portvine, posologyr, POSSA, potools, POUMM, power.transform, powerbiR, ppmlasso, pram, pRecipe, precrec, predhy.GUI, prediction, PReMiuM, PreProcessRecordLinkage, preputils, PressPurt, previsionio, pricelevels, proActiv, processmapR, processpredictR, prodlim, prolific.api, PRONE, PROSPER, Prostar, protGear, protti, prrd, prt, psborrow, PSF, psichomics, psidR, psmineR, psp, psycCleaning, PsychWordVec, ptable, ptairMS, pTITAN2, PTXQC, Publish, PureCN, puremoe, pureseqtmr, pvda, Pviz, pvLRT, qad, QCAcluster, qckitfastq, qdapTools, qgg, qreport, qte, QTL.gCIMapping, QTL.gCIMapping.GUI, qtl2, quickcheck, quickPlot, R.AlphA.Home, r2dii.match, R2DT, r2glmm, R3CPET, r3Cseq, r5r, R62S3, R6causal, RABR, RadEro, RadioGx, radlibs, RALSA, rAmCharts, RaMS, randnet, randomForestExplainer, randomNames, rangeMapper, RankAggSIgFUR, RapidoPGS, RARfreq, rassta, rasterDT, RAthena, ratios, Ravages, RawHummus, rbacon, RBaseX, rBayesianOptimization, rbi, rbi.helpers, rBiasCorrection, rbin, rBiopaxParser, rblt, Rborist, rbtt, RC, Rcan, RCAS, RcextTools, rchemo, RcisTarget, RcisTarget.hg19.motifDBs.cisbpOnly.500bp, RcmdrPlugin.RiskDemo, rCNV, RcppGreedySetCover, rcprd, Rcrawler, rdbnomics, rdfp, Rdiagnosislist, Rdice, RDML, rdracor, rE, readNSx, readtext, realTimeloads, reappraised, rechaRge, reclin2, recorder, RecordLinkage, recount3, redatam, Rediscover, RefBasedMI, RegParallel, regport, regtools, ReinforcementLearning, reinsureR, remiod, REndo, replacer, repmis, reproducible, Require, rerddap, resemble, reslr, RESOLVE, respR, restatapi, ReSurv, rfaRm, rfars, rfisheries, RFplus, rfPred, rgbif, rhdf5client, RIbench, RiboCrypt, RiboDiPA, RiboProfiling, ricu, Rilostat, RImmPort, rintcal, rio, rIsing, riskRegression, RITCH, ritis, rlas, rlist, rllama, RLumShiny, RMassBank, rmcfs, rMIDAS, RNewsflow, rnpn, Rnvd3, roads, robCompositions, RobinCar, robis, robotoolbox, ROCket, rODE, rollinglda, rorcid, rPackedBar, rprev, rqdatatable, rqPen, Rrepest, RSauceLabs, rSPARCS, rsparse, rsurveycto, rsyntax, RTCGA, RTCGAToolbox, rtmpt, RTN, RTNsurvival, rtry, rtsdata, RTTWebClient, runcharter, rush, rusquant, RVA, RWDataPlyr, rWikiPathways, RWildbook, rworkflows, RxnSim, rxode2, ryandexdirect, rzentra, RZooRoH, s3fs, saeHB.twofold, SafeVote, salesforcer, samadb, sampcompR, samplingin, sangeranalyseR, sanityTracker, sansa, SARC, sarp.snowprofile, sarp.snowprofile.alignment, sarp.snowprofile.pyface, saseR, SC2API, scanMiR, scanMiRApp, scATAC.Explorer, scCustomize, SCdeconR, scDiffCom, scifer, scMET, scMitoMut, SCnorm, scoper, scopr, scorecard, ScoreEB, scoringutils, scPipe, scraEP, scRseqApp, scruff, scTreeViz, sdcHierarchies, sdcLog, sdcMicro, sdcTable, SDLfilter, SEA, seAMLess, SeaVal, seedr, segregation, SELF, semgram, semtree, sendigR, sense, sensobol, sentiment.ai, sentimentr, SenTinMixt, sentometrics, sentopics, seqsetvis, SeqVarTools, servosphereR, SEtools, sevenbridges, sevenbridges2, sevenC, sgat, SGP, SGPdata, SHAPforxgboost, shapr, shinyChatR, shinyepico, shinyExprPortal, shinyHugePlot, shinylogs, shinyML, ShinyQuickStarter, shinySbm, shinyWGD, siconvr, sigminer, Signac, signatureSearch, siie, SimBu, simcausal, simDAG, SimEngine, simfinapi, simitation, simPH, simphony, simplePHENOTYPES, simPop, simstudy, SimSurvey, simtrial, SingleCellSignalR, singleCellTK, sitepickR, siteymlgen, sitree, sitreeE, sjtable2df, skm, sleepr, SlideC, slim, smallstuff, SmartEDA, smartsnp, smoothic, snapcount, SNPannotator, SNPhood, snplinkage, snpsettest, socialh, socialmixr, soilDB, SoilHyP, SoilTaxonomy, soiltestcorr, solaR2, solitude, SOMNiBUS, SOMnmR, somspace, soundgen, spacyr, SpaDES.core, SpaDES.tools, sparrow, sparseFLMM, SPARSEMODr, SparseSignatures, spaths, SpatialCPie, SpatialFeatureExperiment, spatialHeatmap, SpatialOmicsOverlay, spatialrisk, SpATS, spatsoc, spDates, speaq, spectr, spectralAnalysis, spellcheckr, spicyR, SpliceWiz, splitstackshape, splutil, spNetwork, spocc, SPONGE, sport, sqlHelpers, SQMtools, squallms, squid, SSHAARP, staggered, standartox, staRdom, statar, statcanR, statgenGWAS, statgenIBD, statgenMPP, statgenQTLxT, Statial, Statsomat, stdReg, stdReg2, stenR, SticsRFiles, stm, stochvolTMB, stplanr, stppSim, strvalidator, subscreen, subSeq, SUMMER, suncalc, SUNGEO, SunsVoc, superml, surveyplanning, surveyPrev, surveysd, survMisc, susographql, SVMDO, SVN, SWATH2stats, sweater, SWMPr, symphony, synlet, synr, table.express, tablecompare, tabledown, tableMatrix, tabulator, tabxplor, takos, TangledFeatures, targeted, targets, taskscheduleR, tatoo, taxize, taxonomizr, TCA, TCGAbiolinks, tcgaViz, tcpl, tdsc, TeXCheckR, text2vec, textclean, textplot, textpress, textrank, textshape, textTinyR, textTools, TheOpenAIR, TideCurves, TideTables, tidyDenovix, TidyDensity, tidyfast, tidyfst, tidyft, tidygapminder, tidyplate, tidyrules, tidytable, tidyterra, tidytransit, tidyvpc, timeplyr, timeseriesdb, TIN, TitanC, tLOH, tmap, TMSig, TnT, toolStability, topdowntimeratio, tosca, totalcensus, ToxicoGx, toxSummary, TPAC, TPACData, tpm, TPP, tracee, trackdf, tradestatistics, TrafficBDE, trainsplit, transcriptogramer, transport, traveltimeR, treediff, treekoR, treemap, TreeMineR, treeshap, triact, TrialEmulation, tRigon, trimmer, tripr, TrumpetPlots, tsbox, tsdistributions, tsgarch, tsmarch, tsmethods, TSstudio, tstests, tstools, TTAinterfaceTrendAnalysis, ttgsea, Tushare, tvtools, twang, twangRDC, twc, twl, twopartm, txshift, uavRmp, ubair, UCell, uci, udpipe, ugatsdb, UKB.COVID19, ukbnmr, ukbtools, UKgrid, Ularcirc, Unico, Uniquorn, uptasticsearch, utilsIPEA, validata, VanillaICE, vardpoor, vein, versioning, versus, VIM, vimp, visualpred, vivainsights, VMDecomp, VOSONDash, vosonSML, vote, votesys, VoxR, vvtableau, walkboutr, wally, WARDEN, weatherOz, WeatherSentiment, WebAnalytics, webtrackR, wehoop, WGScan, WhatsR, whSample, wikitaxa, wilson, word.alignment, worrms, wpa, wpp2017, wpp2019, wTO, wyz.code.metaTesting, wyz.code.offensiveProgramming, wyz.code.rdoc, wyz.code.testthat, xefun, xesreadR, xgb2sql, xgboost, XString, yamss, YEAB, youngSwimmers, zebu, zeitgebr, ZIM4rv, ZIprop, zoltr, zoomGroupStats","","","Statsomat","","nandb","","","riverconn","","censobr","","","","ADPclust, ASMap, aSPU, astrochron, autoFRK, autoimage, aweSOM, BallMapper, BayesLCA, BayesNetBP, bayesPop, biClassify, bioRad, bipartite, biwavelet, bkmr, BMRMM, BoundaryStats, BRcal, Canopy, CCA, ccTensor, chillR, ciftiTools, CJIVE, cmR, cmsafops, cmsafvis, CNLTtsa, CoastlineFD, cocons, convoSPAT, cope, corr2D, CSHShydRology, dartR.sim, dcTensor, demu, DesignCTPB, DshapeR, earlywarnings, eDITH, EGRET, EMD, envi, ETAS, etasFLP, ETRep, ExpAnalysis3d, expectreg, fcfdr, fcr, fdasrvf, fence, FLightR, fnets, foreSIGHT, fpa, fsbrain, funData, gainML, gapfill, gateR, GeneralizedWendland, GeoAdjust, geoelectrics, GEOmap, GeoModels, georob, geospt, geosptdb, GeoThinneR, goeveg, GPFDA, GPvecchia, GRIDCOPULA, gwrr, hero, hglasso, hht, highriskzone, Hmsc, iCNV, infraFDTD.assist, JOPS, klovan, landsepi, LatticeKrig, lgcp, LMERConvenienceFunctions, mappoly, metapone, ModelMap, MOLHD, mousetrap, mrbsizeR, MRFA, MSIMST, MSmix, Mulcom, MultiObjMatch, mvLSW, NCutYX, nnTensor, nucim, oceanmap, OCNet, OptimaRegion, paramGUI, PEIP, pepStat, plsgenomics, ppgm, prevR, pspatreg, qfa, qlcVisualize, RadioSonde, ramps, reslr, RFOC, rhcoclust, RinteractMAPK, rivnet, RnBeads, RNCEP, rNOMADS, robflreg, RPANDA, rsat, RsimMosaic, rworldmap, SafeVote, scBio, scDD, SeBR, Semblance, SensMap, siland, sincell, singleCellTK, smosr, soilphysics, spant, sparrpowR, sparseBC, SpatialExtremes, spatialLIBD, SpatialVS, SpatialVx, SpATS, spatsurv, spBayesSurv, SpeCond, SpectralMap, spm2, spmoran, spThin, stilt, StMoMo, stocc, stopp, stxplore, SUMMER, SurvCorr, SynchWave, telefit, TFARM, TideCurves, TIMP, topologyGSA, TrackReconstruction, treespace, TwoTimeScales, uci, Umatrix, vectorwavelet, verification, voluModel, Voss, vote, warpMix, wavScalogram, waywiser, WRTDStidal, wsyn","","","","","","","causalBatch, CBPS, cem, EpiForsk, jsmodule, MatchThem, netdiffuseR, PSAboot, psBayesborrow, psborrow, RCTrep, SAMprior, sitepickR, vaccineff","CytoGLMM","","","AFM, AUCell, beastt, biodosetools, CBEA, CEDA, CiteFuse, dafishr, ddpcr, DSWE, EntropyMCMC, enviGCMS, extrememix, gemini, genomicInstability, ggrasp, IMIX, INFOSET, kernplus, l1kdeconv, mcrPioda, MiRKAT, MisRepARMA, MixSemiRob, Motif2Site, opGMMassessment, plotmm, pRoloc, QuantileGH, rnaCrosslinkOO, RprobitB, rstanbdp, RTN, scapGNN, scClassify, scone, scPloidy, serocalculator, smartid, UpDown, viper, ZetaSuite, ztpln","mcboost, mlr3fairness, mlr3fda, mlr3shiny, mlr3torch, mlr3verse, sense, spFSR","","","","MALDIcellassay, PvSTATEM","","","","","baytrends, brinton, carpenter, clintools, dataMaid, dataReporter, dbGaPCheckup, desctable, DiffXTables, eclust, epiR, funModeling, gmoTree, goxygen, hagis, IAPWS95, INSPECTumours, jtools, LifeInsureR, LOGAN, metaseqR2, MplusAutomation, mscstexta4r, mscsweblm4r, OPL, packagefinder, PCADSC, qcmetrics, rapport, rapportools, reportRmd, rmda, rosetta, sdsfun, shinyobjects, summarytools, ufs, veccompare, xgxr","","redistverse, wru","","","","tern.rbmi","","ABSurvTDC, accucor, aelab, AgroTech, AHPWR, aire.zmvm, allMT, AlpsNMR, amanida, amerifluxr, autonomics, AvInertia, aweSOM, baizer, batchLLM, BayesianFitForecast, baytrends, bcdata, bestSDP, BGVAR, bibliometrix, bndesr, bootf2, BrazilMet, breathtestcore, bulkreadr, capl, carbonr, CCWeights, ceas, cepumd, cgmanalysis, ChestVolume, chillR, chromConverter, clevRvis, CMapViz, CoastlineFD, COINr, concatipede, covid19.analytics, COVIDIBGE, covidmx, crosswalkr, cTRAP, dafishr, dams, DataLoader, datamods, datazoom.amazonia, dbGaPCheckup, DCEtool, DDIwR, decompTumor2Sig, defineR, dendRoAnalyst, DescTools, dextergui, DIFshiny, DistatisR, DMRcatedata, dmtools, Doscheda, dosedesignR, drawsample, duawranglr, easyr, ECOTOXr, EdSurvey, eHDPrep, eirm, epe4md, eph, epitweetr, esaps, excel2eprime, exceldata, expDB, extractox, fabR, FAST.R, FastRet, fetch, fio, fitPS, flattabler, forgts, formods, FSK2R, gDRimport, GeDi, GEOexplorer, GeomxTools, GeoWeightedModel, GetTDData, GGIRread, ggplotgui, GiniDistance, goat, GOLDprice, googlePublicData, greenfeedr, Greymodels, growthPheno, GWSDAT, HaDeX, hclusteasy, hiReadsProcessor, Holomics, hosm, htrSPRanalysis, htsr, hydrotoolbox, hydroToolkit, icertool, ideanet, IDEATools, IDSL.CSA, IDSL.IPA, IDSL.NPA, IDSL.UFA, IDSL.UFAx, ie2misc, ifo, ifpd, IGoRRR, ihpdr, imageData, immunarch, ImportExport, InflectSSP, insane, INSPEcT, INSPECTumours, interca, InterCellar, InterVA5, iotables, irtawsi, irtGUI, ISAnalytics, isobxr, IsoCorrectoR, IsoCorrectoRGUI, ISRaD, istat, joinXL, jsmodule, jubilee, knfi, KoboconnectR, lavaangui, LearningStats, lehuynh, libr, loadeR, logib, ltertools, Luminescence, maat, macrocol, MACSQuantifyR, MassWateR, matriz, mbQTL, mcmsupply, MDMAPR, metacore, MFO, microbiomeMQC, micromapST, miniMeta, mintyr, mipplot, miRetrieve, mlms, modgetxl, moodef, MSstatsShiny, netShiny, novelqualcodes, ntdr, OlinkAnalyze, OmnipathR, oncmap, One4All, onmaRg, opendatatoronto, OpenMindat, pacta.multi.loanbook, PAMhm, Pandora, papci, PDcIBGE, pdi, Pedixplorer, pedquant, PerRegMod, PesticideLoadIndicator, PhageCocktail, photobiologyInOut, pioneeR, plotGrouper, PNDSIBGE, PNSIBGE, PODES, Poly4AT, pressuRe, prior3D, PUPMCR, puzzle, PvSTATEM, pxmake, qrlabelr, QTL.gCIMapping, qualitycontrol, quicR, QuICSeedR, QurvE, radiant.data, ramlegacy, rbiom, RBNZ, RcmdrMisc, Rdiagnosislist, RDML, readabs, readit, readmoRe, readrba, readtext, reappraised, regions, Relectoral, replicateBE, rexer, rio, RJafroc, RLumShiny, rodeo, rsurveycto, Rtrack, SAKERS, SangerTools, scan, scbursts, scdhlm, scicomptools, SDAR, sdtmval, SEAHORS, seahtrue, secuTrialR, SemNetCleaner, sendigR, shinyHeatmaply, shinyIRT, ShinyLink, sivirep, SoilManageR, SomaDataIO, SpatialOmicsOverlay, sphereML, spiro, SQI, SqueakR, SUSES, tall, TestGenerator, TFutils, threesixtygiving, tidyDenovix, tidygapminder, tidyplate, tidyquant, tidytlg, tidyverse, toxEval, tpfp, TR8, treasury, tRigon, TSAR, ubiquity, unheadr, upstartr, uscoauditlog, vcdExtra, VectorCodeR, visvaR, visvow, vvmover, weed, whippr, ypssc","","","MsQuality, PTXQC","spANOVA","","","MLE, PINstimation, powerHE","","finbipartite, fingraph","","","","","teal.modules.clinical, teal.modules.general","","","alarmdata, censable, geomander, PL94171, redistverse","","TwitterAutomatedTrading","semnar","","","","","CIMTx, MatchThem, mvGPS","deprivateR"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>deadline<\/th>\n      <th>package<\/th>\n      <th>version<\/th>\n      <th>maintainer<\/th>\n      <th>links<\/th>\n      <th>event<\/th>\n      <th>reverse packages (n)<\/th>\n      <th>reverse packages<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":50,"order":[[0,"asc"],[6,"desc"],[1,"asc"]],"columnDefs":[{"className":"dt-right","targets":6},{"name":"deadline","targets":0},{"name":"package","targets":1},{"name":"version","targets":2},{"name":"maintainer","targets":3},{"name":"links","targets":4},{"name":"event","targets":5},{"name":"reverse packages (n)","targets":6},{"name":"reverse packages","targets":7}],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>




<div id="quarto-appendix" class="default"><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><h2 class="anchored quarto-appendix-heading">Footnotes</h2>

<ol>
<li id="fn1"><p>When a package is <em>archived</em>, the archived version is still available in the <a href="https://cran.r-project.org/src/contrib/Archive/">CRAN archive area</a> together with all previous versions.<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




<footer class="footer"><div class="nav-footer"><div class="nav-footer-center"><div class="toc-actions"><ul><li><a href="https://github.com/cranhaven/cranhaven.org/blob/main/dashboard-at-risk.qmd" class="toc-action"><i class="bi bi-github"></i>View source</a></li><li><a href="https://github.com/cranhaven/cranhaven.org/edit/main/dashboard-at-risk.qmd" class="toc-action"><i class="bi empty"></i>Edit this page</a></li><li><a href="https://github.com/cranhaven/cranhaven.org/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></div></div></footer></body></html>