<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.42">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Dashboard: Packages At-Risk of Being Archived – CRANhaven</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-2f5df379a58b258e96c21c0638c20c03.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-8647a4a42273f773479d27c00df3f9ed.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">CRANhaven</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-articles" role="link" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Articles</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-articles">    
        <li>
    <a class="dropdown-item" href="./dashboard-live.html">
 <span class="dropdown-text">Dashboard: Recently Archived CRAN Packages</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./dashboard-at-risk.html">
 <span class="dropdown-text">Dashboard: Packages At-Risk of Being Archived</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./cran-archiving-stats.html">
 <span class="dropdown-text">Study: Many Archived Packages Return to CRAN</span></a>
  </li>  
    </ul>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">About</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/cranhaven/cranhaven.org/"> <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar"><div class="quarto-margin-header"><div class="margin-header-item">
<p><a href="https://cranhaven.r-universe.dev/builds"><img src="https://cranhaven.r-universe.dev/badges/:total" alt="CRANhaven number of packages"></a></p>
</div></div>
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Dashboard: Packages At-Risk of Being Archived</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<style>
tr { vertical-align: top; }
</style>
<p>The below table lists 112 R packages that are at risk of being archived<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a> on CRAN, because they have outstanding unresolved issues.</p>
<p>Some of them might have already submitted a new version to fix the issues identified by CRAN (check the event column).</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="live-dashboard" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="live-dashboard">{"x":{"filter":"none","vertical":false,"data":[["2025-03-20<br><small>(in 14 days)<\/small>","2025-03-06<br><small>(today)<\/small>","2025-03-12<br><small>(in 6 days)<\/small>","2025-03-19<br><small>(in 13 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-12<br><small>(in 6 days)<\/small>","2025-03-19<br><small>(in 13 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-19<br><small>(in 13 days)<\/small>","2025-03-08<br><small>(in 2 days)<\/small>","2025-03-16<br><small>(in 10 days)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-12<br><small>(in 6 days)<\/small>","2025-03-10<br><small>(in 4 days)<\/small>","2025-03-20<br><small>(in 14 days)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-12<br><small>(in 6 days)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-10<br><small>(in 4 days)<\/small>","2025-03-20<br><small>(in 14 days)<\/small>","2025-03-08<br><small>(in 2 days)<\/small>","2025-03-16<br><small>(in 10 days)<\/small>","2025-03-20<br><small>(in 14 days)<\/small>","2025-03-19<br><small>(in 13 days)<\/small>","2025-03-09<br><small>(in 3 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-19<br><small>(in 13 days)<\/small>","2025-03-09<br><small>(in 3 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-09<br><small>(in 3 days)<\/small>","2025-03-19<br><small>(in 13 days)<\/small>","2025-03-14<br><small>(in 8 days)<\/small>","2025-03-19<br><small>(in 13 days)<\/small>","2025-03-20<br><small>(in 14 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-09<br><small>(in 3 days)<\/small>","2025-03-16<br><small>(in 10 days)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-17<br><small>(in 11 days)<\/small>","2025-03-12<br><small>(in 6 days)<\/small>","2025-03-19<br><small>(in 13 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-09<br><small>(in 3 days)<\/small>","2025-03-12<br><small>(in 6 days)<\/small>","2025-03-20<br><small>(in 14 days)<\/small>","2025-03-09<br><small>(in 3 days)<\/small>","2025-03-10<br><small>(in 4 days)<\/small>","2025-03-18<br><small>(in 12 days)<\/small>","2025-03-10<br><small>(in 4 days)<\/small>","2025-03-20<br><small>(in 14 days)<\/small>","2025-03-13<br><small>(in 7 days)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-06<br><small>(today)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-24<br><small>(in 18 days)<\/small>","2025-03-09<br><small>(in 3 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-16<br><small>(in 10 days)<\/small>","2025-03-08<br><small>(in 2 days)<\/small>","2025-03-17<br><small>(in 11 days)<\/small>","2025-03-14<br><small>(in 8 days)<\/small>","2025-03-20<br><small>(in 14 days)<\/small>","2025-03-09<br><small>(in 3 days)<\/small>","2025-03-20<br><small>(in 14 days)<\/small>","2025-03-19<br><small>(in 13 days)<\/small>","2025-03-10<br><small>(in 4 days)<\/small>","2025-03-14<br><small>(in 8 days)<\/small>","2025-03-09<br><small>(in 3 days)<\/small>","2025-03-09<br><small>(in 3 days)<\/small>","2025-03-20<br><small>(in 14 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-20<br><small>(in 14 days)<\/small>","2025-03-14<br><small>(in 8 days)<\/small>","2025-03-10<br><small>(in 4 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-14<br><small>(in 8 days)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-20<br><small>(in 14 days)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-09<br><small>(in 3 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-18<br><small>(in 12 days)<\/small>","2025-03-21<br><small>(in 15 days)<\/small>","2025-03-14<br><small>(in 8 days)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-17<br><small>(in 11 days)<\/small>","2025-03-18<br><small>(in 12 days)<\/small>","2025-03-18<br><small>(in 12 days)<\/small>","2025-03-06<br><small>(today)<\/small>","2025-03-14<br><small>(in 8 days)<\/small>","2025-03-08<br><small>(in 2 days)<\/small>","2025-03-11<br><small>(in 5 days)<\/small>","2025-03-12<br><small>(in 6 days)<\/small>","2025-03-16<br><small>(in 10 days)<\/small>","2025-03-09<br><small>(in 3 days)<\/small>","2025-03-18<br><small>(in 12 days)<\/small>","2025-03-13<br><small>(in 7 days)<\/small>","2025-03-16<br><small>(in 10 days)<\/small>","2025-03-08<br><small>(in 2 days)<\/small>","2025-03-07<br><small>(in 1 day)<\/small>","2025-03-12<br><small>(in 6 days)<\/small>","2025-03-08<br><small>(in 2 days)<\/small>","2025-03-20<br><small>(in 14 days)<\/small>","2025-03-05<br><small>(1 day overdue)<\/small>","2025-03-10<br><small>(in 4 days)<\/small>","2025-03-16<br><small>(in 10 days)<\/small>"],["<a href=\"https://cran.r-project.org/package=accessr\">accessr<\/a>","<a href=\"https://cran.r-project.org/package=AICcmodavg\">AICcmodavg<\/a>","<a href=\"https://cran.r-project.org/package=ale\">ale<\/a>","<a href=\"https://cran.r-project.org/package=BayesChange\">BayesChange<\/a>","<a href=\"https://cran.r-project.org/package=bayesdfa\">bayesdfa<\/a>","<a href=\"https://cran.r-project.org/package=bdsm\">bdsm<\/a>","<a href=\"https://cran.r-project.org/package=bigstep\">bigstep<\/a>","<a href=\"https://cran.r-project.org/package=BRugs\">BRugs<\/a>","<a href=\"https://cran.r-project.org/package=btergm\">btergm<\/a>","<a href=\"https://cran.r-project.org/package=cbq\">cbq<\/a>","<a href=\"https://cran.r-project.org/package=CHNOSZ\">CHNOSZ<\/a>","<a href=\"https://cran.r-project.org/package=clinDataReview\">clinDataReview<\/a>","<a href=\"https://cran.r-project.org/package=CohortCharacteristics\">CohortCharacteristics<\/a>","<a href=\"https://cran.r-project.org/package=CohortSurvival\">CohortSurvival<\/a>","<a href=\"https://cran.r-project.org/package=cxhull\">cxhull<\/a>","<a href=\"https://cran.r-project.org/package=D4TAlink.light\">D4TAlink.light<\/a>","<a href=\"https://cran.r-project.org/package=DataCombine\">DataCombine<\/a>","<a href=\"https://cran.r-project.org/package=DataFakeR\">DataFakeR<\/a>","<a href=\"https://cran.r-project.org/package=datawizard\">datawizard<\/a>","<a href=\"https://cran.r-project.org/package=DGP4LCF\">DGP4LCF<\/a>","<a href=\"https://cran.r-project.org/package=diffEnrich\">diffEnrich<\/a>","<a href=\"https://cran.r-project.org/package=disprofas\">disprofas<\/a>","<a href=\"https://cran.r-project.org/package=dtmapi\">dtmapi<\/a>","<a href=\"https://cran.r-project.org/package=easycsv\">easycsv<\/a>","<a href=\"https://cran.r-project.org/package=ecb\">ecb<\/a>","<a href=\"https://cran.r-project.org/package=EFAtools\">EFAtools<\/a>","<a href=\"https://cran.r-project.org/package=equatiomatic\">equatiomatic<\/a>","<a href=\"https://cran.r-project.org/package=fcirt\">fcirt<\/a>","<a href=\"https://cran.r-project.org/package=fields\">fields<\/a>","<a href=\"https://cran.r-project.org/package=FLightR\">FLightR<\/a>","<a href=\"https://cran.r-project.org/package=float\">float<\/a>","<a href=\"https://cran.r-project.org/package=flocker\">flocker<\/a>","<a href=\"https://cran.r-project.org/package=flowTraceR\">flowTraceR<\/a>","<a href=\"https://cran.r-project.org/package=fucom\">fucom<\/a>","<a href=\"https://cran.r-project.org/package=gimap\">gimap<\/a>","<a href=\"https://cran.r-project.org/package=greatR\">greatR<\/a>","<a href=\"https://cran.r-project.org/package=hacksaw\">hacksaw<\/a>","<a href=\"https://cran.r-project.org/package=hBayesDM\">hBayesDM<\/a>","<a href=\"https://cran.r-project.org/package=hddplot\">hddplot<\/a>","<a href=\"https://cran.r-project.org/package=hilldiv\">hilldiv<\/a>","<a href=\"https://cran.r-project.org/package=hmmr\">hmmr<\/a>","<a href=\"https://cran.r-project.org/package=httr2\">httr2<\/a>","<a href=\"https://cran.r-project.org/package=hutilscpp\">hutilscpp<\/a>","<a href=\"https://cran.r-project.org/package=ICS\">ICS<\/a>","<a href=\"https://cran.r-project.org/package=imt\">imt<\/a>","<a href=\"https://cran.r-project.org/package=isotracer\">isotracer<\/a>","<a href=\"https://cran.r-project.org/package=kmeRs\">kmeRs<\/a>","<a href=\"https://cran.r-project.org/package=LogConcDEAD\">LogConcDEAD<\/a>","<a href=\"https://cran.r-project.org/package=manymodelr\">manymodelr<\/a>","<a href=\"https://cran.r-project.org/package=mapping\">mapping<\/a>","<a href=\"https://cran.r-project.org/package=MatchIt\">MatchIt<\/a>","<a href=\"https://cran.r-project.org/package=maths.genealogy\">maths.genealogy<\/a>","<a href=\"https://cran.r-project.org/package=mbest\">mbest<\/a>","<a href=\"https://cran.r-project.org/package=mgm\">mgm<\/a>","<a href=\"https://cran.r-project.org/package=MicroSEC\">MicroSEC<\/a>","<a href=\"https://cran.r-project.org/package=miWQS\">miWQS<\/a>","<a href=\"https://cran.r-project.org/package=mixtools\">mixtools<\/a>","<a href=\"https://cran.r-project.org/package=mlr3pipelines\">mlr3pipelines<\/a>","<a href=\"https://cran.r-project.org/package=modelbased\">modelbased<\/a>","<a href=\"https://cran.r-project.org/package=mpwR\">mpwR<\/a>","<a href=\"https://cran.r-project.org/package=multipleDL\">multipleDL<\/a>","<a href=\"https://cran.r-project.org/package=ncdf4.helpers\">ncdf4.helpers<\/a>","<a href=\"https://cran.r-project.org/package=nestedmodels\">nestedmodels<\/a>","<a href=\"https://cran.r-project.org/package=noegletalR\">noegletalR<\/a>","<a href=\"https://cran.r-project.org/package=odbc\">odbc<\/a>","<a href=\"https://cran.r-project.org/package=officedown\">officedown<\/a>","<a href=\"https://cran.r-project.org/package=ohtadstats\">ohtadstats<\/a>","<a href=\"https://cran.r-project.org/package=opendatatoronto\">opendatatoronto<\/a>","<a href=\"https://cran.r-project.org/package=pcatsAPIclientR\">pcatsAPIclientR<\/a>","<a href=\"https://cran.r-project.org/package=PLNmodels\">PLNmodels<\/a>","<a href=\"https://cran.r-project.org/package=PubChemR\">PubChemR<\/a>","<a href=\"https://cran.r-project.org/package=qdapRegex\">qdapRegex<\/a>","<a href=\"https://cran.r-project.org/package=qMRI\">qMRI<\/a>","<a href=\"https://cran.r-project.org/package=R2BayesX\">R2BayesX<\/a>","<a href=\"https://cran.r-project.org/package=rbioacc\">rbioacc<\/a>","<a href=\"https://cran.r-project.org/package=Rcrawler\">Rcrawler<\/a>","<a href=\"https://cran.r-project.org/package=readxl\">readxl<\/a>","<a href=\"https://cran.r-project.org/package=rempsyc\">rempsyc<\/a>","<a href=\"https://cran.r-project.org/package=restez\">restez<\/a>","<a href=\"https://cran.r-project.org/package=Rfast\">Rfast<\/a>","<a href=\"https://cran.r-project.org/package=rhub\">rhub<\/a>","<a href=\"https://cran.r-project.org/package=riskmetric\">riskmetric<\/a>","<a href=\"https://cran.r-project.org/package=rmzqc\">rmzqc<\/a>","<a href=\"https://cran.r-project.org/package=rock\">rock<\/a>","<a href=\"https://cran.r-project.org/package=rstanarm\">rstanarm<\/a>","<a href=\"https://cran.r-project.org/package=Rvcg\">Rvcg<\/a>","<a href=\"https://cran.r-project.org/package=sasr\">sasr<\/a>","<a href=\"https://cran.r-project.org/package=scDHA\">scDHA<\/a>","<a href=\"https://cran.r-project.org/package=ScottKnott\">ScottKnott<\/a>","<a href=\"https://cran.r-project.org/package=scRNAstat\">scRNAstat<\/a>","<a href=\"https://cran.r-project.org/package=shiny.info\">shiny.info<\/a>","<a href=\"https://cran.r-project.org/package=skellam\">skellam<\/a>","<a href=\"https://cran.r-project.org/package=SPUTNIK\">SPUTNIK<\/a>","<a href=\"https://cran.r-project.org/package=STRMPS\">STRMPS<\/a>","<a href=\"https://cran.r-project.org/package=synthpop\">synthpop<\/a>","<a href=\"https://cran.r-project.org/package=T2DFitTailor\">T2DFitTailor<\/a>","<a href=\"https://cran.r-project.org/package=TEAM\">TEAM<\/a>","<a href=\"https://cran.r-project.org/package=tern.rbmi\">tern.rbmi<\/a>","<a href=\"https://cran.r-project.org/package=tessellation\">tessellation<\/a>","<a href=\"https://cran.r-project.org/package=TestGenerator\">TestGenerator<\/a>","<a href=\"https://cran.r-project.org/package=tidyformula\">tidyformula<\/a>","<a href=\"https://cran.r-project.org/package=tidyr\">tidyr<\/a>","<a href=\"https://cran.r-project.org/package=tmbstan\">tmbstan<\/a>","<a href=\"https://cran.r-project.org/package=TreatmentPatterns\">TreatmentPatterns<\/a>","<a href=\"https://cran.r-project.org/package=TT\">TT<\/a>","<a href=\"https://cran.r-project.org/package=TukeyC\">TukeyC<\/a>","<a href=\"https://cran.r-project.org/package=TukeyRegion\">TukeyRegion<\/a>","<a href=\"https://cran.r-project.org/package=tzdb\">tzdb<\/a>","<a href=\"https://cran.r-project.org/package=unitizer\">unitizer<\/a>","<a href=\"https://cran.r-project.org/package=wallace\">wallace<\/a>","<a href=\"https://cran.r-project.org/package=weatherOz\">weatherOz<\/a>","<a href=\"https://cran.r-project.org/package=wxgenR\">wxgenR<\/a>"],["1.1.1<br><small>(2025-01-15)<small>","2.3-3<br><small>(2023-11-16)<small>","0.3.0<br><small>(2024-02-14)<small>","1.1.2<br><small>(2025-02-21)<small>","1.3.3<br><small>(2024-02-26)<small>","0.1.0<br><small>(2025-01-21)<small>","1.1.1<br><small>(2023-05-13)<small>","0.9-2.1<br><small>(2024-02-25)<small>","1.10.12<br><small>(2024-03-31)<small>","0.2.0.3<br><small>(2023-04-02)<small>","2.1.0<br><small>(2024-02-11)<small>","1.6.1<br><small>(2024-06-18)<small>","0.4.0<br><small>(2024-11-26)<small>","0.6.2<br><small>(2025-01-22)<small>","0.7.4<br><small>(2023-10-24)<small>","2.1.17<br><small>(2024-12-02)<small>","0.2.21<br><small>(2016-04-13)<small>","0.1.3<br><small>(2023-02-12)<small>","1.0.0<br><small>(2025-01-10)<small>","1.0.0<br><small>(2024-05-28)<small>","0.1.2<br><small>(2022-06-27)<small>","0.2.0<br><small>(2024-06-10)<small>","0.0.2<br><small>(2024-09-17)<small>","1.0.8<br><small>(2018-05-21)<small>","0.4.2<br><small>(2023-04-20)<small>","0.4.5<br><small>(2024-12-22)<small>","0.3.3<br><small>(2024-05-26)<small>","0.1.0<br><small>(2022-02-01)<small>","16.3<br><small>(2024-09-30)<small>","0.5.5<br><small>(2024-07-04)<small>","0.3-2<br><small>(2023-12-10)<small>","1.0-0<br><small>(2024-02-05)<small>","0.1.0<br><small>(2022-06-03)<small>","0.0.3<br><small>(2025-01-15)<small>","1.0.2<br><small>(2025-02-18)<small>","2.0.0<br><small>(2024-04-09)<small>","0.0.2<br><small>(2020-12-15)<small>","1.2.1<br><small>(2022-09-23)<small>","0.59-2<br><small>(2023-09-14)<small>","1.5.1<br><small>(2019-10-01)<small>","1.0-0<br><small>(2021-05-27)<small>","1.1.0<br><small>(2025-01-18)<small>","0.10.7<br><small>(2025-01-08)<small>","1.4-1<br><small>(2023-09-21)<small>","1.0.0<br><small>(2024-09-02)<small>","1.1.7<br><small>(2024-11-05)<small>","2.1.0<br><small>(2024-07-17)<small>","1.6-10<br><small>(2024-09-11)<small>","0.3.7<br><small>(2021-11-15)<small>","1.4.1<br><small>(2023-12-12)<small>","4.7.0<br><small>(2025-01-12)<small>","0.1.1<br><small>(2025-03-03)<small>","0.6<br><small>(2018-05-25)<small>","1.2-14<br><small>(2023-09-05)<small>","2.1.6<br><small>(2024-08-25)<small>","0.4.4<br><small>(2021-04-02)<small>","2.0.0<br><small>(2022-12-05)<small>","0.7.1<br><small>(2024-11-14)<small>","0.9.0<br><small>(2025-02-05)<small>","0.1.5<br><small>(2023-11-13)<small>","1.0.0<br><small>(2023-11-24)<small>","0.3-6<br><small>(2021-10-15)<small>","1.1.0<br><small>(2023-09-30)<small>","0.2.1<br><small>(2024-09-12)<small>","1.6.0<br><small>(2025-02-28)<small>","0.3.3<br><small>(2024-07-16)<small>","2.1.1<br><small>(2019-11-15)<small>","0.1.5<br><small>(2022-04-13)<small>","1.1.0<br><small>(2023-06-23)<small>","1.2.0<br><small>(2024-03-05)<small>","2.1.3<br><small>(2024-12-16)<small>","0.7.8<br><small>(2023-10-17)<small>","1.2.7.9<br><small>(2025-03-06)<small>","1.1-5<br><small>(2023-10-20)<small>","1.2.1<br><small>(2024-02-27)<small>","0.1.9-1<br><small>(2018-11-11)<small>","1.4.4<br><small>(2025-02-27)<small>","0.1.9<br><small>(2025-02-01)<small>","2.1.4<br><small>(2023-10-25)<small>","2.1.5<br><small>(2025-02-27)<small>","2.0.0<br><small>(2024-04-11)<small>","0.2.4<br><small>(2024-01-09)<small>","0.5.4<br><small>(2024-04-16)<small>","0.8.1<br><small>(2024-01-21)<small>","2.32.1<br><small>(2024-01-18)<small>","0.24<br><small>(2024-09-20)<small>","0.1.4<br><small>(2025-02-27)<small>","1.2.2<br><small>(2024-04-02)<small>","1.3-2<br><small>(2023-08-30)<small>","0.1.1<br><small>(2021-09-22)<small>","0.2.0<br><small>(2020-03-23)<small>","0.2.3<br><small>(2024-03-25)<small>","1.4.2<br><small>(2024-04-16)<small>","0.6.8<br><small>(2025-02-22)<small>","1.9-0<br><small>(2024-12-20)<small>","3.0.0<br><small>(2024-04-16)<small>","0.1.0<br><small>(2019-09-13)<small>","0.1.4<br><small>(2024-11-26)<small>","2.3.0<br><small>(2023-12-18)<small>","0.3.3<br><small>(2024-11-08)<small>","0.1.0<br><small>(2023-02-17)<small>","1.3.1<br><small>(2024-01-24)<small>","1.0.91<br><small>(2023-08-26)<small>","3.0.0<br><small>(2025-02-06)<small>","0.98<br><small>(2022-04-05)<small>","1.3-42<br><small>(2023-08-31)<small>","0.1.6.3<br><small>(2023-04-17)<small>","0.4.0<br><small>(2023-05-12)<small>","1.4.21<br><small>(2024-05-18)<small>","2.1.3<br><small>(2024-07-18)<small>","1.0.0<br><small>(2024-07-26)<small>","1.4.2<br><small>(2024-11-02)<small>"],["<a href=\"mailto:p.northrop-at-ucl.ac.uk\">Paul J. Northrop<\/a>","<a href=\"mailto:marc.mazerolle-at-sbf.ulaval.ca\">Marc J. Mazerolle<\/a>","<a href=\"mailto:Chitu.Okoli-at-skema.edu\">Chitu Okoli<\/a>","<a href=\"mailto:l.danese1-at-campus.unimib.it\">Luca Danese<\/a>","<a href=\"mailto:eric.ward-at-noaa.gov\">Eric J. Ward<\/a>","<a href=\"mailto:marcindubel-at-gmail.com\">Marcin Dubel<\/a>","<a href=\"mailto:piotr.michal.szulc-at-gmail.com\">Piotr Szulc<\/a>","<a href=\"mailto:ligges-at-statistik.tu-dortmund.de\">Uwe Ligges<\/a>","<a href=\"mailto:philip.leifeld-at-manchester.ac.uk\">Philip Leifeld<\/a>","<a href=\"mailto:xiao.lu.research-at-gmail.com\">Xiao LU<\/a>","<a href=\"mailto:j3ffdick-at-gmail.com\">Jeffrey Dick<\/a>","<a href=\"mailto:laure.cougnaud-at-openanalytics.eu\">Laure Cougnaud<\/a>","<a href=\"mailto:marti.catalasabate-at-ndorms.ox.ac.uk\">Marti Catala<\/a>","<a href=\"mailto:edward.burn-at-ndorms.ox.ac.uk\">Edward Burn<\/a>","<a href=\"mailto:laurent_step-at-outlook.fr\">Stéphane Laurent<\/a>","<a href=\"mailto:gregoire.thomas-at-SQU4RE.com\">Gregoire Thomas<\/a>","<a href=\"mailto:christopher.gandrud-at-gmail.com\">Christopher Gandrud<\/a>","<a href=\"mailto:krystian8207-at-gmail.com\">Krystian Igras<\/a>","<a href=\"mailto:etienne.bacher-at-protonmail.com\">Etienne Bacher<\/a>","<a href=\"mailto:jiachen.cai-at-mrc-bsu.cam.ac.uk\">Jiachen Cai<\/a>","<a href=\"mailto:harry.smith-at-ucdenver.edu\">Harry Smith<\/a>","<a href=\"mailto:pius.dahinden-at-tillotts.com\">Pius Dahinden<\/a>","<a href=\"mailto:lutran-at-iom.int\">Luong Bang Tran<\/a>","<a href=\"mailto:dror.bogin-at-gmail.com\">Dror Bogin<\/a>","<a href=\"mailto:expersso5-at-gmail.com\">Eric Persson<\/a>","<a href=\"mailto:markus.d.steiner-at-gmail.com\">Markus Steiner<\/a>","<a href=\"mailto:phgrosjean-at-sciviews.org\">Philippe Grosjean<\/a>","<a href=\"mailto:naidantu-at-usf.edu\">Naidan Tu<\/a>","<a href=\"mailto:douglasnychka-at-gmail.com\">Douglas Nychka<\/a>","<a href=\"mailto:eldar.rakhimberdiev-at-uva.nl\">Eldar Rakhimberdiev<\/a>","<a href=\"mailto:wrathematics-at-gmail.com\">Drew Schmidt<\/a>","<a href=\"mailto:jacob.socolar-at-gmail.com\">Jacob B. Socolar<\/a>","<a href=\"mailto:Okdll-at-gmx.net\">Oliver Kardell<\/a>","<a href=\"mailto:mateusvanzetta-at-id.uff.br\">Mateus Vanzetta<\/a>","<a href=\"mailto:cansav09-at-gmail.com\">Candace Savonen<\/a>","<a href=\"mailto:ruth.kristianingsih30-at-gmail.com\">Ruth Kristianingsih<\/a>","<a href=\"mailto:daranzolin-at-gmail.com\">David Ranzolin<\/a>","<a href=\"mailto:wooyoung.ahn-at-gmail.com\">Woo-Young Ahn<\/a>","<a href=\"mailto:jhmaindonald-at-gmail.com\">John Maindonald<\/a>","<a href=\"mailto:antton.alberdi-at-bio.ku.dk\">Antton Alberdi<\/a>","<a href=\"mailto:i.visser-at-uva.nl\">Ingmar Visser<\/a>","<a href=\"mailto:hadley-at-posit.co\">Hadley Wickham<\/a>","<a href=\"mailto:hugh.parsonage-at-gmail.com\">Hugh Parsonage<\/a>","<a href=\"mailto:klausnordhausenR-at-gmail.com\">Klaus Nordhausen<\/a>","<a href=\"mailto:martinezig-at-google.com\">Ignacio Martinez<\/a>","<a href=\"mailto:matthieu.bruneaux-at-gmail.com\">Matthieu Bruneaux<\/a>","<a href=\"mailto:rafal.urniaz-at-cantab.net\">Rafal Urniaz<\/a>","<a href=\"mailto:y.chen101-at-lse.ac.uk\">Yining Chen<\/a>","<a href=\"mailto:gonzabato-at-hotmail.com\">Nelson Gonzabato<\/a>","<a href=\"mailto:srf.alessio-at-gmail.com\">Alessio Serafini<\/a>","<a href=\"mailto:noah.greifer-at-gmail.com\">Noah Greifer<\/a>","<a href=\"mailto:louis.aslett-at-durham.ac.uk\">Louis Aslett<\/a>","<a href=\"mailto:patperry-at-gmail.com\">Patrick O. Perry<\/a>","<a href=\"mailto:jonashaslbeck-at-gmail.com\">Jonas Haslbeck<\/a>","<a href=\"mailto:ikegamitky-at-gmail.com\">Masachika Ikegami<\/a>","<a href=\"mailto:hargartenp-at-alumni.vcu.edu\">Paul M. Hargarten<\/a>","<a href=\"mailto:derek.young-at-uky.edu\">Derek Young<\/a>","<a href=\"mailto:mlr.developer-at-mb706.com\">Martin Binder<\/a>","<a href=\"mailto:dom.makowski-at-gmail.com\">Dominique Makowski<\/a>","<a href=\"mailto:Okdll-at-gmx.net\">Oliver Kardell<\/a>","<a href=\"mailto:yuqitian35-at-gmail.com\">Yuqi Tian<\/a>","<a href=\"mailto:lzeman-at-uvic.ca\">Lee Zeman<\/a>","<a href=\"mailto:ashbythorpe-at-gmail.com\">Ashby Thorpe<\/a>","<a href=\"mailto:frlars21-at-student.aau.dk\">Frederik Larsen<\/a>","<a href=\"mailto:hadley-at-posit.co\">Hadley Wickham<\/a>","<a href=\"mailto:david.gohel-at-ardata.fr\">David Gohel<\/a>","<a href=\"mailto:pfpetrowski-at-gmail.com\">Paul F. Petrowski<\/a>","<a href=\"mailto:sharla.gelfand-at-gmail.com\">Sharla Gelfand<\/a>","<a href=\"mailto:michal.kouril-at-cchmc.org\">Michal Kouril<\/a>","<a href=\"mailto:julien.chiquet-at-inrae.fr\">Julien Chiquet<\/a>","<a href=\"mailto:selcukorkmaz-at-gmail.com\">Selcuk Korkmaz<\/a>","<a href=\"mailto:tyler.rinker-at-gmail.com\">Tyler Rinker<\/a>","<a href=\"mailto:karsten.tabelow-at-wias-berlin.de\">Karsten Tabelow<\/a>","<a href=\"mailto:Nikolaus.Umlauf-at-uibk.ac.at\">Nikolaus Umlauf<\/a>","<a href=\"mailto:virgile.baudrot-at-qonfluens.com\">Virgile Baudrot<\/a>","<a href=\"mailto:khalilsalim1-at-gmail.com\">Salim Khalil<\/a>","<a href=\"mailto:jenny-at-posit.co\">Jennifer Bryan<\/a>","<a href=\"mailto:remi.theriault-at-mail.mcgill.ca\">Rémi Thériault<\/a>","<a href=\"mailto:joelnitta-at-gmail.com\">Joel H. Nitta<\/a>","<a href=\"mailto:rfastofficial-at-gmail.com\">Manos Papadakis<\/a>","<a href=\"mailto:csardi.gabor-at-gmail.com\">Gábor Csárdi<\/a>","<a href=\"mailto:eli.miller-at-atorusresearch.com\">Eli Miller<\/a>","<a href=\"mailto:chris.bielow-at-bsc.fu-berlin.de\">Chris Bielow<\/a>","<a href=\"mailto:rock-at-opens.science\">Gjalt-Jorn Peters<\/a>","<a href=\"mailto:benjamin.goodrich-at-columbia.edu\">Ben Goodrich<\/a>","<a href=\"mailto:zarquon42-at-gmail.com\">Stefan Schlager<\/a>","<a href=\"mailto:clark.liming-at-gmail.com\">Liming Li<\/a>","<a href=\"mailto:hvn0006-at-auburn.edu\">Ha Nguyen<\/a>","<a href=\"mailto:ivanalaman-at-gmail.com\">Ivan Bezerra Allaman<\/a>","<a href=\"mailto:xiayh17-at-gmail.com\">Yonghe Xia<\/a>","<a href=\"mailto:kuba-at-appsilon.com\">Jakub Nowicki<\/a>","<a href=\"mailto:patrick.brown-at-utoronto.ca\">Patrick E. Brown<\/a>","<a href=\"mailto:p.inglese-at-outlook.com\">Paolo Inglese<\/a>","<a href=\"mailto:svilsen-at-math.aau.dk\">Søren B. Vilsen<\/a>","<a href=\"mailto:beata.nowok-at-gmail.com\">Beata Nowok<\/a>","<a href=\"mailto:971264017-at-qq.com\">Cheng Liu<\/a>","<a href=\"mailto:john.pura-at-duke.edu\">John Pura<\/a>","<a href=\"mailto:joe.zhu-at-roche.com\">Joe Zhu<\/a>","<a href=\"mailto:laurent_step-at-outlook.fr\">Stéphane Laurent<\/a>","<a href=\"mailto:c.barboza-at-darwin-eu.org\">Cesar Barboza<\/a>","<a href=\"mailto:damian.pavlyshyn-at-gmail.com\">Damian Pavlyshyn<\/a>","<a href=\"mailto:hadley-at-posit.co\">Hadley Wickham<\/a>","<a href=\"mailto:kaskr-at-dtu.dk\">Kasper Kristensen<\/a>","<a href=\"mailto:m.l.vankessel-at-erasmusmc.nl\">Maarten van Kessel<\/a>","<a href=\"mailto:michal.zielaskowski-at-gmail.com\">Michal Zielaskowski<\/a>","<a href=\"mailto:ivanalaman-at-gmail.com\">Ivan Bezerra Allaman<\/a>","<a href=\"mailto:pavlo.mozharovskyi-at-telecom-paris.fr\">Pavlo Mozharovskyi<\/a>","<a href=\"mailto:davis-at-posit.co\">Davis Vaughan<\/a>","<a href=\"mailto:brodie.gaslam-at-yahoo.com\">Brodie Gaslam<\/a>","<a href=\"mailto:mblair1-at-amnh.org\">Mary E. Blair<\/a>","<a href=\"mailto:rodrigo.pires-at-dpird.wa.gov.au\">Rodrigo Pires<\/a>","<a href=\"mailto:dwoodson-at-usbr.gov\">David Woodson<\/a>"],["<a href=\"https://cran.r-project.org/web/checks/check_results_accessr.html\">checks<\/a><br><a href=\"https://paulnorthrop.github.io/accessr/\">url<\/a><br><a href=\"https://github.com/paulnorthrop/accessr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_AICcmodavg.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ale.html\">checks<\/a><br><a href=\"https://github.com/tripartio/ale\">url<\/a><br><a href=\"https://github.com/tripartio/ale/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_BayesChange.html\">checks<\/a><br><a href=\"https://github.com/lucadanese/BayesChange\">url<\/a><br><a href=\"https://github.com/lucadanese/BayesChange/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_bayesdfa.html\">checks<\/a><br><a href=\"https://fate-ewi.github.io/bayesdfa/\">url<\/a><br><a href=\"https://github.com/fate-ewi/bayesdfa/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_bdsm.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_bigstep.html\">checks<\/a><br><a href=\"https://github.com/pmszulc/bigstep\">url<\/a><br><a href=\"https://github.com/pmszulc/bigstep/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_BRugs.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_btergm.html\">checks<\/a><br><a href=\"https://github.com/leifeld/btergm\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_cbq.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_CHNOSZ.html\">checks<\/a><br><a href=\"https://www.chnosz.net/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_clinDataReview.html\">checks<\/a><br><a href=\"https://github.com/openanalytics/clinDataReview\">url<\/a><br><a href=\"https://github.com/openanalytics/clinDataReview/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_CohortCharacteristics.html\">checks<\/a><br><a href=\"https://darwin-eu.github.io/CohortCharacteristics/\">url<\/a><br><a href=\"https://github.com/darwin-eu/CohortCharacteristics/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_CohortSurvival.html\">checks<\/a><br><a href=\"https://darwin-eu-dev.github.io/CohortSurvival/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_cxhull.html\">checks<\/a><br><a href=\"https://github.com/stla/cxhull\">url<\/a><br><a href=\"https://github.com/stla/cxhull/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_D4TAlink.light.html\">checks<\/a><br><a href=\"https://bitbucket.org/SQ4/d4talink.light\">url<\/a><br><a href=\"https://bitbucket.org/SQ4/d4talink.light/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_DataCombine.html\">checks<\/a><br><a href=\"http://CRAN.R-project.org/package=DataCombine\">url<\/a><br><a href=\"https://github.com/christophergandrud/DataCombine/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_DataFakeR.html\">checks<\/a><br><a href=\"https://github.com/openpharma/DataFakeR\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_datawizard.html\">checks<\/a><br><a href=\"https://easystats.github.io/datawizard/\">url<\/a><br><a href=\"https://github.com/easystats/datawizard/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_DGP4LCF.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_diffEnrich.html\">checks<\/a><br><a href=\"https://github.com/SabaLab/diffEnrich\">url<\/a><br><a href=\"https://github.com/SabaLab/diffEnrich/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_disprofas.html\">checks<\/a><br><a href=\"https://github.com/piusdahinden/disprofas\">url<\/a><br><a href=\"https://github.com/piusdahinden/disprofas/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_dtmapi.html\">checks<\/a><br><a href=\"https://github.com/Displacement-Tracking-Matrix/dtmapi-R\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_easycsv.html\">checks<\/a><br><a href=\"https://github.com/bogind/easycsv\">url<\/a><br><a href=\"https://github.com/bogind/easycsv/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ecb.html\">checks<\/a><br><a href=\"https://sdw.ecb.europa.eu/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_EFAtools.html\">checks<\/a><br><a href=\"https://github.com/mdsteiner/EFAtools\">url<\/a><br><a href=\"https://github.com/mdsteiner/EFAtools/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_equatiomatic.html\">checks<\/a><br><a href=\"https://github.com/datalorax/equatiomatic\">url<\/a><br><a href=\"https://github.com/datalorax/equatiomatic/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_fcirt.html\">checks<\/a><br><a href=\"https://github.com/Naidantu/fcirt\">url<\/a><br><a href=\"https://github.com/Naidantu/fcirt/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_fields.html\">checks<\/a><br><a href=\"https://github.com/dnychka/fieldsRPackage\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_FLightR.html\">checks<\/a><br><a href=\"https://CRAN.R-project.org/package=FLightR\">url<\/a><br><a href=\"https://github.com/eldarrak/FLightR/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_float.html\">checks<\/a><br><a href=\"https://github.com/wrathematics/float\">url<\/a><br><a href=\"https://github.com/wrathematics/float/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_flocker.html\">checks<\/a><br><a href=\"https://github.com/jsocolar/flocker\">url<\/a><br><a href=\"https://github.com/jsocolar/flocker/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_flowTraceR.html\">checks<\/a><br><a href=\"https://github.com/OKdll/flowTraceR\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_fucom.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gimap.html\">checks<\/a><br><a href=\"https://github.com/FredHutch/gimap\">url<\/a><br><a href=\"https://github.com/FredHutch/gimap/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_greatR.html\">checks<\/a><br><a href=\"https://ruthkr.github.io/greatR/\">url<\/a><br><a href=\"https://github.com/ruthkr/greatR/issues/\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hacksaw.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hBayesDM.html\">checks<\/a><br><a href=\"https://github.com/CCS-Lab/hBayesDM\">url<\/a><br><a href=\"https://github.com/CCS-Lab/hBayesDM/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hddplot.html\">checks<\/a><br><a href=\"https://github.com/jhmaindonald/hddplot\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hilldiv.html\">checks<\/a><br><a href=\"https://github.com/anttonalberdi/hilldiv\">url<\/a><br><a href=\"https://github.com/anttonalberdi/hilldiv/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hmmr.html\">checks<\/a><br><a href=\"<https://depmix.github.io/hmmr/>\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_httr2.html\">checks<\/a><br><a href=\"https://httr2.r-lib.org\">url<\/a><br><a href=\"https://github.com/r-lib/httr2/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hutilscpp.html\">checks<\/a><br><a href=\"https://github.com/hughparsonage/hutilscpp\">url<\/a><br><a href=\"https://github.com/hughparsonage/hutilscpp/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ICS.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_imt.html\">checks<\/a><br><a href=\"https://github.com/google/imt\">url<\/a><br><a href=\"https://github.com/google/imt/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_isotracer.html\">checks<\/a><br><a href=\"https://gitlab.com/matthieu-bruneaux/isotracer\">url<\/a><br><a href=\"https://gitlab.com/matthieu-bruneaux/isotracer/-/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_kmeRs.html\">checks<\/a><br><a href=\"https://github.com/urniaz/kmeRs\">url<\/a><br><a href=\"https://github.com/urniaz/kmeRs/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_LogConcDEAD.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_manymodelr.html\">checks<\/a><br><a href=\"https://github.com/Nelson-Gon/manymodelr\">url<\/a><br><a href=\"https://github.com/Nelson-Gon/manymodelr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mapping.html\">checks<\/a><br><a href=\"https://mappinguniverse.github.io/mapping/index.html\">url<\/a><br><a href=\"https://github.com/mappinguniverse/mapping/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_MatchIt.html\">checks<\/a><br><a href=\"https://kosukeimai.github.io/MatchIt/\">url<\/a><br><a href=\"https://github.com/kosukeimai/MatchIt/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_maths.genealogy.html\">checks<\/a><br><a href=\"https://genealogy.louisaslett.com/\">url<\/a><br><a href=\"https://github.com/louisaslett/maths.genealogy/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mbest.html\">checks<\/a><br><a href=\"https://github.com/patperry/r-mbest\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mgm.html\">checks<\/a><br><a href=\"https://www.jstatsoft.org/article/view/v093i08\">url<\/a><br><a href=\"https://github.com/jmbh/mgm/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_MicroSEC.html\">checks<\/a><br><a href=\"https://github.com/MANO-B/MicroSEC/\">url<\/a><br><a href=\"https://github.com/MANO-B/MicroSEC/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_miWQS.html\">checks<\/a><br><a href=\"https://github.com/phargarten2/miWQS/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mixtools.html\">checks<\/a><br><a href=\"https://github.com/dsy109/mixtools\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mlr3pipelines.html\">checks<\/a><br><a href=\"https://mlr3pipelines.mlr-org.com\">url<\/a><br><a href=\"https://github.com/mlr-org/mlr3pipelines/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_modelbased.html\">checks<\/a><br><a href=\"https://easystats.github.io/modelbased/\">url<\/a><br><a href=\"https://github.com/easystats/modelbased/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mpwR.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_multipleDL.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ncdf4.helpers.html\">checks<\/a><br><a href=\"https://www.r-project.org\">url<\/a><br><a href=\"https://github.com/pacificclimate/ncdf4.helpers/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_nestedmodels.html\">checks<\/a><br><a href=\"https://github.com/ashbythorpe/nestedmodels\">url<\/a><br><a href=\"https://github.com/ashbythorpe/nestedmodels/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_noegletalR.html\">checks<\/a><br><a href=\"https://github.com/FrLars21/noegletalR\">url<\/a><br><a href=\"https://github.com/FrLars21/noegletalR/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_odbc.html\">checks<\/a><br><a href=\"https://odbc.r-dbi.org\">url<\/a><br><a href=\"https://github.com/r-dbi/odbc/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_officedown.html\">checks<\/a><br><a href=\"https://ardata-fr.github.io/officeverse/\">url<\/a><br><a href=\"https://github.com/davidgohel/officedown/discussions\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ohtadstats.html\">checks<\/a><br><a href=\"https://github.com/pfpetrowski/OhtaDStats\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_opendatatoronto.html\">checks<\/a><br><a href=\"https://sharlagelfand.github.io/opendatatoronto/\">url<\/a><br><a href=\"https://github.com/sharlagelfand/opendatatoronto/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_pcatsAPIclientR.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_PLNmodels.html\">checks<\/a><br><a href=\"https://pln-team.github.io/PLNmodels/\">url<\/a><br><a href=\"https://github.com/pln-team/PLNmodels/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_PubChemR.html\">checks<\/a><br><a href=\"https://github.com/selcukorkmaz/PubChemR\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_qdapRegex.html\">checks<\/a><br><a href=\"https://github.com/trinker/qdapRegex\">url<\/a><br><a href=\"https://github.com/trinker/qdapRegex/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_qMRI.html\">checks<\/a><br><a href=\"https://www.wias-berlin.de/research/ats/imaging/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_R2BayesX.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rbioacc.html\">checks<\/a><br><a href=\"https://gitlab.in2p3.fr/mosaic-software/rbioacc\">url<\/a><br><a href=\"https://gitlab.in2p3.fr/mosaic-software/rbioacc/-/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_Rcrawler.html\">checks<\/a><br><a href=\"https://github.com/salimk/Rcrawler/\">url<\/a><br><a href=\"https://github.com/salimk/Rcrawler/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_readxl.html\">checks<\/a><br><a href=\"https://readxl.tidyverse.org\">url<\/a><br><a href=\"https://github.com/tidyverse/readxl/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rempsyc.html\">checks<\/a><br><a href=\"https://rempsyc.remi-theriault.com\">url<\/a><br><a href=\"https://github.com/rempsyc/rempsyc/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_restez.html\">checks<\/a><br><a href=\"https://github.com/ropensci/restez\">url<\/a><br><a href=\"https://github.com/ropensci/restez/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_Rfast.html\">checks<\/a><br><a href=\"https://github.com/RfastOfficial/Rfast\">url<\/a><br><a href=\"https://github.com/RfastOfficial/Rfast/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rhub.html\">checks<\/a><br><a href=\"https://github.com/r-hub/rhub\">url<\/a><br><a href=\"https://github.com/r-hub/rhub/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_riskmetric.html\">checks<\/a><br><a href=\"https://pharmar.github.io/riskmetric/\">url<\/a><br><a href=\"https://github.com/pharmaR/riskmetric/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rmzqc.html\">checks<\/a><br><a href=\"https://github.com/MS-Quality-hub/rmzqc\">url<\/a><br><a href=\"https://github.com/MS-Quality-hub/rmzqc/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rock.html\">checks<\/a><br><a href=\"https://rock.opens.science\">url<\/a><br><a href=\"https://gitlab.com/r-packages/rock/-/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rstanarm.html\">checks<\/a><br><a href=\"https://mc-stan.org/rstanarm/\">url<\/a><br><a href=\"https://github.com/stan-dev/rstanarm/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_Rvcg.html\">checks<\/a><br><a href=\"https://github.com/zarquon42b/Rvcg\">url<\/a><br><a href=\"https://github.com/zarquon42b/Rvcg/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_sasr.html\">checks<\/a><br><a href=\"https://github.com/insightsengineering/sasr/\">url<\/a><br><a href=\"https://github.com/insightsengineering/sasr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_scDHA.html\">checks<\/a><br><a href=\"https://github.com/duct317/scDHA\">url<\/a><br><a href=\"https://github.com/duct317/scDHA/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ScottKnott.html\">checks<\/a><br><a href=\"https://github.com/ivanalaman/ScottKnott\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_scRNAstat.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_shiny.info.html\">checks<\/a><br><a href=\"https://github.com/Appsilon/shiny.info/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_skellam.html\">checks<\/a><br><a href=\"https//r-forge.r-project.org/projects/healthqueues/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_SPUTNIK.html\">checks<\/a><br><a href=\"https://github.com/paoloinglese/SPUTNIK\">url<\/a><br><a href=\"https://github.com/paoloinglese/SPUTNIK/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_STRMPS.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_synthpop.html\">checks<\/a><br><a href=\"<https://www.synthpop.org.uk/>\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_T2DFitTailor.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_TEAM.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tern.rbmi.html\">checks<\/a><br><a href=\"https://github.com/insightsengineering/tern.rbmi\">url<\/a><br><a href=\"https://github.com/insightsengineering/tern.rbmi/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tessellation.html\">checks<\/a><br><a href=\"https://github.com/stla/tessellation\">url<\/a><br><a href=\"https://github.com/stla/tessellation/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_TestGenerator.html\">checks<\/a><br><a href=\"https://github.com/darwin-eu/TestGenerator\">url<\/a><br><a href=\"https://github.com/darwin-eu/TestGenerator/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tidyformula.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tidyr.html\">checks<\/a><br><a href=\"https://tidyr.tidyverse.org\">url<\/a><br><a href=\"https://github.com/tidyverse/tidyr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tmbstan.html\">checks<\/a><br><a href=\"https://github.com/kaskr/tmbstan/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_TreatmentPatterns.html\">checks<\/a><br><a href=\"https://github.com/darwin-eu/TreatmentPatterns\">url<\/a><br><a href=\"https://github.com/darwin-eu/TreatmentPatterns/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_TT.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_TukeyC.html\">checks<\/a><br><a href=\"https://github.com/jcfaria/TukeyC\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_TukeyRegion.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tzdb.html\">checks<\/a><br><a href=\"https://tzdb.r-lib.org\">url<\/a><br><a href=\"https://github.com/r-lib/tzdb/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_unitizer.html\">checks<\/a><br><a href=\"https://github.com/brodieG/unitizer\">url<\/a><br><a href=\"https://github.com/brodieG/unitizer/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_wallace.html\">checks<\/a><br><a href=\"http://wallaceecomod.github.io/wallace/,\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_weatherOz.html\">checks<\/a><br><a href=\"https://github.com/ropensci/weatherOz/\">url<\/a><br><a href=\"https://github.com/ropensci/weatherOz/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_wxgenR.html\">checks<\/a><br><a href=\"NA\">url2<\/a>"],[null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#cbq\">resubmitted/pretest<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#httr2\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#Rfast\">resubmitted/UL<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#tern.rbmi\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#TreatmentPatterns\">resubmitted/UL<\/a><\/small>",null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#wallace\">resubmitted/archive<\/a><\/small>",null,null],[0,8,0,0,0,0,1,0,2,0,3,0,1,0,3,0,4,0,16,0,0,0,0,4,0,1,0,0,187,0,6,0,1,0,0,0,0,0,0,0,0,184,2,6,0,0,0,1,0,0,14,0,1,3,0,1,43,8,2,0,0,1,0,0,8,4,0,0,0,0,0,11,0,0,0,0,300,0,0,133,1,0,2,0,11,19,0,2,1,0,0,3,0,0,0,0,0,0,1,0,0,2254,2,0,0,0,0,5,0,0,0,0],["","AdIsMF, enmSdmX, longit, OpenStats, Petersen, sars, soiltestcorr, temperatureresponse","","","","","stabiliser","","ergMargins, netmediate","","accucor, ecipex, LipidMS","","PhenotypeR","","archeoViz, gyro, tessellation","","E4tools, gesttools, psData, rties","","bayestestR, correlation, easystats, effectsize, ggeffects, ggstatsplot, modelbased, parameters, performance, report, see, sjlabelled, sjmisc, sjPlot, sjstats, statsExpressions","","","","","gimmeTools, SemNetCleaner, SemNetDictionaries, uniset","","EFA.dimensions","","","ADPclust, ASMap, aSPU, astrochron, autoFRK, autoimage, aweSOM, BallMapper, BayesLCA, BayesNetBP, bayesPop, biClassify, bioRad, bipartite, biwavelet, bkmr, BMRMM, BoundaryStats, BRcal, Canopy, CCA, ccTensor, chillR, ciftiTools, CJIVE, cmR, cmsafops, cmsafvis, CNLTtsa, CoastlineFD, cocons, convoSPAT, cope, corr2D, CSHShydRology, dartR.sim, dcTensor, demu, DesignCTPB, DshapeR, earlywarnings, eDITH, EGRET, EMD, envi, ETAS, etasFLP, ETRep, ExpAnalysis3d, expectreg, fcfdr, fcr, fdasrvf, fence, FLightR, fnets, foreSIGHT, fpa, fsbrain, funData, gainML, gapfill, gateR, GeneralizedWendland, GeoAdjust, geoelectrics, GEOmap, GeoModels, georob, geospt, geosptdb, GeoThinneR, goeveg, GPFDA, GPvecchia, GRIDCOPULA, gwrr, hero, hglasso, hht, highriskzone, Hmsc, iCNV, infraFDTD.assist, JOPS, klovan, landsepi, LatticeKrig, lgcp, LMERConvenienceFunctions, mappoly, metapone, ModelMap, MOLHD, mousetrap, mrbsizeR, MRFA, MSIMST, MSmix, Mulcom, MultiObjMatch, mvLSW, NCutYX, nnTensor, nucim, oceanmap, OCNet, OptimaRegion, paramGUI, PEIP, pepStat, plsgenomics, popsom7, ppgm, prevR, pspatreg, qfa, qlcVisualize, RadioSonde, ramps, reslr, RFOC, rhcoclust, RinteractMAPK, rivnet, RnBeads, RNCEP, rNOMADS, robflreg, RPANDA, rsat, RsimMosaic, rworldmap, SafeVote, scBio, scDD, SeBR, Semblance, SensMap, siland, sincell, singleCellTK, smosr, soilphysics, spant, sparrpowR, sparseBC, SpatialExtremes, spatialLIBD, SpatialVS, SpatialVx, SpATS, spatsurv, spBayesSurv, SpeCond, SpectralMap, spm2, spmoran, spThin, stilt, StMoMo, stocc, stopp, stxplore, SUMMER, SurvCorr, SynchWave, telefit, TFARM, TideCurves, TIMP, topologyGSA, TrackReconstruction, treespace, TwoTimeScales, uci, Umatrix, vectorwavelet, verification, voluModel, Voss, vote, warpMix, wavScalogram, waywiser, WRTDStidal, wsyn","","GPoM, MatrixExtra, pbdMPI, recometrics, recosystem, rsparse","","mpwR","","","","","","","","","adobeanalyticsr, aebdata, airnow, AIscreenR, AnVILAz, AnVILBase, appsheet, arcgisgeocode, arcgislayers, arcgisplaces, arcgisutils, askgpt, atrrr, awdb, bbk, BFS, BigDataPE, BiocCheck, biomaRt, blackmarbler, brickster, BRVM, bskyr, bugphyzz, CatastRo, CDSE, chattr, cjar, climaemet, clinicalomicsdbR, comtradr, congress, connectcreds, conrad, consibiocloudclient, CopernicusMarine, cvdprevent, dataonderivatives, dataRetrieval, dawaR, depmap, diario, dpkg, dsmSearch, dtmapi, dtreg, easybio, ECOTOXr, ellmer, envstat, eurostat, extractox, f1dataR, faers, fanyi, feltr, fenr, fitzRoy, FlickrAPI, fmpapi, foghorn, g3viz, galah, gdldata, gemini.R, genesysr, geocodebr, GEOquery, gh, GHCNr, GIFT, GitAI, GitStats, govinfoR, gptstudio, gptzeror, gtexr, gypsum, happign, hdar, healthatlas, healthyR.data, HoloFoodR, httptest2, HuBMAPR, hyd1d, hydflood, I14Y, khisr, kisopenapi, kiwisR, KoboconnectR, lehdr, link, LLMR, locateip, lulab.utils, mapme.biodiversity, maths.genealogy, mergenstudio, mgi.report.reader, microinverterdata, MolgenisArmadillo, mskcc.oncotree, N2H4, ncaavolleyballr, nixtlar, ntdr, ntfy, octopusR, ollamar, openaistream, openeo, openFDA, osmapiR, osmdata, owidapi, paws.common, peermodels, photon, pkgdown, planscorer, plug, promptr, proverbs, PurpleAir, pysparklyr, QBMS, qbr, racademyocean, Rapi, raqs, RAQSAPI, rchroma, Rcpi, readMDTable, remmy, resmush, restatis, riem, Rigma, ringostat, rirods, RKorAPClient, rlinkedinads, RMassBank, rnpn, rollama, rols, ropenmeteo, rosv, rqti, rtgstat, rtweet, selenium, sg, shiny.telemetry, shinylive, simfinapi, sivirep, solaR2, sotkanet, spanishoddata, stortingscrape, susographql, tapLock, tidyhydat, tidyllm, tidyprompt, tidyquant, tidywikidatar, tRdbImport, treasury, trud, ucimlrepo, umbridge, vcr, wbids, wbwdi, webdav, whatthreewords, wikkitidy, worldbank, zendown","grattan, healthyAddress","ICSClust, ICSNP, ICSOutlier, ICSShiny, ICtest, MNM","","","","detectR","","","causalBatch, CBPS, cem, EpiForsk, jsmodule, MatchThem, netdiffuseR, PSAboot, psBayesborrow, psborrow, RCTrep, SAMprior, sitepickR, vaccineff","","CytoGLMM","bootnet, mDAG, MultiGroupO","","marlod","AFM, AUCell, beastt, biodosetools, CBEA, CEDA, CiteFuse, dafishr, ddpcr, DSWE, EntropyMCMC, enviGCMS, extrememix, gemini, genomicInstability, ggrasp, IMIX, INFOSET, kernplus, l1kdeconv, mcrPioda, MiRKAT, MisRepARMA, MixSemiRob, Motif2Site, opGMMassessment, plotmm, pRoloc, QuantileGH, rnaCrosslinkOO, RprobitB, rstanbdp, RTN, scapGNN, scClassify, scone, scPloidy, serocalculator, smartid, UpDown, viper, ZetaSuite, ztpln","mcboost, mlr3fairness, mlr3fda, mlr3shiny, mlr3torch, mlr3verse, sense, spFSR","easystats, see","","","eurocordexr","","","datrProfile, epidm, MCOE, pgTools, SangerTools, shiny.telemetry, sqlHelpers, virtuoso","D4TAlink.light, ratlas, TestAnaAPP, unhcrdown","","","","","","BAwiR, edgar, ggfootball, qdap, rdhs, sgat, textclean, UniprotR, vegtable, WhatsR, worldfootballR","","","","","ABSurvTDC, accucor, aelab, AgroTech, AHPWR, aire.zmvm, allMT, AlpsNMR, amanida, amerifluxr, autonomics, AvInertia, aweSOM, baizer, batchLLM, BayesianFitForecast, baytrends, bcdata, bestSDP, BGVAR, bibliometrix, bndesr, bootf2, BrazilMet, breathtestcore, bulkreadr, capl, carbonr, CCWeights, ceas, cepumd, cgmanalysis, ChestVolume, chillR, chromConverter, clevRvis, CMapViz, CoastlineFD, COINr, concatipede, covid19.analytics, COVIDIBGE, covidmx, crosswalkr, cTRAP, dafishr, dams, DataLoader, datamods, datazoom.amazonia, dbGaPCheckup, DCEtool, DDIwR, decompTumor2Sig, defineR, dendRoAnalyst, DescTools, dextergui, DIFshiny, DistatisR, DMRcatedata, dmtools, Doscheda, dosedesignR, drawsample, duawranglr, easyr, ECOTOXr, EdSurvey, eHDPrep, eirm, epe4md, eph, epitweetr, esaps, excel2eprime, exceldata, expDB, extractox, fabR, FAST.R, FastRet, fetch, fio, fitPS, flattabler, forgts, formods, FSK2R, gDRimport, GeDi, GEOexplorer, GeomxTools, GeoWeightedModel, GetTDData, GGIRread, ggplotgui, GiniDistance, goat, GOLDprice, googlePublicData, greenfeedr, Greymodels, growthPheno, GWSDAT, HaDeX, hclusteasy, hiReadsProcessor, Holomics, hosm, htrSPRanalysis, htsr, hydrotoolbox, hydroToolkit, icertool, ideanet, IDEATools, IDSL.CSA, IDSL.IPA, IDSL.NPA, IDSL.UFA, IDSL.UFAx, ie2misc, ifo, ifpd, IGoRRR, ihpdr, imageData, immunarch, ImportExport, InflectSSP, insane, INSPEcT, INSPECTumours, interca, InterCellar, InterVA5, iotables, irtawsi, irtGUI, ISAnalytics, isobxr, IsoCorrectoR, IsoCorrectoRGUI, ISRaD, istat, joinXL, jsmodule, jubilee, knfi, KoboconnectR, lavaangui, LearningStats, lehuynh, libr, loadeR, logib, ltertools, maat, macrocol, MACSQuantifyR, MassWateR, matriz, mbQTL, mbX, mcmsupply, MDMAPR, metacore, metaumbrella, MFO, microbiomeMQC, micromapST, miniMeta, mintyr, mipplot, miRetrieve, mlms, modgetxl, moodef, MSstatsShiny, netShiny, novelqualcodes, ntdr, oglcnac, OlinkAnalyze, OmnipathR, oncmap, One4All, onmaRg, opendatatoronto, OpenMindat, pacta.multi.loanbook, PAMhm, papci, PDcIBGE, pdi, Pedixplorer, pedquant, PerRegMod, PesticideLoadIndicator, PhageCocktail, PhenoSpectra, photobiologyInOut, pioneeR, plotGrouper, PNDSIBGE, PNSIBGE, PODES, Poly4AT, pressuRe, prior3D, PUPMCR, puzzle, PvSTATEM, pxmake, qrlabelr, QTL.gCIMapping, qualitycontrol, quicR, QuICSeedR, QurvE, radiant.data, ramlegacy, rb3, rbiom, RBNZ, RcmdrMisc, Rdiagnosislist, RDML, readabs, readit, readmoRe, readrba, readtext, reappraised, regions, Relectoral, replicateBE, rexer, rio, RJafroc, RLumShiny, rodeo, rsurveycto, Rtrack, SAKERS, SangerTools, scan, scbursts, scdhlm, scicomptools, SDAR, sdtmval, SEAHORS, seahtrue, secuTrialR, SemNetCleaner, sendigR, shinyHeatmaply, shinyIRT, ShinyLink, sivirep, SoilManageR, SomaDataIO, SpatialOmicsOverlay, sphereML, spiro, SQI, SqueakR, SUSES, tab2xml, tall, TestGenerator, TFutils, threesixtygiving, tidyDenovix, tidygapminder, tidyplate, tidyquant, tidytlg, tidyverse, toxEval, tpfp, TR8, Trading, treasury, tRigon, TSAR, ubiquity, unheadr, upstartr, uscoauditlog, vcdExtra, VectorCodeR, visvaR, visvow, vvmover, weed, whippr, ypssc","","","adass, ALDEx2, autoMFA, bivpois, bltm, CatsCradle, cauchypca, CCd, CICA, clusterMI, cols, Compositional, CompositionalML, CompositionalRF, consICA, consrq, corrfuns, COTAN, CPC, cpmr, cpss, CptNonPar, Cronbach, datanugget, dcortools, dcorVS, dCovTS, deepMOU, DER, Directional, DrDimont, dyads, epilogi, fastFMM, fitlandr, frscore, fsn, funcharts, glmtoolbox, gMOIP, gomp, gp, GPCERF, handwriter, highd2means, imbalanceDatRel, IMIFA, iNETgrate, irtQ, jenga, kDGLM, LACE, lmap, manydist, MAPFX, mbr, metaGE, metaplus, methylscaper, MetSizeR, mfpp, micd, mixgb, mixpoissonreg, MLE, mlmts, MN, mrfse, mvcauchy, mvcor, mvhtests, naive, nda, nebula, nestedcv, NetFACS, NetSci, NGBVS, nlgm, NNS, obfuscatoR, octad, OPI, optimalFlow, pchc, pdcor, phylotypr, phylter, ppsbm, PR, PRSPGx, psichomics, QUALYPSO, RAINBOWR, randomLCA, RankAggSIgFUR, rcccd, RDM, regda, Rfast2, robqda, Rogue, Rtropical, rvMF, scoredec, segen, SelectBoost, sensobol, shinyExprPortal, simDAG, skedastic, smartsnp, SmoothWin, SMOTEWB, spaceNet, sparseCov, speaq, Spectrum, SteppedPower, success, tame, tetragon, TML, truh, TSCI, twl, univariateML, VERSO, volcano3D, VoxR, WRI, wTO, zenith","packager","","MsQuality, PTXQC","","AuxSurvey, BayesERtools, BayesPostEst, bayesrules, evidence, fbst, IRexamples, jmBIG, SIMPLE.REGRESSION, tidyposterior, webSDM","Arothron, ashapesampler, concom, cxhull, doolkit, espadon, ETRep, gyro, habtools, jacobi, JFM, JSDNE, molaR, Morpho, morphomap, pressuRe, RCDT, tessellation, Toothnroll","","scCAN, ScRIMM","spANOVA","","","MLE, PINstimation, powerHE","","","","","","","polyhedralCubature","","","abjutils, abstr, academictwitteR, acledR, ActFrag, activAnalyzer, activPAL, actogrammr, actxps, adabag, adas.utils, adepro, adheRenceRX, admiral, admiraldev, admiralophtha, admiralvaccine, adobeanalyticsr, adproclus, adventr, aeddo, AEenrich, aelab, AeRobiology, AeroSampleR, afdx, AgePopDenom, agricolaeplotr, agriutilities, agua, agvgd, AHM, ahpsurvey, AHPWR, AICcPermanova, aihuman, aire.zmvm, airship, airt, AIscreenR, alakazam, ale, alfred, alien, AlignLV, allestimates, allofus, alookr, alphavantager, AlpsNMR, amadeus, amanida, amp, amt, anabel, andurinha, animalcules, AnimalSequences, animbook, anipaths, annotater, anomalize, anomaly, AntibodyForests, AnVIL, AnVILGCP, AnVILWorkflow, anyflights, aoristic, APAlyzer, APCtools, aplotExtra, apmx, applicable, AQEval, AquaticLifeHistory, archiveRetriever, arcpullr, ARDECO, arealDB, arena2r, ARGOS, ARPALData, artMS, arulesViz, ARUtools, ASML, ASpli, aspline, auctestr, auk, autoFC, autoGO, autoharp, autoimport, autoMrP, autonomics, autoReg, AutoScore, autoslider.core, autostats, autoTS, avidaR, AvInertia, AzureKusto, baclava, baguette, BaHZING, baizer, bakR, baldur, bambooHR, bambu, bandle, banffIT, banter, barcodetrackR, bartMan, baseballr, basecallQC, basket, BasketballAnalyzeR, BatchGetSymbols, batchmix, BatchQC, batchtma, BAwiR, bayesammi, bayesEO, BayesERtools, BayesianLaterality, BayesianReasoning, BayesMortalityPlus, bayesmove, BayesMultiMode, bayesnec, BayesPostEst, BayesPPDSurv, BayesSpace, baystability, bdl, bdlim, bdsm, beadplexr, beastt, beaver, BeeGUTS, beezdiscounting, bennu, bestSDP, betaHMM, bettr, BgeeDB, BGPhazard, bibliometrix, biclust, biclustermd, BifactorIndicesCalculator, bigergm, bigtime, BindingSiteFinder, biobroom, bioCancer, BiocFHIR, BiocSet, biodosetools, biogrowth, bioRad, bioregion, biostats101, BioTIMEr, BioVizSeq, bipartiteD3, birdscanR, bistablehistory, BiVariAn, bkmr, blackmarbler, BMisc, bmm, bnmonitor, BNPdensity, BOBaFIT, boiwsa, BOJ, bonsaiforest, bootnet, bp, bpmnR, BrailleR, BrazilCrime, breathtestcore, breathteststan, brendaDb, bridger, brisk, brms.mmrm, brokenstick, brolgar, broom, broom.helpers, broom.mixed, bruceR, BRVM, bSi, bskyr, bssm, bsynth, bugphyzz, bulkAnalyseR, bulkreadr, bullseye, bumbl, bunching, bupaR, burnr, BUSpaRse, BVARverse, bvhar, bwsTools, c2z, cacc, caffsim, calibmsm, calibrationband, campsis, CaMutQC, canceR, cancerscreening, cansim, capm, carbonr, card, cards, cardx, care4cmodel, CARNIVAL, carpenter, CaseBasedReasoning, casino, cassandRa, catlearn, catmaply, cats, causact, CB2, CBEA, cBioPortalData, cbioportalR, CBNplot, cccrm, ccdR, CCPlotR, ccpsyc, CDMConnector, cdom, cdrcR, cellbaseR, CellBench, CellMixS, censcyt, censored, cepumd, ceRnetsim, ceRtainty, Certara.ModelResults, cesR, cfda, cft, CGPfunctions, ChangePointTaylor, charlesschwabapi, cheese, chemodiv, chessboard, ChestVolume, childsds, chillR, ChIPpeakAnno, chromConverter, ChromSCape, chronochrt, cia, cicero, CIMICE, cimir, CIMTx, CINE, CircSeqAlignTk, cisp, cities, cjbart, CKMRpop, ClassifyR, Cleanet, clidatajp, climaemet, clinDR, clinicalsignificance, clockify, cloudos, CLSIEP15, clustermole, clusterProfiler, clustifyr, ClustImpute, cmcR, CMHSU, cmstatr, CNVScope, CoastlineFD, cocktailApp, COCOA, codebook, codebookr, CodelistGenerator, codified, cogena, cogmapr, cohortBuilder, CohortCharacteristics, CohortConstructor, CohortPlat, cohorts, CohortSurvival, CohortSymmetry, ColOpenData, colorist, comapr, ComBatFamQC, combiroc, COMBO, COMIX, compareDF, COMPASS, comperes, CompositeReliability, compositeReliabilityInNestedDesigns, ComPrAn, concordance, ConfidenceEllipse, confintROB, ConfusionTableR, congress, CoNI, conquestr, ConsRank, conText, contribution, convdistr, convergEU, CopernicusMarine, CopSens, corporaexplorer, correlationfunnel, corrmeta, CoSIA, COTAN, Counternull, countries, countsimQC, coveR2, covid19br, covid19france, covid19tunisia, covid19us, covidcast, covidmx, CoxAIPW, coxed, Coxmos, cpss, CRABS, cricketdata, crimeutils, CRMetrics, cropDemand, cropZoning, crosshap, crossnma, crosstable, crosstalkr, crossurr, crt2power, CRTspat, crypto2, csodata, csquares, CTexploreR, cthist, ctxR, cubble, cuperdec, curatedMetagenomicData, currr, cutpointr, cvap, cvasi, cvdprevent, cvms, CytoGLMM, cytominer, czechrates, d3r, dabestr, dados, dafishr, damAOI, dampack, Damsel, daqapo, dar, dartR.base, dartR.spatial, data.validator, datadictionary, datadogr, dataframeexplorer, DataMetProcess, dataSDA, dataspice, datazoom.amazonia, datos, daymetr, DBarcodeCompatibility, dbGaPCheckup, dbglm, dbplyr, DCEtool, DCLEAR, dcm2, DCPO, deaR, decisionSupport, deconvR, decoupleR, DEGreport, demodelr, DemoKin, dendRoAnalyst, dendroNetwork, denguedatahub, DEP, depigner, deprivateR, describedata, descriptr, descstat, desctable, deseats, designit, destiny, dexter, dextergui, dexterMST, dfoliatR, dgpsi, DHSr, DiagrammeR, diathor, dibble, diceplot, diceR, did, didimputation, diffcyt, diffeR, digiRhythm, dilp, DImodelsVis, dinoR, disbayes, DiscreteGapStatistic, diseasystore, DisImpact, distillML, disto, distrr, div, divent, dlmtree, dlookr, dm, dmai, DMCfun, dmtools, dndR, do, doBy, dotsViolin, doudpackage, dowser, dpasurv, dplyrAssist, drawProteins, drcte, DrDimont, dreamer, dreamlet, drifter, drpop, DrugExposureDiagnostics, drugprepr, drugsens, DrugSim2DR, DrugUtilisation, dscore, DSSAT, dst, dtrackr, DTSEA, duet, dumbbell, Dune, DuoClustering2018, DuplexDiscovereR, dycdtools, dynamic, dynamicSDM, dyngen, dynparam, dynwrap, easier, easyalluvial, easycensus, easysurv, eatRep, EBASE, ec50estimator, ecan, ech, echoice2, echor, echos, ecocomDP, ECOTOXr, EDCimport, edeaR, eDjoint, EEAaq, EFAtools, EFDR, egor, eHDPrep, eiExpand, EIX, elaborator, ELMER, emayili, embed, emery, EmiR, emuR, ENMeval, EnrichDO, EntropicStatistics, epe4md, eph, epiCleanr, epicmodel, epidata, epifitter, EpiForsk, epikit, EpiMix, EpiReport, epitrix, epocakir, epubr, eq5dsuite, ErlangC, ern, ERSA, esaps, escalation, esem, esmtools, estadistica, ESTER, esvis, eudract, eurlex, euroleaguer, europepmc, eurostat, evinf, evprof, evsim, excel2eprime, ExcelFunctionsR, excessmort, excluder, ExPanDaR, expDB, expertsurv, explainer, explor, ExploreModelMatrix, extraChIPs, extRatum, exuber, eye, eyelinkReader, eyeRead, eyetrackingR, ezEDA, ezmmek, f1dataR, fable, fableCount, fabletools, fabR, factoextra, factR, faersquarterlydata, FAIRmaterials, FairMclus, fakir, famat, FAMetA, farr, fasstr, FAST.R, fastml, fastqcr, fastrep, fastRG, fastRhockey, FAVA, fbar, fcaR, fdapaceShiny, feasts, FedData, feltr, fenr, ferrn, ffp, ffscrapr, fftab, fgeo.analyze, fido, figma, file2meco, filebin, finalfit, FindIT2, findSVI, finetune, finnsurveytext, finnts, fishualize, fitbitr, fitlandr, fitzRoy, fixerapi, FLAMES, flametree, flashier, flashlight, flexFitR, flexlsx, flexsurv, flipbookr, flowchart, flowmapper, flowml, FlowSOM, flowTraceR, fluxible, FMAT, fmpcloudr, fmriqa, fobitools, foodingraph, footBayes, forecastML, forensIT, forestecology, ForestElementsR, forestinventory, forestmangr, forestr, forgts, ForIT, FORTLS, FourWayHMM, fpp3, fqar, frenchdata, frostr, fsemipar, FSK2R, fst4pg, ftExtra, fude, funcharts, functiondepends, funkycells, funkyheatmap, funneljoin, funspotr, funStatTest, fusen, fuzzyjoin, FuzzyPovertyR, GA4GHshiny, GABB, galah, galigor, galvanizer, gapclosing, gasmodel, GauPro, GBScleanR, gcplyr, GDPuc, gdverse, gemma.R, gen5helper, geneHapR, genekitr, geneplast.data, GeneSelectR, geneticae, GeneTonic, geneviewer, genieBPC, genogeographer, GenomicDataCommons, GenomicPlot, GenoTriplo, geoAr, geocausal, geogenr, geomander, geomultistar, GEOquery, geotoolsR, GeoTox, GerminaR, gesttools, GetDFPData2, GetTDData, gfer, gg4way, ggalluvial, GGally, ggarchery, ggautomap, ggblanket, ggbrain, ggbump, ggcleveland, ggedit, ggESDA, ggfacto, ggFishPlots, ggflowchart, ggfortify, gggenomes, ggheatmap, ggHoriPlot, ggip, ggiraphExtra, gglm, ggmanh, ggmap, ggmcmc, ggmice, ggmosaic, ggmsa, ggmugs, ggmulti, GGoutlieR, ggpcp, ggplate, ggPMX, ggpubr, ggQC, ggquickeda, ggRandomForests, ggreveal, ggsankeyfier, ggsc, ggsector, ggseg, ggseg3d, ggseqplot, ggsolvencyii, ggsom, ggspatial, ggstackplot, ggstats, ggstatsplot, ggstream, ggsurveillance, ggsurvfit, ggtaxplot, ggtibble, ggtree, ggtreeSpace, GHCNr, GIFT, GillespieSSA2, gimap, GimmeMyPlot, GiniDecompLY, gitdown, gitear, gitlabr, GladiaTOX, globaltrends, gluvarpro, gmgm, gMOIP, goat, godley, gompertztrunc, Goodreader, googleAnalyticsR, gosset, govinfoR, gpbStat, gprofiler2, gptzeror, grafify, GRaNIE, granovaGG, granulator, graph4lg, graphTweets, gratia, gravitas, gravity, greeks, greenfeedr, Greg, gregRy, grizbayr, groupdata2, groupr, grouprar, growthcleanr, GrowthCurveME, GRShiny, grwat, gscramble, gsDesign, gsDesign2, gson, gtexr, gtexture, GTFSwizard, gtreg, gtsummary, guideR, GUIDEseq, gwaRs, gwasrapidd, gwavr, GWE, GWlasso, h3jsr, hacksig, HaDeX, halk, handwriter, handwriterApp, handwriterRF, harrietr, hbamr, hca, hdImpute, hdpGLM, HDStIM, HDXBoxeR, hdxmsqc, headliner, healthdb, healthyR.ai, healthyR.data, healthyR.ts, heiscore, hemispheR, hermes, HEssR, hettx, heuristicsmineR, hhmR, HicAggR, HiCDCPlus, HiContacts, hidecan, highcharter, highlightr, HiLDA, himach, historicalborrow, historicalborrowlong, hlaR, hlt, HMDHFDplus, hockeystick, Holomics, hoopR, hot.deck, HPiP, HRTnomaly, htrSPRanalysis, hubEnsembles, HuBMAPR, huito, HuraultMisc, hurricaneexposure, HVT, hydroloom, HYPEtools, i2extras, iadf, IBCF.MTME, IBRtools, ICD10gm, iCNV, IDE, ideamdb, ideanet, idiffomix, idiogramFISH, idmc, ifo, IgGeneUsage, iglu, IGoRRR, ihpdr, immunarch, immunotation, IMP, implicitMeasures, imprinting, imputeREE, imputeTestbench, imt, ImVol, incidence2, IncidencePrevalence, industRial, iNETgrate, iNEXT.beta3D, infer, infercnv, infiltrodiscR, InflectSSP, influxdbr, InjurySeverityScore, injurytools, insane, inspectdf, INSPECTumours, InsuSensCalc, INTACT, intensegRid, InteractionPoweR, InterCellar, interpretCI, inti, inventorize, invivoPKfit, iNZightTools, iNZightTS, IONiseR, iotables, IPEDS, IPEDSuploadables, ipfr, ipolygrowth, iPRISM, ir, iraceplot, irtQ, irtrees, ISAnalytics, Isinglandr, isobxr, IsoformSwitchAnalyzeR, isomiRs, isoorbi, isotracer, ISRaD, itdr, itol.toolkit, itraxR, itscalledsoccer, ivo.table, jackstrap, janitor, japanstat, jarbes, jmastats, JohnsonKinaseData, jpgrid, jpstat, jrt, jstor, kaigiroku, kappaGold, karel, karyotapR, katdetectr, kayadata, keyATM, khisr, kibior, KMunicate, knfi, kntnr, kollaR, konfound, KrakenR, labelled, LACE, lactater, LAD, LadderFuelsR, LAGOSNE, lans2r, lares, layer, LBSPR, lcsm, LDABiplots, LDAShiny, lddmm, LeadSense, ledger, lefser, LegATo, lessSEM, LexisPlotR, lfproQC, LifeInsureR, lifelogr, LifemapR, Lifertable, LightLogR, lillies, limpca, lineartestr, linkspotter, lipidomeR, LipidomicsR, lipidr, lisaClust, litteR, lmDiallel, lmeresampler, lnmixsurv, localsp, loewesadditivity, logrx, LongDat, longitudinalcascade, longke, longreadvqs, longsurr, lookout, LorMe, lsirm12pl, LTASR, lterpalettefinder, ltertools, LTFHPlus, macrosyntR, MadanText, MadanTextNetwork, madshapR, MagmaClustR, maicChecks, MAICtools, MainExistingDatasets, malariaAtlas, maldipickr, MantaID, manydata, manydist, mapbayr, mapboxapi, mapindiatools, mapme.biodiversity, mapmixture, maraca, margaret, MarketMatching, marketr, markovmix, markowitz, mase, MassWateR, mastR, matman, MatrixHMM, MatrixQCvis, matrixset, matsindf, matuR, maxaltall, MBECS, mbQTL, mbRes, mbX, mcmsupply, mcp, mcradds, mcStats, mde, measr, medfateland, meerva, meltt, mem, memapp, memes, memoria, mergingTools, MesKit, metabCombiner, metabolic, MetabolicSurv, MetabolSSMF, metaconfoundr, metacore, MetaCyto, metaGE, MetaHD, metajam, MetAlyzer, metamicrobiomeR, metan, MetaNeighbor, metaplot, metapower, MetaScope, metasnf, metatools, MetaUtility, meteoland, meteospain, methodical, MethReg, methylclock, MetNet, metools, metrica, MetricGraph, metricminer, MF.beta4, MFO, mgwrhw, MHQoL, mia, miaViz, MIC, mice, miceafter, microbial, microbiome, microbiomeExplorer, microbiomeMarker, MicrobiomeSurv, MicrobiotaProcess, microhaplot, microinverterdata, micsr, migest, migrate, MigrationDetectR, mildsvm, miloR, MIMER, mipplot, miRetrieve, mispitools, missCompare, mistyR, mitre, miWQS, mixOmics, mixtur, MixviR, MLFS, mlstrOpalr, mMARCH.AC, Mmcsd, MMUPHin, mnreadR, mobileR, mobr, MOCHA, modACDC, modelr, modeltime, modeltime.ensemble, modeltime.resample, moderndive, MOFA2, mojson, MolgenisArmadillo, MOMA, monoClust, moodef, Moonlight2R, moranajp, morse, MortalityLaws, mosaic, mosaicCore, mosaicModel, MouseFM, mousetrap, movegroup, movementsync, mpathr, mpathsenser, MPI, mpindex, mplot, MplusLGM, mpoly, mpower, MPTmultiverse, mpwR, mrf2d, mrgsim.sa, MSA2dist, mshap, msImpute, MSiP, mskcc.oncotree, MSPrep, MsQuality, msSPChelpR, MSstatsLiP, MSstatsShiny, mtdesign, mtvc, mulgar, MultiAssayExperiment, multicrispr, multilevelmediation, multilevLCA, multimedia, multimorbidity, multinma, multistateQTL, multitool, multiverse, MuPETFlow, musicatk, MutationalPatterns, mvMAPIT, mxfda, mxnorm, myClim, naniar, NanoMethViz, nanostringr, naryn, NBAloveR, nbTransmission, ncaavolleyballr, ncappc, ncmeta, ndexr, ndi, nemtr, neo4r, neonPlantEcology, neonSoilFlux, neonUtilities, neotoma2, nestedmodels, NetFACS, nettskjemar, NetworkChange, NetworkExtinction, netZooR, NeuralNetTools, NeuroDecodeR, neuromplex, nevada, newsanchor, nfl4th, nflfastR, nflseedR, ngramr, ngsReports, nhdplusTools, nichetools, NiLeDAM, NIMAA, nixtlar, nlpsem, nlrx, nls.multstart, nmarank, nmrrr, NMTox, nncc, noegletalR, nonmemica, NoRCE, normfluodbf, NPARC, npcs, npi, nplyr, nrba, ntdr, nuggets, nullabor, numbat, NVAR, nzffdr, nzilbb.vowels, obcost, obliqueRSF, occCite, ocf, oddnet, oddsapiR, oddstream, odeGUTS, ofpetrial, ogrdbstats, OGRE, OhdsiReportGenerator, ojsr, OlinkAnalyze, OLStrajr, OmicsMLRepoR, OMICsPCA, omicwas, OmnipathR, omopgenerics, OmopSketch, OmopViewer, omu, omXplore, OnboardClient, OncoBayes2, oncomsm, oncrawlR, OneArmTTE, ontologics, ontophylo, OOS, openair, openairmaps, openappr, openbankeR, opencage, OpenLand, openmeteo, openMSE, OpeNoise, OpenRepGrid, OpenRepGrid.ic, opitools, OPL, optic, optimLanduse, OralOpioids, ordbetareg, orderanalyzer, ordinalsimr, ordr, ORFID, orthos, otargen, ouladFormat, OutliersO3, outqrf, outreg, overviewR, owmr, oysteR, packagefinder, pacta.multi.loanbook, PACVr, palaeoSig, pammtools, PAMpal, PAMscapes, panelr, parafac4microbiome, parcats, parsermd, parsnip, partition, partR2, passt, pathlinkR, PathoStat, paths, pathviewr, PatientLevelPrediction, PatientProfiles, patterncausality, pcaExplorer, pcutils, pdfetch, pdi, peacesciencer, Pedixplorer, pedtricks, pepe, performanceEstimation, PERK, pestr, Petersen, phantasus, phantSEM, PHEindicatormethods, phenomis, phenopath, PhenoSpectra, Phenotype, PheVis, philr, phoenics, PhosR, photobiology, photobiologyInOut, photosynthesisLRC, phyr, phytoclass, PieGlyph, pivotea, piwikproR, PKbioanalysis, PKNCA, planr, planscorer, Platypus, play, PlayerChart, PLNmodels, plotBart, plotdap, plotGrouper, plotly, plotthis, plsmod, plume, plutor, plyxp, pmartR, pmc, pmetar, pmxcode, podcleaner, pointblank, poissonreg, PolarCAP, poldis, polished, pollster, polymatching, POMA, POMADE, pomcheckr, pooledpeaks, popstudy, PopulateR, portalr, portvine, postGGIR, powdR, powerjoin, ppcseq, ppcSpatial, ppendemic, PPforest, ppitables, ppmf, ppseq, PracticalEquiDesign, prcr, precisely, precmed, predict3d, predictNMB, preference, presenter, prettyglm, priceR, primer, primirTSS, PrInCE, prioriactions, priorityelasticnet, prisonbrief, probably, processanimateR, processcheckR, processmapR, processpredictR, profileplyr, progeny, PRONE, prophet, ProtE, ProteinGymR, ProteoBayes, ProteoDisco, protGear, protti, proustr, psfmi, psHarmonize, psidread, psm3mkv, psmineR, psycCleaning, psychmeta, psychonetrics, psycModel, psyntur, ptetools, PubChemR, public.ctn0094extra, PublicWorksFinanceIT, puls, PUMP, PupillometryR, PupilPre, pvda, pwr2ppl, pxmake, pysparklyr, qacBase, qbr, qdap, QFeatures, qPCRtools, qPLEXanalyzer, qpNCA, qPRAentry, qsub, qtl2ggplot, qtl2pattern, QTLExperiment, quadVAR, qualtRics, quantiseqr, queuecomputer, quicR, QuICSeedR, quid, quincunx, QurvE, R.AlphA.Home, r2dii.analysis, r2dii.match, R6P, r6qualitytools, racademyocean, ractivecampaign, rADA, RadialVisGadgets, radiant.basics, radiant.data, radiant.model, rainette, ralger, random.cdisc.data, randomForestVIP, RandomWalker, rappsflyer, rar, RareComb, RARfreq, ratlas, rATTAINS, rattle, ravelRy, RavenR, rawKS, rb3, rbbnp, rbedrock, rcausim, RClimacell, rcontroll, RCPA, RCT, RCTrep, RCTS, Rcurvep, rdflib, rdracor, rdss, ReactomeGSA, readabs, readapra, readrba, reappraised, recalibratiNN, receptLoss, recipes, recodeflow, REDCapCAST, REDCapDM, REDCapR, REDCapTidieR, REDI, refund.shiny, RegDDM, regions, registr, regressinator, regrrr, reliabilitydiag, reproducer, reslr, resourcecode, ReSurv, retimer, retroharmonize, reveneraR, RevGadgets, ReviewR, revulyticsR, rexer, rfacebookstat, rfars, rfieldclimate, RFishBC, rfPermute, RFPM, rFSA, Rgff, rgho, rgoogleads, RHRV, riAFTBART, ribor, RIdeogram, Riex, Rigma, ripc, riskCommunicator, risks, RItools, riverconn, RivRetrieve, rjaf, rKolada, RKorAPClient, rlinkedinads, rmapzen, RmarineHeatWaves, rmdl, rmlnomogram, Rmonize, rmweather, rmytarget, RnaSeqSampleSize, RNeXML, rnpn, roadoi, robCompositions, RobinCar, robis, robomit, robotoolbox, robvis, Robyn, rockx, roclang, rolap, rollup, romic, ropenmeteo, ropercenter, rPACI, rpcss, rplanes, rplec, rprev, RQdeltaCT, Rrepest, rsample, rscopus, rscorecard, RseqQC, Rsmlx, rsnell, rsofun, rsppfp, RSSL, rstanemax, rstatix, RStoolbox, rStrava, rTCRBCRr, rtgstat, rticulate, RTL, Rtpca, rtpcr, rtrek, rtrends, rtry, rtypeform, rubias, ruler, rules, RulesTools, ruminate, rusk, RVA, rvkstat, rwa, rwalkr, rWCVP, RWDataPlyr, rWikiPathways, ryandexdirect, rytstat, sabre, saccadr, saens, safetyGraphics, sampbias, sampcompR, sampler, saotd, sapfluxnetr, saqgetr, saros, saros.base, SBMTrees, sbo, scanstatistics, scatterbar, scatterpie, sccomp, scCustomize, SCDB, scDotPlot, scFeatures, SCFMonitor, scGOclust, SchoolDataIT, schtools, scicomptools, scimo, scipub, scoper, SCORPIUS, scpi, scPipe, scPloidy, scpoisson, scR, ScreenR, scrutiny, scSpatialSIM, SCtools, SDGdetector, SDModels, sdtm.oak, sdtmval, seahtrue, secrettext, sectorgap, secuTrialR, sedproxy, SeedMaker, seeds, SEERaBomb, sejmRP, semtree, SensIAT, SenTinMixt, seqCAT, serocalculator, sfc, sfcr, sffdr, sfislands, sGMRFmix, sgsR, sgstar, ShapleyOutlier, SharkDemography, shazam, SHELF, ShellChron, sherlock, shiny.telemetry, shinyepico, shinyExprPortal, shinyHugePlot, ShinyItemAnalysis, ShinyLink, shinyML, shinymodels, shinyobjects, shinyrecipes, shinySIR, ShinyTester, shinyWGD, ShortForm, shorts, SIAmodules, SIBER, siera, sift, sigminer, Signac, signeR, simaerep, SimBIID, SimBu, simET, simhelpers, simlandr, simmer.plot, SimNPH, simpleFDR, SimplyAgree, simpr, simrel, simTool, simts, SingleCaseES, singleCellTK, SingleMoleculeFootprinting, singscore, SISIR, sitar, sitepickR, sits, SiZer, sjPlot, skater, skimr, SLCARE, sleeperapi, slendr, SMAD, smartid, smartsheetr, smdi, smer, smoothic, smoothy, smosr, snpAIMeR, SNPassoc, snSMART, soc.ca, socialrisk, sociome, SoilManageR, soiltestcorr, SomaDataIO, SOMNiBUS, sorvi, sovereign, sparklyr, sparsegl, SPARTAAS, SpatFD, SpatialCPie, spatialRF, spatialTIME, spatPomp, speccurvieR, specr, spectralR, Spectran, speech, speechbr, speedytax, SpiceFP, spicyR, Spillover, spillR, splithalf, spnaf, SPONGE, spqdep, sqlhelper, squallms, sreg, srppp, srvyr, ssimparser, sspm, sSPPY, ssrn, stabiliser, stability, staccuracy, stacks, standR, StanMoMo, staRdom, starschemar, starvz, statforbiology, Statial, stationaRy, statnipokladna, statsearchanalyticsr, statsExpressions, statsr, StepReg, stevemisc, STICr, SticsRFiles, STMr, stochLAB, stodom, stoRy, stppSim, strand, STRMPS, stRoke, studentlife, stxplore, subSeq, suddengains, sugarbag, summarytools, Superpower, supersigs, supportR, SurfR, Surrogate, surveil, surveydata, surveyexplorer, survHE, survivalAnalysis, survivoR, survminer, survobj, survParamSim, SurvSparse, survstan, sweep, swfscAirDAS, swfscDAS, swfscMisc, swimplot, swissparl, SWMPr, SWMPrExtension, SWTools, syllabifyr, symphony, symptomcheckR, synergyfinder, sysAgNPs, syuzhet, T2Qv, tabbycat, tabledown, tablet, tabnet, tabr, tabshiftr, tabulator, tabxplor, TADCompare, talkr, tall, tame, TAPseq, tardis, tatooheene, tbrf, TCGAbiolinks, tcgaViz, TCIU, tcpl, tdcmStan, teal.modules.general, teal.transform, teamcolors, tectonicr, telemetR, telraamStats, Temporal, tenm, tern, tern.mmrm, terraTCGAdata, TestAnaAPP, TestDimorph, TestGardener, tetraclasse, text, text2sdg, text2speech, textanalyzer, texter, TextForecast, TextMiningGUI, textreuse, tfestimators, tfrmt, theft, theftdlc, ThermalSampleR, TheSFACE, thinkr, THREC, threesixtygiving, thurstonianIRT, tidier, tidyAML, tidybayes, tidyBdE, tidyboot, tidybulk, tidycat, tidyCDISC, tidycensus, tidychangepoint, tidyclust, tidycmprsk, tidycomm, tidyCoverage, tidyDenovix, TidyDensity, tidydice, tidyedgar, tidyEmoji, tidyfinance, tidyfit, tidyFlowCore, tidygam, tidygapminder, tidygate, tidygenomics, tidygraph, tidyHeatmap, tidyheatmaps, tidyhydat, tidyindex, tidyjson, tidylda, tidylog, tidyMC, tidymodels, tidymodlr, tidync, tidypaleo, tidyplots, tidyplus, tidyposterior, tidypredict, tidyquant, tidyrates, tidyREDCap, tidyrgee, tidyseurat, tidySingleCellExperiment, tidysmd, tidySpatialExperiment, tidystats, tidySummarizedExperiment, tidyterra, tidytidbits, tidytlg, tidytof, tidytreatment, tidytree, tidytuesdayR, tidyverse, tidywater, tidywikidatar, tigger, timbeR, timeOmics, timetk, tintin, TissueEnrich, tmod, tna, tntpr, TOAST, tongfen, TOP, topChef, topics, topr, tosr, TOSTER, toxEval, ToxicoGx, toxpiR, Tplyr, TPP, TPP2D, track2KBA, tradepolicy, trainR, traitstrap, transcriptogramer, TransProR, traumar, treasury, TreatmentPatterns, treeclimbR, treeheatr, treekoR, trelliscopejs, TriDimRegression, triptych, TSAR, tsdataleaks, tsgc, tsnet, TSstudio, tuberculosis, TUFLOWR, tune, Twitmo, twoxtwo, ubair, UCSCXenaShiny, ukbtools, ulex, ulrb, umiAnalyzer, UnalR, understandBPMN, unheadr, unpivotr, unstruwwel, unusualprofile, UpAndDownPlots, ura, usdampr, usedthese, usemodels, ushr, uxr, vachette, valhallr, validata, varitas, vcdExtra, vDiveR, VectorCodeR, vectorsurvR, veesa, viafr, vici, vidger, viewpoly, vigicaen, virtualPollen, Virusparies, viscomp, visdat, ViSe, visOmopResults, visvaR, visvow, visxhclust, vital, vivainsights, vivaldi, vmeasur, volcanoPlot, volker, voson.tcn, vosonSML, votesmart, vpc, vprr, vvauditor, vvcanvas, vvtableau, VWPre, walkboutr, walkscore, WARDEN, WASP, waves, wbids, wbstats, wbwdi, wcde, wcep, wcox, WeatherSentiment, webr, weed, wehoop, whippr, whitewater, whoa, whomds, widyr, wikilake, windows.pls, wingen, winr, WoodSimulatR, workflowsets, worldfootballR, worldmet, wpa, wppi, wqtrends, wrangle, wrappedtools, WRTDStidal, wru, wyz.code.metaTesting, wyz.code.offensiveProgramming, wyz.code.rdoc, wyz.code.testthat, x3ptools, xadmix, Xcertainty, xesreadR, xmap, xml2relational, xmrr, xpose, xpose.nlmixr2, xpose.xtras, xxdi, yamlet, ycevo, yfR, zdeskR, zendown, zenith, zFPKM, zipcodeR, zippeR, zitools, zoomerjoin, zoomr","BayesGP, hmmTMB","","","","","clock, minty, readr, shide, vroom","","","",""]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>deadline<\/th>\n      <th>package<\/th>\n      <th>version<\/th>\n      <th>maintainer<\/th>\n      <th>links<\/th>\n      <th>event<\/th>\n      <th>reverse packages (n)<\/th>\n      <th>reverse packages<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":50,"order":[[0,"asc"],[6,"desc"],[1,"asc"]],"columnDefs":[{"className":"dt-right","targets":6},{"name":"deadline","targets":0},{"name":"package","targets":1},{"name":"version","targets":2},{"name":"maintainer","targets":3},{"name":"links","targets":4},{"name":"event","targets":5},{"name":"reverse packages (n)","targets":6},{"name":"reverse packages","targets":7}],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>




<div id="quarto-appendix" class="default"><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><h2 class="anchored quarto-appendix-heading">Footnotes</h2>

<ol>
<li id="fn1"><p>When a package is <em>archived</em>, the archived version is still available in the <a href="https://cran.r-project.org/src/contrib/Archive/">CRAN archive area</a> together with all previous versions.<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




<footer class="footer"><div class="nav-footer"><div class="nav-footer-center"><div class="toc-actions"><ul><li><a href="https://github.com/cranhaven/cranhaven.org/blob/main/dashboard-at-risk.qmd" class="toc-action"><i class="bi bi-github"></i>View source</a></li><li><a href="https://github.com/cranhaven/cranhaven.org/edit/main/dashboard-at-risk.qmd" class="toc-action"><i class="bi empty"></i>Edit this page</a></li><li><a href="https://github.com/cranhaven/cranhaven.org/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></div></div></footer></body></html>