<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Dashboard: Packages At-Risk of Being Archived â€“ CRANhaven</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<link href="site_libs/htmltools-fill-0.5.8/fill.css" rel="stylesheet">
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="site_libs/datatables-binding-0.32/datatables.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">CRANhaven</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-articles" role="link" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Articles</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-articles">    
        <li>
    <a class="dropdown-item" href="./dashboard-live.html">
 <span class="dropdown-text">Dashboard: Recently Archived CRAN Packages</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./dashboard-at-risk.html">
 <span class="dropdown-text">Dashboard: Packages At-Risk of Being Archived</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./cran-archiving-stats.html">
 <span class="dropdown-text">Study: Many Archived Packages Return to CRAN</span></a>
  </li>  
    </ul>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">About</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/cranhaven/cranhaven.org/"> <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar"><div class="quarto-margin-header"><div class="margin-header-item">
<p><a href="https://cranhaven.r-universe.dev/builds"><img src="https://cranhaven.r-universe.dev/badges/:total" alt="CRANhaven number of packages"></a></p>
</div></div>
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Dashboard: Packages At-Risk of Being Archived</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<style>
tr { vertical-align: top; }
</style>
<p>The below table lists 72 R packages that are at risk of being archived<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a> on CRAN, because they have outstanding unresolved issues.</p>
<p>Some of them might have already submitted a new version to fix the issues identified by CRAN (check the event column).</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="live-dashboard" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="live-dashboard">{"x":{"filter":"none","vertical":false,"data":[["2024-11-18<br><small>(in 4 days)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-28<br><small>(in 14 days)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-25<br><small>(in 11 days)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-25<br><small>(in 11 days)<\/small>","2024-11-18<br><small>(in 4 days)<\/small>","2024-11-25<br><small>(in 11 days)<\/small>","2024-11-25<br><small>(in 11 days)<\/small>","2024-11-26<br><small>(in 12 days)<\/small>","2024-11-26<br><small>(in 12 days)<\/small>","2024-11-28<br><small>(in 14 days)<\/small>","2024-11-24<br><small>(in 10 days)<\/small>","2024-11-24<br><small>(in 10 days)<\/small>","2024-11-28<br><small>(in 14 days)<\/small>","2024-11-21<br><small>(in 7 days)<\/small>","2024-11-24<br><small>(in 10 days)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-19<br><small>(in 5 days)<\/small>","2024-11-24<br><small>(in 10 days)<\/small>","2024-11-22<br><small>(in 8 days)<\/small>","2024-11-25<br><small>(in 11 days)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-17<br><small>(in 3 days)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-18<br><small>(in 4 days)<\/small>","2024-11-27<br><small>(in 13 days)<\/small>","2024-11-25<br><small>(in 11 days)<\/small>","2024-11-24<br><small>(in 10 days)<\/small>","2024-11-26<br><small>(in 12 days)<\/small>","2024-11-19<br><small>(in 5 days)<\/small>","2024-11-19<br><small>(in 5 days)<\/small>","2024-11-25<br><small>(in 11 days)<\/small>","2024-11-16<br><small>(in 2 days)<\/small>","2024-11-28<br><small>(in 14 days)<\/small>","2024-11-24<br><small>(in 10 days)<\/small>","2024-11-24<br><small>(in 10 days)<\/small>","2024-11-24<br><small>(in 10 days)<\/small>","2024-11-26<br><small>(in 12 days)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-18<br><small>(in 4 days)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-21<br><small>(in 7 days)<\/small>","2024-11-24<br><small>(in 10 days)<\/small>","2024-11-28<br><small>(in 14 days)<\/small>","2024-11-19<br><small>(in 5 days)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-26<br><small>(in 12 days)<\/small>","2024-11-20<br><small>(in 6 days)<\/small>","2024-11-26<br><small>(in 12 days)<\/small>","2024-11-24<br><small>(in 10 days)<\/small>","2024-11-24<br><small>(in 10 days)<\/small>","2024-11-21<br><small>(in 7 days)<\/small>","2024-11-25<br><small>(in 11 days)<\/small>","2024-11-19<br><small>(in 5 days)<\/small>","2024-11-26<br><small>(in 12 days)<\/small>","2024-11-28<br><small>(in 14 days)<\/small>","2024-11-19<br><small>(in 5 days)<\/small>","2024-11-28<br><small>(in 14 days)<\/small>","2024-11-25<br><small>(in 11 days)<\/small>","2024-11-28<br><small>(in 14 days)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-25<br><small>(in 11 days)<\/small>","2024-11-28<br><small>(in 14 days)<\/small>","2024-11-28<br><small>(in 14 days)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>","2024-11-12<br><small>(2 days overdue)<\/small>"],["<a href=\"https://cran.r-project.org/package=algaeClassify\">algaeClassify<\/a>","<a href=\"https://cran.r-project.org/package=aphylo\">aphylo<\/a>","<a href=\"https://cran.r-project.org/package=autometric\">autometric<\/a>","<a href=\"https://cran.r-project.org/package=bayesGAM\">bayesGAM<\/a>","<a href=\"https://cran.r-project.org/package=cleaner\">cleaner<\/a>","<a href=\"https://cran.r-project.org/package=D4TAlink.light\">D4TAlink.light<\/a>","<a href=\"https://cran.r-project.org/package=data.table\">data.table<\/a>","<a href=\"https://cran.r-project.org/package=dhis2r\">dhis2r<\/a>","<a href=\"https://cran.r-project.org/package=diffudist\">diffudist<\/a>","<a href=\"https://cran.r-project.org/package=dint\">dint<\/a>","<a href=\"https://cran.r-project.org/package=distfreereg\">distfreereg<\/a>","<a href=\"https://cran.r-project.org/package=dm\">dm<\/a>","<a href=\"https://cran.r-project.org/package=drape\">drape<\/a>","<a href=\"https://cran.r-project.org/package=easyclimate\">easyclimate<\/a>","<a href=\"https://cran.r-project.org/package=EGRET\">EGRET<\/a>","<a href=\"https://cran.r-project.org/package=erfe\">erfe<\/a>","<a href=\"https://cran.r-project.org/package=fedregs\">fedregs<\/a>","<a href=\"https://cran.r-project.org/package=fluxfinder\">fluxfinder<\/a>","<a href=\"https://cran.r-project.org/package=FuzzyStatTraEOO\">FuzzyStatTraEOO<\/a>","<a href=\"https://cran.r-project.org/package=galah\">galah<\/a>","<a href=\"https://cran.r-project.org/package=gdm\">gdm<\/a>","<a href=\"https://cran.r-project.org/package=geoidep\">geoidep<\/a>","<a href=\"https://cran.r-project.org/package=ggpicrust2\">ggpicrust2<\/a>","<a href=\"https://cran.r-project.org/package=GiniDecompLY\">GiniDecompLY<\/a>","<a href=\"https://cran.r-project.org/package=gnm\">gnm<\/a>","<a href=\"https://cran.r-project.org/package=googleCloudRunner\">googleCloudRunner<\/a>","<a href=\"https://cran.r-project.org/package=gsheet\">gsheet<\/a>","<a href=\"https://cran.r-project.org/package=gWidgets2\">gWidgets2<\/a>","<a href=\"https://cran.r-project.org/package=hgutils\">hgutils<\/a>","<a href=\"https://cran.r-project.org/package=HTSSIP\">HTSSIP<\/a>","<a href=\"https://cran.r-project.org/package=httr2\">httr2<\/a>","<a href=\"https://cran.r-project.org/package=igraph\">igraph<\/a>","<a href=\"https://cran.r-project.org/package=kergp\">kergp<\/a>","<a href=\"https://cran.r-project.org/package=lmls\">lmls<\/a>","<a href=\"https://cran.r-project.org/package=MBNMAdose\">MBNMAdose<\/a>","<a href=\"https://cran.r-project.org/package=MBNMAtime\">MBNMAtime<\/a>","<a href=\"https://cran.r-project.org/package=MetaLonDA\">MetaLonDA<\/a>","<a href=\"https://cran.r-project.org/package=metaumbrella\">metaumbrella<\/a>","<a href=\"https://cran.r-project.org/package=mobsim\">mobsim<\/a>","<a href=\"https://cran.r-project.org/package=network\">network<\/a>","<a href=\"https://cran.r-project.org/package=networkDynamic\">networkDynamic<\/a>","<a href=\"https://cran.r-project.org/package=orthGS\">orthGS<\/a>","<a href=\"https://cran.r-project.org/package=osmapiR\">osmapiR<\/a>","<a href=\"https://cran.r-project.org/package=panelvar\">panelvar<\/a>","<a href=\"https://cran.r-project.org/package=pct\">pct<\/a>","<a href=\"https://cran.r-project.org/package=PerformanceAnalytics\">PerformanceAnalytics<\/a>","<a href=\"https://cran.r-project.org/package=pkgmaker\">pkgmaker<\/a>","<a href=\"https://cran.r-project.org/package=progressr\">progressr<\/a>","<a href=\"https://cran.r-project.org/package=projections\">projections<\/a>","<a href=\"https://cran.r-project.org/package=PSGD\">PSGD<\/a>","<a href=\"https://cran.r-project.org/package=quantreg\">quantreg<\/a>","<a href=\"https://cran.r-project.org/package=R6P\">R6P<\/a>","<a href=\"https://cran.r-project.org/package=RadEro\">RadEro<\/a>","<a href=\"https://cran.r-project.org/package=ResearchAssociate\">ResearchAssociate<\/a>","<a href=\"https://cran.r-project.org/package=rgeoboundaries\">rgeoboundaries<\/a>","<a href=\"https://cran.r-project.org/package=RPANDA\">RPANDA<\/a>","<a href=\"https://cran.r-project.org/package=rPanglaoDB\">rPanglaoDB<\/a>","<a href=\"https://cran.r-project.org/package=RRBoost\">RRBoost<\/a>","<a href=\"https://cran.r-project.org/package=sdpt3r\">sdpt3r<\/a>","<a href=\"https://cran.r-project.org/package=SensIAT\">SensIAT<\/a>","<a href=\"https://cran.r-project.org/package=serocalculator\">serocalculator<\/a>","<a href=\"https://cran.r-project.org/package=serp\">serp<\/a>","<a href=\"https://cran.r-project.org/package=sgapi\">sgapi<\/a>","<a href=\"https://cran.r-project.org/package=sleev\">sleev<\/a>","<a href=\"https://cran.r-project.org/package=soilassessment\">soilassessment<\/a>","<a href=\"https://cran.r-project.org/package=stepSplitReg\">stepSplitReg<\/a>","<a href=\"https://cran.r-project.org/package=tidysynth\">tidysynth<\/a>","<a href=\"https://cran.r-project.org/package=tidyvpc\">tidyvpc<\/a>","<a href=\"https://cran.r-project.org/package=TreeRingShape\">TreeRingShape<\/a>","<a href=\"https://cran.r-project.org/package=usethis\">usethis<\/a>","<a href=\"https://cran.r-project.org/package=W4MRUtils\">W4MRUtils<\/a>","<a href=\"https://cran.r-project.org/package=wooldridge\">wooldridge<\/a>"],["2.0.3<br><small>(2024-10-11)<small>","0.3-3<br><small>(2023-09-07)<small>","0.1.1<br><small>(2024-10-31)<small>","0.0.2<br><small>(2022-03-17)<small>","1.5.4<br><small>(2022-10-29)<small>","2.1.14<br><small>(2023-10-04)<small>","1.16.2<br><small>(2024-10-10)<small>","0.2.1<br><small>(2024-11-04)<small>","1.0.1<br><small>(2023-02-27)<small>","2.1.4<br><small>(2022-10-17)<small>","1.0<br><small>(2024-11-11)<small>","1.0.10<br><small>(2024-01-21)<small>","0.0.1<br><small>(2023-09-18)<small>","0.2.1<br><small>(2023-07-11)<small>","3.0.9<br><small>(2023-04-18)<small>","0.0.1<br><small>(2022-12-16)<small>","1.0.0<br><small>(2019-09-16)<small>","1.0.0<br><small>(2024-10-22)<small>","1.0<br><small>(2022-12-16)<small>","2.0.2<br><small>(2024-04-12)<small>","1.6.0-3<br><small>(2024-11-01)<small>","0.3.0<br><small>(2024-09-22)<small>","1.7.3<br><small>(2023-11-08)<small>","1.0.0<br><small>(2024-01-19)<small>","1.1-5<br><small>(2023-09-16)<small>","0.5.0<br><small>(2022-02-28)<small>","0.4.5<br><small>(2020-04-07)<small>","1.0-9<br><small>(2022-01-10)<small>","0.2.11<br><small>(2022-03-31)<small>","1.4.1<br><small>(2019-09-13)<small>","1.0.6<br><small>(2024-11-04)<small>","2.1.1<br><small>(2024-10-19)<small>","0.5.7<br><small>(2024-02-05)<small>","0.1.0<br><small>(2022-01-18)<small>","0.4.3<br><small>(2024-04-18)<small>","0.2.4<br><small>(2023-10-14)<small>","1.1.8<br><small>(2019-12-18)<small>","1.0.11<br><small>(2024-03-08)<small>","0.3.1<br><small>(2024-03-22)<small>","1.18.2<br><small>(2023-12-05)<small>","0.11.4<br><small>(2023-12-10)<small>","0.1.5<br><small>(2024-09-17)<small>","0.2.1<br><small>(2024-09-05)<small>","0.5.5<br><small>(2023-01-05)<small>","0.9.9<br><small>(2023-05-19)<small>","2.0.4<br><small>(2020-02-06)<small>","0.32.10<br><small>(2023-05-03)<small>","0.15.0<br><small>(2024-10-29)<small>","0.6.0<br><small>(2023-03-23)<small>","1.0.3<br><small>(2023-04-24)<small>","5.99<br><small>(2024-10-22)<small>","0.3.0<br><small>(2022-10-02)<small>","1.0.4<br><small>(2024-11-11)<small>","1.0.1<br><small>(2020-12-15)<small>","1.3<br><small>(2024-03-12)<small>","2.3<br><small>(2024-02-21)<small>","0.2.1<br><small>(2021-05-12)<small>","0.1<br><small>(2020-10-13)<small>","0.3<br><small>(2019-02-11)<small>","0.1.0<br><small>(2024-11-01)<small>","1.0.3<br><small>(2024-07-21)<small>","0.2.4<br><small>(2022-02-16)<small>","1.0.2<br><small>(2024-03-28)<small>","1.0.3<br><small>(2023-07-10)<small>","0.2.6<br><small>(2023-07-08)<small>","1.0.3<br><small>(2022-11-22)<small>","0.2.0<br><small>(2023-05-21)<small>","1.5.1<br><small>(2024-01-18)<small>","3.0.3<br><small>(2024-04-22)<small>","3.0.0<br><small>(2024-07-29)<small>","1.0.0<br><small>(2023-09-08)<small>","1.4-3<br><small>(2023-05-03)<small>"],["<a href=\"mailto:vpatil-at-usgs.gov\">Vijay Patil<\/a>","<a href=\"mailto:g.vegayon-at-gmail.com\">George Vega Yon<\/a>","<a href=\"mailto:will.landau.oss-at-gmail.com\">William Michael Landau<\/a>","<a href=\"mailto:samthoma-at-alumni.iu.edu\">Samuel Thomas<\/a>","<a href=\"mailto:m.s.berends-at-umcg.nl\">Matthijs S. Berends<\/a>","<a href=\"mailto:gregoire.thomas-at-SQU4RE.com\">Gregoire Thomas<\/a>","<a href=\"mailto:t.barrett88-at-gmail.com\">Tyson Barrett<\/a>","<a href=\"mailto:amanyiraho-at-gmail.com\">Robinson Amanyiraho<\/a>","<a href=\"mailto:giulia.bertagnolli-at-gmail.com\">Giulia Bertagnolli<\/a>","<a href=\"mailto:stefan.b.fleck-at-gmail.com\">Stefan Fleck<\/a>","<a href=\"mailto:mill9116-at-umn.edu\">Jesse Miller<\/a>","<a href=\"mailto:kirill-at-cynkra.com\">Kirill MÃ¼ller<\/a>","<a href=\"mailto:hklyne-at-g.harvard.edu\">Harvey Klyne<\/a>","<a href=\"mailto:veronica.cral-at-gmail.com\">VerÃ³nica Cruz-Alonso<\/a>","<a href=\"mailto:ldecicco-at-usgs.gov\">Laura DeCicco<\/a>","<a href=\"mailto:barryhafia-at-gmail.com\">Amadou Barry<\/a>","<a href=\"mailto:scott.large-at-noaa.gov\">Scott Large<\/a>","<a href=\"mailto:sjw22120-at-gmail.com\">Stephanie Wilson<\/a>","<a href=\"mailto:uo270115-at-uniovi.es\">Andrea GarcÃ­a Cernuda<\/a>","<a href=\"mailto:martin.westgate-at-csiro.au\">Martin Westgate<\/a>","<a href=\"mailto:mfitzpatrick-at-umces.edu\">Matt Fitzpatrick<\/a>","<a href=\"mailto:geografo.pe-at-gmail.com\">antony barja<\/a>","<a href=\"mailto:cafferychen7850-at-gmail.com\">Chen Yang<\/a>","<a href=\"mailto:abdessamad.ambarek-at-gmail.com\">Abdessamad Ait Mbarek<\/a>","<a href=\"mailto:ht-at-heatherturner.net\">Heather Turner<\/a>","<a href=\"mailto:r-at-sunholo.com\">Mark Edmondson<\/a>","<a href=\"mailto:conway.max1-at-gmail.com\">Max Conway<\/a>","<a href=\"mailto:jverzani-at-gmail.com\">John Verzani<\/a>","<a href=\"mailto:hvdboorn-at-gmail.com\">H.G. van den Boorn<\/a>","<a href=\"mailto:seb369-at-cornell.edu\">Samuel Barnett<\/a>","<a href=\"mailto:hadley-at-posit.co\">Hadley Wickham<\/a>","<a href=\"mailto:kirill-at-cynkra.com\">Kirill MÃ¼ller<\/a>","<a href=\"mailto:roustant-at-insa-toulouse.fr\">Olivier Roustant<\/a>","<a href=\"mailto:hriebl-at-uni-goettingen.de\">Hannes Riebl<\/a>","<a href=\"mailto:hugopedder-at-gmail.com\">Hugo Pedder<\/a>","<a href=\"mailto:hugopedder-at-gmail.com\">Hugo Pedder<\/a>","<a href=\"mailto:ametwall-at-stanford.edu\">Ahmed A. Metwally<\/a>","<a href=\"mailto:cgosling-at-parisnanterre.fr\">Corentin J Gosling<\/a>","<a href=\"mailto:felix.may-at-fu-berlin.de\">Felix May<\/a>","<a href=\"mailto:buttsc-at-uci.edu\">Carter T. Butts<\/a>","<a href=\"mailto:skyebend-at-uw.edu\">Skye Bender-deMoll<\/a>","<a href=\"mailto:elenaaledoesteban-at-gmail.com\">Elena Aledo<\/a>","<a href=\"mailto:joanmaspons-at-gmail.com\">Joan Maspons<\/a>","<a href=\"mailto:robert.ferstl-at-ur.de\">Robert Ferstl<\/a>","<a href=\"mailto:rob00x-at-gmail.com\">Robin Lovelace<\/a>","<a href=\"mailto:brian-at-braverock.com\">Brian G. Peterson<\/a>","<a href=\"mailto:renozao-at-protonmail.com\">Renaud Gaujoux<\/a>","<a href=\"mailto:henrikb-at-braju.com\">Henrik Bengtsson<\/a>","<a href=\"mailto:thibautjombart-at-gmail.com\">Thibaut Jombart<\/a>","<a href=\"mailto:anthony.christidis-at-stat.ubc.ca\">Anthony Christidis<\/a>","<a href=\"mailto:rkoenker-at-illinois.edu\">Roger Koenker<\/a>","<a href=\"mailto:tidylab-at-gmail.com\">Harel Lustiger<\/a>","<a href=\"mailto:acatalae-at-gmail.com\">Arturo Catala<\/a>","<a href=\"mailto:MohmedSoudy2009-at-gmail.com\">Mohamed Soudy<\/a>","<a href=\"mailto:team-at-geoboundaries.org\">Ahmadou Dicko<\/a>","<a href=\"mailto:helene.morlon-at-bio.ens.psl.eu\">HÃ©lÃ¨ne Morlon<\/a>","<a href=\"mailto:daniecos-at-uio.no\">Daniel Osorio<\/a>","<a href=\"mailto:xiaomeng.ju-at-stat.ubc.ca\">Xiaomeng Ju<\/a>","<a href=\"mailto:a45rahma-at-uwaterloo.ca\">Adam Rahman<\/a>","<a href=\"mailto:andrew.redd-at-hsc.utah.edu\">Andrew Redd<\/a>","<a href=\"mailto:kwlai-at-ucdavis.edu\">Kristina Lai<\/a>","<a href=\"mailto:ejike.ugba-at-outlook.com\">Ejike R. Ugba<\/a>","<a href=\"mailto:OR_Hub-at-defra.gov.uk\">Andrew Christy<\/a>","<a href=\"mailto:r.tao-at-vanderbilt.edu\">Ran Tao<\/a>","<a href=\"mailto:thineomuto-at-yahoo.com\">Christian Thine Omuto<\/a>","<a href=\"mailto:anthony.christidis-at-stat.ubc.ca\">Anthony Christidis<\/a>","<a href=\"mailto:ed769-at-georgetown.edu\">Eric Dunford<\/a>","<a href=\"mailto:james.craig-at-certara.com\">James Craig<\/a>","<a href=\"mailto:ishidam-at-sanchikanri.com\">Megumi ISHIDA<\/a>","<a href=\"mailto:jenny-at-posit.co\">Jennifer Bryan<\/a>","<a href=\"mailto:lain.pavot-at-inrae.fr\">Lain Pavot<\/a>","<a href=\"mailto:jshea-at-roosevelt.edu\">Justin M. Shea<\/a>"],["<a href=\"https://cran.r-project.org/web/checks/check_results_algaeClassify.html\">checks<\/a><br><a href=\"https://doi.org/10.5066/F7S46Q3F\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_aphylo.html\">checks<\/a><br><a href=\"https://github.com/USCbiostats/aphylo\">url<\/a><br><a href=\"https://github.com/USCbiostats/aphylo/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_autometric.html\">checks<\/a><br><a href=\"https://wlandau.github.io/autometric/\">url<\/a><br><a href=\"https://github.com/wlandau/autometric/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_bayesGAM.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_cleaner.html\">checks<\/a><br><a href=\"https://msberends.github.io/cleaner/\">url<\/a><br><a href=\"https://github.com/msberends/cleaner/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_D4TAlink.light.html\">checks<\/a><br><a href=\"https://bitbucket.org/SQ4/d4talink.light\">url<\/a><br><a href=\"https://bitbucket.org/SQ4/d4talink.light/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_data.table.html\">checks<\/a><br><a href=\"https://r-datatable.com\">url<\/a><br><a href=\"https://github.com/Rdatatable/data.table/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_dhis2r.html\">checks<\/a><br><a href=\"https://github.com/amanyiraho/dhis2r\">url<\/a><br><a href=\"https://github.com/amanyiraho/dhis2r/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_diffudist.html\">checks<\/a><br><a href=\"https://gbertagnolli.github.io/diffudist/\">url<\/a><br><a href=\"https://github.com/gbertagnolli/diffudist/issues/\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_dint.html\">checks<\/a><br><a href=\"https://github.com/s-fleck/dint\">url<\/a><br><a href=\"https://github.com/s-fleck/dint/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_distfreereg.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_dm.html\">checks<\/a><br><a href=\"https://dm.cynkra.com/\">url<\/a><br><a href=\"https://github.com/cynkra/dm/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_drape.html\">checks<\/a><br><a href=\"https://github.com/harveyklyne/drape/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_easyclimate.html\">checks<\/a><br><a href=\"https://github.com/VeruGHub/easyclimate\">url<\/a><br><a href=\"https://github.com/VeruGHub/easyclimate/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_EGRET.html\">checks<\/a><br><a href=\"https://pubs.usgs.gov/tm/04/a10/\">url<\/a><br><a href=\"https://github.com/DOI-USGS/EGRET/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_erfe.html\">checks<\/a><br><a href=\"https://arxiv.org/abs/2108.04737\">url<\/a><br><a href=\"https://github.com/amadoudiogobarry/erfe/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_fedregs.html\">checks<\/a><br><a href=\"https://github.com/slarge/fedregs/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_fluxfinder.html\">checks<\/a><br><a href=\"https://github.com/COMPASS-DOE/fluxfinder\">url<\/a><br><a href=\"https://github.com/COMPASS-DOE/fluxfinder/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_FuzzyStatTraEOO.html\">checks<\/a><br><a href=\"https://bellman.ciencias.uniovi.es/smire+codire/FuzzyStatTraRpackage.html\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_galah.html\">checks<\/a><br><a href=\"https://galah.ala.org.au/R/\">url<\/a><br><a href=\"https://github.com/AtlasOfLivingAustralia/galah-R/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gdm.html\">checks<\/a><br><a href=\"https://mfitzpatrick.al.umces.edu/gdm/\">url<\/a><br><a href=\"https://github.com/fitzLab-AL/gdm/issues/\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_geoidep.html\">checks<\/a><br><a href=\"https://geografo.pe/geoidep/\">url<\/a><br><a href=\"https://github.com/ambarja/geoidep/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ggpicrust2.html\">checks<\/a><br><a href=\"https://github.com/cafferychen777/ggpicrust2\">url<\/a><br><a href=\"https://github.com/cafferychen777/ggpicrust2/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_GiniDecompLY.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gnm.html\">checks<\/a><br><a href=\"https://github.com/hturner/gnm\">url<\/a><br><a href=\"https://github.com/hturner/gnm/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_googleCloudRunner.html\">checks<\/a><br><a href=\"https://code.markedmondson.me/googleCloudRunner/\">url<\/a><br><a href=\"https://github.com/MarkEdmondson1234/googleCloudRunner/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gsheet.html\">checks<\/a><br><a href=\"https://github.com/maxconway/gsheet\">url<\/a><br><a href=\"https://github.com/maxconway/gsheet/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gWidgets2.html\">checks<\/a><br><a href=\"https://github.com/gWidgets3/gWidgets2\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hgutils.html\">checks<\/a><br><a href=\"https://github.com/hvdboorn/hgutils\">url<\/a><br><a href=\"https://github.com/hvdboorn/hgutils/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_HTSSIP.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_httr2.html\">checks<\/a><br><a href=\"https://httr2.r-lib.org\">url<\/a><br><a href=\"https://github.com/r-lib/httr2/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_igraph.html\">checks<\/a><br><a href=\"https://r.igraph.org/\">url<\/a><br><a href=\"https://github.com/igraph/rigraph/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_kergp.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_lmls.html\">checks<\/a><br><a href=\"https://hriebl.github.io/lmls/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_MBNMAdose.html\">checks<\/a><br><a href=\"https://hugaped.github.io/MBNMAdose/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_MBNMAtime.html\">checks<\/a><br><a href=\"https://hugaped.github.io/MBNMAtime/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_MetaLonDA.html\">checks<\/a><br><a href=\"https://github.com/aametwally/MetaLonDA\">url<\/a><br><a href=\"https://github.com/aametwally/MetaLonDA/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_metaumbrella.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_mobsim.html\">checks<\/a><br><a href=\"https://github.com/MoBiodiv/mobsim\">url<\/a><br><a href=\"https://github.com/MoBiodiv/mobsim/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_network.html\">checks<\/a><br><a href=\"https://statnet.org/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_networkDynamic.html\">checks<\/a><br><a href=\"https://statnet.org/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_orthGS.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_osmapiR.html\">checks<\/a><br><a href=\"https://docs.ropensci.org/osmapiR/\">url<\/a><br><a href=\"https://github.com/ropensci/osmapiR/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_panelvar.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_pct.html\">checks<\/a><br><a href=\"https://itsleeds.github.io/pct/\">url<\/a><br><a href=\"https://github.com/ITSLeeds/pct/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_PerformanceAnalytics.html\">checks<\/a><br><a href=\"https://github.com/braverock/PerformanceAnalytics\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_pkgmaker.html\">checks<\/a><br><a href=\"https://renozao.github.io/pkgmaker/\">url<\/a><br><a href=\"https://github.com/renozao/pkgmaker/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_progressr.html\">checks<\/a><br><a href=\"https://progressr.futureverse.org\">url<\/a><br><a href=\"https://github.com/futureverse/progressr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_projections.html\">checks<\/a><br><a href=\"https://www.repidemicsconsortium.org/projections/\">url<\/a><br><a href=\"https://github.com/reconhub/projections/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_PSGD.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_quantreg.html\">checks<\/a><br><a href=\"https://www.r-project.org\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_R6P.html\">checks<\/a><br><a href=\"https://tidylab.github.io/R6P/\">url<\/a><br><a href=\"https://github.com/tidylab/R6P/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_RadEro.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ResearchAssociate.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rgeoboundaries.html\">checks<\/a><br><a href=\"https://github.com/wmgeolab/rgeoboundaries\">url<\/a><br><a href=\"https://github.com/wmgeolab/rgeoboundaries/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_RPANDA.html\">checks<\/a><br><a href=\"https://github.com/hmorlon/PANDA\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rPanglaoDB.html\">checks<\/a><br><a href=\"https://github.com/dosorio/rPanglaoDB/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_RRBoost.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_sdpt3r.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_SensIAT.html\">checks<\/a><br><a href=\"https://github.com/UofUEpiBio/SensIAT\">url<\/a><br><a href=\"https://github.com/UofUEpiBio/SensIAT/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_serocalculator.html\">checks<\/a><br><a href=\"https://github.com/UCD-SERG/serocalculator\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_serp.html\">checks<\/a><br><a href=\"https://github.com/ejikeugba/serp\">url<\/a><br><a href=\"https://github.com/ejikeugba/serp/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_sgapi.html\">checks<\/a><br><a href=\"https://defra-data-science-centre-of-excellence.github.io/sgapi/\">url<\/a><br><a href=\"https://github.com/Defra-Data-Science-Centre-of-Excellence/sgApi/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_sleev.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_soilassessment.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_stepSplitReg.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tidysynth.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tidyvpc.html\">checks<\/a><br><a href=\"https://github.com/certara/tidyvpc\">url<\/a><br><a href=\"https://github.com/certara/tidyvpc/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_TreeRingShape.html\">checks<\/a><br><a href=\"https://github.com/ishidamgm/TreeRingShape\">url<\/a><br><a href=\"https://github.com/ishidamgm/TreeRingShape/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_usethis.html\">checks<\/a><br><a href=\"https://usethis.r-lib.org\">url<\/a><br><a href=\"https://github.com/r-lib/usethis/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_W4MRUtils.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_wooldridge.html\">checks<\/a><br><a href=\"https://justinmshea.github.io/wooldridge/\">url<\/a><br><a href=\"https://github.com/JustinMShea/wooldridge/issues/\">url2<\/a>"],[null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#distfreereg\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#GiniDecompLY\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#lmls\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#ResearchAssociate\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#sleev\">resubmitted/archive<\/a><\/small>",null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#TreeRingShape\">resubmitted/archive<\/a><\/small>",null,null,null],[0,0,1,0,1,0,1676,0,0,1,0,4,0,0,1,0,0,0,0,0,0,0,0,0,13,0,4,8,0,1,157,887,0,0,0,0,0,0,0,88,11,0,0,0,0,20,0,86,0,0,146,3,0,0,0,0,0,0,2,0,0,0,0,0,0,0,0,1,0,95,0,0],["","","crew","","clean","","Ac3net, accessibility, acdcR, Achilles, acoRn, actel, ActivePathways, ActivityIndex, actuaRE, ADAMgui, adductomicsR, AdhereR, AdhereRViz, ADImpute, AeRobiology, AF, AFM, africamonitor, agcounts, AggregateR, agriutilities, ahw, AICcPermanova, akc, akiFlagger, aLFQ, alpaca, AlphaBeta, amadeus, amapGeocode, AMAPVox, AMIM, amplican, amt, Anaconda, animint2, antaresEditObject, antaresRead, AnxietySleep, aopdata, aorsf, AOV1R, APCI, aPEAR, APIS, aqp, arf, artMS, asciiSetupReader, ascotraceR, aslib, ASpli, ASRgenomics, assertable, atime, AtmChile, atSNP, attrib, AUCell, aum, autoCovariateSelection, autonewsmd, autonomics, AutoPlots, autostats, autostsm, aweSOM, bagged.outliertrees, bakR, bambu, BANDITS, Banksy, baseballr, basecallQC, BatchExperiments, BatchJobs, BatchQC, batchtools, bayesLife, BayesLN, bayesMeanScale, bayesMig, bayesPop, bayesTFR, BBmisc, bbmix, bbotk, bcputility, bdots, bea.R, BEclear, bedr, BeeGUTS, behavr, benford.analysis, bfw, BgeeCall, BgeeDB, BGmisc, BiasCorrector, biblionetwork, bife, BIGL, bigreadr, bigsnpr, binancer, binomialRF, binsegRcpp, biomartr, BIOMASS, biopixR, BioR, bioregion, birankr, biscuiteer, bkmrhat, BKTR, BLA, blorr, BMisc, bnbc, BootstrapQTL, BPRMeth, bRacatus, brainGraph, branchpointer, brandwatchR, bratteli, bread, BREADR, bridger2, BRINDA, brmsmargins, brpop, bruceR, bsseq, BTSPAS, BTYDplus, bupaR, BuyseTest, c3, CAGEr, CALF, calmr, CamelUp, camtrapR, CaMutQC, caretEnsemble, cartogramR, casebase, CAST, CATALYST, categoryEncodings, CatEncoders, CatsCradle, causalCmprsk, CausalGPS, cbl, CBRT, ccdR, ccmap, ccostr, cdparcoord, ceas, celda, CellBarcode, cellbaseR, CelliD, cellKey, CEMiTool, Census2016, Certara.NLME8, Certara.R, Certara.VPCResults, cffdrs, cglm, changepoint.influence, ChannelAttributionApp, ChemoSpecUtils, Chicago, chicane, chimeraviz, ChineseNames, ChIPexoQual, ChIPpeakAnno, CHO, chronicle, cicero, circhelp, citationchaser, citecorp, ciw, clam, classifierplots, cleanTS, ClickHouseHTTP, clickstream, climate, clinDataReview, clinspacy, clinUtils, CluMP, Clustering, clustra, CLVTools, cmapR, cmsaf, CNVRanger, CNVScope, COCOA, coder, CoDi, coffee, cohorts, coinmarketcapr, coloc, colocPropTest, colorednoise, Colossus, comorbidity, CompareCausalNetworks, compareDF, COMPASS, ComPrAn, compSPOT, CONFESS, CoNI, conleyreg, consensusDE, consortr, ConsReg, constellation, container, CoOL, cooltools, CooRTweet, cophescan, coRdon, CoreGx, CornerstoneR, corporaexplorer, corpustools, corrcoverage, correspondenceTables, CoSMoS, counterfactuals, coveffectsplot, COVID19, covid19br, covid19india, covidnor, CovRegRF, cppRouting, CRE, creditmodel, crew, crfsuite, CRISPRseek, CropScapeR, crosshap, crumble, csa, csalert, csdata, csmaps, CSMES, cstidy, cstime, CSTools, csutil, csv, csvy, ctbi, ctmm, cTRAP, ctsem, ctxR, cubar, curtailment, cusum, cusumcharter, cvAUC, cvms, CWT, cxhull, cyclestreets, cylcop, cytoKernel, CytoML, D3partitionR, daiquiri, daiR, damr, dartR, dartR.base, dartR.popgen, dartR.sim, dartR.spatial, dat, data.table.threads, data360r, DataCombine, dataCompare, datadictionary, DataExplorer, dataframeexplorer, datamods, dataMojo, dataprep, dataPreparation, DataSpaceR, DATAstudio, datazoom.amazonia, dateutils, dblr, dbnR, dbWebForms, ddtlcm, decompTumor2Sig, deconvR, dedupewider, deepdive, DEFormats, DegNorm, delayed, deltaccd, DemoKin, DEScan2, DescTools, designit, DESpace, detectRUNS, DEWSeq, dfmeta, DGM, diathor, diceplot, did2s, Diderot, DiDforBigData, didimputation, DIDmultiplegtDYN, difconet, DifferentialRegulation, DiscoRhythm, DiSCos, discSurv, DisImpact, disk.frame, distantia, distillML, DIZtools, DIZutils, DMCFB, do, DominoEffect, doremi, DoubleML, DQAgui, DQAstats, dQTG.seq, DramaAnalysis, dreamlet, drgee, dropR, droptest, dsos, DTAT, dtplyr, DTSg, dtts, DTwrappers, DTwrappers2, dumbbell, dvmisc, dynamite, dynsurv, DysPIA, DYTE, E4tools, Eagle, easybio, easycsv, easyr, easysurv, eatGADS, eatTools, EBPRS, eclust, ecocomDP, ecoregime, edeaR, edl, EdSurvey, eeptools, efdm, EGM, EHR, EIAapi, EIX, eixport, electionsBR, eltr, EmiR, EmissV, ENCODExplorerData, enderecobr, endogeneity, endoSwitch, enmSdmX, ensr, enviGCMS, eoR, Epi, epialleleR, EpiCompare, epidm, EpiForsk, epigraHMM, EpiMix, EpiNow2, epiomics, epistasisGA, eplusr, epwshiftr, equateMultiple, eSIR, estadistica, EthSEQ, etm, etwfe, eurocordexr, eurodata, europeanaR, eurostat, evapoRe, eventstudyr, evoper, EWCE, exiftoolr, explainer, exploratory, expowo, expss, exvatools, eyeRead, eyeTrackR, ezr, facmodCS, facmodTS, FACT, factR, faers, faersquarterlydata, FairMclus, familiar, FAOSTAT, farrell, fastdid, fastDummies, fastLink, fastreeR, fastRhockey, fastText, fastverse, fbRads, fcfdr, FCO, fdq, FeatureImpCluster, fedmatch, fedstatAPIr, fedup, felp, fgdr, Fgmutils, fgsea, fhircrackr, FIESTA, FIESTAutils, finch, finnsurveytext, fishmethods, fitbitViz, FitHiC, flextable, flightsbr, flippant, FLOPART, florabr, flowGraph, flowml, flowWorkspace, Fluidigm, FMAT, fmeffects, FOCI, forceplate, forecastML, forestError, formatdown, formulaic, FORTLS, fossilbrush, FourWayHMM, fplot, fplyr, FRASER, fruclimadapt, fsbrain, FuncDiv, funchir, FuzzyDBScan, fwlplot, fy, GALLO, GARCOM, gasfluxes, gatom, gbp, gcite, gde, gDRcore, gDRimport, gDRtestData, gDRutils, gdxdt, gemma.R, gems, GeneGeneInteR, geneplast, GENESIS, GeneStructureTools, geneXtendeR, genomation, GenomAutomorphism, genomic.autocorr, GenomicDistributions, GenomicDistributionsData, GenomicInteractions, GenomicTools.fileHandler, GenomicTuples, GenoScan, GenVisR, geobr, geocausal, GeoMongo, GeomxTools, geoperu, GEOquery, GeoTcgaData, GeoThinneR, germinationmetrics, getDTeval, GFE, gfer, gfoRmula, ggbrain, ggcyto, ggdmc, ggetho, ggfacto, ggfigdone, GGIR, GGIRread, ggkegg, ggmugs, ggperiodic, ggPMX, ggpval, ggquickeda, ggsom, ggTimeSeries, GHap, githubinstall, gittargets, GladiaTOX, GmicR, GMMAT, gmoTree, gnrprod, goat, gompertztrunc, googleCloudVisionR, GOTHiC, gpboost, GRaNIE, grattan, grattanInflators, greatR, GREENeR, greport, growthcleanr, GRShiny, GSALightning, gsDesign2, GSODR, gtfs2gps, gtfsio, gtfsrouter, gtfstools, GTFSwizard, GUIDEseq, gunsales, gwaRs, gwascatData, gwasforest, GWASinspector, GWASTools, gwid, HACSim, HaDeX, hagis, hahmmr, hal9001, haldensify, hBayesDM, HCD, HCR, hdd, hdpGLM, healthdb, healthyAddress, heatwaveR, heims, hereR, HERON, hesim, heuristicsmineR, HicAggR, HiCBricks, HiCcompare, HiCDCPlus, HiCDOC, hicp, highfrequency, hilldiv, himach, hisse, Hmisc, HMMcopy, HMP2Data, hoardeR, HodgesTools, hollr, hoopR, HospitalNetwork, hot.deck, HPLB, hR, hsrecombi, httk, hudr, hurricaneexposure, hutils, hutilscpp, hwig, hydroloom, hyfo, hypervolume, HYPEtools, i2extras, IBRtools, iCAMP, ICAMS, iCellR, IceSat2R, iCNV, IDEATools, IDmining, ie2misc, ieegio, ifCNVR, iGC, iimi, imabc, imcRtools, iml, immunarch, immuneSIM, importinegi, imputeMulti, inbreedR, IncDTW, incidence2, inferr, InflectSSP, influential, Information, insurancerating, InterCellar, intervalaverage, inTrees, IOHanalyzer, ips, irg, IrregLong, ISCA, iscoCrosswalks, isdparser, IsoBayes, isobxr, isotracer, iSTATS, itol.toolkit, itscalledsoccer, ivdoctr, ivtools, jage, JointFPM, joinXL, journalabbr, joyn, jrSiCKLSNMF, JSmediation, jsmodule, jstable, jsTree, jubilee, JWileymisc, kangar00, kdry, kerastuneR, kesernetwork, kgen, kgraph, kibior, kim, KinSwingR, kisopenapi, KMD, kmeRtone, knfi, KnockoffScreen, koRpus, kosis, KPC, lab2clean, LabourMarketAreas, labourR, LACE, lacunr, lar, latrend, LCMSQA, ldamatch, ldaPrototype, ldsr, leafR, leakyIV, LearningStats, LegATo, lexicon, LexisNexisTools, lgrExtra, libbib, libr, lidR, lightgbm, LilRhino, limexhub, limorhyde2, lineagespot, LinkHD, lipidr, lisaClust, live, LKT, llama, LMPdata, lmtp, loadeR, loci2path, logiBin, LOLA, LOMAR, lookupTable, LOPART, lori, LPDynR, LSPFP, Luminescence, LWFBrook90R, lwqs, LymphoSeq, m5, maditr, maftools, MAGAR, magclass, MAGEE, maicChecks, MANOVA.RM, MantaID, mapi, marginaleffects, margins, mariner, marked, maser, MAST, matman, MatrixHMM, mau, maxaltall, maxent.ot, mbr, mbRes, mcauchyd, mcboost, medrxivr, Melissa, memisc, MesKit, metabolomicsWorkbenchR, MetaComp, metaforest, metaGE, metagene2, MetAlyzer, MetaScope, meteo, methimpute, methrix, methylKit, MethylMix, methylPipe, methylscaper, metR, mggd, mgi.report.reader, miceFast, miceRanger, MicrobiotaProcess, microeco, microseq, midfieldr, miesmuschel, miLineage, MIMER, mimi, mind, minfi, MinimumDistance, minSNPs, mipplot, MIRA, mirTarRnaSeq, miscset, missCompare, misty, mixgb, mixIndependR, mknapsack, mlbplotR, mldr.resampling, mlexperiments, mllrnrs, mlr, mlr3, mlr3batchmark, mlr3benchmark, mlr3cluster, mlr3db, mlr3fairness, mlr3fda, mlr3filters, mlr3fselect, mlr3hyperband, mlr3learners, mlr3mbo, mlr3misc, mlr3oml, mlr3pipelines, mlr3resampling, mlr3shiny, mlr3spatial, mlr3spatiotempcv, mlr3summary, mlr3superlearner, mlr3torch, mlr3tuning, mlr3tuningspaces, mlr3verse, mlr3viz, mlrintermbo, mlrMBO, mlsurvlrnrs, mltools, mmaqshiny, mnem, mobileR, MOCHA, modACDC, ModCon, ModelMetrics, modelsummary, MoleculeExperiment, MouseFM, MPAC, mpactr, MplusAutomation, mpwR, mrMLM, mrMLM.GUI, mrregression, MsBackendSql, mschart, msImpute, mslp, MSmix, MSnID, MSstats, MSstatsConvert, mstate, mstclustering, mstrio, mstudentd, mtb, mulea, multicrispr, multifamm, MultiFit, multiHiCcompare, multilevelcoda, multilevelTools, multistateQTL, multiWGC, MungeSumstats, muscat, musica, musicatk, mutationtypes, mutualinf, mvQuad, mvSUSY, mxnorm, myClim, naaccr, naptanr, nc, ndjson, neatR, neatRanges, neatStats, neo4r, NEONiso, neonOS, neonPlantEcology, neonUtilities, nestedcv, net4pg, netgsa, NetLogoR, netSmooth, networkR, netZooR, neuroimaGene, nflfastR, nflplotR, nflreadr, nflseedR, NFLSimulatoR, nlmixr2extra, nlpred, NMcalc, NMdata, NMsim, nngeo, NNS, noaastormevents, noctua, node2vec, noncompliance, nonmem2rx, normfluodbf, nosoi, npcurePK, nRegression, nullranges, numbat, o2geosocial, OBIC, obliqueRSF, occupationMeasurement, octad, octopus, oddsapiR, ofpetrial, ogrdbstats, OGRE, okxAPI, OlinkAnalyze, OmicNavigator, OMICsPCA, omicwas, ompr, OnboardClient, OncoSimulR, ondisc, One4All, openCyto, opendataformat, openEBGM, OpenML, OpenSpecy, opentripplanner, optiSel, ORFhunteR, ORFik, orgR, origami, orthogene, osmclass, OUTRIDER, overviewR, owdbr, owidR, paar, packageRank, pairkat, PakPMICS2014Ch, PakPMICS2014HH, PakPMICS2014HL, PakPMICS2014Wm, PAMpal, PAMscapes, panelaggregation, PanelMatch, panelWranglR, panstarrs, paradox, ParBayesianOptimization, parseRPDR, parSim, partCNV, partools, pathviewr, pathwayTMB, patternator, pbox, pcadapt, PCBS, PCMBase, PCMBaseCpp, PCRA, pcvr, pda, pdynmc, PeakSegDisk, Pedixplorer, pedquant, penaltyLearning, PepMapViz, pepr, pepStat, PGRdup, pgTools, phantasus, phantasusLite, PharmacoGx, pheble, phenofit, phenomis, PhenotypeSimulator, phers, phm, phonics, PhyloProfile, phyloseq, pingers, pinyin, pivottabler, pk4adi, pkggraph, pkgnet, PKPDsim, pksensi, PlanetNICFI, plinkQC, plmmr, plnr, plotgardener, plotly, pmartR, pmd, pmparser, pmxpartab, pointdensityP, poisFErobust, politeness, polle, polmineR, polypharmacy, pomp, POMS, poolfstat, pooling, popEpi, portvine, posologyr, POSSA, potools, POUMM, power.transform, powerbiR, ppmlasso, PPtreeregViz, pram, pRecipe, precrec, predhy.GUI, prediction, PReMiuM, PreProcessRecordLinkage, preputils, PressPurt, previsionio, pricelevels, proActiv, processmapR, processpredictR, prodlim, prolific.api, PRONE, PROSPER, Prostar, protGear, protti, prrd, prt, psborrow, PSF, psichomics, psidR, psmineR, psp, psycCleaning, PsychWordVec, ptable, ptairMS, pTITAN2, PTXQC, Publish, PureCN, puremoe, pureseqtmr, pvda, Pviz, pvLRT, qad, QCAcluster, qckitfastq, qdapTools, qgg, qreport, qte, QTL.gCIMapping, QTL.gCIMapping.GUI, qtl2, quickcheck, quickPlot, r2dii.match, R2DT, r2glmm, R3CPET, r3Cseq, r5r, R62S3, R6causal, RABR, RadEro, RadioGx, radlibs, RALSA, rAmCharts, RaMS, randnet, randomForestExplainer, randomNames, rangeMapper, RankAggSIgFUR, RapidoPGS, RARfreq, rassta, rasterDT, RAthena, ratios, Ravages, RawHummus, rbacon, RBaseX, rBayesianOptimization, rbi, rbi.helpers, rBiasCorrection, rbin, rBiopaxParser, rblt, Rborist, rbtt, Rcan, RCAS, RcextTools, rchemo, RcisTarget, RcisTarget.hg19.motifDBs.cisbpOnly.500bp, RcmdrPlugin.RiskDemo, rCNV, RcppGreedySetCover, rcprd, Rcrawler, rdbnomics, rdfp, Rdiagnosislist, Rdice, RDML, rdracor, rE, readNSx, readtext, realTimeloads, reappraised, rechaRge, reclin2, recorder, RecordLinkage, recount3, Rediscover, RefBasedMI, RegParallel, regport, regtools, ReinforcementLearning, reinsureR, remiod, REndo, replacer, repmis, reproducible, Require, rerddap, resemble, reslr, RESOLVE, respR, restatapi, ReSurv, rfaRm, rfars, rfisheries, rfPred, RFpredInterval, rgbif, rhdf5client, RIbench, RiboCrypt, RiboDiPA, RiboProfiling, ricu, Rilostat, RImmPort, rintcal, rio, rIsing, riskRegression, RITCH, ritis, rlas, rlist, rllama, RLumShiny, RMassBank, rmcfs, rMIDAS, RNewsflow, rnpn, Rnvd3, roads, robCompositions, RobinCar, robis, robotoolbox, ROCket, rODE, rollinglda, rorcid, rPackedBar, rprev, rqdatatable, rqPen, Rrepest, RSauceLabs, rSPARCS, rsparse, rsurveycto, rsyntax, RTCGA, RTCGAToolbox, rtmpt, RTN, RTNsurvival, rtry, rtsdata, RTTWebClient, runcharter, rush, rusquant, RVA, RWDataPlyr, rWikiPathways, RWildbook, rworkflows, RxnSim, rxode2, ryandexdirect, rzentra, RZooRoH, s3fs, SafeVote, salesforcer, samadb, sampcompR, samplingin, sangeranalyseR, sanityTracker, sansa, SARC, sarp.snowprofile, sarp.snowprofile.alignment, sarp.snowprofile.pyface, saseR, SC2API, scanMiR, scanMiRApp, scATAC.Explorer, scCustomize, SCdeconR, scDiffCom, scifer, scMET, scMitoMut, SCnorm, scoper, scorecard, ScoreEB, scoringutils, scPipe, scraEP, scRseqApp, scruff, scTreeViz, sdcHierarchies, sdcLog, sdcMicro, sdcTable, SDLfilter, SEA, seAMLess, SeaVal, seedr, seeker, segregation, SELF, semgram, semtree, sendigR, sense, sensobol, sentiment.ai, sentimentr, SenTinMixt, sentometrics, sentopics, seqsetvis, SeqVarTools, servosphereR, SEtools, sevenbridges, sevenbridges2, sevenC, sgat, SGP, SGPdata, SHAPforxgboost, shapr, shinyChatR, shinyepico, shinyExprPortal, shinyHugePlot, shinylogs, shinyML, ShinyQuickStarter, shinySbm, shinyWGD, siconvr, sigminer, Signac, signatureSearch, siie, SimBu, simcausal, simDAG, SimEngine, simfinapi, simitation, simPH, simphony, simplePHENOTYPES, simPop, simstudy, SimSurvey, simtrial, SingleCellSignalR, singleCellTK, SIPmg, sitepickR, siteymlgen, sitree, sitreeE, sjtable2df, skm, sleepr, slim, smallstuff, SmartEDA, smartsnp, smoothic, snapcount, SNPannotator, SNPhood, snplinkage, snpsettest, socialh, socialmixr, soilDB, SoilHyP, SoilTaxonomy, soiltestcorr, solaR2, solitude, SOMnmR, somspace, soundgen, spacyr, SpaDES.core, SpaDES.tools, sparrow, sparseFLMM, SPARSEMODr, SparseSignatures, spaths, SpatialCPie, SpatialFeatureExperiment, spatialHeatmap, SpatialOmicsOverlay, spatialrisk, SpATS, spatsoc, spDates, speaq, spectr, spectralAnalysis, spellcheckr, spicyR, SpliceWiz, splitstackshape, splutil, spocc, SPONGE, sport, sqlHelpers, SQMtools, squallms, squid, SSHAARP, staggered, standartox, staRdom, statar, statcanR, statgenGWAS, statgenIBD, statgenMPP, statgenQTLxT, Statial, Statsomat, stdReg, stdReg2, stenR, SticsRFiles, stm, stopdetection, stplanr, stppSim, strvalidator, subscreen, subSeq, SUMMER, suncalc, SUNGEO, SunsVoc, superml, surveyplanning, surveyPrev, surveysd, survMisc, susographql, SVMDO, SVN, SWATH2stats, sweater, SWMPr, symphony, synlet, synr, synthACS, table.express, tablecompare, tabledown, tableMatrix, tabulator, tabxplor, takos, TangledFeatures, targeted, targets, taskscheduleR, tatoo, taxonomizr, TCA, TCGAbiolinks, tcgaViz, tcpl, tdsc, TeXCheckR, text2vec, textclean, textplot, textpress, textrank, textshape, textTinyR, textTools, TheOpenAIR, TideCurves, TideTables, tidyDenovix, TidyDensity, tidyfast, tidyfst, tidyft, tidygapminder, tidyplate, tidyrules, tidytable, tidyterra, tidytransit, tidyvpc, timeplyr, timeseriesdb, TIN, TitanC, tLOH, TMSig, TnT, toolStability, topdowntimeratio, tosca, totalcensus, ToxicoGx, toxSummary, TPAC, TPACData, tpm, TPP, tracee, trackdf, tradestatistics, TrafficBDE, trainsplit, transcriptogramer, transport, traveltimeR, treediff, treekoR, treemap, TreeMineR, treeshap, triact, TrialEmulation, tRigon, trimmer, tripr, TrumpetPlots, tsbox, tsdistributions, tsgarch, tsmethods, TSstudio, tstests, tstools, TTAinterfaceTrendAnalysis, ttgsea, Tushare, tvtools, twang, twangRDC, twl, twopartm, txshift, uavRmp, UCell, uci, udpipe, ugatsdb, UKB.COVID19, ukbnmr, ukbtools, UKgrid, Ularcirc, Unico, Uniquorn, uptasticsearch, utilsIPEA, validata, VanillaICE, vardpoor, vein, versioning, versus, VIM, vimp, visualpred, vivainsights, VMDecomp, VOSONDash, vosonSML, vote, votesys, VoxR, vvtableau, walkboutr, wally, weatherOz, WeatherSentiment, WebAnalytics, webtrackR, wehoop, WGScan, WhatsR, whSample, wikitaxa, wilson, word.alignment, worrms, wpa, wpp2017, wpp2019, wTO, wyz.code.metaTesting, wyz.code.offensiveProgramming, wyz.code.rdoc, wyz.code.testthat, xefun, xesreadR, xgb2sql, xgboost, XString, yamss, youngSwimmers, zebu, zeitgebr, ZIprop, zoltr, zoomGroupStats","","","rotor","","robotoolbox, rolap, srppp, when","","","EGRETci","","","","","","","","","","CausalGPS, CvmortalityMult, flexmixNL, logmult, MetaCycle, multgee, perARMA, prefmod, SCCS, StMoMo, vcdExtra, VFP, VIRF","","BRVM, casabourse, gfer, odk","asbio, extremevalues, gWidgets2tcltk, ie2misc, migui, plfMA, relMix, strvalidator","","SIPmg","aebdata, airnow, AIscreenR, AnVILAz, AnVILBase, appsheet, arcgisgeocode, arcgislayers, arcgisutils, askgpt, atrrr, bbk, BFS, BiocCheck, biomaRt, brickster, BRVM, bskyr, bugphyzz, CatastRo, CDSE, chattr, climaemet, clinicalomicsdbR, comtradr, congress, conrad, consibiocloudclient, CopernicusMarine, cvdprevent, databraryr, dataonderivatives, dawaR, depmap, dhis2r, dpkg, dsmSearch, dtmapi, dtreg, ECOTOXr, envstat, eurostat, f1dataR, faers, fanyi, feltr, fenr, fitzRoy, FlickrAPI, foghorn, g3viz, galah, gdldata, gemini.R, genesysr, GEOquery, gh, GIFT, GitStats, govinfoR, gptstudio, gptzeror, gtexr, gypsum, happign, hdar, healthatlas, healthyR.data, HoloFoodR, httptest2, HuBMAPR, hyd1d, hydflood, khisr, kisopenapi, kiwisR, KoboconnectR, lehdr, link, LLMR, locateip, lulab.utils, mapme.biodiversity, mergenstudio, mgi.report.reader, microinverterdata, MolgenisArmadillo, mregions2, mskcc.oncotree, N2H4, nixtlar, ntdr, ntfy, octopusR, ollamar, openaistream, openeo, openFDA, osmapiR, osmdata, peermodels, photon, pkgdown, planscorer, promptr, proverbs, PurpleAir, pysparklyr, QBMS, qbr, racademyocean, Rapi, raqs, RAQSAPI, Rcpi, readMDTable, remmy, resmush, restatis, riem, Rigma, ringostat, rirods, rlinkedinads, RMassBank, rollama, rols, ropenmeteo, rosv, rqti, rtgstat, rtweet, selenium, sg, shiny.telemetry, shinylive, simfinapi, sivirep, sotkanet, stortingscrape, susographql, tapLock, tidyhydat, tidyllm, tidyquant, tidywikidatar, tRdbImport, treasury, trud, ucimlrepo, umbridge, vcr, webdav, whatthreewords, wikkitidy, worldbank, zendown","adegenet, adproclus, AFM, AHPWR, akc, alakazam, AlphaBeta, ANF, AnimalHabitatNetwork, AnimalSequences, anipaths, aniS, aPEAR, apisensr, archeofrag, arlclustering, arulesViz, ASpli, AurieLSHGaussian, autoharp, autostats, backbone, backShift, BallMapper, bamm, Banksy, bartMan, BASiNET, BASiNETEntropy, basket, batchelor, baycn, BayesNetBP, BayesSUR, bbnet, bc3net, bcgam, BCT, BDgraph, BeeBDC, BGmisc, bibliometrix, bigergm, BiocPkgTools, Bioi, BioM2, BioNERO, BioNet, BioR, bioregion, Bios2cor, BioTIP, bipartite, BiRewire, BlockmodelingGUI, bluster, bnma, bnmonitor, BNSL, bnstruct, bnviewer, BoolNet, bootcluster, bootnet, Boptbd, BoundaryStats, brainGraph, btergm, bwsTools, c3net, C443, CAMERA, cancerGI, Canek, cape, CARBayes, CARNIVAL, cartograflow, Cascade, cassowaryr, CatsCradle, causact, causalDisco, causaleffect, causaloptim, CausalR, CBNplot, cccd, CCPlotR, ccTensor, CellNOptR, CellTrails, CEMiTool, centiserve, CePa, ceRnetsim, CeRSeek, CeTF, cglasso, chatAI4R, checked, chemodiv, cholera, CHRONOS, cia, cicero, CIMICE, CIN, CiteFuse, CITMIC, CKMRpop, clevRvis, clickstream, clipper, cliqueMS, CliquePercolation, clustAnalytics, ClustAssess, ClusterFoldSimilarity, clusterProfiler, ClustIRR, clustNet, ClusTorus, clustree, clustringr, CoDi, cogeqc, comato, communication, concatipede, concorR, condiments, CoNI, ConnectednessApproach, conos, ConsensusClustering, CooRTweet, corclass, coreCT, corHMM, corkscrew, corpustools, corrViz, corTest, cosmosR, covEB, crandep, cranly, criticalpath, crosstalkr, csurvey, CTD, CTDquerier, CurricularAnalytics, cytometree, DART, dc3net, dcanr, dce, DCLEAR, DDP, deaR, debrowser, degreenet, delayed, DemoKin, dendroNetwork, DEsubs, dexterMST, DiagrammeR, Diderot, diffcoexp, DiffCorr, diffee, diffeRenTES, DiffNet, diffudist, diffusionMap, diffuStats, DIscBIO, discourseGT, disparityfilter, distinctiveness, dm, DmelSGI, DNLC, dnr, dogesr, dominoSignal, doolkit, drake, DrDimont, DrugSim2DR, DRviaSPCN, dsem, DTSEA, DuplexDiscovereR, dupNodes, dynetNLAResistance, dyngen, dynwrap, easybgm, ebdbNet, ecoCopula, ecodist, ECoL, econet, EcoNetGen, econetwork, economiccomplexity, EcotoneFinder, edgebundle, edgebundleR, editrules, edmcr, EDOIF, eff2, EGAD, EGAnet, eHDPrep, einet, EmbedSOM, EnMCB, enrichplot, enviGCMS, epiCo, epicontacts, EpiILMCT, epiNEM, epinetr, epiregulon.extra, epistasisGA, erah, ERPM, esATAC, espadon, ess, EstimateGroupNetwork, etree, EventPointer, evolqg, FactorCopula, fairadapt, fake, fakemake, FamAgg, fanovaGraph, fasjem, fastnet, fastRG, fbnet, FCMapper, FedData, FELLA, fgga, FGNet, FindIt, finnsurveytext, FLAMES, flare, Fletcher2013b, flowGraph, FlowSOM, fnets, foodingraph, ForestGapR, forestRK, fossilbrush, FrF2, fssemR, func2vis, GADAG, GALLO, gamCopula, gatom, gdistance, GeDi, gemtc, genBaRcode, GeneGeneInteR, genekitr, geneplast, geneplast.data, GENESIS, GeNetIt, GeneTonic, genlasso, GenomicInteractions, geomeTriD, geonetwork, GephiForR, ggdag, ggenealogy, ggflowchart, ggkegg, ggm, ggnetwork, ggraph, ggsem, ggtangle, gimme, GISSB, GMPro, GNET2, goat, GOCompare, GoodFitSBM, gor, GOxploreR, GR, gRain, grainscape, grand, GRaNIE, graph4lg, graphclust, graphicalExtremes, graphicalVAR, graphkernels, graphlayouts, graphsim, graphTweets, gRbase, gRc, GREMLINS, gRim, gromovlab, GSAR, GSD, gtexture, GWE, habCluster, handwriter, heterocop, HeteroGGM, heteromixgm, hglasso, highcharter, hipathia, hisse, HOasso, Holomics, HospitalNetwork, HPiP, HTSSIP, HTT, huge, hypeR, HyperG, iCARH, ICDS, iCellR, icmstate, icosa, iDINGO, IDMIR, idopNetwork, IDSpatialStats, ig.degree.betweenness, igraphinshiny, iGraphMatch, igraphtosonia, igraphwalshdata, ILSM, imager, imcRtools, IMEC, IMMAN, immcp, immunarch, immunaut, immuneSIM, imsig, incidentally, INDEED, iNETgrate, infercnv, influenceR, influential, intensitynet, InterCellar, intergraph, interplex, IntramiRExploreR, iPRISM, IR, iSEE, iSEEfier, ITNr, JANE, Jdmbs, jeek, jewel, JGL, jrSiCKLSNMF, jti, kangar00, Karen, KEGGlincs, keyplayer, kgraph, kknn, KMD, kstMatrix, L1centrality, l1spectral, LACE, latenetwork, lavaangui, lconnect, leastcostpath, leiden, leidenAlg, leidenbase, levi, lexRankr, linkcomm, linkprediction, Linnorm, LncPath, locits, LogisticCopula, LoopDetectR, LorMe, LPKsample, LSVAR, LTFHPlus, macrosyntR, MadanTextNetwork, madrat, MAGAR, malan, manet, manymome, manynet, MAPFX, mappeR, margaret, markovchain, martini, massiveGST, matrixcut, mau, maxmatching, mazeGen, MBNMAdose, mcMST, mcvis, meconetcomp, MEGE, Mercator, MetaboSignal, MetaLandSim, MetaNeighbor, MetaNet, metanetwork, MetID, MetricGraph, mfpp, microeco, micropan, miloR, MineICA, miniCRAN, miRSM, miRspongeR, missSBM, mitre, mixOmics, MLCOPULA, mlmts, mmibain, MMOC, mnda, MNS, MODA, modelbpp, modMax, modnets, MOGAMUN, monocle, moranajp, morph, mosbi, MOSClip, motifcluster, motifr, movementsync, MPAC, mppR, MRFcov, mRMRe, MRReg, MRS, MSCquartets, mstknnclust, MTA, mully, multichull, multilaterals, multilink, multinet, multinets, multinma, MultIS, multivariance, multiWGC, mumosa, MWASTools, mwcsr, mwTensor, mycaas, nat, nat.templatebrains, nda, neat, neatmaps, Neighboot, neo4r, NEpiC, netassoc, netCoin, netcom, NetCoupler, netdiffuseR, NetFACS, netgsa, netgwas, netmap, NetMix, NetOrigin, NetPathMiner, netplot, netrankr, netresponse, netropy, nets, NetSAM, NetSci, netseer, netseg, netShiny, NetSwan, netUtils, NetworkChange, NetworkComparr, networkD3, NetworkDistance, NetworkExtinction, networkGen, NetworkToolbox, networktools, netZooR, nevada, NIMAA, nimble, nlnet, nlrx, nncc, node2vec, noisysbmGGM, NoRCE, numbat, nutriNetwork, OCNet, oddnet, ohun, OmnipathR, OncoSimulR, ontoProc, OpenRepGrid, OpenRepGrid.ic, oposSOM, optbdmaeAT, optrcdmaeAT, OrdCD, ORION, orthGS, OUwie, PAC, PACEA, PAFit, pageRank, pagoda2, pairkat, pald, pandaR, PanomiR, PANR, parsec, particles, Path2PPI, pathfindR, pathlinkR, PathwaySpace, Patterns, pcalg, pcFactorStan, PCGII, pcSteiner, pcvr, pedtricks, pencopulaCond, perturbR, PGRdup, pGRN, phangorn, PhaseTypeR, PhenoGeneRanker, phenomis, PhosR, phylepic, phylopath, phyloregion, phyloseq, phyloseqGraphTest, phylosignal, phyloTop, piano, piecewiseSEM, PIUMA, pkggraph, pkgnet, PlackettLuce, plantTracker, PLEXI, plinkQC, PLNmodels, pmd, polymapR, PolyTree, pomdp, pop, poppr, POSetR, postHoc, PPInfer, PR, priorCON, prioritizr, ProgModule, ProjectManagement, provGraphR, PSMatch, psSubpathway, ptools, PUGMM, PWIR, qdap, QFeatures, qgraph, qmtools, qtlnet, quanteda.textplots, R.temis, R3CPET, R6causal, RaceID, Racmacs, Radviz, rags2ridges, RavenR, RCA, rcausim, RcextTools, RcmdrPlugin.RMTCJags, rcrimeanalysis, RCSL, RCX, rD3plot, rdracor, RDS, ReactomePA, RedeR, regmed, regnet, ReliabilityTheory, remify, rEMM, RepertoiR, ResIN, RevEcoR, rflexscan, rgexf, RGraphSpace, rhcoclust, RITAN, riverconn, RLassoCox, rmcfs, rnaCrosslinkOO, RNetLogo, RNewsflow, rnmamod, Rnmr1D, roads, robber, robin, RobustIV, robustrao, roots, roughnet, RPANDA, rPref, rscc, rScudo, rSDI, rsemmed, RseqNet, rsetse, RSP, rSpectral, rsppfp, rsyntax, RTN, rTRM, rTwig, RWgraph, sand, SANTA, saotd, SARP.compo, saseR, satdad, SBGNview, sBIC, sbm, SBN, scalpel, scapGNN, scClassify, sccore, scDblFinder, scDHA, SCFA, scHOT, scider, scINSIGHT, scistreer, scMerge, scMultiSim, scoredec, ScorePlus, SCORPION, scran, scrapper, scRepertoire, scRtools, scTensor, scTGIF, scTreeViz, secrlinear, SelectBoost, SEMdeep, SEMgraph, SEMID, SemNeT, semPlot, seqcombo, SeqDetect, seqHMM, SeqNet, Seurat, sfcr, sfnetworks, SFSI, sglasso, sgraph, SGSeq, ShapePattern, sharp, shazam, shock, shp2graph, SID, sidier, sigmajs, SignacX, signnet, simcausal, simDAG, simdata, simona, simplifyNet, simule, sincell, sindyr, SingleCellSignalR, singleCellTK, SIRE, slingshot, smallstuff, SmCCNet, SMITE, smotefamily, snahelper, snowboot, SOAs, SoilR, solitude, SOMbrero, SOMMD, soptdmaeA, spacejamr, SpaDES.core, Spaniel, SpatialCPie, SpatialGraph, spatialHeatmap, spatsoc, specr, spinner, SplicingGraphs, splineTimeR, SPONGE, spreadr, ssifs, ssMutPA, SSNbler, sSPPY, StabMap, statGraph, stemmatology, stminsights, STraTUS, streamDAG, STRINGdb, SubpathwayLNCE, SubtypeDrug, SuperCell, SurvHiDim, SVN, synaptome.db, syntenet, SystemicR, TangledFeatures, targets, TAShiny, TDA, ternarynet, text2map, textrank, threejs, tidygraph, tidySEM, tilemaps, timeordered, tip, Tlasso, tnet, TOHM, TOP, topologyGSA, tosr, TPEA, tradeSeq, TrajectoryUtils, transcriptogramer, TreeAndLeaf, TreeDimensionTest, treefit, treemap, TRONCO, TSCAN, UNCOVER, uSORT, VARDetect, veloviz, VertexSimilarity, VertexWiseR, vissE, vivainsights, vivid, VOSONDash, WayFindR, wdnet, WebGestaltR, webSDM, wfg, whitechapelR, whomds, wpa, wppi, wTO, XI, xLLiM, ZygosityPredictor","","","","","","","","BasketballAnalyzeR, Bergm, BGGM, bigergm, bingat, BlockmodelingGUI, bootcluster, btergm, calmr, CEMiTool, CeTF, CIN, CluMSID, ClusTCR2, collpcm, conserveR, degreenet, discourseGT, dnr, EGAnet, EloRating, EpiModel, epinet, ergm, ergm.count, ergm.ego, ergm.multi, ergm.rank, ergMargins, ergmgp, ergmito, fergm, fglsnet, gatom, genBaRcode, GGMncv, GGMnonreg, ggnetwork, GGPA, GOxploreR, GUEST, haplotypes, InflectSSP, intergraph, interplex, ITNr, latentnet, Linkage, lolog, manynet, metanetwork, mlergm, motifr, MRPC, ndtv, netdiffuseR, nethet, netmap, netmediate, netplot, networkABC, NetworkChange, NetworkDistance, networkDynamic, networkDynamicData, NetworkExtinction, networkLite, PAFit, PDN, PhosR, pkggraph, quanteda.textplots, RDS, RHMS, SBICgraph, sidier, sna, SparseTSCGM, statnet, statnetWeb, tergm, tergmLite, tip, tsna, VBLPCM, WOTPLY, WRSS, xergm.common","EpiModel, ergmgp, ITNr, ndtv, netdiffuseR, networkDynamicData, PAFit, statnet, tergm, tergmLite, tsna","","","","","AssetAllocation, ConnectednessApproach, facmodCS, facmodTS, HeckmanEM, highOrderPortfolios, methylclock, OMICsPCA, PCRA, pedquant, PortfolioAnalytics, portfolioBacktest, RMOPI, rmsfuns, RTL, scRtools, Semblance, SlidingWindows, SMNCensReg, tidyquant","","adestr, AIPW, ale, antaresEditObject, ao, basksim, bayesmove, beer, bolasso, calmr, campsis, canaper, cccrm, collinear, crumble, cSEM, cyclestreets, dapper, deseats, dipsaus, drugdevelopR, easyalluvial, ecic, econet, EFAtools, EGAnet, elevatr, EpiForsk, epwshiftr, fabletools, fastRhockey, fdacluster, funGp, futureverse, fxTWAPLS, geocausal, geocmeans, GeoModels, gtfs2gps, hbamr, hoopR, JANE, kmeRtone, lava, lightr, lmtp, LWFBrook90R, mcmcensemble, metasnf, modeltime.resample, nflfastR, nflseedR, nlrx, opentripplanner, parseRPDR, pavo, polle, poppr, powRICLPM, PWIR, rainette, rangeMapper, readsdr, receptiviti, rechaRge, remiod, restriktor, saeczi, Seurat, SeuratObject, simaerep, SimDesign, skpr, smoots, squat, stppSim, targeted, tidyfit, tidySEM, tsdistributions, tsgarch, vital, vmeasur, wehoop, WeightedCluster, ycevo","","","abc, ACE.CoCo, aftsem, alphaOutlier, asm, atRisk, batchtma, bgumbel, binsreg, boostrq, BwQuant, car, caROC, chipPCR, cluscov, ClusPred, cmcR, cmprskQR, cobs, Compositional, ConnectednessApproach, Counterfactual, cqn, cqrReg, crs, ddpca, diagL1, diveMove, EdSurvey, emplik, erah, esreg, EXRQ, fairadapt, ForecastComb, ForecastCombinations, gapfill, geeVerse, GEInter, georob, ggpmisc, ggquickeda, gJLS2, HDRFA, hettx, HIMA, hrqglas, invacost, iNZightPlots, JWileymisc, locpolExpectile, lqmix, lqr, lss2, marlod, matrisk, MCMCpack, MECfda, MendelianRandomization, midasr, MiRKAT, modeLLtest, MRZero, MTE, multid, MultiKink, mvctm, MXM, NCA, newIMVC, nlmeVPC, np, NSM3, OPTtesting, pcvr, PDMIF, perryExamples, pez, pfa, pheno, plaqr, plotluck, profExtrema, pwr2ppl, qcauchyreg, qcpm, qfa, QRank, QregBB, QRegVCM, QRIpkg, qris, qrjoint, qrLMM, qrmix, qrNLMM, qte, QTE.RD, QTOCen, Qtools, quantCurves, quantdr, quantilogram, quantoptr, quantreg.nonpar, quantregGrowth, quantsmooth, quantspec, RATest, Rearrangement, reverseR, riv, rlme, rms, robmed, RobPer, RPPanalyzer, RPPASPACE, rqcanon, rqPen, rrat, scde, SCnorm, scRepertoire, SeBR, SGP, SIM, SimplyAgree, siqr, smoothAPC, sn, SortedEffects, SpatialRegimes, STMr, stR, subsampling, SystemicR, tidyvpc, TSrepr, tsrobprep, TTCA, turboEM, unitquantreg, vdg, weightQuant, WRTDStidal","BKTR, PTXQC, rmzqc","","","","","","","edmcr, PlaneGeometry","","","","","","","","","Certara.VPCResults","","academictwitteR, anyflights, autoMFA, badger, BiocBook, biocthis, BiocWorkflowTools, bootCT, buildr, cmdfun, covidsymptom, DataPackageR, DecomposeR, depigner, devtools, distops, do, dockerfiler, doolkit, easysurv, ensembleTax, epidatr, etl, exampletestr, fabR, FastUtils, fec16, fledge, flipr, fusen, ggseqplot, gitgadget, googleAnalyticsR, googleCloudRunner, gripp, hgutils, hlidacr, hoopR, HubPub, iheiddown, IRTest, k5, latte, lehuynh, leprechaun, m2r, manydata, margaret, memes, memoiR, metabolic, mnda, moodleR, multilinguer, natmanager, nettskjemar, OCSdata, oeli, ontoFAST, OpenMindat, packagepal, packer, pathlit, personalr, phrases, pkgverse, plnr, pocketapi, polite, popbayes, populR, RadEro, rcompendium, rDataPipeline, readme2vignette, Revticulate, rfold, roxygen2md, rsf, rUM, sendgridr, SIAtools, SomaDataIO, surveydown, testthis, tidytuesdayR, trainR, Trendtwosub, understandBPMN, vprr, washr, wehoop, whitewater, worcs, ymlthis","",""]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>deadline<\/th>\n      <th>package<\/th>\n      <th>version<\/th>\n      <th>maintainer<\/th>\n      <th>links<\/th>\n      <th>event<\/th>\n      <th>reverse packages (n)<\/th>\n      <th>reverse packages<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":50,"order":[[0,"asc"],[6,"desc"],[1,"asc"]],"columnDefs":[{"className":"dt-right","targets":6},{"name":"deadline","targets":0},{"name":"package","targets":1},{"name":"version","targets":2},{"name":"maintainer","targets":3},{"name":"links","targets":4},{"name":"event","targets":5},{"name":"reverse packages (n)","targets":6},{"name":"reverse packages","targets":7}],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>




<div id="quarto-appendix" class="default"><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><h2 class="anchored quarto-appendix-heading">Footnotes</h2>

<ol>
<li id="fn1"><p>When a package is <em>archived</em>, the archived version is still available in the <a href="https://cran.r-project.org/src/contrib/Archive/">CRAN archive area</a> together with all previous versions.<a href="#fnref1" class="footnote-back" role="doc-backlink">â†©ï¸Ž</a></p></li>
</ol>
</section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "î§‹";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




<footer class="footer"><div class="nav-footer"><div class="nav-footer-center"><div class="toc-actions"><ul><li><a href="https://github.com/cranhaven/cranhaven.org/blob/main/dashboard-at-risk.qmd" class="toc-action"><i class="bi bi-github"></i>View source</a></li><li><a href="https://github.com/cranhaven/cranhaven.org/edit/main/dashboard-at-risk.qmd" class="toc-action"><i class="bi empty"></i>Edit this page</a></li><li><a href="https://github.com/cranhaven/cranhaven.org/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></div></div></footer></body></html>