<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.42">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Dashboard: Packages At-Risk of Being Archived – CRANhaven</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-2f5df379a58b258e96c21c0638c20c03.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-8647a4a42273f773479d27c00df3f9ed.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">CRANhaven</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-articles" role="link" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Articles</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-articles">    
        <li>
    <a class="dropdown-item" href="./dashboard-live.html">
 <span class="dropdown-text">Dashboard: Recently Archived CRAN Packages</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./dashboard-at-risk.html">
 <span class="dropdown-text">Dashboard: Packages At-Risk of Being Archived</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./cran-archiving-stats.html">
 <span class="dropdown-text">Study: Many Archived Packages Return to CRAN</span></a>
  </li>  
    </ul>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">About</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/cranhaven/cranhaven.org/"> <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar"><div class="quarto-margin-header"><div class="margin-header-item">
<p><a href="https://cranhaven.r-universe.dev/builds"><img src="https://cranhaven.r-universe.dev/badges/:total" alt="CRANhaven number of packages"></a> <a href="https://github.com/CRANhaven/cranhaven.r-universe.dev/actions/workflows/update.yml"><img src="https://github.com/CRANhaven/cranhaven.r-universe.dev/actions/workflows/update.yml/badge.svg" alt=""></a></p>
</div></div>
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Dashboard: Packages At-Risk of Being Archived</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<style>
tr { vertical-align: top; }
</style>
<p>The below table lists 111 R packages that are at risk of being archived<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a> on CRAN, because they have outstanding unresolved issues.</p>
<p>Some of them might have already submitted a new version to fix the issues identified by CRAN (check the event column).</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="live-dashboard" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="live-dashboard">{"x":{"filter":"none","vertical":false,"data":[["2025-04-05<br><small>(in 14 days)<\/small>","2025-04-09<br><small>(in 18 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-11<br><small>(in 20 days)<\/small>","2025-03-28<br><small>(in 6 days)<\/small>","2025-03-29<br><small>(in 7 days)<\/small>","2025-03-30<br><small>(in 8 days)<\/small>","2025-04-09<br><small>(in 18 days)<\/small>","2025-04-02<br><small>(in 11 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-03-26<br><small>(in 4 days)<\/small>","2025-04-02<br><small>(in 11 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-12<br><small>(in 21 days)<\/small>","2025-04-12<br><small>(in 21 days)<\/small>","2025-04-11<br><small>(in 20 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-03-29<br><small>(in 7 days)<\/small>","2025-03-29<br><small>(in 7 days)<\/small>","2025-03-29<br><small>(in 7 days)<\/small>","2025-04-02<br><small>(in 11 days)<\/small>","2025-04-09<br><small>(in 18 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-03-26<br><small>(in 4 days)<\/small>","2025-04-03<br><small>(in 12 days)<\/small>","2025-03-28<br><small>(in 6 days)<\/small>","2025-04-07<br><small>(in 16 days)<\/small>","2025-04-03<br><small>(in 12 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-03-31<br><small>(in 9 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-11<br><small>(in 20 days)<\/small>","2025-03-30<br><small>(in 8 days)<\/small>","2025-03-31<br><small>(in 9 days)<\/small>","2025-04-02<br><small>(in 11 days)<\/small>","2025-03-30<br><small>(in 8 days)<\/small>","2025-04-11<br><small>(in 20 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-03-24<br><small>(in 2 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-11<br><small>(in 20 days)<\/small>","2025-03-31<br><small>(in 9 days)<\/small>","2025-04-03<br><small>(in 12 days)<\/small>","2025-04-11<br><small>(in 20 days)<\/small>","2025-04-03<br><small>(in 12 days)<\/small>","2025-03-26<br><small>(in 4 days)<\/small>","2025-03-31<br><small>(in 9 days)<\/small>","2025-03-30<br><small>(in 8 days)<\/small>","2025-04-09<br><small>(in 18 days)<\/small>","2025-04-12<br><small>(in 21 days)<\/small>","2025-04-09<br><small>(in 18 days)<\/small>","2025-04-12<br><small>(in 21 days)<\/small>","2025-03-26<br><small>(in 4 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-03-27<br><small>(in 5 days)<\/small>","2025-04-03<br><small>(in 12 days)<\/small>","2025-03-31<br><small>(in 9 days)<\/small>","2025-03-28<br><small>(in 6 days)<\/small>","2025-04-03<br><small>(in 12 days)<\/small>","2025-03-29<br><small>(in 7 days)<\/small>","2025-04-09<br><small>(in 18 days)<\/small>","2025-04-12<br><small>(in 21 days)<\/small>","2025-04-03<br><small>(in 12 days)<\/small>","2025-04-12<br><small>(in 21 days)<\/small>","2025-04-12<br><small>(in 21 days)<\/small>","2025-04-12<br><small>(in 21 days)<\/small>","2025-04-12<br><small>(in 21 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-03-23<br><small>(in 1 day)<\/small>","2025-04-03<br><small>(in 12 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-11<br><small>(in 20 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-03<br><small>(in 12 days)<\/small>","2025-03-26<br><small>(in 4 days)<\/small>","2025-04-09<br><small>(in 18 days)<\/small>","2025-03-30<br><small>(in 8 days)<\/small>","2025-03-30<br><small>(in 8 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-01<br><small>(in 10 days)<\/small>","2025-04-11<br><small>(in 20 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-12<br><small>(in 21 days)<\/small>","2025-04-02<br><small>(in 11 days)<\/small>","2025-04-03<br><small>(in 12 days)<\/small>","2025-04-02<br><small>(in 11 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-04<br><small>(in 13 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-01<br><small>(in 10 days)<\/small>","2025-04-07<br><small>(in 16 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-03-31<br><small>(in 9 days)<\/small>","2025-04-02<br><small>(in 11 days)<\/small>","2025-04-09<br><small>(in 18 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-01<br><small>(in 10 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-03<br><small>(in 12 days)<\/small>","2025-03-30<br><small>(in 8 days)<\/small>","2025-04-12<br><small>(in 21 days)<\/small>","2025-03-31<br><small>(in 9 days)<\/small>","2025-03-29<br><small>(in 7 days)<\/small>","2025-03-29<br><small>(in 7 days)<\/small>","2025-03-31<br><small>(in 9 days)<\/small>","2025-04-05<br><small>(in 14 days)<\/small>","2025-04-11<br><small>(in 20 days)<\/small>"],["<a href=\"https://cran.r-project.org/package=adespatial\">adespatial<\/a>","<a href=\"https://cran.r-project.org/package=adlift\">adlift<\/a>","<a href=\"https://cran.r-project.org/package=AlgDesign\">AlgDesign<\/a>","<a href=\"https://cran.r-project.org/package=analogue\">analogue<\/a>","<a href=\"https://cran.r-project.org/package=anticlust\">anticlust<\/a>","<a href=\"https://cran.r-project.org/package=archeofrag\">archeofrag<\/a>","<a href=\"https://cran.r-project.org/package=bayesPop\">bayesPop<\/a>","<a href=\"https://cran.r-project.org/package=BonEV\">BonEV<\/a>","<a href=\"https://cran.r-project.org/package=brulee\">brulee<\/a>","<a href=\"https://cran.r-project.org/package=C50\">C50<\/a>","<a href=\"https://cran.r-project.org/package=CAESAR.Suite\">CAESAR.Suite<\/a>","<a href=\"https://cran.r-project.org/package=campsismod\">campsismod<\/a>","<a href=\"https://cran.r-project.org/package=cbcTools\">cbcTools<\/a>","<a href=\"https://cran.r-project.org/package=ClassDiscovery\">ClassDiscovery<\/a>","<a href=\"https://cran.r-project.org/package=CloneSeeker\">CloneSeeker<\/a>","<a href=\"https://cran.r-project.org/package=cocorresp\">cocorresp<\/a>","<a href=\"https://cran.r-project.org/package=CohensdpLibrary\">CohensdpLibrary<\/a>","<a href=\"https://cran.r-project.org/package=CohortSymmetry\">CohortSymmetry<\/a>","<a href=\"https://cran.r-project.org/package=coppeCosenzaR\">coppeCosenzaR<\/a>","<a href=\"https://cran.r-project.org/package=corncob\">corncob<\/a>","<a href=\"https://cran.r-project.org/package=crplyr\">crplyr<\/a>","<a href=\"https://cran.r-project.org/package=crunchy\">crunchy<\/a>","<a href=\"https://cran.r-project.org/package=Cubist\">Cubist<\/a>","<a href=\"https://cran.r-project.org/package=cxhull\">cxhull<\/a>","<a href=\"https://cran.r-project.org/package=DataCombine\">DataCombine<\/a>","<a href=\"https://cran.r-project.org/package=datawizard\">datawizard<\/a>","<a href=\"https://cran.r-project.org/package=datetimeoffset\">datetimeoffset<\/a>","<a href=\"https://cran.r-project.org/package=disprofas\">disprofas<\/a>","<a href=\"https://cran.r-project.org/package=distillML\">distillML<\/a>","<a href=\"https://cran.r-project.org/package=divvy\">divvy<\/a>","<a href=\"https://cran.r-project.org/package=dnapath\">dnapath<\/a>","<a href=\"https://cran.r-project.org/package=doRNG\">doRNG<\/a>","<a href=\"https://cran.r-project.org/package=easycsv\">easycsv<\/a>","<a href=\"https://cran.r-project.org/package=enmSdmX\">enmSdmX<\/a>","<a href=\"https://cran.r-project.org/package=fHMM\">fHMM<\/a>","<a href=\"https://cran.r-project.org/package=FIESTA\">FIESTA<\/a>","<a href=\"https://cran.r-project.org/package=FLASHMM\">FLASHMM<\/a>","<a href=\"https://cran.r-project.org/package=gaawr2\">gaawr2<\/a>","<a href=\"https://cran.r-project.org/package=gdm\">gdm<\/a>","<a href=\"https://cran.r-project.org/package=geodist\">geodist<\/a>","<a href=\"https://cran.r-project.org/package=geofi\">geofi<\/a>","<a href=\"https://cran.r-project.org/package=gie\">gie<\/a>","<a href=\"https://cran.r-project.org/package=glmmTMB\">glmmTMB<\/a>","<a href=\"https://cran.r-project.org/package=gmwmx2\">gmwmx2<\/a>","<a href=\"https://cran.r-project.org/package=hacksaw\">hacksaw<\/a>","<a href=\"https://cran.r-project.org/package=hacksig\">hacksig<\/a>","<a href=\"https://cran.r-project.org/package=happign\">happign<\/a>","<a href=\"https://cran.r-project.org/package=hilldiv\">hilldiv<\/a>","<a href=\"https://cran.r-project.org/package=hsphase\">hsphase<\/a>","<a href=\"https://cran.r-project.org/package=innsight\">innsight<\/a>","<a href=\"https://cran.r-project.org/package=io\">io<\/a>","<a href=\"https://cran.r-project.org/package=ISwR\">ISwR<\/a>","<a href=\"https://cran.r-project.org/package=jamba\">jamba<\/a>","<a href=\"https://cran.r-project.org/package=jiebaR\">jiebaR<\/a>","<a href=\"https://cran.r-project.org/package=leidenbase\">leidenbase<\/a>","<a href=\"https://cran.r-project.org/package=lme4\">lme4<\/a>","<a href=\"https://cran.r-project.org/package=magick\">magick<\/a>","<a href=\"https://cran.r-project.org/package=MethEvolSIM\">MethEvolSIM<\/a>","<a href=\"https://cran.r-project.org/package=odbc\">odbc<\/a>","<a href=\"https://cran.r-project.org/package=opendatatoronto\">opendatatoronto<\/a>","<a href=\"https://cran.r-project.org/package=outlierensembles\">outlierensembles<\/a>","<a href=\"https://cran.r-project.org/package=pak\">pak<\/a>","<a href=\"https://cran.r-project.org/package=partsm\">partsm<\/a>","<a href=\"https://cran.r-project.org/package=PKPDsim\">PKPDsim<\/a>","<a href=\"https://cran.r-project.org/package=Polychrome\">Polychrome<\/a>","<a href=\"https://cran.r-project.org/package=PreProcess\">PreProcess<\/a>","<a href=\"https://cran.r-project.org/package=proteus\">proteus<\/a>","<a href=\"https://cran.r-project.org/package=PRSim\">PRSim<\/a>","<a href=\"https://cran.r-project.org/package=PublicWorksFinanceIT\">PublicWorksFinanceIT<\/a>","<a href=\"https://cran.r-project.org/package=qdapRegex\">qdapRegex<\/a>","<a href=\"https://cran.r-project.org/package=R2BayesX\">R2BayesX<\/a>","<a href=\"https://cran.r-project.org/package=RAMClustR\">RAMClustR<\/a>","<a href=\"https://cran.r-project.org/package=rankrate\">rankrate<\/a>","<a href=\"https://cran.r-project.org/package=raster\">raster<\/a>","<a href=\"https://cran.r-project.org/package=Rcrawler\">Rcrawler<\/a>","<a href=\"https://cran.r-project.org/package=retistruct\">retistruct<\/a>","<a href=\"https://cran.r-project.org/package=Rfast2\">Rfast2<\/a>","<a href=\"https://cran.r-project.org/package=rim\">rim<\/a>","<a href=\"https://cran.r-project.org/package=RITCH\">RITCH<\/a>","<a href=\"https://cran.r-project.org/package=RJSONIO\">RJSONIO<\/a>","<a href=\"https://cran.r-project.org/package=rkriging\">rkriging<\/a>","<a href=\"https://cran.r-project.org/package=rlang\">rlang<\/a>","<a href=\"https://cran.r-project.org/package=RMSD\">RMSD<\/a>","<a href=\"https://cran.r-project.org/package=rmzqc\">rmzqc<\/a>","<a href=\"https://cran.r-project.org/package=ropenmeteo\">ropenmeteo<\/a>","<a href=\"https://cran.r-project.org/package=rsvg\">rsvg<\/a>","<a href=\"https://cran.r-project.org/package=rtweet\">rtweet<\/a>","<a href=\"https://cran.r-project.org/package=Rtwobitlib\">Rtwobitlib<\/a>","<a href=\"https://cran.r-project.org/package=sasr\">sasr<\/a>","<a href=\"https://cran.r-project.org/package=sdmTMB\">sdmTMB<\/a>","<a href=\"https://cran.r-project.org/package=sgat\">sgat<\/a>","<a href=\"https://cran.r-project.org/package=shapr\">shapr<\/a>","<a href=\"https://cran.r-project.org/package=skellam\">skellam<\/a>","<a href=\"https://cran.r-project.org/package=smooth\">smooth<\/a>","<a href=\"https://cran.r-project.org/package=SplitSoftening\">SplitSoftening<\/a>","<a href=\"https://cran.r-project.org/package=spqdep\">spqdep<\/a>","<a href=\"https://cran.r-project.org/package=stelfi\">stelfi<\/a>","<a href=\"https://cran.r-project.org/package=stringi\">stringi<\/a>","<a href=\"https://cran.r-project.org/package=SuppDists\">SuppDists<\/a>","<a href=\"https://cran.r-project.org/package=symengine\">symengine<\/a>","<a href=\"https://cran.r-project.org/package=tabnet\">tabnet<\/a>","<a href=\"https://cran.r-project.org/package=terra\">terra<\/a>","<a href=\"https://cran.r-project.org/package=tesseract\">tesseract<\/a>","<a href=\"https://cran.r-project.org/package=TestGenerator\">TestGenerator<\/a>","<a href=\"https://cran.r-project.org/package=timetools\">timetools<\/a>","<a href=\"https://cran.r-project.org/package=tinytable\">tinytable<\/a>","<a href=\"https://cran.r-project.org/package=Trading\">Trading<\/a>","<a href=\"https://cran.r-project.org/package=tswge\">tswge<\/a>","<a href=\"https://cran.r-project.org/package=vein\">vein<\/a>","<a href=\"https://cran.r-project.org/package=VineCopula\">VineCopula<\/a>","<a href=\"https://cran.r-project.org/package=winfapReader\">winfapReader<\/a>"],["0.3-26<br><small>(2025-03-21)<small>","1.4-5<br><small>(2023-03-22)<small>","1.2.1.1<br><small>(2024-09-21)<small>","0.17-7<br><small>(2024-09-21)<small>","0.8.10<br><small>(2025-03-13)<small>","1.1.0<br><small>(2025-03-07)<small>","11.0-2<br><small>(2025-02-22)<small>","1.0<br><small>(2016-02-13)<small>","0.4.0<br><small>(2025-01-30)<small>","0.1.8<br><small>(2023-02-08)<small>","0.2.0<br><small>(2025-03-03)<small>","1.2.1<br><small>(2025-03-17)<small>","0.5.0<br><small>(2023-07-12)<small>","3.4.5<br><small>(2024-04-28)<small>","1.0.11<br><small>(2022-06-30)<small>","0.4-5<br><small>(2024-10-06)<small>","0.5.11<br><small>(2024-08-22)<small>","0.2.3<br><small>(2025-03-01)<small>","0.1.3<br><small>(2017-10-28)<small>","0.4.1<br><small>(2024-01-10)<small>","0.4.2<br><small>(2025-03-18)<small>","0.3.3<br><small>(2021-01-13)<small>","0.4.4<br><small>(2024-07-02)<small>","0.7.4<br><small>(2023-10-24)<small>","0.2.21<br><small>(2016-04-13)<small>","1.0.1<br><small>(2025-03-07)<small>","0.4.1<br><small>(2025-01-11)<small>","0.2.0<br><small>(2024-06-10)<small>","0.1.0.13<br><small>(2023-03-25)<small>","1.0.0<br><small>(2023-10-26)<small>","0.7.6<br><small>(2025-02-21)<small>","1.8.6.1<br><small>(2025-01-13)<small>","1.0.8<br><small>(2018-05-21)<small>","1.2.10<br><small>(2024-12-11)<small>","1.4.1<br><small>(2024-09-16)<small>","3.7.0<br><small>(2025-02-04)<small>","1.1.0<br><small>(2025-03-11)<small>","0.0.2<br><small>(2025-02-21)<small>","1.6.0-6<br><small>(2025-01-31)<small>","0.1.0<br><small>(2024-05-23)<small>","1.0.18<br><small>(2025-02-11)<small>","0.1.2<br><small>(2024-06-02)<small>","1.1.10<br><small>(2024-09-26)<small>","0.0.1<br><small>(2025-03-10)<small>","0.0.2<br><small>(2020-12-15)<small>","0.1.2<br><small>(2022-02-17)<small>","0.3.2<br><small>(2025-01-24)<small>","1.5.1<br><small>(2019-10-01)<small>","2.0.3<br><small>(2024-02-16)<small>","0.3.1<br><small>(2024-11-26)<small>","0.3.2<br><small>(2019-12-17)<small>","2.0-9<br><small>(2024-10-10)<small>","1.0.2<br><small>(2025-03-10)<small>","0.11<br><small>(2019-12-13)<small>","0.1.32<br><small>(2025-02-06)<small>","1.1-36<br><small>(2025-01-11)<small>","2.8.5<br><small>(2024-09-20)<small>","0.2<br><small>(2025-03-17)<small>","1.6.0<br><small>(2025-02-28)<small>","0.1.5<br><small>(2022-04-13)<small>","0.1.2<br><small>(2025-02-22)<small>","0.8.0.1<br><small>(2025-01-16)<small>","1.1-3<br><small>(2020-11-25)<small>","1.4.0<br><small>(2024-08-19)<small>","1.5.1<br><small>(2022-05-03)<small>","3.1.7<br><small>(2019-05-06)<small>","1.1.4<br><small>(2023-10-21)<small>","1.5<br><small>(2024-04-08)<small>","0.3.1<br><small>(2024-12-12)<small>","0.7.8<br><small>(2023-10-17)<small>","1.1-5<br><small>(2023-10-20)<small>","1.3.1<br><small>(2023-10-20)<small>","1.2.0<br><small>(2023-08-17)<small>","3.6-31<br><small>(2025-01-16)<small>","0.1.9-1<br><small>(2018-11-11)<small>","0.8.0<br><small>(2025-03-10)<small>","0.1.5.2<br><small>(2024-03-10)<small>","0.7.0<br><small>(2024-10-18)<small>","0.1.26<br><small>(2024-01-15)<small>","1.3-1.9<br><small>(2023-11-27)<small>","1.0.1<br><small>(2024-07-19)<small>","1.1.5<br><small>(2025-01-17)<small>","0.1.0<br><small>(2023-11-06)<small>","0.5.4<br><small>(2024-04-16)<small>","0.1<br><small>(2024-09-02)<small>","2.6.1<br><small>(2024-09-20)<small>","2.0.0<br><small>(2024-02-24)<small>","0.3.9<br><small>(2025-03-20)<small>","0.1.4<br><small>(2025-02-27)<small>","0.6.0<br><small>(2024-05-30)<small>","0.9<br><small>(2021-02-26)<small>","1.0.2<br><small>(2025-02-07)<small>","0.2.3<br><small>(2024-03-25)<small>","4.1.1<br><small>(2025-02-03)<small>","2.1-1<br><small>(2024-10-27)<small>","0.1.3.5<br><small>(2025-01-08)<small>","1.0.1<br><small>(2023-10-24)<small>","1.8.4<br><small>(2024-05-06)<small>","1.1-9.8<br><small>(2024-09-03)<small>","0.2.6<br><small>(2024-02-25)<small>","0.6.0<br><small>(2024-06-15)<small>","1.8-29<br><small>(2025-02-26)<small>","5.2.2<br><small>(2024-10-04)<small>","0.3.3<br><small>(2024-11-08)<small>","1.15.3<br><small>(2023-08-29)<small>","0.7.0<br><small>(2025-01-24)<small>","3.1<br><small>(2025-02-22)<small>","2.1.0<br><small>(2023-01-31)<small>","1.1.3<br><small>(2024-05-01)<small>","2.6.0<br><small>(2024-12-17)<small>","0.1-5.1<br><small>(2024-06-19)<small>"],["<a href=\"mailto:aurelie.siberchicot-at-univ-lyon1.fr\">Aurélie Siberchicot<\/a>","<a href=\"mailto:nunesrpackages-at-gmail.com\">Matt Nunes<\/a>","<a href=\"mailto:jvbraun.statistics-at-gmail.com\">Jerome Braun<\/a>","<a href=\"mailto:ucfagls-at-gmail.com\">Gavin L. Simpson<\/a>","<a href=\"mailto:martin.papenberg-at-hhu.de\">Martin Papenberg<\/a>","<a href=\"mailto:sebastien.plutniak-at-posteo.net\">Sebastien Plutniak<\/a>","<a href=\"mailto:hanas-at-uw.edu\">Hana Sevcikova<\/a>","<a href=\"mailto:dongmei_li-at-urmc.rochester.edu\">Dongmei Li<\/a>","<a href=\"mailto:max-at-posit.co\">Max Kuhn<\/a>","<a href=\"mailto:mxkuhn-at-gmail.com\">Max Kuhn<\/a>","<a href=\"mailto:zhangxiao1994-at-cuhk.edu.cn\">Xiao Zhang<\/a>","<a href=\"mailto:nicolas.luyckx-at-calvagone.com\">Nicolas Luyckx<\/a>","<a href=\"mailto:john.helveston-at-gmail.com\">John Helveston<\/a>","<a href=\"mailto:krc-at-silicovore.com\">Kevin R. Coombes<\/a>","<a href=\"mailto:krc-at-silicovore.com\">Kevin R. Coombes<\/a>","<a href=\"mailto:ucfagls-at-gmail.com\">Gavin L. Simpson<\/a>","<a href=\"mailto:denis.cousineau-at-uottawa.ca\">Denis Cousineau<\/a>","<a href=\"mailto:xihang.chen-at-ndorms.ox.ac.uk\">Xihang Chen<\/a>","<a href=\"mailto:ptaranti-at-gmail.com\">Pier Taranti<\/a>","<a href=\"mailto:adwillis-at-uw.edu\">Amy D Willis<\/a>","<a href=\"mailto:greg-at-crunch.io\">Greg Freedman Ellis<\/a>","<a href=\"mailto:greg-at-crunch.io\">Greg Freedman Ellis<\/a>","<a href=\"mailto:mxkuhn-at-gmail.com\">Max Kuhn<\/a>","<a href=\"mailto:laurent_step-at-outlook.fr\">Stéphane Laurent<\/a>","<a href=\"mailto:christopher.gandrud-at-gmail.com\">Christopher Gandrud<\/a>","<a href=\"mailto:etienne.bacher-at-protonmail.com\">Etienne Bacher<\/a>","<a href=\"mailto:trevor.l.davis-at-gmail.com\">Trevor L. Davis<\/a>","<a href=\"mailto:pius.dahinden-at-tillotts.com\">Pius Dahinden<\/a>","<a href=\"mailto:theo_s-at-berkeley.edu\">Theo Saarinen<\/a>","<a href=\"mailto:gawainantell-at-gmail.com\">Gawain Antell<\/a>","<a href=\"mailto:tylerlukegrimes-at-gmail.com\">Tyler Grimes<\/a>","<a href=\"mailto:renozao-at-protonmail.com\">Renaud Gaujoux<\/a>","<a href=\"mailto:dror.bogin-at-gmail.com\">Dror Bogin<\/a>","<a href=\"mailto:adam.smith-at-mobot.org\">Adam B. Smith<\/a>","<a href=\"mailto:oelschlaeger.lennart-at-gmail.com\">Lennart Oelschläger<\/a>","<a href=\"mailto:graysonwhite13-at-gmail.com\">Grayson White<\/a>","<a href=\"mailto:changjiang.xu-at-utoronto.ca\">Changjiang Xu<\/a>","<a href=\"mailto:jinghuazhao-at-hotmail.com\">Jing Hua Zhao<\/a>","<a href=\"mailto:mfitzpatrick-at-umces.edu\">Matt Fitzpatrick<\/a>","<a href=\"mailto:mark.padgham-at-email.com\">Mark Padgham<\/a>","<a href=\"mailto:markus.kainu-at-kapsi.fi\">Markus Kainu<\/a>","<a href=\"mailto:ybuhl-at-posteo.de\">Yannik Buhl<\/a>","<a href=\"mailto:mollieebrooks-at-gmail.com\">Mollie Brooks<\/a>","<a href=\"mailto:lionelvoirol-at-hotmail.com\">Lionel Voirol<\/a>","<a href=\"mailto:daranzolin-at-gmail.com\">David Ranzolin<\/a>","<a href=\"mailto:andrea.carenzo-at-gmail.com\">Andrea Carenzo<\/a>","<a href=\"mailto:carteronpaul-at-gmail.com\">Paul Carteron<\/a>","<a href=\"mailto:antton.alberdi-at-bio.ku.dk\">Antton Alberdi<\/a>","<a href=\"mailto:mhferdosi-at-yahoo.com\">Mohammad Ferdosi<\/a>","<a href=\"mailto:niklas.koenen-at-gmail.com\">Niklas Koenen<\/a>","<a href=\"mailto:djh.shih-at-gmail.com\">David J. H. Shih<\/a>","<a href=\"mailto:pd.mes-at-cbs.dk\">Peter Dalgaard<\/a>","<a href=\"mailto:jmw86069-at-gmail.com\">James M. Ward<\/a>","<a href=\"mailto:mail-at-qinwenfeng.com\">Qin Wenfeng<\/a>","<a href=\"mailto:bge-at-uw.edu\">Brent Ewing<\/a>","<a href=\"mailto:bbolker+lme4-at-gmail.com\">Ben Bolker<\/a>","<a href=\"mailto:jeroenooms-at-gmail.com\">Jeroen Ooms<\/a>","<a href=\"mailto:castillo-at-bio.lmu.de\">Sara Castillo Vicente<\/a>","<a href=\"mailto:hadley-at-posit.co\">Hadley Wickham<\/a>","<a href=\"mailto:sharla.gelfand-at-gmail.com\">Sharla Gelfand<\/a>","<a href=\"mailto:sevvandik-at-gmail.com\">Sevvandi Kandanaarachchi<\/a>","<a href=\"mailto:csardi.gabor-at-gmail.com\">Gábor Csárdi<\/a>","<a href=\"mailto:Matthieu.Stigler-at-gmail.com\">Matthieu Stigler<\/a>","<a href=\"mailto:ron-at-insight-rx.com\">Ron Keizer<\/a>","<a href=\"mailto:krc-at-silicovore.com\">Kevin R. Coombes<\/a>","<a href=\"mailto:krc-at-silicovore.com\">Kevin R. Coombes<\/a>","<a href=\"mailto:giancarlo.vercellino-at-gmail.com\">Giancarlo Vercellino<\/a>","<a href=\"mailto:manuela.brunner-at-env.ethz.ch\">Manuela Brunner<\/a>","<a href=\"mailto:lorena.ricciotti-at-uniba.it\">Lorena Ricciotti<\/a>","<a href=\"mailto:tyler.rinker-at-gmail.com\">Tyler Rinker<\/a>","<a href=\"mailto:Nikolaus.Umlauf-at-uibk.ac.at\">Nikolaus Umlauf<\/a>","<a href=\"mailto:helge.hecht-at-recetox.muni.cz\">Helge Hecht<\/a>","<a href=\"mailto:pearce790-at-gmail.com\">Michael Pearce<\/a>","<a href=\"mailto:r.hijmans-at-gmail.com\">Robert J. Hijmans<\/a>","<a href=\"mailto:khalilsalim1-at-gmail.com\">Salim Khalil<\/a>","<a href=\"mailto:david.c.sterratt-at-ed.ac.uk\">David C. Sterratt<\/a>","<a href=\"mailto:rfastofficial-at-gmail.com\">Manos Papadakis<\/a>","<a href=\"mailto:stemmler.eric-at-gmail.com\">Eric Stemmler<\/a>","<a href=\"mailto:david_j_zimmermann-at-hotmail.com\">David Zimmermann-Kollenda<\/a>","<a href=\"mailto:CRAN-at-r-project.org\">CRAN Team<\/a>","<a href=\"mailto:10billhuang01-at-gmail.com\">Chaofan Huang<\/a>","<a href=\"mailto:lionel-at-posit.co\">Lionel Henry<\/a>","<a href=\"mailto:kazwd2008-at-gmail.com\">Kazumi Wada<\/a>","<a href=\"mailto:chris.bielow-at-bsc.fu-berlin.de\">Chris Bielow<\/a>","<a href=\"mailto:rqthomas-at-vt.edu\">Quinn Thomas<\/a>","<a href=\"mailto:jeroenooms-at-gmail.com\">Jeroen Ooms<\/a>","<a href=\"mailto:lluis.revilla-at-gmail.com\">Lluís Revilla Sancho<\/a>","<a href=\"mailto:hpages.on.github-at-gmail.com\">Hervé Pagès<\/a>","<a href=\"mailto:clark.liming-at-gmail.com\">Liming Li<\/a>","<a href=\"mailto:sean-at-seananderson.ca\">Sean C. Anderson<\/a>","<a href=\"mailto:matias.poullain-at-gmail.com\">Matias Poullain<\/a>","<a href=\"mailto:Martin.Jullum-at-nr.no\">Martin Jullum<\/a>","<a href=\"mailto:patrick.brown-at-utoronto.ca\">Patrick E. Brown<\/a>","<a href=\"mailto:ivan-at-svetunkov.com\">Ivan Svetunkov<\/a>","<a href=\"mailto:JakubDvorak-at-email.cz\">Jakub Dvorak<\/a>","<a href=\"mailto:fernando.lopez-at-upct.es\">Fernando Lopez<\/a>","<a href=\"mailto:cmjonestodd-at-gmail.com\">Charlotte M. Jones-Todd<\/a>","<a href=\"mailto:marek-at-gagolewski.com\">Marek Gagolewski<\/a>","<a href=\"mailto:thorsten.pohlert-at-gmx.de\">Thorsten Pohlert<\/a>","<a href=\"mailto:marlin-at-inventati.org\">Jialin Ma<\/a>","<a href=\"mailto:christophe.regouby-at-free.fr\">Christophe Regouby<\/a>","<a href=\"mailto:r.hijmans-at-gmail.com\">Robert J. Hijmans<\/a>","<a href=\"mailto:jeroenooms-at-gmail.com\">Jeroen Ooms<\/a>","<a href=\"mailto:c.barboza-at-darwin-eu.org\">Cesar Barboza<\/a>","<a href=\"mailto:vnavel-at-yahoo.fr\">Vladislav Navel<\/a>","<a href=\"mailto:vincent.arel-bundock-at-umontreal.ca\">Vincent Arel-Bundock<\/a>","<a href=\"mailto:info-at-openriskcalculator.com\">Tasos Grivas<\/a>","<a href=\"mailto:bsadler-at-smu.edu\">Bivin Sadler<\/a>","<a href=\"mailto:zergioibarra-at-gmail.com\">Sergio Ibarra-Espinosa<\/a>","<a href=\"mailto:mail-at-tnagler.com\">Thomas Nagler<\/a>","<a href=\"mailto:prosdocimi.ilaria-at-gmail.com\">Ilaria Prosdocimi<\/a>"],["<a href=\"https://cran.r-project.org/web/checks/check_results_adespatial.html\">checks<\/a><br><a href=\"https://github.com/adeverse/adespatial\">url<\/a><br><a href=\"https://github.com/adeverse/adespatial/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_adlift.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_AlgDesign.html\">checks<\/a><br><a href=\"https://github.com/jvbraun/AlgDesign\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_analogue.html\">checks<\/a><br><a href=\"https://github.com/gavinsimpson/analogue\">url<\/a><br><a href=\"https://github.com/gavinsimpson/analogue/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_anticlust.html\">checks<\/a><br><a href=\"https://github.com/m-Py/anticlust\">url<\/a><br><a href=\"https://github.com/m-Py/anticlust/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_archeofrag.html\">checks<\/a><br><a href=\"https://github.com/sebastien-plutniak/archeofrag\">url<\/a><br><a href=\"https://github.com/sebastien-plutniak/archeofrag/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_bayesPop.html\">checks<\/a><br><a href=\"https://bayespop.csss.washington.edu\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_BonEV.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_brulee.html\">checks<\/a><br><a href=\"https://github.com/tidymodels/brulee\">url<\/a><br><a href=\"https://github.com/tidymodels/brulee/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_C50.html\">checks<\/a><br><a href=\"https://topepo.github.io/C5.0/\">url<\/a><br><a href=\"https://github.com/topepo/C5.0/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_CAESAR.Suite.html\">checks<\/a><br><a href=\"https://github.com/XiaoZhangryy/CAESAR.Suite\">url<\/a><br><a href=\"https://github.com/XiaoZhangryy/CAESAR.Suite/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_campsismod.html\">checks<\/a><br><a href=\"https://github.com/Calvagone/campsismod\">url<\/a><br><a href=\"https://github.com/Calvagone/campsismod/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_cbcTools.html\">checks<\/a><br><a href=\"https://github.com/jhelvy/cbcTools\">url<\/a><br><a href=\"https://github.com/jhelvy/cbcTools/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ClassDiscovery.html\">checks<\/a><br><a href=\"http://oompa.r-forge.r-project.org/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_CloneSeeker.html\">checks<\/a><br><a href=\"http://oompa.r-forge.r-project.org/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_cocorresp.html\">checks<\/a><br><a href=\"https://gavinsimpson.github.io/cocorresp/\">url<\/a><br><a href=\"https://github.com/gavinsimpson/cocorresp/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_CohensdpLibrary.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_CohortSymmetry.html\">checks<\/a><br><a href=\"https://ohdsi.github.io/CohortSymmetry/\">url<\/a><br><a href=\"https://github.com/OHDSI/CohortSymmetry/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_coppeCosenzaR.html\">checks<\/a><br><a href=\"https://github.com/ptaranti/coppeCosenzaR\">url<\/a><br><a href=\"https://github.com/ptaranti/coppeCosenzaR/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_corncob.html\">checks<\/a><br><a href=\"https://github.com/statdivlab/corncob\">url<\/a><br><a href=\"https://github.com/statdivlab/corncob/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_crplyr.html\">checks<\/a><br><a href=\"https://crunch.io/r/crplyr/\">url<\/a><br><a href=\"https://github.com/Crunch-io/crplyr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_crunchy.html\">checks<\/a><br><a href=\"https://crunch.io/r/crunchy/\">url<\/a><br><a href=\"https://github.com/Crunch-io/crunchy/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_Cubist.html\">checks<\/a><br><a href=\"https://topepo.github.io/Cubist/\">url<\/a><br><a href=\"https://github.com/topepo/Cubist/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_cxhull.html\">checks<\/a><br><a href=\"https://github.com/stla/cxhull\">url<\/a><br><a href=\"https://github.com/stla/cxhull/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_DataCombine.html\">checks<\/a><br><a href=\"http://CRAN.R-project.org/package=DataCombine\">url<\/a><br><a href=\"https://github.com/christophergandrud/DataCombine/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_datawizard.html\">checks<\/a><br><a href=\"https://easystats.github.io/datawizard/\">url<\/a><br><a href=\"https://github.com/easystats/datawizard/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_datetimeoffset.html\">checks<\/a><br><a href=\"https://trevorldavis.com/R/datetimeoffset/\">url<\/a><br><a href=\"https://github.com/trevorld/r-datetimeoffset/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_disprofas.html\">checks<\/a><br><a href=\"https://github.com/piusdahinden/disprofas\">url<\/a><br><a href=\"https://github.com/piusdahinden/disprofas/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_distillML.html\">checks<\/a><br><a href=\"https://github.com/forestry-labs/distillML\">url<\/a><br><a href=\"https://github.com/forestry-labs/distillML/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_divvy.html\">checks<\/a><br><a href=\"https://gawainantell.github.io/divvy/\">url<\/a><br><a href=\"https://github.com/GawainAntell/divvy/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_dnapath.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_doRNG.html\">checks<\/a><br><a href=\"https://renozao.github.io/doRNG/\">url<\/a><br><a href=\"https://github.com/renozao/doRNG/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_easycsv.html\">checks<\/a><br><a href=\"https://github.com/bogind/easycsv\">url<\/a><br><a href=\"https://github.com/bogind/easycsv/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_enmSdmX.html\">checks<\/a><br><a href=\"https://github.com/adamlilith/enmSdmX\">url<\/a><br><a href=\"https://github.com/adamlilith/enmSdmX/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_fHMM.html\">checks<\/a><br><a href=\"https://loelschlaeger.de/fHMM/\">url<\/a><br><a href=\"https://github.com/loelschlaeger/fHMM/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_FIESTA.html\">checks<\/a><br><a href=\"https://usdaforestservice.github.io/FIESTA/\">url<\/a><br><a href=\"https://github.com/USDAForestService/FIESTA/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_FLASHMM.html\">checks<\/a><br><a href=\"https://github.com/BaderLab/FLASHMM\">url<\/a><br><a href=\"https://github.com/BaderLab/FLASHMM/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gaawr2.html\">checks<\/a><br><a href=\"https://jinghuazhao.github.io/gaawr2/\">url<\/a><br><a href=\"https://github.com/jinghuazhao/gaawr2/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gdm.html\">checks<\/a><br><a href=\"https://mfitzpatrick.al.umces.edu/gdm/\">url<\/a><br><a href=\"https://github.com/fitzLab-AL/gdm/issues/\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_geodist.html\">checks<\/a><br><a href=\"https://github.com/hypertidy/geodist\">url<\/a><br><a href=\"https://github.com/hypertidy/geodist/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_geofi.html\">checks<\/a><br><a href=\"https://ropengov.github.io/geofi/\">url<\/a><br><a href=\"https://github.com/rOpenGov/geofi/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gie.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_glmmTMB.html\">checks<\/a><br><a href=\"https://github.com/glmmTMB/glmmTMB\">url<\/a><br><a href=\"https://github.com/glmmTMB/glmmTMB/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_gmwmx2.html\">checks<\/a><br><a href=\"https://github.com/SMAC-Group/gmwmx2\">url<\/a><br><a href=\"https://github.com/SMAC-Group/gmwmx2/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hacksaw.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hacksig.html\">checks<\/a><br><a href=\"https://github.com/Acare/hacksig\">url<\/a><br><a href=\"https://github.com/Acare/hacksig/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_happign.html\">checks<\/a><br><a href=\"https://github.com/paul-carteron\">url<\/a><br><a href=\"https://github.com/paul-carteron/happign/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hilldiv.html\">checks<\/a><br><a href=\"https://github.com/anttonalberdi/hilldiv\">url<\/a><br><a href=\"https://github.com/anttonalberdi/hilldiv/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_hsphase.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_innsight.html\">checks<\/a><br><a href=\"https://bips-hb.github.io/innsight/\">url<\/a><br><a href=\"https://github.com/bips-hb/innsight/issues/\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_io.html\">checks<\/a><br><a href=\"https://bitbucket.org/djhshih/io\">url<\/a><br><a href=\"https://bitbucket.org/djhshih/io/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ISwR.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_jamba.html\">checks<\/a><br><a href=\"https://jmw86069.github.io/jamba/\">url<\/a><br><a href=\"https://github.com/jmw86069/jamba/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_jiebaR.html\">checks<\/a><br><a href=\"https://github.com/qinwf/jiebaR/\">url<\/a><br><a href=\"https://github.com/qinwf/jiebaR/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_leidenbase.html\">checks<\/a><br><a href=\"https://github.com/cole-trapnell-lab/leidenbase\">url<\/a><br><a href=\"https://github.com/cole-trapnell-lab/leidenbase/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_lme4.html\">checks<\/a><br><a href=\"https://github.com/lme4/lme4/\">url<\/a><br><a href=\"https://github.com/lme4/lme4/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_magick.html\">checks<\/a><br><a href=\"https://docs.ropensci.org/magick/\">url<\/a><br><a href=\"https://github.com/ropensci/magick/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_MethEvolSIM.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_odbc.html\">checks<\/a><br><a href=\"https://odbc.r-dbi.org\">url<\/a><br><a href=\"https://github.com/r-dbi/odbc/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_opendatatoronto.html\">checks<\/a><br><a href=\"https://sharlagelfand.github.io/opendatatoronto/\">url<\/a><br><a href=\"https://github.com/sharlagelfand/opendatatoronto/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_outlierensembles.html\">checks<\/a><br><a href=\"https://sevvandi.github.io/outlierensembles/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_pak.html\">checks<\/a><br><a href=\"https://pak.r-lib.org/\">url<\/a><br><a href=\"https://github.com/r-lib/pak/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_partsm.html\">checks<\/a><br><a href=\"https://github.com/MatthieuStigler/partsm\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_PKPDsim.html\">checks<\/a><br><a href=\"https://github.com/InsightRX/PKPDsim\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_Polychrome.html\">checks<\/a><br><a href=\"http://oompa.r-forge.r-project.org/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_PreProcess.html\">checks<\/a><br><a href=\"http://oompa.r-forge.r-project.org/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_proteus.html\">checks<\/a><br><a href=\"https://rpubs.com/giancarlo_vercellino/proteus\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_PRSim.html\">checks<\/a><br><a href=\"https://git.math.uzh.ch/reinhard.furrer/PRSim-devel\">url<\/a><br><a href=\"https://git.math.uzh.ch/reinhard.furrer/PRSim-devel/-/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_PublicWorksFinanceIT.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_qdapRegex.html\">checks<\/a><br><a href=\"https://github.com/trinker/qdapRegex\">url<\/a><br><a href=\"https://github.com/trinker/qdapRegex/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_R2BayesX.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_RAMClustR.html\">checks<\/a><br><a href=\"https://github.com/cbroeckl/RAMClustR\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rankrate.html\">checks<\/a><br><a href=\"https://pearce790.github.io/rankrate/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_raster.html\">checks<\/a><br><a href=\"https://rspatial.org/raster\">url<\/a><br><a href=\"https://github.com/rspatial/raster/issues/\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_Rcrawler.html\">checks<\/a><br><a href=\"https://github.com/salimk/Rcrawler/\">url<\/a><br><a href=\"https://github.com/salimk/Rcrawler/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_retistruct.html\">checks<\/a><br><a href=\"http://davidcsterratt.github.io/retistruct/\">url<\/a><br><a href=\"https://github.com/davidcsterratt/retistruct/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_Rfast2.html\">checks<\/a><br><a href=\"https://github.com/RfastOfficial/Rfast2\">url<\/a><br><a href=\"https://github.com/RfastOfficial/Rfast2/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rim.html\">checks<\/a><br><a href=\"https://rcst.github.io/rim/\">url<\/a><br><a href=\"https://github.com/rcst/rim/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_RITCH.html\">checks<\/a><br><a href=\"https://davzim.github.io/RITCH/\">url<\/a><br><a href=\"https://github.com/DavZim/RITCH/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_RJSONIO.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rkriging.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rlang.html\">checks<\/a><br><a href=\"https://rlang.r-lib.org\">url<\/a><br><a href=\"https://github.com/r-lib/rlang/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_RMSD.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rmzqc.html\">checks<\/a><br><a href=\"https://github.com/MS-Quality-hub/rmzqc\">url<\/a><br><a href=\"https://github.com/MS-Quality-hub/rmzqc/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_ropenmeteo.html\">checks<\/a><br><a href=\"http://flare-forecast.org/ropenmeteo/\">url<\/a><br><a href=\"https://github.com/flare-forecast/ropenmeteo/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rsvg.html\">checks<\/a><br><a href=\"https://docs.ropensci.org/rsvg/\">url<\/a><br><a href=\"https://github.com/ropensci/rsvg/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_rtweet.html\">checks<\/a><br><a href=\"https://docs.ropensci.org/rtweet/\">url<\/a><br><a href=\"https://github.com/ropensci/rtweet/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_Rtwobitlib.html\">checks<\/a><br><a href=\"https://github.com/hpages/Rtwobitlib\">url<\/a><br><a href=\"https://github.com/hpages/Rtwobitlib/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_sasr.html\">checks<\/a><br><a href=\"https://github.com/insightsengineering/sasr/\">url<\/a><br><a href=\"https://github.com/insightsengineering/sasr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_sdmTMB.html\">checks<\/a><br><a href=\"https://pbs-assess.github.io/sdmTMB/\">url<\/a><br><a href=\"https://github.com/pbs-assess/sdmTMB/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_sgat.html\">checks<\/a><br><a href=\"https://github.com/matiaspoullain/sgat\">url<\/a><br><a href=\"https://github.com/matiaspoullain/sgat/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_shapr.html\">checks<\/a><br><a href=\"https://norskregnesentral.github.io/shapr/\">url<\/a><br><a href=\"https://github.com/NorskRegnesentral/shapr/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_skellam.html\">checks<\/a><br><a href=\"https//r-forge.r-project.org/projects/healthqueues/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_smooth.html\">checks<\/a><br><a href=\"https://github.com/config-i1/smooth\">url<\/a><br><a href=\"https://github.com/config-i1/smooth/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_SplitSoftening.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_spqdep.html\">checks<\/a><br><a href=\"https://f8l5h9.github.io/spqdep/\">url<\/a><br><a href=\"https://github.com/f8l5h9/spqdep/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_stelfi.html\">checks<\/a><br><a href=\"https://github.com/cmjt/stelfi/\">url<\/a><br><a href=\"https://github.com/cmjt/stelfi/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_stringi.html\">checks<\/a><br><a href=\"https://stringi.gagolewski.com/\">url<\/a><br><a href=\"https://github.com/gagolews/stringi/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_SuppDists.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_symengine.html\">checks<\/a><br><a href=\"https://github.com/symengine/symengine.R\">url<\/a><br><a href=\"https://github.com/symengine/symengine.R/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tabnet.html\">checks<\/a><br><a href=\"https://mlverse.github.io/tabnet/\">url<\/a><br><a href=\"https://github.com/mlverse/tabnet/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_terra.html\">checks<\/a><br><a href=\"https://rspatial.org/\">url<\/a><br><a href=\"https://github.com/rspatial/terra/issues/\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tesseract.html\">checks<\/a><br><a href=\"https://docs.ropensci.org/tesseract/\">url<\/a><br><a href=\"https://github.com/ropensci/tesseract/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_TestGenerator.html\">checks<\/a><br><a href=\"https://github.com/darwin-eu/TestGenerator\">url<\/a><br><a href=\"https://github.com/darwin-eu/TestGenerator/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_timetools.html\">checks<\/a><br><a href=\"https://sourceforge.net/projects/timetools/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tinytable.html\">checks<\/a><br><a href=\"https://vincentarelbundock.github.io/tinytable/\">url<\/a><br><a href=\"https://github.com/vincentarelbundock/tinytable/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_Trading.html\">checks<\/a><br><a href=\"https://openriskcalculator.com/\">url<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_tswge.html\">checks<\/a><br><a href=\"NA\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_vein.html\">checks<\/a><br><a href=\"https://github.com/atmoschem/vein\">url<\/a><br><a href=\"https://github.com/atmoschem/vein/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_VineCopula.html\">checks<\/a><br><a href=\"https://github.com/tnagler/VineCopula\">url<\/a><br><a href=\"https://github.com/tnagler/VineCopula/issues\">url2<\/a>","<a href=\"https://cran.r-project.org/web/checks/check_results_winfapReader.html\">checks<\/a><br><a href=\"https://ilapros.github.io/winfapReader/\">url<\/a><br><a href=\"https://github.com/ilapros/winfapReader/issues\">url2<\/a>"],[null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#CAESAR.Suite\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#corncob\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#enmSdmX\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#odbc\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#qdapRegex\">resubmitted/inspect<\/a><\/small>",null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#rim\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#symengine\">resubmitted/archive<\/a><\/small>",null,null,null,null,null,"<small><a href=\"https://nx10.github.io/cransubs/pkg#tinytable\">resubmitted/pending<\/a><\/small>",null,null,null,null,null],[2,5,12,2,1,1,0,0,0,10,0,1,0,7,0,0,0,0,0,1,0,0,7,2,4,16,1,0,0,0,0,82,4,0,0,0,0,0,0,20,0,0,18,0,0,0,0,0,1,0,0,0,0,4,2,331,95,0,8,0,0,4,0,0,14,0,0,0,0,11,0,0,0,244,0,0,22,0,0,43,0,3003,0,2,0,28,5,0,0,1,0,0,3,3,0,0,0,420,16,2,0,304,0,0,0,1,0,0,0,16,0],["CommEcol, OCNet","binhf, CNLTreg, CNLTtsa, liftLRD, nlt","agricolae, bioCancer, cbcTools, choiceDes, conjoint, crossdes, dexisensitivity, DoE.wrapper, edibble, radiant.design, SensoMineR, STPGA","cops, FuseSOM","CEGO","archeofrag.gui","","","","baguette, ebmc, fmf, imbalance, MetabolomicsBasics, ModTools, noisemodel, pheble, Pigengene, StratifiedRF","","campsis","","GenAlgo, Mercator, Modeler, PCDimension, rnaCrosslinkOO, RPointCloud, Thresher","","","","","","benchdamic","","","C50, dendroTools, finnts, geomod, rminer, tsensembler, utsf","archeoViz, gyro","E4tools, gesttools, psData, rties","bayestestR, correlation, easystats, effectsize, ggeffects, ggstatsplot, modelbased, parameters, performance, report, see, sjlabelled, sjmisc, sjPlot, sjstats, statsExpressions","xmpdf","","","","","adjustedCurves, ANCOMBC, archeofrag.gui, autoMrP, BANDITS, bestNormalize, bhmbasket, bigsnpr, BSL, bumphunter, CDatanet, CICI, clustcurv, cola, condSURV, Counterfactual, dcanr, deltaccd, DifferentialRegulation, distinct, drugDemand, DynForest, ellipticalsymmetry, envi, EPX, evinf, ewoc, exuber, fabisearch, fect, FLORAL, funGp, gbts, GeDS, gratis, gsynth, hwep, hybridModels, IFAA, ILoReg, IsoBayes, kmBlock, latentcor, lcc, LSEbootLS, mclustAddons, MCPModBC, Mediana, midasml, missForest, momentuHMM, mplot, mslp, nebula, PADOG, PartialNetwork, ramr, randomLCA, redist, RobGARCHBoot, robmixglm, Robyn, RolDE, rpm, rsides, SC3, segtest, seqimpute, skpr, sparrpowR, SparseMDC, sphunif, SPONGE, StochBlock, survidm, tglkmeans, TRexSelector, updog, varclust, WebGestaltR, WpProj, wxgenR","gimmeTools, SemNetCleaner, SemNetDictionaries, uniset","","","","","","","distanceto, gtfsrouter, GWnnegPCA, GWpcor, gwpcormapper, IRexamples, MazamaLocationUtils, MazamaTimeSeries, metro, MigConnectivity, opentripplanner, sfcentral, simodels, spotoroo, tidytransit, topdowntimeratio, traipse, trip, ulex, zetadiv","","","cv, DEGRE, depower, GLMMcosinor, glmmSeq, gmvjoint, iccCounts, lefko3, LongDat, Maaslin2, mecoturn, mtscr, multilevelmediation, muscat, pastaPlot, pdR, predictmeans, twosigma","","","","","","hsrecombi","","","","","BioM2, chinese.misc, CTM, TangPoemR","monocle, Seurat","abn, actuaRE, afex, agRee, agriutilities, AgroR, altmeta, ANCOMBC, arm, ARTool, attrib, autoMrP, bapred, bayesammi, BayesCACE, BayesLN, BayesSenMC, baystability, BCClong, BClustLonG, benchdamic, BFpack, blme, blmeco, boot.pval, BradleyTerry2, breakaway, buildmer, cAIC4, car, carcass, catalytic, ceas, censcyt, cgam, chngpt, CIMPLE, cito, ciTools, CleaningValidation, climwin, clinicalsignificance, clintools, CLME, clustTMB, CMatching, ComBatFamQC, CompositeReliability, compositeReliabilityInNestedDesigns, confintROB, conversim, cpr, CR2, crctStepdown, CRTspat, cv, cvms, dataquieR, daySupply, DClusterm, demic, designr, dfmeta, DHARMa, diagmeta, diffcyt, difR, dmlalg, doremi, doseR, dplR, dreamlet, eatRep, EdSurvey, eefAnalytics, effects, eirm, epr, EQUALSTATS, ESTER, EstimateBreed, evolvability, ez, faraway, fastFMM, faux, fence, finalfit, fishmethods, fullfact, galamm, gamm4, gammi, geeasy, geex, GeoDiff, glm.predict, glmertree, glmm.hp, GLMMcosinor, glmmEP, glmmPen, GLMMRR, glmmSeq, glmmTMB, GLMpack, gorica, grafify, gtheory, gvcR, HDANOVA, HeritSeq, ib, iccbeta, iccmult, IDmeasurer, IMAS, IMTest, inferference, influence.ME, INLAjoint, INSPECTumours, inti, intRvals, ISLET, isni, IVAS, jlctree, jlmerclusterperm, joineRML, jomo, jstable, JWileymisc, KenSyn, konfound, lefko3, LEGIT, likelihoodR, LKT, lme4breeding, LMERConvenienceFunctions, lmerTest, LongDat, longpower, longsurr, LSAmitR, Maaslin2, macc, MAGMWAR, manymodelr, MargCond, MariNET, marked, MBECS, MCM, MDMA, MDMR, MECfda, mecor, MedianaDesigner, mediation, MEMSS, merDeriv, mermboost, merTools, meta, metadeconfoundR, metamicrobiomeR, metamisc, metan, metaplus, MethodCompare, MF.beta4, micemd, MIGEE, MiRKAT, misty, mixAK, mixedbiastest, MixedPsy, MixRF, MLID, mlma, mlmhelpr, mlmpower, mlmRev, mlmtools, MLMusingR, mlVAR, MMeM, modnets, msqrob2, MSstats, MSstatsTMT, mult.latent.reg, multid, multiDimBio, multilevelmod, multilevelTools, multisite.accuracy, MultisiteMediation, multitool, multpois, muscat, mvMISE, mxnorm, netmediate, omics, pamm, panelr, partR2, pastaPlot, PBImisc, pbkrtest, pcgen, pcvr, pdR, pedigreemm, PepSetTest, peramo, Phenotype, PhenStat, PheVis, phoenics, phyr, piecewiseSEM, plgraphics, PLmixed, pmm, POMA, PoolTestR, powerbydesign, pprof, predictmeans, predint, PrevMap, projpred, psfmi, PStrata, PSweight, psycModel, ptmixed, PUMP, pvca, pwlmm, qape, qra, quest, r2mlm, rADA, raincin, Rcmdr, RcmdrPlugin.TeachStat, refund, refund.shiny, reghelper, registr, regplot, REndo, repmod, reproducer, rewie, rexposome, RLRsim, robustBLME, robustlmm, rockchalk, rosetta, rpql, rptR, rr2, RRreg, rsq, rstanarm, rties, RVAideMemoire, sae, saeczi, samplr, SBMTrees, sdmTMB, semEff, ShinyItemAnalysis, SIAmodules, SignacX, siland, SimplyAgree, simr, skpr, smicd, snm, SoyM, spAbundance, SPCDAnalyze, specr, splithalf, spOccupancy, spsurvey, squid, stabiliser, stability, standardize, statgenGxE, statgenSTA, steppedwedge, stevemisc, structree, SumcaVer1, Surrogate, surrosurv, svylme, swaprinc, swCRTdesign, TcGSA, trace, treekoR, trouBBlme4SolveR, tukeytrend, VAJointSurv, variancePartition, varTestnlme, VCA, vcPB, VetResearchLMM, vglmer, warpMix, WebPower, weights, welchADF, WeMix, ZLAvian","add2ggplot, animation, asciicast, basemaps, BioCartaImage, BioImageDbs, biopixR, camcorder, carbonate, chimeraviz, chronochrt, colocr, colorizer, convevol, cropcircles, daiR, DiscoRhythm, doconv, drhutools, DynareR, eCerto, eoffice, eyetools, fsbrain, gdiff, GenAI, ggfx, ggimage, ggkegg, ggpath, gofigR, handwriter, handwriterApp, huito, image.binarization, image.textlinedetector, imagefluency, imageseg, jdenticon, kerastuneR, kollaR, LearnClust, LearningRlab, lterpalettefinder, magickGUI, mapboxapi, mapscanner, mathpix, MCOE, meme, metaDigitise, mlbplotR, moodef, movegroup, MtreeRing, multipanelfigure, NCC, nflplotR, ODT, OMICsPCA, openSkies, pavo, plot4fun, plothelper, PoPdesign, pressuRe, protein8k, PubChemR, r4lineups, rfaRm, ROMDB, SDGdetector, seqArchRplus, SpatialExperiment, spatialLIBD, SpatialOmicsOverlay, SRTsim, summarytools, svgtools, terrainr, texPreview, tidycharts, tidySpatialExperiment, timeLineGraphics, tinysnapshot, tvthemes, UAHDataScienceSF, UMI4Cats, unhcrdown, UniprotR, vprr, WaverideR, weatherOz, ZooID, zoomGroupStats","","datrProfile, epidm, MCOE, pgTools, SangerTools, shiny.telemetry, sqlHelpers, virtuoso","","","dockerfiler, guideR, natmanager, PaRe","","","Apollonius, ASML, CliquePercolation, colorrepel, DiscreteGapStatistic, gyro, Mercator, plasma, quickcode, RCDT, SillyPutty, simona, vimpclust, YEAB","","","","","BAwiR, edgar, ggfootball, qdap, rdhs, sgat, textclean, UniprotR, vegtable, WhatsR, worldfootballR","","","","abmR, acdcR, affinity, agricolaeplotr, angstroms, arcpullr, bamm, basecallQC, basf, bioRad, birdring, bRacatus, cartography, cdlTools, CensSpatial, chillR, clhs, CLimd, climetrics, cmsaf, cmsafops, cmsafvis, colorist, CONFESS, cooltools, coreCT, CropScapeR, cruts, csa, CSHShydRology, ctmcmove, ctmm, cubeview, cytomapper, dartR.base, dartR.spatial, DEPONS2R, diffeR, dismo, dispeRse, dissever, DMMF, dPCP, DRHotNet, ecochange, EEAaq, eixport, elevatr, elsa, EmissV, ENMTools, epizootic, eRTG3D, esmisc, evapoRe, exactextractr, ExtractTrainData, fasterize, fgdr, fitbitViz, FjordLight, ForestGapR, FORTLS, gaiah, GCD, gdistance, gen3sis, geomerge, geomod, GeoRange, geotopbricks, geoTS, geouy, GeoWeightedModel, gfcanalysis, gimms, GInSARCorW, glcm, GLCMTextures, googletraffic, gps.track, grainscape, graph4lg, graticule, grec, GWSDAT, habCluster, habtools, HDSpatialScan, hillshader, hydflood, hypervolume, icardaFIGSr, infinityFlow, intkrige, ipdw, isocat, itsdm, kehra, kissmig, KMLtoSHAPE, lakemorpho, layer, leafem, leaflet, leafR, lgcp, lue, lulcc, macroBiome, mapboxapi, mapedit, Mapinguari, mapscanner, mapview, marmap, maskRangeR, MaxentVariableSelection, maxlike, memgene, meteo, meteoForecast, mgc, ModelMap, momentuHMM, monographaR, move, movecost, movegroup, multimark, MultiscaleDTM, naturaList, NicheBarcoding, nlrx, nodiv, nosoi, oceanmap, OpenLand, OpenStreetMap, osc, patternize, PersomicsArray, phenopix, plainview, plotdap, poems, PointedSDMs, PopGenReport, pRecipe, pressuRe, PrevMap, prioritizr, ptools, QRAGadget, qrencoder, qrlabelr, quadmesh, RAC, rangeMapper, rasterDT, rasterKernelEstimates, rasterList, rasterVis, rayrender, rayshader, rbgm, RCGLS, RchivalTag, rcrimeanalysis, ReadDIM, rechaRge, recolorize, redlistr, remote, rivnet, rleafmap, Rlibkdv, RmarineHeatWaves, RPANDA, RPyGeo, rsat, rSPARCS, rTensor2, rts, rwc, RWmisc, rworldmap, sabre, samc, satellite, scBio, sdcSpatial, sdm, sdmpredictors, sdmvspecies, SEAHORS, secr, ShapePattern, ShapeSelectForest, siland, sirad, slideview, slippymath, soilassessment, SoilSaltIndex, SpatGRID, SpatialKDE, SpatialPosition, spatsurv, spDates, spdynmod, sperich, spex, SPIAT, spup, SSDM, sta, steps, stfit, stJoincount, stppSim, SUNGEO, surveyPrev, tabularaster, TCHazaRds, telefit, TensorTools, theiaR, tiler, topoDistance, track2KBA, trackdem, trackeR, trip, TUFLOWR, twc, ulex, usdm, virtualspecies, VoxR, WCM, WEGE, wingen, zipcodeR","","","cauchypca, choosepc, cols, Compositional, CompositionalML, corrfuns, crwbmetareg, dCovTS, DER, Directional, geppe, gomp, MLE, mvcauchy, mvhtests, nlgm, pchc, pdcor, PR, regda, robqda, TPXG","","","ajv, animint2, BaseSpaceR, CirceR, cmmr, CodelistGenerator, CohortGenerator, CropScapeR, DIZutils, FAOSTAT, GladiaTOX, gpboost, haploR, KEGGlincs, kerastuneR, LDAvis, LightningR, lipidmapsR, ltertools, Macarron, mitre, mojson, PhageCocktail, PubChemR, pubmed.mineR, pxR, R4CouchDB, rbiouml, RCy3, RedditExtractoR, restimizeapi, revgeo, rfigshare, Rgff, ropensecretsapi, RSDA, RTCGAToolbox, RXKCD, rYoutheria, SignacX, switchr, tsmp, wilson","","abjutils, academictwitteR, AcceptReject, accrualPlot, accucor, Achilles, acledR, activatr, actxps, adas.utils, additive, adepro, adheRenceRX, adjclust, adjustedCurves, admiral, admiraldev, admiralmetabolic, admiralonco, admiralophtha, admiralpeds, admiralvaccine, admixr, adnuts, adobeanalyticsr, adproclus, aeddo, aedseo, AEenrich, afcharts, AFR, ageutils, aggregateBioVar, aghq, agriutilities, agrostab, ags, agua, agvgd, aifeducation, airGRiwrm, AirMonitor, airnow, airpart, airship, airt, akc, alakazam, alarmdata, ale, alevinQC, alfred, alien, AlignLV, allofus, almanac, alookr, AlphaMissenseR, AlpsNMR, amadeus, ambient, Amelia, ami, amp, amt, anabel, AnanseSeurat, and, Andromeda, andurinha, animalEKF, AnimalSequences, animbook, animl, aniS, annotater, anomalize, AnthropMMD, AntibodyForests, AnVIL, AnVILGCP, AnVILWorkflow, apache.sedona, APackOfTheClones, APCalign, apexcharter, APIS, APL, aplotExtra, applicable, appsheet, apyramid, AquaBEHER, aquodom, arcgisgeocode, arcgislayers, arcgisplaces, arcgisutils, archive, arcpbf, arcpullr, areal, arealDB, arena2r, ARPALData, arpr, ArrayExpress, arrow, artpack, ARUtools, askgpt, ASML, aspline, asremlPlus, assertions, assertr, assignPOP, assignR, ast2ast, ATE.ERROR, ATQ, atrrr, attempt, audubon, auk, AutoAds, autodb, autogam, autoharp, autoimport, autoMrP, autonomics, autoReg, AutoScore, autoslider.core, autostats, autothresholdr, autoTS, AutoWMM, AuxSurvey, awdb, AzureAppInsights, AzureKusto, babelgene, babette, baguette, baizer, baldur, bambooHR, bandle, banffIT, banter, barcodetrackR, bartcs, bartMan, baRulho, baseballr, basedosdados, BaseSet, BasketballAnalyzeR, batchLLM, batchtma, BATSS, bayesAB, bayesammi, bayesassurance, bayesdfa, BayesERtools, bayesian, BayesianLaterality, BayesMallows, bayesmove, bayesnec, bayesplot, BayesPostEst, BayesSpace, BayesTools, baymedr, baystability, bbknnR, bcdata, BCEA, bcrm, bdrc, bdsm, beadplexr, beastier, beastt, beautier, beaver, bench, bennu, besthr, bettr, BFF, BGPhazard, biblioverlap, bigergm, bignum, bigrquery, bigrquerystorage, billboarder, BindingSiteFinder, BiocBook, BiocPkgTools, BiocSet, biocthis, biodosetools, BioGA, biogrowth, bioinactivation, biometryassist, BioNERO, bioRad, bioregion, bioseq, BiostatsUHNplus, biovizBase, birdie, birdscanR, bistablehistory, bitmexr, BiVariAn, bivariateLeaflet, bizicount, blastula, blit, blob, BlockCov, blocs, blsR, bnmonitor, bodsr, boiwsa, bonsai, boomer, bootnet, borealis, box.linters, box.lsp, boxly, boxr, bpmnR, bpmnVisualizationR, brclimr, brendaDb, BREW3R.r, brickster, bridgr, BRINDA, brisk, brms, brms.mmrm, brolgar, broom, broom.helpers, brpop, brulee, BRVM, bs4cards, bsicons, bsitar, bskyr, bslib, bssm, bsynth, bulkAnalyseR, bulkreadr, bullseye, bumbl, bundle, bupaR, bupaverse, burnr, butcher, BuyseTest, BVARverse, bwsTools, c2z, cacc, cachem, cageminer, CAGEr, calibrationband, calidad, calmr, camcorder, campfin, campsis, campsismod, canadamaps, cancensus, cancerscreening, cansim, capesR, carbonr, card, cards, cardx, care4cmodel, caretEnsemble, carrier, cartograflow, cartographer, cascadess, cassandRa, catalytic, catfun, catmaply, causact, CausalGPS, causalOT, CausalQueries, cbcTools, CBEA, cbioportalR, CBNplot, cbpManager, cccm, ccdR, ccmReportR, ccoptimalmatch, ccostr, ccpsyc, CDMConnector, cdrcR, celaref, CellBench, cellKey, cellpypes, cellularautomata, censable, censcyt, censored, censusapi, Cepo, cepumd, ceramic, cereal, ceRnetsim, Certara.ModelResults, Certara.R, Certara.VPCResults, Certara.Xpose.NLME, cfdnakit, cfDPro, cft, CGPfunctions, ChangePointTaylor, chantrics, Characterization, chattr, checkthat, cheese, chem16S, chemodiv, chessboard, chevron, chilemapas, chillR, ChIPseeker, chk, CHOIRBM, cholera, chores, chromote, chronicle, chronicler, chronochrt, chunked, cia, cimir, circRprofiler, CircSeqAlignTk, circumplex, CiteFuse, citrus, cjar, cjbart, clap, clarify, ClassifyR, clc, cleanepi, cleaner, CleaningValidation, clidatajp, cliff, climaemet, clinicalsignificance, clock, cloudfs, CluMP, ClussCluster, ClustAssess, clustermole, clusterProfiler, clustifyr, ClustImpute, clustMixType, ClusTorus, clustree, clustringr, cmcR, cmdfun, cmhc, cmpsR, cms, cmstatr, cmstatrExt, cobalt, cocoon, codebook, codebookr, CodelistGenerator, codified, cofad, cogeqc, CohortAlgebra, cohortBuilder, CohortCharacteristics, CohortConstructor, CohortExplorer, CohortGenerator, CohortPathways, CohortSurvival, CohortSymmetry, COINr, ColOpenData, colorist, ColorNameR, Colossus, comapr, combinedevents, CoMiRe, COMIX, compareDF, compareMCMCs, COMPASS, comperank, comperes, ComPrAn, comtradr, concordance, concordexR, condathis, condformat, ConfidenceEllipse, conflicted, conflr, congress, CongressData, CoNI, connectapi, connectcreds, connectwidgets, conos, conquestr, conserveR, consortr, ConsReg, constructive, ContourFunctions, contrastable, contribution, contsurvplot, convergEU, cookiecutter, cookies, CopernicusMarine, copulaSim, CoreGx, corncob, coro, corporaexplorer, correlationfunnel, corrr, cosmosR, COTAN, countland, coursekata, covid19br, covidcast, covidmx, covTestR, cowplot, cowsay, cpam, cpss, cpsvote, crane, crawl, crew, crew.aws.batch, crew.cluster, crimedata, crimeutils, crisprVerse, CropBreeding, crosshap, crossmap, crossnma, crosstable, crosstalkr, crossurr, crplyr, cSEM, csmGmm, cspp, csquares, csvwr, CTexploreR, cthist, cTRAP, ctrdata, ctrialsgov, ctxR, cubar, cubble, cubelyr, cucumber, cuda.ml, cuperdec, CuratedAtlasQueryR, curatedMetagenomicData, curatedPCaData, curatedTBData, customiser, cusumcharter, cutpointr, cvap, cvasi, cvCovEst, cvms, Cyclops, cylcop, cypress, CytoGLMM, cytoKernel, CytoMDS, cytominer, CytoPipeline, CytoSimplex, czso, d3po, dabestr, dados, dafishr, DAISIEprep, dampack, Damsel, dann, daqapo, dar, dartRverse, data.validator, DatabaseConnector, datacleanr, datacutr, DataMetProcess, datamods, dataquieR, dataset, datefixR, datos, dawaR, daySupply, dbflobr, dbGaPCheckup, dbglm, dbi.table, DBItest, dbmss, dbplot, dbplyr, dce, DCLEAR, dcm2, dcurves, dearseq, debkeepr, DeclareDesign, decoupleR, decp, deepredeff, deeptime, deflist, DEGreport, deident, delayed, deltaccd, deltatest, demodelr, denguedatahub, densityarea, depigner, deprivateR, describedata, descriptio, descriptr, DescrTab2, descstat, desctable, deseats, designit, DesignLibrary, desirability2, desplot, destiny, detourr, detrendr, devtools, dexter, dextergui, dexterMST, df2yaml, dfdr, dfeR, dfoliatR, DFplyr, dgpsi, DHS.rates, DHSr, DiagrammeR, dials, dibble, diceplot, Dict, did2s, didec, DIDHAD, diffdfs, diffEnrich, diffeR, dilp, DImodelsVis, dinoR, dipsaus, DirectEffects, dirttee, discAUC, disclosuR, discoveR, discrim, diseasystore, DisImpact, disk.frame, disprofas, Distance, distcomp, Distributacalcul, distributional, distributions3, distrr, dittodb, div, divent, diversityForest, divseg, diyar, dlim, dlookr, dlr, dm, doBy, doctest, donutsk, dots, dotsViolin, dotwhisker, doubletrouble, douconca, downlit, dowser, dpasurv, dpkg, dplyr, dpm, dr4pl, drake, DrDimont, dreamer, dreamlet, driveR, DRomics, drugDemand, DrugExposureDiagnostics, drugprepr, DrugUtilisation, DSAIDE, DSAIRM, dsdp, dsims, DSSAT, dst, dsTidyverse, dsTidyverseClient, dStruct, dtplyr, dtrackr, dtwclust, duckplyr, duet, dumbbell, dunlin, DuplexDiscovereR, dupree, dverse, dwctaxon, dyn.log, dynamite, DynaRankR, dynConfiR, dyngen, dynprog, dySEM, easier, easy.utils, easyalluvial, EasyCellType, easycensus, easydb, easylabel, easyr, easyreporting, easysurv, ebirdst, ebnm, ecan, ech, echoice2, echor, echos, ecocbo, ecochange, ecocomDP, ecode, econid, EcotoneFinder, ECOTOXr, EDCimport, edeaR, edecob, edibble, ediblecity, editbl, edwards97, EEAaq, EFAtools, EGM, egor, eHDPrep, eider, eiExpand, elaborator, ellipsis, ellmer, EloOptimized, embed, emdi, EmpiricalCalibration, emuR, encryptr, enderecobr, ENMeval, enrichplot, entropart, envvar, eph, epiCleanr, epicmodel, epiCo, epidatr, EpiForsk, epikit, EpiMix, EpiModel, EpiNow2, epiparameter, epitrix, epitweetr, epo, epocakir, eponge, epoxy, eq5d, eq5dsuite, EquiTrends, era, ergm, ergm.multi, ERSA, escheR, esci, eSDM, esem, esquisse, estatapi, ESTER, estimatr, EstimDiagnostics, esvis, etl, eufmdis.adapt, Eunomia, eurlex, europepmc, eurostat, evalITR, EventPredInCure, eventstudyr, EvidenceSynthesis, evinf, evprof, evsim, exampletestr, excessmort, excluder, ExPanDaR, expDB, expertsurv, expirest, explore, expstudy, extraChIPs, extrasteps, exuber, eye, eyelinkReader, eyetools, eyetrackingR, ezcox, ezEDA, ezmmek, ezplot, f1dataR, fable, fable.ata, fable.prophet, fabletools, fabR, fabricatr, facerec, factory, factR, factset.analyticsapi.engines, faers, familiar, Families, fanyi, farr, farrell, fastGLCM, fastkqr, fastml, fastplyr, fastqcr, fastRhockey, fastTS, FastUtils, faux, fauxnaif, FAVA, favawesome, fbar, fcaR, fcci, fdacluster, fdasrvf, feasts, FeatureExtraction, felp, feltr, fenr, ferrn, ffp, ffscrapr, fgdr, fgeo, fgeo.analyze, fgeo.plot, fgeo.tool, fidelius, fido, fiery, figma, filecacher, filesstrings, FindIT2, findSVI, finetune, finnts, fio, fishtree, fishualize, fitbitr, fitdistrplus, fitlandr, fitzRoy, fixes, fixtuRes, fkbma, flashlight, fledge, flevr, flexFitR, flexlsx, flexsurv, flextable, FlickrAPI, flightplot, flipr, flowchart, flowGate, flowml, FlowSOM, flowTime, fluxible, fmesher, FMM, foghorn, fontawesome, foodingraph, footBayes, footprint, forcats, forecasteR, forecastML, ForeComp, foreSIGHT, forestat, forestecology, ForestElementsR, forestly, forestmangr, forestmodel, forge, forgts, ForIT, formods, forsearch, forstringr, FossilSim, fPASS, fqacalc, fqar, fr, framecleaner, fredr, frenchdata, freqtables, frictionless, frscore, FSK2R, fsr, fst4pg, ftExtra, fude, funcharts, funneljoin, FunnelPlotR, furrr, fwb, fxl, gadget3, gaiah, galah, galvanizer, gamma, gander, gargle, garma, gateR, gcplyr, GDAtools, gdim, GDPuc, gemma.R, gen5helper, geneHapR, genekitr, GeneSelectR, geneticae, GeneTonic, geneviewer, genieBPC, genogeographer, GenomeAdmixR, GenomicDataCommons, GenoTriplo, geocodebr, geodiv, geodl, geofacet, geofi, geoheatmap, geomander, GeomArchetypal, geomtextpath, geomultistar, GeomxTools, GeoTox, geouy, gestalt, getLattes, gfoRmulaICE, gg1d, gg4way, ggalign, ggalignment, ggalluvial, GGally, gganimate, ggarchery, ggarrow, ggautomap, ggbio, ggblanket, ggblend, ggborderline, ggbrain, ggbreak, ggcharts, ggcleveland, ggcyto, ggdag, ggdist, ggDoubleHeat, ggeasy, ggedit, ggESDA, ggetho, ggfacto, ggfields, ggFishPlots, ggflowchart, ggfocus, ggforce, ggformula, ggfoundry, ggfun, ggfx, gggenes, gggenomes, ggh4x, gghdx, gghighlight, gghilbertstrings, gginference, gginnards, ggip, ggiraph, gglgbtq, gglm, ggmanh, ggmap, ggmapcn, ggmapinset, ggmice, ggmosaic, GGoutlieR, ggpackets, ggpage, ggparty, ggpath, ggpattern, ggpca, ggpcp, ggpie, ggplate, ggplot.multistats, ggplot2, ggplotlyExtra, ggpmisc, ggPMX, ggpointless, ggpol, ggpolar, ggpp, ggprism, ggpubr, ggquickeda, ggragged, ggrain, ggraph, ggrepel, ggreveal, ggsankeyfier, ggsc, ggseas, ggsector, ggseqplot, ggshadow, ggside, ggsoccer, ggspatial, ggspectra, ggstackplot, ggstance, ggstats, ggstatsplot, ggsurveillance, ggsurvfit, ggtangle, ggtaxplot, ggtern, ggtext, ggtibble, ggtrace, ggtree, ggtreeExtra, ggtreeSpace, ggupset, ggvis, gh, ghclass, GHCNr, ghypernet, gibasa, gibble, GillespieSSA2, gips, GISSB, GitAI, gitlink, GitStats, glinvci, glmGamPoi, GLMMcosinor, glmSparseNet, globaltrends, GloScope, glossr, gmailr, gmgm, Gmisc, gMOIP, gmoTree, gms, gmwmx2, godley, goldfish, goldilocks, golem, gompertztrunc, Goodreader, googleAnalyticsR, googleAuthR, googledrive, googlenlp, googlesheets4, GOSemSim, GPCERF, gptstudio, GPvam, GR, GrafGen, grandR, GRaNIE, granulator, graphicalVAR, graphite, graphTweets, gratia, gRaven, gravitas, gravity, Greg, greta, greta.dynamics, greta.gam, greta.gp, gridpattern, gridtext, grizbayr, groupdata2, groupr, growR, GrowthCurveME, grwat, gscramble, gsDesign, GSEAmining, gson, gspcr, gt, gtable, gtexr, gtExtras, gtexture, GTFSwizard, gto, gtreg, gtsummary, guardianapi, guideR, GUIDEseq, guildai, guiplot, gvcR, gwasrapidd, gwavr, GWlasso, hacksaw, hacksig, haldensify, halk, handyFunctions, Haplin, hardhat, harmony, harrietr, hatchR, haven, hbamr, hdf5r.Extra, hdme, headliner, healthdb, healthequal, healthyR, healthyR.ai, healthyR.data, healthyR.ts, healthyverse, heapsofpapers, HEDA, heddlr, hedgehog, heemod, heiscore, helixvis, HelpersMG, hermes, heumilkr, heuristicsmineR, hfhub, hgnc, hhmR, HicAggR, HiCDCPlus, highcharter, highlighter, HIPPO, hipread, historicalborrow, historicalborrowlong, histoslider, HLMdiag, HMDHFDplus, hmer, hms, Hmsc, hmstimer, holodeck, HonestDiD, hoodscanR, hoopR, howzatR, hpiR, htmltools, htrSPRanalysis, httptest2, httr2, hubEnsembles, HuBMAPR, hubUtils, hurricaneexposure, huxtable, HybridExpress, HybridMicrobiomes, hydroloom, HyMETT, hypeR, hyperSpec, HYPEtools, hypothesis, I14Y, i2extras, iadf, iai, ibawds, ibb, IBMPopSim, ICD10gm, icecream, icesSAG, iCNV, iCOBRA, icpack, ideal, ideanet, IDEATools, idiogramFISH, idmact, idpr, igcop, igraph, iheiddown, ihpdr, ijtiff, imaginator, imcRtools, IMD, imgrec, immcp, ImML, immunarch, immunaut, immunotation, impactflu, impactr, implyr, imputeREE, imt, incase, incidence2, IncidencePrevalence, inctools, incubate, indiedown, industRial, infer, influxdbr, InjurySeverityScore, injurytools, inlabru, inlamemi, inlcolor, insee, inspectdf, INSPECTumours, instantiate, intensegRid, InteractionPoweR, interactions, InterCellar, interfacer, interpretCI, intRinsic, invctr, invivoPKfit, iNZightPlots, iNZightTools, iNZightTS, iotables, iotarelr, ipaddress, ipeadatar, ipeaplot, IPEDS, IPEDSuploadables, ipmr, ipolygrowth, ipumsr, ir, IR, iraceplot, irtQ, ISAnalytics, iSEEfier, Isinglandr, isobxr, isomiRs, isoorbi, isotracer, iterors, itraxR, itscalledsoccer, itsdm, ivitr, ivs, jackstrap, jagstargets, JANE, janitor, japanstat, jetty, jinjar, jmastats, jmvcore, jobqueue, JointAI, JointFPM, joyn, jpcity, jpgrid, jpmesh, jpstat, jrSiCKLSNMF, JSmediation, jsonstat, jstable, jstor, jtools, juicedown, JWileymisc, k5, kanjistat, kappaGold, karyotapR, katdetectr, keras, keras3, kesernetwork, keyATM, keyholder, kgrams, khisr, kindisperse, kitesquare, knfi, knowYourCG, kntnr, KoboconnectR, kokudosuuchi, konfound, KrakenR, kssa, labelled, lablaster, lactater, lacunr, LAD, LAGOSNE, laminr, lans2r, lares, later, LATERmodel, latrend, lavaanPlot, lay, lcra, lcsm, ldblock, learnr, ledger, lefko3, LegATo, legendry, lehdr, lemur, lenses, lessSEM, levitate, lifecycle, LifeInsureR, LifemapR, LightLogR, lightparser, likelihoodTools, lillies, liminal, linelist, link, linne, lipidr, listcomp, listr, litteR, LLIC, LLMR, LMMstar, lnmixsurv, loadeR, lobstr, localIV, locuszoomr, lodi, loewesadditivity, LOGAN, logrx, LongDat, longevity, longitudinalcascade, loon.ggplot, LRTesteR, lsirm12pl, LSTbook, LTASR, LTFHPlus, luz, lvmisc, mable, MachineShop, macrosyntR, madgrad, madshapR, maestro, MAGMA.R, MagmaClustR, MAICtools, MainExistingDatasets, makemyprior, malariaAtlas, malaytextr, maldipickr, mall, mapbayr, mapboxapi, mapgl, mapindia, mapindiatools, Mapinguari, mapmixture, mappp, margaret, marginaleffects, mariner, mark, markets, markmyassignment, markovmix, marquee, marr, maskr, matahari, MatchIt, matchmaker, matchMulti, matchr, MatchThem, maths.genealogy, matricks, MatrixQCvis, matrixset, matriz, matsbyname, matsindf, maxcombo, MazamaCoreUtils, MazamaLocationUtils, MazamaSpatialPlots, MazamaSpatialUtils, MazamaTimeSeries, mcmsupply, mcp, MCPModBC, mcradds, mcStats, mcunit, mcvis, meantables, measr, medfateland, medrxivr, meltr, memes, memoise, mergenstudio, mergingTools, meshed, messaging, messy, metabCombiner, metaboData, MetabolomicsBasics, metacoder, metaconfoundr, metacore, metadeconfoundR, metagam, metalite, metalite.sl, MetAlyzer, metan, MetaNet, metanetwork, metaplot, metapower, MetaScope, metasnf, metatools, MetaUtility, metawho, meteoland, meteospain, methcon5, MethodOpt, MethReg, MetMashR, MetNet, metrica, mevr, mfdb, mgi.report.reader, mgwrsar, mia, MIAmaxent, miaViz, MIC, miceafter, microbial, microbiomeExplorer, microbiomeMarker, MicrobiotaProcess, microclass, microCRAN, microdiluteR, microeco, microinverterdata, micropan, microseq, microSTASIS, micsr, midasHLA, midi, midoc, migrate, mikropml, mildsvm, MIMER, mineSweepR, mintyr, mipplot, miRetrieve, MIRit, MiscMetabar, missCompare, missSBM, mistyR, mitre, mixedbiastest, MixedPsy, MixOptim, mixpoissonreg, mixtur, mixvlmc, mizer, mkin, mlbplotR, mlfit, mlflow, mlr3fairness, mlstrOpalr, mlts, mlVAR, MMVBVS, mobileR, mobr, mockr, mockthat, modeldata, modeldb, modelenv, modelr, modeltime, modeltime.ensemble, modeltime.resample, moderndive, ModEstM, modsem, moexer, MoleculeExperiment, MOMA, momentuHMM, monashtipr, mongopipe, monoClust, MonotonicityTest, MOODE, moodleR, moranajp, moreparty, morepls, morphemepiece, mort, mortAAR, mosaic, mosaicCalc, mosaicCore, mosdef, MOSim, motifr, motifTestR, mousetrap, move2, movegroup, movementsync, mpathr, mpathsenser, mpower, MPTmultiverse, mrgsim.sa, mrgsolve, MSA2dist, MSCA, msgr, mshap, msigdbr, mskcc.oncotree, MSmix, MSnID, MSPrep, MsQuality, msSPChelpR, mstate, MSTest, mtdesign, mtscr, mulea, multibias, multibiasmeta, multibreakeR, multidplyr, multiGSEA, multilevelmod, multimedia, multimorbidity, multinma, MultiObjMatch, MultIS, multitool, multivarious, multiverse, muscat, musicatk, mvgam, mvMonitoring, mvnimpute, mwlaxeref, mxfda, mxnorm, mycaas, naijR, NaileR, name, nandb, naniar, NanoMethViz, nanostringr, nasapower, nascaR.data, natstrat, naturaList, navigatr, nbTransmission, ncaavolleyballr, ncappc, NCC, ncdump, ncmeta, nearBynding, necountries, nemsqar, neo4r, NEONiso, neotoma2, nestedcv, nestr, netcom, NetCoupler, NetFACS, netgsa, netmap, netseer, NetworkChange, NetworkExtinction, neuroUp, nevada, nfl4th, nflfastR, nflplotR, nflreadr, nflseedR, nflverse, ngramr, ngsReports, nhdR, NHSRplotthedots, nichetools, NiLeDAM, nipalsMCIA, nixtlar, nls.multstart, nmarank, Nmisc, nncc, noah, NobBS, nofrills, nomisr, nomnoml, nonet, nonmemica, normaliseR, normalr, normfluodbf, NPARC, NPBayesImputeCat, npde, npi, nplyr, nrba, ntdr, nuggets, nullranges, nuts, NVAR, nzffdr, nzilbb.vowels, obliqueRSF, occCite, oceanexplorer, octopusR, ODataQuery, odbc, oddnet, oddsapiR, ODRF, oesr, officedown, offsetreg, OhdsiReportGenerator, OhdsiShinyAppBuilder, ohun, ojsr, OlinkAnalyze, OmicsMLRepoR, omicwas, OmnipathR, omock, omopgenerics, OmopSketch, OmopViewer, ompr, omsvg, onbrand, OncoBayes2, oncomsm, OncoSubtype, oncrawlR, One4All, OneSampleLogRankTest, OneSampleMR, ontologics, openair, openairmaps, openappr, opencage, opencpu, openeo, openFDA, openscoring, opGMMassessment, opinAr, optedr, optic, opticskxi, optimalFlow, optimall, OralOpioids, orbital, ordinalsimr, ordr, ORFID, Organism.dplyr, organizr, orthos, osfr, otargen, otpr, overtureR, overviewR, OVtool, owidapi, OxSR, oysteR, packageRank, pacta.loanbook, pacta.multi.loanbook, padr, pafr, pagoda2, palaeoSig, paletteer, palettes, pam, pammtools, PAMpal, PAMscapes, pandoc, panelr, panelsummary, PanomiR, parafac4microbiome, ParamANOVA, pargasite, paropt, parquetize, parsel, parsermd, parsnip, partialised, particles, partition, partR2, parttree, passt, PAST, patchwork, patentr, path.chain, pathling, PatientLevelPrediction, PatientProfiles, patrick, pcvr, pdp, pdxTrees, peacesciencer, PEIMAN2, pems.utils, penppml, pense, PepMapViz, permubiome, personr, peruse, PesticideLoadIndicator, pestr, Petersen, phacking, PHEindicatormethods, PhenoSpectra, PhenotypeR, PHENTHAUproc, PhosR, photobiology, photosynthesis, phsmethods, phylepic, phylosamp, piecemaker, piecenorms, piecepackr, PieGlyph, pikchr, pillar, pins, pioneeR, pivotea, piwikproR, PKbioanalysis, pkgdown, pkgload, pkgnet, PKNCA, planscorer, plater, PLNmodels, plotbb, plotcli, plotDK, PlotFTIR, plotgardener, plotGrouper, plotly, plotor, plotROC, plotthis, plsmmLasso, plsmod, plumber, plumbertableau, plume, plutor, plyinteractions, plyranges, plyxp, pmev, pmxcode, podcleaner, pointblank, poissonreg, polimetrics, polished, pollster, polmineR, polyglotr, polymatching, polypoly, POMA, pomcheckr, pool, poolABC, pooledpeaks, PoolTestR, PopGenHelpR, poppr, PopulateR, populR, poputils, portalr, portfolioBacktest, portvine, posterior, potions, power.transform, powerjoin, powerly, powerNLSEM, powRICLPM, ppcseq, ppmf, pprof, precisely, precmed, precommit, precondition, precrec, predict3d, predictNMB, predictoR, predRupdate, presenter, prevtoinc, pricesensitivitymeter, primarycensored, prioriactions, prioritizr, priorsense, prnsamplr, proActiv, probably, ProbBreed, processanimateR, processcheckR, processmapR, processpredictR, prodigenr, profile, profileplyr, profvis, projects, promises, prophet, protein8k, ProteoBayes, ProteoDisco, protGear, protti, proustr, prozor, PRROC, prt, pseudobibeR, pseudoCure, psHarmonize, psm3mkv, psmineR, psycCleaning, psychmeta, psychonetrics, psycModel, psyntur, ptable, ptairMS, PTXQC, PublicationBias, PublicWorksFinanceIT, pubmedtk, puls, PUMP, PupillometryR, PupilPre, PurpleAir, purrr, pushoverr, pvda, pxmake, pysparklyr, qacBase, qckitfastq, qgisprocess, qmj, qmtools, qPLEXanalyzer, qsvaR, qtkit, qtl2ggplot, qtl2pattern, qtl2pleio, QTLExperiment, quadVAR, qualmap, qualtRics, quantilogram, quantiseqr, quarto, queryBuilder, questionr, quickReg, QuICSeedR, quid, quincunx, quollr, r2dii.analysis, r2dii.match, r2dii.plot, r2mlm, r2r, r5r, racademyocean, Racmacs, RadialVisGadgets, radiant.basics, radiant.data, radiant.model, radiant.multivariate, radsafer, Radviz, rai, RAIDS, rainette, rairtable, rando, random.cdisc.data, randomizeR, RandomWalker, Rapi, rapsimng, RAQSAPI, rar, rassta, rater, ratlas, rATTAINS, ravepipeline, rawDiag, rb3, rbedrock, RBesT, rbgm, rbw, Rcatch22, rcbayes, RClimacell, RclusTool, rcoder, RCPA, Rcpi, RcppJagger, RCT, RCTS, Rcurvep, rddi, rdecision, rdhs, rdss, rdtLite, react, reactable.extras, readabs, readapra, readr, readrba, readsdr, ready4, reappraised, rearrr, recipes, reconstructKM, REDCapDM, REDCapR, REDCapTidieR, REddyProc, REDI, redist, redistmetrics, RegDDM, reghelper, regions, regport, RegrCoeffsExplorer, regressinator, regressoR, reliabilitydiag, rempsyc, reportRmd, reprex, reservoirnet, reservr, resourcecode, respirometry, ResultModelManager, rethnicity, reticulate, retroharmonize, retry, reval, reveneraR, ReviewR, rextendr, rfars, rFIA, rfishbase, RFishBC, rfm, rfPermute, RGCCA, rgeomstats, Rgff, rgho, rgoogleads, RiboCrypt, ribor, rice, Ricrt, ricu, riem, rifi, rifiComparative, Rigma, riingo, ripc, riskCommunicator, risks, riverconn, RivRetrieve, rjaf, RJcluster, rjtools, rKolada, rliger, rlinkedinads, rlowdb, RMariaDB, RmarineHeatWaves, rmcmc, rmdl, rmdpartials, rminizinc, RMM, Rmonize, RNeXML, rnpn, roadoi, roads, RobinCar, robis, RoBMA, RobMixReg, robotoolbox, RoBSA, robustlmm, rockx, roclang, rocnp, rolap, rollama, rollup, romic, ropenblas, rosm, rotl, roundwork, roxygen2, roxygen2md, rPACI, rpact, Rpadrino, rpaleoclim, rplanes, RplotterPkg, RPresto, RprobitB, rredlist, rrtable, Rsagacmd, rsample, rsconnect, RSDA, RseqCovarImpute, rsetse, rsf, rsi, rsimsum, RSQLite, rSRD, rstatix, rstudio.prefs, rsurveycto, rsyntax, rtern, rticulate, RTL, RtsEva, rtweet, rTwig, rtypeform, rubias, ruler, rules, rUM, ruminate, RVA, rvec, RVenn, rvest, rvg, rWCVP, rytstat, s3, sabre, saccadr, sae.projection, saebnocov, saeczi, saemix, saens, saeTrafo, safestats, safetensors, safetyCharts, safetyGraphics, SAiVE, salesforcer, sampbias, sampcompR, samplingin, SangerTools, santoku, sapfluxnetr, saros, saros.base, sass, sbm, sbo, sboost, scales, scater, scatterpie, sccomp, sccore, scCustomize, SCDA, SCDB, scdhlm, scDiagnostics, scDotPlot, scenes, scFeatureFilter, schex, scholar, SchoolDataIT, schtools, scider, scifer, scimo, sclr, scMultiSim, scoper, scorematchingad, scpi, scPipe, scPipeline, scPloidy, ScreenR, screenshot, scregclust, scRepertoire, scriptName, scrutiny, sctransform, sdcHierarchies, sdcTable, SDModels, sdmTMB, SDMtune, sdtm.oak, sdtmval, seahtrue, secrettext, secsse, secuTrialR, sedproxy, SeedMaker, segclust2d, SEIRfansy, selenider, selenium, semfindr, semhelpinghands, semlbci, semptools, SensIAT, sensobol, seqCAT, SeqNet, seqtrie, serocalculator, servosphereR, SESraster, Seurat, SeuratObject, sevenbridges2, sfcr, sfd, sfdep, sfhotspot, sfnetworks, sg, sgboost, sglr, shapviz, shazam, sherlock, shide, shiny, shiny.destroy, shiny.gosling, shiny.react, shiny.router, shiny.telemetry, shinyauthr, shinybrms, shinyChakraUI, shinychat, shinyCohortBuilder, shinyepico, shinyExprPortal, shinyGizmo, shinyHugePlot, shinyInvoice, ShinyItemAnalysis, shinyjqui, shinylive, shinyloadtest, shinymeta, shinymodels, shinyNotes, shinyobjects, shinyQueryBuilder, shinytest, shinytest2, shinyvalidate, shinyWidgets, shrinkGPR, SIAmodules, SIAtools, sift, sigminer, Signac, signs, silicate, sim.BA, simaerep, simDAG, SimEngine, simET, simglm, simITS, simlandr, SimNPH, simodels, simPIC, simpr, simrel, simstandard, SimSurvey, simulist, SingleCaseES, singleCellTK, SingleMoleculeFootprinting, SISIR, sistec, sistmr, sitar, sivirep, SiZer, sjmisc, sjPlot, sjtable2df, skater, sketch, sketchR, SKFCPD, skimr, slackr, slap, SLCARE, sleeperapi, slider, sloop, smartsheetr, smcfcs, smoothic, smplot2, snapcount, snvecR, soc.ca, socialranking, socialrisk, sociome, socviz, soiltestcorr, SOMbrero, SOMnmR, sortable, sorvi, sourcoise, spanishoddata, SpaNorm, sparklyr, sparklyr.flint, sparklyr.nested, sparkxgb, SparseBiplots, sparsediscrim, sparsegl, sparseR, sparsevctrs, SpatialCPie, SpatialDDLS, SpatialFeatureExperiment, SpatialKDE, spatialLIBD, SpatialRDD, spatialRF, spatialsample, spatPomp, spbabel, spDates, specr, spectralR, Spectran, speechbr, SPIAT, SPIChanges, spicyR, spind, splatter, splithalfr, spnaf, SPONGE, sportyR, SpotClean, spotifyr, spRingsteen, spsur, sqlhelper, SqlRender, sqltargets, sRdpData, sreg, srvyr, ssd4mosaic, ssdtools, SSEparser, SSLR, sspm, sSPPY, ssrn, SSVS, stability, staccuracy, stackgbm, stacks, stacomiR, stagedtrees, stampr, standR, stars, starschemar, starsTileServer, starter, starvz, statar, statcheck, states, statgenGWAS, statgenGxE, statgenHTP, statgenIBD, statgenMPP, statgenSTA, statnipokladna, statsExpressions, statswalesr, stbl, stdmod, stenographer, stenR, stevemisc, stevethemes, stickyr, SticsRFiles, stochLAB, stors, stoRy, stplanr, strand, strata, stratamatch, streamy, streetscape, strex, stringr, StructuralVariantAnnotation, stxplore, styler, subincomeR, subscreen, sudachir, suddengains, sugarbag, sugrrants, summclust, SUMO, SUNGEO, SunsVoc, SuperCell, supernova, supersigs, superspreading, supportR, surtvep, surveil, surveydata, surveyexplorer, survivalAnalysis, survminer, survParamSim, survstan, susographql, svaNUMT, svaRetro, svMisc, swaprinc, sweep, swfscAirDAS, swfscDAS, swfscMisc, SWMPrExtension, SWTools, symphony, SynergyLMM, syntenet, SynthETIC, syrup, sysAgNPs, Sysrecon, systemPipeShiny, syuzhet, tabbycat, table.express, tablecompare, tablespan, tablet, tabnet, tabshiftr, tabularmaps, tabulator, tabxplor, tadar, tall, tame, tarchetypes, tardis, targets, taxa, taxadb, taylor, tbl2xts, tbrf, tcgaViz, tcl, tclust, tcpl, tdcmStan, teal, teal.code, teal.data, teal.slice, teal.transform, tealeaves, telraamStats, term, tern, tern.mmrm, terrainr, testdat, TestGenerator, testthat, text, text2map, textrecipes, tf, tfdatasets, tfestimators, tfevents, tfrmt, tfrmtbuilder, tfruns, theft, theftdlc, thematic, themis, ThermalSampleR, thredds, thurstonianIRT, tibble, tibbletime, tibblify, tidier, tidyAML, tidybayes, tidybins, tidyboot, tidybulk, tidyCDISC, tidycensus, tidychangepoint, tidycharts, tidyclust, tidycmprsk, tidycode, tidycomm, tidyCoverage, tidydelta, TidyDensity, tidydr, tidyEdSurvey, tidyestimate, tidyfinance, tidyfit, tidyFlowCore, tidygam, tidygate, tidygenomics, tidygeoRSS, tidygraph, tidyHeatmap, tidyheatmaps, tidyhte, tidyhydat, tidyindex, tidyjson, tidylda, tidyllm, tidylo, tidylog, tidyMC, tidymodels, TidyMultiqc, tidync, tidyomics, tidypaleo, tidyplate, tidyplots, tidyplus, tidyposterior, tidypredict, tidyquant, tidyquery, tidyr, tidyrates, tidyREDCap, tidyrgee, tidyRSS, tidyrules, tidysdm, tidyselect, tidyseurat, tidySingleCellExperiment, tidysmd, tidySpatialExperiment, tidystringdist, tidySummarizedExperiment, tidytable, tidyterra, tidytext, tidytidbits, tidytlg, tidytof, tidytransit, tidytreatment, tidytree, tidytuesdayR, tidyverse, tidyvpc, tidywater, tidywikidatar, tigerstats, tigger, timbr, timeplyr, timetk, tintin, tinylens, tinyscholar, tip, tipr, TKCat, tmap, tmod, tmt, tna, tntpr, toastui, tomledit, tongfen, topics, toposort, tor, torch, torchvision, torchvisionlib, tornado, toscutil, tosr, tower, toxpiR, toxSummary, Tplyr, track2KBA, traitstrap, transformr, transmogR, TransProR, transx, traudem, traumar, treasury, treeclimbR, treediff, treeio, treesitter, TreeSummarizedExperiment, trelliscopejs, trialr, triangulr, tricolore, trip, triplesmatch, triptych, troopdata, trud, tsibble, tsibbletalk, tsnet, tuberculosis, TUFLOWR, tune, tv, tweenr, Twitmo, twn, TwoWayFEWeights, twoxtwo, typetracer, ubair, ubms, UCSCXenaShiny, UCSCXenaTools, uisapi, ulrb, UMI4Cats, UnalR, uncertainUCDP, uncmbb, unheadr, univariateML, universalmotif, unjoin, unpivotr, unstruwwel, unusualprofile, updateme, ura, usedthese, usemodels, usethis, usmap, usmapdata, ustfd, utile.tables, uxr, vaccine, vaccineff, vachette, valentine, valhallr, validata, validateIt, valr, valueSetCompare, VancouvR, vannstats, variancePartition, vchartr, vctrs, vdiffr, vDiveR, vecmatch, vegawidget, versus, vetiver, viafr, VicmapR, viewscape, vigicaen, vimp, vimpclust, Virusparies, visae, visOmopResults, vistime, visualFields, visvaR, vitae, vital, vmeasur, voiceR, vol2birdR, volcano3D, volker, voronoiTreemap, voson.tcn, vosonSML, Voyager, vroom, vsp, vvcanvas, vvfiller, vvshiny, VWPre, waffle, waldo, walker, wallace, washr, watcher, Wats, waves, waywiser, wbids, wbstats, wbwdi, wdnr.gis, weaana, webchem, WebGestaltR, webmockr, webr, wehoop, WeightIt, weird, weitrix, when, whippr, whomds, widyr, wikkitidy, wingen, WoodSimulatR, worcs, wordpiece, workflows, workflowsets, worldfootballR, wpm, wppi, WpProj, wqspt, wrangle, wrappedtools, WRI, writer, wru, WVPlots, xadmix, xdvir, xlr, xmap, xml2, xml2relational, xmpdf, xmrr, xpectr, xportr, xpose, xpose.nlmixr2, xpose.xtras, xrf, xtsum, yamlet, yardstick, ycevo, ymlthis, yodel, yulab.utils, zctaCrosswalk, zcurve, zenstats, zephyr, zipangu, zipcodeR, zoltr, zoomr, ZygosityPredictor","","MsQuality, PTXQC","","actel, AdhereR, camcorder, ChemmineR, chess, cohorttools, colorfindr, dtrackr, favawesome, gofigR, grPipe, integr, LearnVizLMM, netSEM, ODT, PaRe, pikchr, PRISMA2020, PubChemR, rfaRm, RIdeogram, rphylopic, SBGNview, spatialHeatmap, svgtools, tidycharts, vtree, wikiprofiler","carbonate, epitweetr, needmining, saotd, Twitmo","","","tinyVAST","","","MLE, PINstimation, powerHE","lablaster, legion, MAPA","","","","abjutils, abn, acoRn, act, actel, ADAMgui, aifeducation, airr, alakazam, amadeus, aMNLFA, AmpGram, annotater, antaresRead, APCalign, audubon, augmentedRCBD, auk, authoritative, autonomics, BatchJobs, batchtools, BAwiR, bdc, bea.R, bibliometrix, BINtools, biobtreeR, biometryassist, bioseq, BiostatsUHNplus, birdie, bskyr, BUMHMM, bupaR, BWGS, CAGEr, CancerGram, canprot, CAOP.RAA.2024, CatastRo, CDMConnector, CellNOptR, ChemmineR, chinese.misc, cicero, circRprofiler, CITAN, cleanNLP, clidatajp, climate, clintools, clockify, clustringr, CodelistGenerator, CohortGenerator, comorbidity, conflr, cookiemonster, CooRTweet, corporaexplorer, corpustools, CoTiMA, covidsymptom, CrossVA, cryptotrackr, csv, CTNote, ctrdata, ctrialsgov, cultevo, cyclestreets, cymruservices, czso, daewr, dartR.sim, data.tree, dataCompareR, dataMaid, dataone, dataReporter, DatastreamDSWS2R, datazoom.amazonia, dbi.table, dbWebForms, debrowser, DEGreport, Diderot, diffUTR, disclosuR, discoverableresearch, DistPlotter, doc2concrete, doudpackage, dplR, dscore, dundermifflin, dynr, eatGADS, eatTools, Ecfun, edgar, EGSEA, ELISAtools, emayili, EMJMCMC, enderecobr, epidata, epidm, epitrix, eplusr, eurostat, evinf, exams.mylearn, exams2sakai, eyelinker, farff, fastLink, felp, fiery, filebin, filesstrings, findInFiles, fixtuRes, flexlsx, flipbookr, flippant, flowGraph, forceplate, formatters, fscache, fusen, fuzzySim, gasanalyzer, gbm.auto, gDRimport, genekitr, geneset, gfer, gganimate, ggfittext, ggfootball, GHap, gibasa, gINTomics, gitdown, gmwmx, goxygen, GreedyExperimentalDesign, growthPheno, healthyR.ts, highlightr, huxtable, IAcsSPCR, ICAMS, ICD10gm, IDEATools, ie2misc, imaginator, immApex, immunarch, imola, inldata, inpdfr, IOHanalyzer, ipa, ISO11784Tools, itscalledsoccer, janitor, japanstat, journalabbr, jrc, jSDM, kanjistat, kerntools, klsh, kmeRtone, KOR.addrlink, LACE, languageserver, LexisNexisTools, LilRhino, lime, logrx, LSTbook, LSX, m5, MadanText, MadanTextNetwork, madrat, mapboxapi, margaret, mastif, mclm, medExtractR, memapp, messydates, midasHLA, miRecSurv, mlms, mlr, mlr3oml, mlrCPO, mnis, Modstrings, MOFA2, monolix2rx, moranajp, MOSim, mpoly, mRpostman, MSA2dist, mscstexta4r, MSstatsConvert, MSstatsPTM, mulea, multicrispr, multipanelfigure, musicatk, naaccr, naijR, naturaList, necountries, newsmap, njtr1, nodbi, NormalityAssessment, nzffdr, OmnipathR, omopgenerics, onc.api, OpenMindat, OpenML, openxlsx, openxlsx2, optigrab, osfr, osmclass, oxcAAR, pacs, path.chain, pdfsearch, pedquant, PGRdup, pgTools, phonetisr, PhosR, phylotypr, piecemaker, pk.unit.trans, plumber, plumbertableau, podcleaner, poldis, politeness, polmineR, polyRAD, potions, ppgm, presenter, priceR, processcheckR, psmineR, PvSTATEM, pxmake, QCAcluster, qdapRegex, qlcData, Qtlizer, quanteda, quanteda.textmodels, quanteda.textplots, quanteda.textstats, R.temis, r2dii.match, radiant.data, radiant.model, ralger, RALSA, rangeBuilder, rassta, rattle, RclusTool, RcmdrPlugin.temis, RCy3, rdflib, rdtLite, re, Reacnorm, readabs, readODS, readsdr, readtext, ready4, receptiviti, redatam, REDCapDM, REDCapTidieR, refinr, RepertoiR, reporter, reqres, restez, retrosheet, rextendr, rfaRm, Rgff, RJalaliDate, rmcfs, RNewsflow, RNeXML, robotoolbox, routr, roxygen2, RPresto, rprime, rprintf, Rsgf, rslp, RSqlParser, rSWeeP, rsyntax, rtables, rtables.officer, RTCGA, Rtrack, rwhatsapp, samplesizeestimator, sanketphonetictranslator, saros, saros.base, scanMiR, scCustomize, scitb, scoper, scorecard, sejmRP, SemNetCleaner, sensitivityCalibration, sentimentr, sentometrics, seqtrie, sergeant, SerolyzeR, sgraph, shazam, shiny.react, shiny.reglog, shinyMixR, ShinyQuickStarter, shinyquiz, Signac, sinew, sistec, sketchy, snakecase, sonicscrewdriver, SpliceWiz, sqlHelpers, SSHAARP, statcheck, statgenIBD, statnipokladna, strex, stringr, stringx, strs, Structstrings, supportR, table.glue, tabxplor, tangram, tardis, tatoo, taxadb, taxize, taxlist, testthis, text, text2map, text2vec, textclean, textpress, textshape, textshaping, textstem, tfrmt, TheOpenAIR, thinkr, ThomasJeffersonUniv, thriftr, tidybulk, tidyplus, tigger, timetk, tinycodet, tipitaka, TMDb, tokenbrowser, tokenizers, transltr, tsentiment, Twitmo, ulex, vegtable, wakefield, webdeveloper, WhatsR, wiesbaden, wikilake, wildcard, wordmap, wordpiece, wordvector, worldfootballR, xmpdf, Xplortext, XString, zipangu, zoolog","Bayenet, causal.decomp, glmtoolbox, holiglm, kSamples, NSM3, PHENIX, PMCMRplus, psbcGroup, PTSR, sft, sgdGMF, survcomp, ufs, vartest, WAnova","nlmixr2est, nlmixr2extra","","adw, ag5Tools, AgePopDenom, agriwater, amadeus, amt, AquaBEHER, arcgislayers, arcpullr, ascotraceR, assignR, atakrig, atpolR, aws.wrfsmn, barrks, basemaps, BAT, bayesEO, bespatial, BIEN, bioclim, BIOMASS, biomod2, bivariatemaps, blackmarbler, BoundaryStats, CAST, CatastRo, CCAMLRGIS, cdlTools, CDSE, ceramic, cercospoRa, cffdrs, chirps, cholera, chooseGCM, clc, climateStability, climetrics, CMIP6VisR, cnmap, CoordinateCleaner, coveR2, cropDemand, cropZoning, ctmm, damAOI, daymetr, deforestable, DEPONS2R, diffeR, disaggregation, diseasemapping, dismo, divraster, divvy, dots, downscale, dsmSearch, dynamAedes, dynamicSDM, dynatopGIS, easyclimate, ebirdst, ebvcube, ecospat, ecotrends, eDITH, elevatr, ElevDistr, enerscape, ENMeval, enmpa, enmSdmX, ENMTools, envi, envirem, epm, eseis, Euclimatch, eva3dm, ExposR, fasterRaster, FedData, fgdr, FIESTAutils, fitbitViz, florabr, ForestTools, FORTLS, foto, gateR, gatoRs, gdistance, gecko, GeNetIt, GeoAdjust, geocausal, geocmeans, geocomplexity, geodata, geodiv, geodl, geomerge, geostatsp, GeoThinneR, geotopbricks, gfcanalysis, ggmapcn, GHCNr, GLCMTextures, glossa, grec, gtfs2emis, gtfs2gps, gwavr, habtools, happign, hemispheR, heterogen, HurreconR, hwsdr, hydflood, hypervolume, hypsoLoop, ICvectorfields, INLAspacetime, inldata, intSDM, IsoriX, ISRaD, itcSegment, LAIr, landform, landscapemetrics, landscapeR, ldmppr, leastcostpath, letsR, lidaRtRee, lidR, lifeR, linemap, link2GI, LPDynR, LST, macroBiome, makePalette, malariaAtlas, mapgl, mapme.biodiversity, mapmisc, mapsRinteractive, maptiles, MBHdesign, mddmaps, medfateland, MetaLandSim, metaRange, meteo, MIAmaxent, MigConnectivity, mkde, mlr3spatial, modEvA, modisfast, MODISTools, MoleculeExperiment, monographaR, mop, move, movecost, movegroup, multilandr, MultiscaleDTM, NetLogoR, nichevol, npphen, OCNet, ofpetrial, Orcs, ows4R, paleobioDB, pastclim, phenomap, phenopix, PHENTHAUproc, phyloraster, phyloregion, phylospatial, PlanetNICFI, pliman, PointedSDMs, predicts, prior3D, priorCON, prioritizr, prioritizrdata, QBMS, quadtree, quaxnat, quickPlot, raceland, rangeBuilder, rangeModelMetadata, rangr, raptr, rassta, raster, rasterbc, rasterdiv, rasterpic, rasterVis, rcaiman, rcontroll, rcrimeanalysis, Recocrop, red, reddPrec, redlistr, regional, restoptr, rflsgen, RFplus, RiskMap, rivnet, rnaturalearth, roads, rPAex, rpaleoclim, RPhosFate, rpostgis, Rsagacmd, rsat, rsi, RStoolbox, rsyncrosim, Rtrack, rtrend, rts, rworldmap, S4DM, SAiVE, samc, sampbias, sara4r, satellite, satres, scapesClassification, sdm, sdmpredictors, SDMtune, secr, SESraster, sgsR, ShapePattern, shar, shoredate, sigugr, simpleSeg, SiMRiv, simulariatools, sits, smoothr, smosr, SnowData, soilassessment, SpaDES.core, SpaDES.tools, sparrpowR, spatialEco, SpatialFeatureExperiment, spatialising, spEDM, SSIMmap, StormR, stppSim, SUMMER, SUNGEO, supercells, SurfaceTortoise, SurfRough, tanaka, TCHazaRds, tectonicr, tenm, terrainr, tidysdm, tidyterra, TOC, TroublemakeR, uavRmp, usdm, viewscape, virtualspecies, voluModel, Voyager, waterquality, wdiEF, weatherOz, wflo, windfarmGA, wingen","","","","modelsummary","","","","AssetCorr, CDVineCopulaConditional, CondCopulas, CopulaCenR, FactorCopula, gamCopula, GJRM, gofCopula, kdevine, LocalCop, LogisticCopula, MMDCopula, pacotest, parTimeROC, VC2copula, vccp",""]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>deadline<\/th>\n      <th>package<\/th>\n      <th>version<\/th>\n      <th>maintainer<\/th>\n      <th>links<\/th>\n      <th>event<\/th>\n      <th>reverse packages (n)<\/th>\n      <th>reverse packages<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":50,"order":[[0,"asc"],[6,"desc"],[1,"asc"]],"columnDefs":[{"className":"dt-right","targets":6},{"name":"deadline","targets":0},{"name":"package","targets":1},{"name":"version","targets":2},{"name":"maintainer","targets":3},{"name":"links","targets":4},{"name":"event","targets":5},{"name":"reverse packages (n)","targets":6},{"name":"reverse packages","targets":7}],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>




<div id="quarto-appendix" class="default"><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><h2 class="anchored quarto-appendix-heading">Footnotes</h2>

<ol>
<li id="fn1"><p>When a package is <em>archived</em>, the archived version is still available in the <a href="https://cran.r-project.org/src/contrib/Archive/">CRAN archive area</a> together with all previous versions.<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




<footer class="footer"><div class="nav-footer"><div class="nav-footer-center"><div class="toc-actions"><ul><li><a href="https://github.com/cranhaven/cranhaven.org/blob/main/dashboard-at-risk.qmd" class="toc-action"><i class="bi bi-github"></i>View source</a></li><li><a href="https://github.com/cranhaven/cranhaven.org/edit/main/dashboard-at-risk.qmd" class="toc-action"><i class="bi empty"></i>Edit this page</a></li><li><a href="https://github.com/cranhaven/cranhaven.org/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></div></div></footer></body></html>